
<!DOCTYPE html>

<html class="no-js" lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width,initial-scale=1" name="viewport"/>
<link href="https://ccms-ucsd.github.io/GNPSDocumentation/featurebasedmolecularnetworking-with-openms/" rel="canonical"/>
<link href="../assets/images/favicon.png" rel="shortcut icon"/>
<meta content="mkdocs-1.1.2, mkdocs-material-7.0.2" name="generator"/>
<title>FBMN with OpenMS - GNPS Documentation</title>
<link href="../assets/stylesheets/main.1655a90d.min.css" rel="stylesheet"/>
<link href="../assets/stylesheets/palette.7fa14f5b.min.css" rel="stylesheet"/>
<link crossorigin="" href="https://fonts.gstatic.com" rel="preconnect"/>
<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700%7CRoboto+Mono&amp;display=fallback" rel="stylesheet"/>
<style>:root{--md-text-font-family:"Roboto";--md-code-font-family:"Roboto Mono"}</style>
<script>window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)},ga.l=+new Date,ga("create","UA-8412213-5","ccms-ucsd.github.io"),ga("set","anonymizeIp",!0),ga("send","pageview"),document.addEventListener("DOMContentLoaded",function(){document.forms.search&&document.forms.search.query.addEventListener("blur",function(){var e;this.value&&(e=document.location.pathname,ga("send","pageview",e+"?q="+this.value))}),"undefined"!=typeof location$&&location$.subscribe(function(e){ga("send","pageview",e.pathname)})})</script>
<script async="" src="https://www.google-analytics.com/analytics.js"></script>
</head>
<body data-md-color-accent="none" data-md-color-primary="none" data-md-color-scheme="" dir="ltr">
<input autocomplete="off" class="md-toggle" data-md-toggle="drawer" id="__drawer" type="checkbox"/>
<input autocomplete="off" class="md-toggle" data-md-toggle="search" id="__search" type="checkbox"/>
<label class="md-overlay" for="__drawer"></label>
<div data-md-component="skip">
<a class="md-skip" href="#introduction-to-fbmn-with-openms">
          Skip to content
        </a>
</div>
<div data-md-component="announce">
</div>
<header class="md-header" data-md-component="header">
<nav aria-label="Header" class="md-header__inner md-grid">
<a aria-label="GNPS Documentation" class="md-header__button md-logo" href="https://ccms-ucsd.github.io/GNPSDocumentation/" title="GNPS Documentation">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"></path></svg>
</a>
<label class="md-header__button md-icon" for="__drawer">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2z"></path></svg>
</label>
<div class="md-header__title" data-md-component="header-title">
<div class="md-header__ellipsis">
<div class="md-header__topic">
<span class="md-ellipsis">
            GNPS Documentation
          </span>
</div>
<div class="md-header__topic" data-md-component="header-topic">
<span class="md-ellipsis">
            
              FBMN with OpenMS
            
          </span>
</div>
</div>
</div>
<div class="md-header__options">
</div>
<label class="md-header__button md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"></path></svg>
</label>
<div class="md-search" data-md-component="search" role="dialog">
<label class="md-search__overlay" for="__search"></label>
<div class="md-search__inner" role="search">
<form class="md-search__form" name="search">
<input aria-label="Search" autocapitalize="off" autocomplete="off" autocorrect="off" class="md-search__input" data-md-component="search-query" data-md-state="active" name="query" placeholder="Search" required="" spellcheck="false" type="text"/>
<label class="md-search__icon md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"></path></svg>
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"></path></svg>
</label>
<button aria-label="Clear" class="md-search__icon md-icon" tabindex="-1" type="reset">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"></path></svg>
</button>
</form>
<div class="md-search__output">
<div class="md-search__scrollwrap" data-md-scrollfix="">
<div class="md-search-result" data-md-component="search-result">
<div class="md-search-result__meta">
            Initializing search
          </div>
<ol class="md-search-result__list"></ol>
</div>
</div>
</div>
</div>
</div>
<div class="md-header__source">
<a class="md-source" data-md-component="source" href="https://github.com/CCMS-UCSD/GNPSDocumentation/" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><path d="M439.55 236.05L244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    CCMS-UCSD/GNPSDocumentation
  </div>
</a>
</div>
</nav>
</header>
<div class="md-container" data-md-component="container">
<main class="md-main" data-md-component="main">
<div class="md-main__inner md-grid">
<div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Navigation" class="md-nav md-nav--primary" data-md-level="0">
<label class="md-nav__title" for="__drawer">
<a aria-label="GNPS Documentation" class="md-nav__button md-logo" href="https://ccms-ucsd.github.io/GNPSDocumentation/" title="GNPS Documentation">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"></path></svg>
</a>
    GNPS Documentation
  </label>
<div class="md-nav__source">
<a class="md-source" data-md-component="source" href="https://github.com/CCMS-UCSD/GNPSDocumentation/" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><path d="M439.55 236.05L244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    CCMS-UCSD/GNPSDocumentation
  </div>
</a>
</div>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="..">
        GNPS Introduction
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../isgnpsright/">
        Is my data compatible with GNPS?
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../faq/">
        Frequently Asked Questions
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../superquickstart/">
        Superquick Start Guide
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../quickstart/">
        Quick Start Guide
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gnpsanalysisoverview/">
        GNPS Analysis Tools Overview
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../lexicon/">
        GNPS Lexicon
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_8" id="__nav_8" type="checkbox"/>
<label class="md-nav__link" for="__nav_8">
        Mass Spec/Computational Background
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Mass Spec/Computational Background" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_8">
<span class="md-nav__icon md-icon"></span>
          Mass Spec/Computational Background
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../massspecbackground/networkingtheory/">
        Molecular Networking
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_9" id="__nav_9" type="checkbox"/>
<label class="md-nav__link" for="__nav_9">
        Data Preparation/Upload
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Data Preparation/Upload" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_9">
<span class="md-nav__icon md-icon"></span>
          Data Preparation/Upload
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../fileconversion/">
        Mass Spectrometry File Conversion
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fileupload/">
        GNPS File Upload
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../metadata/">
        Metadata
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_10" id="__nav_10" type="checkbox"/>
<label class="md-nav__link" for="__nav_10">
        Recommended Data Analysis
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Recommended Data Analysis" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_10">
<span class="md-nav__icon md-icon"></span>
          Recommended Data Analysis
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../networking/">
        Molecular Networking
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_10_2" id="__nav_10_2" type="checkbox"/>
<label class="md-nav__link" for="__nav_10_2">
        GC-MS Molecular Networking
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="GC-MS Molecular Networking" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_10_2">
<span class="md-nav__icon md-icon"></span>
          GC-MS Molecular Networking
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../gcanalysis/">
        GC-MS on GNPS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gc-ms-deconvolution/">
        GC-MS Deconvolution
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gc-ms-library-molecular-network/">
        GC-MS Library Search and Molecular Networking
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../networkingviews/">
        Molecular Networking Views
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../librarysearch/">
        Spectral Library Search
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../masst/">
        MASST Search (Mass Search Tool)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../resultviews/">
        Result Views Navigation
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_11" id="__nav_11" type="checkbox"/>
<label class="md-nav__link" for="__nav_11">
        Mass Spec Data Visualization
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Mass Spec Data Visualization" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_11">
<span class="md-nav__icon md-icon"></span>
          Mass Spec Data Visualization
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../lcms-dashboard/">
        GNPS Dashboard
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gnpsdatasetexplorer/">
        GNPS Dataset Explorer
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--active md-nav__item--nested">
<input checked="" class="md-nav__toggle md-toggle" data-md-toggle="__nav_12" id="__nav_12" type="checkbox"/>
<label class="md-nav__link" for="__nav_12">
        Advanced Data Analysis
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Advanced Data Analysis" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_12">
<span class="md-nav__icon md-icon"></span>
          Advanced Data Analysis
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../networkingfiles/">
        Molecular Networking Files
      </a>
</li>
<li class="md-nav__item md-nav__item--active md-nav__item--nested">
<input checked="" class="md-nav__toggle md-toggle" data-md-toggle="__nav_12_2" id="__nav_12_2" type="checkbox"/>
<label class="md-nav__link" for="__nav_12_2">
        Feature Based Molecular Networking
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Feature Based Molecular Networking" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_12_2">
<span class="md-nav__icon md-icon"></span>
          Feature Based Molecular Networking
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking/">
        FBMN Workflow
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-mzmine2/">
        FBMN with MZmine
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-ms-dial/">
        FBMN with MS-DIAL
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-xcms3/">
        FBMN with XCMS
      </a>
</li>
<li class="md-nav__item md-nav__item--active">
<input class="md-nav__toggle md-toggle" data-md-toggle="toc" id="__toc" type="checkbox"/>
<label class="md-nav__link md-nav__link--active" for="__toc">
          FBMN with OpenMS
          <span class="md-nav__icon md-icon"></span>
</label>
<a class="md-nav__link md-nav__link--active" href="./">
        FBMN with OpenMS
      </a>
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#introduction-to-fbmn-with-openms">
    Introduction to FBMN with OpenMS
  </a>
<nav aria-label="Introduction to FBMN with OpenMS" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#citations">
    Citations
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#development">
    Development
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#feature-detection-with-openms-for-the-fbmn">
    Feature Detection with OpenMS for the FBMN
  </a>
<nav aria-label="Feature Detection with OpenMS for the FBMN" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#running-the-openms-gnps-pipeline-on-gnps-web-platform">
    Running the OpenMS-GNPS pipeline on GNPS web-platform
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#running-the-openms-gnps-pipeline-with-openms-topp-tools">
    Running the OpenMS-GNPS pipeline with OpenMS TOPP tools
  </a>
<nav aria-label="Running the OpenMS-GNPS pipeline with OpenMS TOPP tools" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#installation-and-requirement">
    Installation and requirement
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#representative-openms-gnps-workflow">
    Representative OpenMS-GNPS workflow
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#requirements-for-the-openms-gnps-pipeline">
    Requirements for the OpenMS-GNPS pipeline
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#the-gnpsexport-topp-tool">
    The GNPSExport TOPP tool
  </a>
<nav aria-label="The GNPSExport TOPP tool" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#usage-and-options-for-gnpsexport-topp-tool">
    Usage and Options for GNPSExport TOPP tool:
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#options-for-the-consensus-msms-spectra-output_type">
    Options for the consensus MS/MS spectra: output_type
  </a>
</li>
</ul>
</nav>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#running-the-fbmn-workflow-with-openms-files">
    Running the FBMN workflow with OpenMS files
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#join-the-gnps-community">
    Join the GNPS Community !
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#page-contributors">
    Page Contributors
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-metaboscape/">
        FBMN with MetaboScape
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-progenesisQI/">
        FBMN with Progenesis QI
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-mztab-m/">
        FBMN with mzTab-M
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-cytoscape/">
        FBMN export to Cytoscape
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_12_3" id="__nav_12_3" type="checkbox"/>
<label class="md-nav__link" for="__nav_12_3">
        FBMN - Ion Identity Networking
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="FBMN - Ion Identity Networking" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_12_3">
<span class="md-nav__icon md-icon"></span>
          FBMN - Ion Identity Networking
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin/">
        IIN Workflow
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin-mzmine/">
        IIN with MZmine2
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin-xcms/">
        IIN with XCMS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin-msdial/">
        IIN with MS-DIAL
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../ms2lda/">
        MS2LDA and MotifDB Substructure Discovery
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../cytoscape/">
        Molecular Networking in Cytoscape
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../dereplicator/">
        DEREPLICATOR
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../nap/">
        Network Annotation Propagation (NAP)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../ms2lda/">
        MS2LDA and MotifDB Substructure Discovery
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../molnetenhancer/">
        MolNetEnhancer
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../metaminer/">
        MetaMiner
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../sirius/">
        Sirius
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../ReDU/">
        Reanalysis of Data User Interface (ReDU)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../qemistree/">
        Qemistree
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../mmvec/">
        MMVEC
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../mergepolarity/">
        Merge Network Polarity
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../chemdir/">
        Chemical Directionality
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../usi/">
        Spectrum Resolover
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_13" id="__nav_13" type="checkbox"/>
<label class="md-nav__link" for="__nav_13">
        Spectral Libraries
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Spectral Libraries" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_13">
<span class="md-nav__icon md-icon"></span>
          Spectral Libraries
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../browselibraries/">
        Browsing Reference Spectral Libraries
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../spectrumcuration/">
        Add/Curate Reference Spectrum in the Library
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../batchupload/">
        Reference Spectral Library Batch Addition
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../downloadlibraries/">
        Downloading Reference Spectral Libraries
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_13_5" id="__nav_13_5" type="checkbox"/>
<label class="md-nav__link" for="__nav_13_5">
        Open-source Method for Reference Library Generation
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Open-source Method for Reference Library Generation" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_13_5">
<span class="md-nav__icon md-icon"></span>
          Open-source Method for Reference Library Generation
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../msmschooser/">
        MSMS-Chooser Workflow
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../msmschooserdataacquisitioncomputational/">
        Computational Data Acquisition Prep
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../msmschooserdataacquisitionQE/">
        Data Acqusition - Thermo QExactive (v1.0)
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tag_info/">
        GNPS Tag Contribution and Analyses
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_14" id="__nav_14" type="checkbox"/>
<label class="md-nav__link" for="__nav_14">
        Create a Public Datasets
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Create a Public Datasets" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_14">
<span class="md-nav__icon md-icon"></span>
          Create a Public Datasets
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../datasets/">
        Dataset Creation/Sharing
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../continuousid/">
        Continuous Identification
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../moleculeexplorer/">
        Molecule Explorer
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_15" id="__nav_15" type="checkbox"/>
<label class="md-nav__link" for="__nav_15">
        Community/Social Features
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Community/Social Features" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_15">
<span class="md-nav__icon md-icon"></span>
          Community/Social Features
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../user/">
        User Page
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../troubleshooting/">
        Troubleshooting
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_17" id="__nav_17" type="checkbox"/>
<label class="md-nav__link" for="__nav_17">
        Tutorial Guides
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Tutorial Guides" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_17">
<span class="md-nav__icon md-icon"></span>
          Tutorial Guides
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/bacterianetworking/">
        Molecular Networking - Analog Discovery in Microbial Extracts
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/americangutmzmine/">
        Feature-Based Molecular Networking (FBMN) with MZmine2 - Microbiome Molecules in Fecal Samples in American Gut Project
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/americangut-ms-dial/">
        FBMN with MS-DIAL - Microbiome Molecules in Fecal Samples in American Gut Project
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/coffee-tutorial-metaboscape/">
        FBMN with MetaboScape - Coffee Analysis
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/featurebasedgnps/">
        FBMN analysis at GNPS
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../changelog/">
        Change Log
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gnps_community/">
        Join the GNPS community
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../workshops/">
        GNPS Workshops/Training
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../logos/">
        GNPS Logos
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../contact/">
        Contact/Office Hours
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../toolindex/">
        GNPS Tool Index
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../api/">
        API
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../developers/">
        For Developers
      </a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#introduction-to-fbmn-with-openms">
    Introduction to FBMN with OpenMS
  </a>
<nav aria-label="Introduction to FBMN with OpenMS" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#citations">
    Citations
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#development">
    Development
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#feature-detection-with-openms-for-the-fbmn">
    Feature Detection with OpenMS for the FBMN
  </a>
<nav aria-label="Feature Detection with OpenMS for the FBMN" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#running-the-openms-gnps-pipeline-on-gnps-web-platform">
    Running the OpenMS-GNPS pipeline on GNPS web-platform
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#running-the-openms-gnps-pipeline-with-openms-topp-tools">
    Running the OpenMS-GNPS pipeline with OpenMS TOPP tools
  </a>
<nav aria-label="Running the OpenMS-GNPS pipeline with OpenMS TOPP tools" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#installation-and-requirement">
    Installation and requirement
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#representative-openms-gnps-workflow">
    Representative OpenMS-GNPS workflow
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#requirements-for-the-openms-gnps-pipeline">
    Requirements for the OpenMS-GNPS pipeline
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#the-gnpsexport-topp-tool">
    The GNPSExport TOPP tool
  </a>
<nav aria-label="The GNPSExport TOPP tool" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#usage-and-options-for-gnpsexport-topp-tool">
    Usage and Options for GNPSExport TOPP tool:
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#options-for-the-consensus-msms-spectra-output_type">
    Options for the consensus MS/MS spectra: output_type
  </a>
</li>
</ul>
</nav>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#running-the-fbmn-workflow-with-openms-files">
    Running the FBMN workflow with OpenMS files
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#join-the-gnps-community">
    Join the GNPS Community !
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#page-contributors">
    Page Contributors
  </a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-content" data-md-component="content">
<article class="md-content__inner md-typeset">
<a class="md-content__button md-icon" href="https://github.com/CCMS-UCSD/GNPSDocumentation/edit/master/docs/featurebasedmolecularnetworking-with-openms.md" title="Edit this page">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25z"></path></svg>
</a>
<h1>FBMN with OpenMS</h1>
<h2 id="introduction-to-fbmn-with-openms">Introduction to FBMN with OpenMS<a class="headerlink" href="#introduction-to-fbmn-with-openms" title="Permanent link">¶</a></h2>
<p>The main documentation for <strong>Feature-Based Molecular Networking</strong> <a href="../featurebasedmolecularnetworking/">can be accessed here</a>. See <a href="https://www.nature.com/articles/s41592-020-0933-6">our article</a>.</p>
<p>Below we are describing how to use <strong>OpenMS</strong> with the FBMN workflow on GNPS. </p>
<h3 id="citations">Citations<a class="headerlink" href="#citations" title="Permanent link">¶</a></h3>
<div class="admonition quote">
<p class="admonition-title">Recommended Citations</p>
<p>This work builds on the efforts and tools from our many colleagues, please cite their work:</p>
<p>Nothias, L.-F., Petras, D., Schmid, R. et al. <a href="https://www.nature.com/articles/s41592-020-0933-6">Feature-based molecular networking in the GNPS analysis environment</a>. Nat. Methods 17, 905–908 (2020).</p>
<p>Wang, M. et al. <a href="https://doi.org/10.1038/nbt.3597">Sharing and community curation of mass spectrometry data with Global Natural Products Social Molecular Networking</a>. Nat. Biotechnol. 34, 828–837 (2016).</p>
<p>Röst, H. L. et al. OpenMS: a flexible open-source software platform for mass spectrometry data analysis. Nat. Methods 13, 741–748 (2016). <a href="https://doi.org/10.1038/nmeth.3959">https://doi.org/10.1038/nmeth.3959</a></p>
</div>
<h3 id="development">Development<a class="headerlink" href="#development" title="Permanent link">¶</a></h3>
<p>The OpenMS website at <a href="https://www.openms.de/">https://www.openms.de/</a></p>
<p>The GNPSExport TOPP tool code can be found on <a href="https://github.com/OpenMS/OpenMS">OpenMS GitHub repository</a> and the lastest version on <a href="https://github.com/Bioinformatic-squad-DorresteinLab/OpenMS/blob/develop/src/topp/GNPSExport.cpp">this fork repository</a>.</p>
<p>The code for the Open-GNPS pipeline (ProteoSAFe workflow and python wrapper) is available on <a href="https://github.com/Bioinformatic-squad-DorresteinLab/openms-gnps-workflow">this GitHub repository</a>.</p>
<h2 id="feature-detection-with-openms-for-the-fbmn">Feature Detection with OpenMS for the FBMN<a class="headerlink" href="#feature-detection-with-openms-for-the-fbmn" title="Permanent link">¶</a></h2>
<p>We have developed an OpenMS-GNPS pipeline that can be used for the processing of metabolomics LC-MS data with OpenMS and Feature Based Molecular Networking (FBMN). This pipeline uses OpenMS tools and the GNPSExport (part of OpenMS TOPP tools).</p>
<p>In brief, after running an <em>OpenMS "metabolomics" pipeline</em>, the <em>GNPSExport TOPP tool</em> can be used on the consensusXML file and corresponding mzML files to generate the files needed for FBMN on GNPS. These two files are:</p>
<ul>
<li>The <strong>feature quantification table</strong> (.TXT format) which is generated with the TextExport tool.</li>
<li>The <strong>MS2 spectral summary</strong> file (.MGF format) which is generated with the GNPSExport tool.</li>
</ul>
<h3 id="running-the-openms-gnps-pipeline-on-gnps-web-platform">Running the OpenMS-GNPS pipeline on GNPS web-platform<a class="headerlink" href="#running-the-openms-gnps-pipeline-on-gnps-web-platform" title="Permanent link">¶</a></h3>
<p>The OpenMS-GNPS pipeline is an experimetal workflow deployed currently on <em>proteomics2.ucsd.edu</em> [https://proteomics2.ucsd.edu/ProteoSAFe/]). The job can be configured as follows:</p>
<p><img alt="img" src="../img/openms/workflow_1.png"/></p>
<ol>
<li>Log into <a href="https://proteomics2.ucsd.edu/">https://proteomics2.ucsd.edu</a> (Create an account if necessary).</li>
<li>Access the lastest version of the OpenMS + GNPS workflow (release 20) at <a href="https://proteomics2.ucsd.edu/ProteoSAFe/index.jsp?params=%7B%22workflow%22%3A%20%22OPENMS%22%2C%20%22workflow_version%22%3A%20%22release_20%22%7D">this address</a>.</li>
<li>In Import Data Files, select the input mzML files (prefered) or mzXML files (not recommended, because the pipeline would have to perform conversion to mzML)</li>
<li>[TO BE RELEASED] Select the parameters from the presets <code>HPLC-Q-Exactive, UHPLC-Q-Exactive, HPLC-QTOF, UHPLC-Q-Exactive</code>.
The corresponding OpenMS configuration files (.INI files) are available from that [GitHub repository] (https://github.com/Bioinformatic-squad-DorresteinLab/openms-gnps-workflow/presets/)). Alternatively, you can upload your OpenMS TOPP tool <em>parameter file</em> (.INI files). Note that <em>parameter files</em> can be updated with a text editor or with the <em>INIFileEditor</em> TOPP tool.</li>
<li>Wait for the job to complete. You will get a notification.</li>
<li>Download the main output files with "Download OpenMS Output Files" and/or all the files with "Download Workflow Files".</li>
<li>Unzip the archive.</li>
<li>Examine the consensusXML file (in the <em>filefilter</em> folder) with TOPPAS.</li>
<li>The <code>gnpsexport</code> folder contains the <em>MS/MS spectral summary file</em> (.MGF file)</li>
<li>The <code>textexporter</code> folder contains the <em>feature quantification table</em> (.TXT file).</li>
<li>Click a FBMN workflow on GNPS by clicking on "<em>Run a Feature-Based Molecular Networking at GNPS</em>" Note that you have to be logged in to GNPS first, <a href="https://gnps.ucsd.edu">https://gnps.ucsd.edu</a>). See documentation for <a href="https://ccms-ucsd.github.io/GNPSDocumentation/featurebasedmolecularnetworking/">the FBMN workflow on GNPS</a>.</li>
<li>Alternatively, upload these files via FTP for <a href="https://ccms-ucsd.github.io/GNPSDocumentation/featurebasedmolecularnetworking/">FBMN on GNPS</a>, or use them for other annotation tools (<a href="../dereplicator/">DEREPLICATOR</a>, <a href="../ms2lda/">Mass2Motif</a>, <a href="../nap/">NAP</a>.</li>
</ol>
<p><img alt="img" src="../img/openms/results_1.png"/></p>
<h3 id="running-the-openms-gnps-pipeline-with-openms-topp-tools">Running the OpenMS-GNPS pipeline with OpenMS TOPP tools<a class="headerlink" href="#running-the-openms-gnps-pipeline-with-openms-topp-tools" title="Permanent link">¶</a></h3>
<h5 id="installation-and-requirement">Installation and requirement<a class="headerlink" href="#installation-and-requirement" title="Permanent link">¶</a></h5>
<p>Installing OpenMS TOPP tools: See the OpenMS website at <a href="https://www.openms.de/">https://www.openms.de/</a></p>
<h5 id="representative-openms-gnps-workflow">Representative OpenMS-GNPS workflow<a class="headerlink" href="#representative-openms-gnps-workflow" title="Permanent link">¶</a></h5>
<p>A representative OpenMS-GNPS workflow would sequencially use these OpenMS TOPP tools:</p>
<ol>
<li>Input the mzML files.</li>
<li>Run the <em>FeatureFinderMetabo</em> tool on the mzML files.</li>
<li>Run the <em>IDMapper</em> tool on the featureXML and mzML files.</li>
<li>Run the <em>MapAlignerPoseClustering</em> tool on the featureXML files.</li>
<li>Run the <em>MetaboliteAdductDecharger</em> tool on the featureXML files.</li>
<li>Run the <em>FeatureLinkerUnlabeledKD</em> tool or <em>FeatureLinkerUnlabeledQT</em>, on the featureXML files and output a consensusXML file.</li>
<li>Run the <em>FileFilter</em> tool on the consensusXML file to keep only consensusElements with at least MS/MS scan (peptide annotation).  </li>
<li>Run the <em>GNPSExport</em> tool on the "filtered consensusXML file" to export an .MGF file.</li>
<li>Run the <em>TextExport</em> tool on the "filtered consensusXML file" to export a .TXT file.</li>
<li>Upload your files to GNPS and run the Feature-Based Molecular Networking workflow. See the main documentation for <a href="https://ccms-ucsd.github.io/GNPSDocumentation/featurebasedmolecularnetworking/">using FBMN on GNPS</a>.</li>
</ol>
<h5 id="requirements-for-the-openms-gnps-pipeline">Requirements for the OpenMS-GNPS pipeline<a class="headerlink" href="#requirements-for-the-openms-gnps-pipeline" title="Permanent link">¶</a></h5>
<ul>
<li>The <em>IDMapper</em> has to be run on the featureXML files, in order to associate MS2 scan(s) (peptide annotation) with each feature. These peptide annotations are used by the GNPSExport.</li>
<li>The <em>FileFilter</em> has to be run on the consensusXML file, prior to the GNPSExport, in order to remove consensusElements without MS2 scans (peptide annotation).</li>
<li>Note that mass accuracy and the retention time window for the pairing between MS2 scan(s) and a LC-MS feature
or consensusElement is defined at the IDMapper tool step for features.</li>
</ul>
<h4 id="the-gnpsexport-topp-tool">The GNPSExport TOPP tool<a class="headerlink" href="#the-gnpsexport-topp-tool" title="Permanent link">¶</a></h4>
<p>The <em>GNPSExport TOPP tool</em> can be run on the consensusXML file and corresponding mzML files to generate a MS/MS spectral file (MGF format) and corresponding feature quantification table (.TXT format) that contains LC-MS peak area intensities.</p>
<p>For each <em>consensusElement</em> in the consensusXML file, <em>GNPSExport</em> produces one representative <em>consensus MS/MS spectrum</em> (named <em>peptide annotation</em> in OpenMS jargon) saved to the <em>MS/MS spectral summary file</em> (.MGF file). Several modes for the generation of the <em>consensus MS/MS spectrum</em> are available and described below. Note that these parameters are defined in the GNPSExport parameters file (.INI file, <a href="../openms_gnpsexport/GNPSExport.ini">available with that link</a>).</p>
<h5 id="usage-and-options-for-gnpsexport-topp-tool">Usage and Options for GNPSExport TOPP tool:<a class="headerlink" href="#usage-and-options-for-gnpsexport-topp-tool" title="Permanent link">¶</a></h5>
<p><code>GNPSExport --help</code></p>
<div class="highlight"><pre><span></span><code>GNPSExport -ini iniFile-GNPSExport.ini
           -in_cm filefilter.consensusXML
           -in_mzml inputFile0.mzML inputFile1.mzML
           -out GNPSExport_output.mgf
</code></pre></div>
<h5 id="options-for-the-consensus-msms-spectra-output_type">Options for the consensus MS/MS spectra: <code>output_type</code><a class="headerlink" href="#options-for-the-consensus-msms-spectra-output_type" title="Permanent link">¶</a></h5>
<ul>
<li>
<p><strong>Most intense</strong> [default]: <code>most_intense</code> - For each <em>consensusElement</em>, the GNPSExport will output the <em>most intense</em> MS/MS scan (with the highest precursor ion intensity) as <em>representative MS/MS spectrum</em>.</p>
</li>
<li>
<p><strong>All MS/MS</strong>: <code>full_spectra</code> - For each <em>consensusElement</em>, the GNPSExport will output <em>All MS/MS scans</em>.</p>
</li>
<li>
<p><strong>Merge</strong> [Experimental]: <code>merged_spectra</code> - For each <em>consensusElement</em>, the GNPSExport will merge all eligible MS/MS scans into one representative <em>MS/MS spectrum</em>. Eligible MS/MS scans have a pairwise <em>cosine similarity</em> with the MS/MS scan of highest precursor intensity above the <em>Cosine Similarity Treshold</em>. The fragment ions of merged MS/MS scans are binned in m/z (or Da) range defined by the <em>Binning width</em> parameter.</p>
<ul>
<li><strong>Cosine Similarity Treshold</strong>: <code>merged_spectra:cos_similarity</code> (float, default: 0.9) - Parameter that defines <em>Cosine Similarity Treshold</em> for the pairwise <em>cosine similarity</em> between the MS/MS scan with the highest precursor intensity and the other MS/MS scans.</li>
<li><strong>Binning width</strong>: <code>merged_spectra:ms2_binned_size</code> (float, default: 0.02 Daltons) -  Parameter that defines the <em>Binning width</em> of fragment ions during the merging of eligible MS/MS spectra.</li>
</ul>
</li>
</ul>
<h3 id="running-the-fbmn-workflow-with-openms-files">Running the FBMN workflow with OpenMS files<a class="headerlink" href="#running-the-fbmn-workflow-with-openms-files" title="Permanent link">¶</a></h3>
<p>After the processing with OpenMS, the output files can be used to run the Feature-Based Molecular Networking workflow on GNPS either using the [Superquick FBMN start page] (http://dorresteinappshub.ucsd.edu:5050/featurebasednetworking) or <a href="https://gnps.ucsd.edu/ProteoSAFe/index.jsp?params=%7B%22workflow%22:%22FEATURE-BASED-MOLECULAR-NETWORKING%22,%22library_on_server%22:%22d.speclibs;%22%7D">the standard interface of the FBMN workflow</a> (you need to be logged in GNPS first).</p>
<p>Make sure to select the correct table source (OpenMS).</p>
<p>The main documentation for FBMN <a href="../featurebasedmolecularnetworking/">can be accessed here</a>. For metadata format requirements, see <a href="../metadata/">this page</a></p>
<h3 id="join-the-gnps-community">Join the GNPS Community !<a class="headerlink" href="#join-the-gnps-community" title="Permanent link">¶</a></h3>
<ul>
<li>For feature request, or to report bugs, please open an "Issue" on the <a href="https://github.com/CCMS-UCSD/GNPS_Workflows"><em>CCMS-UCSD/GNPS_Workflows</em> GitHub repository</a>.</li>
<li>To contribute to the GNPS documentation, please use GitHub by forking the <a href="https://github.com/CCMS-UCSD/GNPSDocumentation"><em>CCMS-UCSD/GNPSDocumentation</em></a> repository, and make a "Pull Request" with the changes.</li>
</ul>
<h2 id="page-contributors">Page Contributors<a class="headerlink" href="#page-contributors" title="Permanent link">¶</a></h2>
<p><span class="git-page-authors git-authors"><a href="mailto:mwang87@gmail.com">Ming Wang</a> (7.2%), <a href="mailto:louisfelix.nothias@gmail.com">lfnothias</a> (84.0%), <a href="mailto:iprotsyuk@google.com">Ivan Protsyuk</a> (8.8%)</span></p>
<hr/>
<div class="md-source-date">
<small>
    
      Last update: <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-datetime">October 13, 2020 22:14:44</span>
</small>
</div>
</article>
</div>
</div>
</main>
<footer class="md-footer">
<nav aria-label="Footer" class="md-footer__inner md-grid">
<a class="md-footer__link md-footer__link--prev" href="../featurebasedmolecularnetworking-with-xcms3/" rel="prev">
<div class="md-footer__button md-icon">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"></path></svg>
</div>
<div class="md-footer__title">
<div class="md-ellipsis">
<span class="md-footer__direction">
                Previous
              </span>
              FBMN with XCMS
            </div>
</div>
</a>
<a class="md-footer__link md-footer__link--next" href="../featurebasedmolecularnetworking-with-metaboscape/" rel="next">
<div class="md-footer__title">
<div class="md-ellipsis">
<span class="md-footer__direction">
                Next
              </span>
              FBMN with MetaboScape
            </div>
</div>
<div class="md-footer__button md-icon">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4z"></path></svg>
</div>
</a>
</nav>
<div class="md-footer-meta md-typeset">
<div class="md-footer-meta__inner md-grid">
<div class="md-footer-copyright">
        
        Made with
        <a href="https://squidfunk.github.io/mkdocs-material/" rel="noopener" target="_blank">
          Material for MkDocs
        </a>
</div>
</div>
</div>
</footer>
</div>
<div class="md-dialog" data-md-component="dialog">
<div class="md-dialog__inner md-typeset"></div>
</div>
<script id="__config" type="application/json">{"base": "..", "features": [], "translations": {"clipboard.copy": "Copy to clipboard", "clipboard.copied": "Copied to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.placeholder": "Type to start searching", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.term.missing": "Missing"}, "search": "../assets/javascripts/workers/search.f5903571.min.js", "version": null}</script>
<script src="../assets/javascripts/bundle.3981fc68.min.js"></script>
</body>
</html>