


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
        <link rel="canonical" href="https://ccms-ucsd.github.io/GNPSDocumentation/qemistree/">
      
      
      <link rel="shortcut icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.1.2, mkdocs-material-5.3.3">
    
    
      
        <title>Qemistree - GNPS Documentation</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.fe0cca5b.min.css">
      
      
    
    
    
      
        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700%7CRoboto+Mono&display=fallback">
        <style>body,input{font-family:"Roboto",-apple-system,BlinkMacSystemFont,Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono",SFMono-Regular,Consolas,Menlo,monospace}</style>
      
    
    
    
    
      
        
<script>window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)},ga.l=+new Date,ga("create","UA-8412213-5","ccms-ucsd.github.io"),ga("set","anonymizeIp",!0),ga("send","pageview"),document.addEventListener("DOMContentLoaded",function(){document.forms.search&&document.forms.search.query.addEventListener("blur",function(){if(this.value){var e=document.location.pathname;ga("send","pageview",e+"?q="+this.value)}})}),document.addEventListener("DOMContentSwitch",function(){ga("send","pageview")})</script>
<script async src="https://www.google-analytics.com/analytics.js"></script>
      
    
    
  </head>
  
  
    <body dir="ltr">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#page-contributors" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
      <header class="md-header" data-md-component="header">
  <nav class="md-header-nav md-grid" aria-label="Header">
    <a href="https://ccms-ucsd.github.io/GNPSDocumentation/" title="GNPS Documentation" class="md-header-nav__button md-logo" aria-label="GNPS Documentation">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 003-3 3 3 0 00-3-3 3 3 0 00-3 3 3 3 0 003 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"/></svg>

    </a>
    <label class="md-header-nav__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2z"/></svg>
    </label>
    <div class="md-header-nav__title" data-md-component="header-title">
      
        <div class="md-header-nav__ellipsis">
          <span class="md-header-nav__topic md-ellipsis">
            GNPS Documentation
          </span>
          <span class="md-header-nav__topic md-ellipsis">
            
              Qemistree
            
          </span>
        </div>
      
    </div>
    
      <label class="md-header-nav__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0116 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 019.5 16 6.5 6.5 0 013 9.5 6.5 6.5 0 019.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg>
      </label>
      
<div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" data-md-state="active">
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0116 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 019.5 16 6.5 6.5 0 013 9.5 6.5 6.5 0 019.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
      </label>
      <button type="reset" class="md-search__icon md-icon" aria-label="Clear" data-md-component="search-reset" tabindex="-1">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"/></svg>
      </button>
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header-nav__source">
        
<a href="https://github.com/CCMS-UCSD/GNPSDocumentation/" title="Go to repository" class="md-source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path d="M439.55 236.05L244 40.45a28.87 28.87 0 00-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 01-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 000 40.81l195.61 195.6a28.86 28.86 0 0040.8 0l194.69-194.69a28.86 28.86 0 000-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    CCMS-UCSD/GNPSDocumentation
  </div>
</a>
      </div>
    
  </nav>
</header>
    
    <div class="md-container" data-md-component="container">
      
        
      
      
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              <div class="md-sidebar md-sidebar--primary" data-md-component="navigation">
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    <nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="https://ccms-ucsd.github.io/GNPSDocumentation/" title="GNPS Documentation" class="md-nav__button md-logo" aria-label="GNPS Documentation">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 003-3 3 3 0 00-3-3 3 3 0 00-3 3 3 3 0 003 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"/></svg>

    </a>
    GNPS Documentation
  </label>
  
    <div class="md-nav__source">
      
<a href="https://github.com/CCMS-UCSD/GNPSDocumentation/" title="Go to repository" class="md-source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path d="M439.55 236.05L244 40.45a28.87 28.87 0 00-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 01-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 000 40.81l195.61 195.6a28.86 28.86 0 0040.8 0l194.69-194.69a28.86 28.86 0 000-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    CCMS-UCSD/GNPSDocumentation
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      


  <li class="md-nav__item">
    <a href=".." title="GNPS Introduction" class="md-nav__link">
      GNPS Introduction
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../isgnpsright/" title="Is my data compatible with GNPS?" class="md-nav__link">
      Is my data compatible with GNPS?
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../faq/" title="Frequently Asked Questions" class="md-nav__link">
      Frequently Asked Questions
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../superquickstart/" title="Superquick Start Guide" class="md-nav__link">
      Superquick Start Guide
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../quickstart/" title="Quick Start Guide" class="md-nav__link">
      Quick Start Guide
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../gnpsanalysisoverview/" title="GNPS Analysis Tools Overview" class="md-nav__link">
      GNPS Analysis Tools Overview
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../lexicon/" title="GNPS Lexicon" class="md-nav__link">
      GNPS Lexicon
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-nav__toggle md-toggle" data-md-toggle="nav-8" type="checkbox" id="nav-8">
    
    <label class="md-nav__link" for="nav-8">
      Mass Spec/Computational Background
      <span class="md-nav__icon md-icon">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M8.59 16.58L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.42z"/></svg>
      </span>
    </label>
    <nav class="md-nav" aria-label="Mass Spec/Computational Background" data-md-level="1">
      <label class="md-nav__title" for="nav-8">
        <span class="md-nav__icon md-icon">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
        </span>
        Mass Spec/Computational Background
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../massspecbackground/networkingtheory/" title="Molecular Networking" class="md-nav__link">
      Molecular Networking
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-nav__toggle md-toggle" data-md-toggle="nav-9" type="checkbox" id="nav-9">
    
    <label class="md-nav__link" for="nav-9">
      Data Preparation/Upload
      <span class="md-nav__icon md-icon">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M8.59 16.58L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.42z"/></svg>
      </span>
    </label>
    <nav class="md-nav" aria-label="Data Preparation/Upload" data-md-level="1">
      <label class="md-nav__title" for="nav-9">
        <span class="md-nav__icon md-icon">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
        </span>
        Data Preparation/Upload
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../fileconversion/" title="Mass Spectrometry File Conversion" class="md-nav__link">
      Mass Spectrometry File Conversion
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../fileupload/" title="GNPS File Upload" class="md-nav__link">
      GNPS File Upload
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../metadata/" title="Metadata" class="md-nav__link">
      Metadata
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-nav__toggle md-toggle" data-md-toggle="nav-10" type="checkbox" id="nav-10">
    
    <label class="md-nav__link" for="nav-10">
      Recommended Data Analysis
      <span class="md-nav__icon md-icon">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M8.59 16.58L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.42z"/></svg>
      </span>
    </label>
    <nav class="md-nav" aria-label="Recommended Data Analysis" data-md-level="1">
      <label class="md-nav__title" for="nav-10">
        <span class="md-nav__icon md-icon">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
        </span>
        Recommended Data Analysis
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../networking/" title="Molecular Networking" class="md-nav__link">
      Molecular Networking
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-nav__toggle md-toggle" data-md-toggle="nav-10-2" type="checkbox" id="nav-10-2">
    
    <label class="md-nav__link" for="nav-10-2">
      GC-MS Molecular Networking
      <span class="md-nav__icon md-icon">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M8.59 16.58L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.42z"/></svg>
      </span>
    </label>
    <nav class="md-nav" aria-label="GC-MS Molecular Networking" data-md-level="2">
      <label class="md-nav__title" for="nav-10-2">
        <span class="md-nav__icon md-icon">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
        </span>
        GC-MS Molecular Networking
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../gcanalysis/" title="GC-MS on GNPS" class="md-nav__link">
      GC-MS on GNPS
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../gc-ms-deconvolution/" title="GC-MS Deconvolution" class="md-nav__link">
      GC-MS Deconvolution
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../gc-ms-library-molecular-network/" title="GC-MS Library Search and Molecular Networking" class="md-nav__link">
      GC-MS Library Search and Molecular Networking
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../networkingviews/" title="Molecular Networking Views" class="md-nav__link">
      Molecular Networking Views
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../librarysearch/" title="Spectral Library Search" class="md-nav__link">
      Spectral Library Search
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../masst/" title="MASST Search (Mass Search Tool)" class="md-nav__link">
      MASST Search (Mass Search Tool)
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../resultviews/" title="Result Views Navigation" class="md-nav__link">
      Result Views Navigation
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      

  


  <li class="md-nav__item md-nav__item--active md-nav__item--nested">
    
      <input class="md-nav__toggle md-toggle" data-md-toggle="nav-11" type="checkbox" id="nav-11" checked>
    
    <label class="md-nav__link" for="nav-11">
      Advanced Data Analysis
      <span class="md-nav__icon md-icon">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M8.59 16.58L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.42z"/></svg>
      </span>
    </label>
    <nav class="md-nav" aria-label="Advanced Data Analysis" data-md-level="1">
      <label class="md-nav__title" for="nav-11">
        <span class="md-nav__icon md-icon">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
        </span>
        Advanced Data Analysis
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-nav__toggle md-toggle" data-md-toggle="nav-11-1" type="checkbox" id="nav-11-1">
    
    <label class="md-nav__link" for="nav-11-1">
      Feature Based Molecular Networking
      <span class="md-nav__icon md-icon">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M8.59 16.58L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.42z"/></svg>
      </span>
    </label>
    <nav class="md-nav" aria-label="Feature Based Molecular Networking" data-md-level="2">
      <label class="md-nav__title" for="nav-11-1">
        <span class="md-nav__icon md-icon">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
        </span>
        Feature Based Molecular Networking
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking/" title="FBMN Workflow" class="md-nav__link">
      FBMN Workflow
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-mzmine2/" title="FBMN with MZmine" class="md-nav__link">
      FBMN with MZmine
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-ms-dial/" title="FBMN with MS-DIAL" class="md-nav__link">
      FBMN with MS-DIAL
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-xcms3/" title="FBMN with XCMS" class="md-nav__link">
      FBMN with XCMS
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-openms/" title="FBMN with OpenMS" class="md-nav__link">
      FBMN with OpenMS
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-metaboscape/" title="FBMN with MetaboScape" class="md-nav__link">
      FBMN with MetaboScape
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-progenesisQI/" title="FBMN with Progenesis QI" class="md-nav__link">
      FBMN with Progenesis QI
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-mztab-m/" title="FBMN with mzTab-M" class="md-nav__link">
      FBMN with mzTab-M
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-cytoscape/" title="FBMN export to Cytoscape" class="md-nav__link">
      FBMN export to Cytoscape
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

        
          
          
          


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-nav__toggle md-toggle" data-md-toggle="nav-11-2" type="checkbox" id="nav-11-2">
    
    <label class="md-nav__link" for="nav-11-2">
      FBMN - Ion Identity Networking
      <span class="md-nav__icon md-icon">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M8.59 16.58L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.42z"/></svg>
      </span>
    </label>
    <nav class="md-nav" aria-label="FBMN - Ion Identity Networking" data-md-level="2">
      <label class="md-nav__title" for="nav-11-2">
        <span class="md-nav__icon md-icon">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
        </span>
        FBMN - Ion Identity Networking
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../fbmn-iin/" title="IIN Workflow" class="md-nav__link">
      IIN Workflow
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../fbmn-iin-mzmine/" title="IIN with MZmine2" class="md-nav__link">
      IIN with MZmine2
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../fbmn-iin-xcms/" title="IIN with XCMS" class="md-nav__link">
      IIN with XCMS
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../fbmn-iin-msdial/" title="IIN with MS-DIAL" class="md-nav__link">
      IIN with MS-DIAL
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../ms2lda/" title="MS2LDA and MotifDB Substructure Discovery" class="md-nav__link">
      MS2LDA and MotifDB Substructure Discovery
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../cytoscape/" title="Molecular Networking in Cytoscape" class="md-nav__link">
      Molecular Networking in Cytoscape
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../dereplicator/" title="DEREPLICATOR" class="md-nav__link">
      DEREPLICATOR
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../nap/" title="Network Annotation Propagation (NAP)" class="md-nav__link">
      Network Annotation Propagation (NAP)
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../ms2lda/" title="MS2LDA and MotifDB Substructure Discovery" class="md-nav__link">
      MS2LDA and MotifDB Substructure Discovery
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../molnetenhancer/" title="MolNetEnhancer" class="md-nav__link">
      MolNetEnhancer
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../metaminer/" title="MetaMiner" class="md-nav__link">
      MetaMiner
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../ReDU/" title="Reanalysis of Data User Interface (ReDU)" class="md-nav__link">
      Reanalysis of Data User Interface (ReDU)
    </a>
  </li>

        
          
          
          

  


  <li class="md-nav__item md-nav__item--active">
    
    <input class="md-nav__toggle md-toggle" data-md-toggle="toc" type="checkbox" id="__toc">
    
    
      <label class="md-nav__link md-nav__link--active" for="__toc">
        Qemistree
        <span class="md-nav__icon md-icon">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 9h14V7H3v2m0 4h14v-2H3v2m0 4h14v-2H3v2m16 0h2v-2h-2v2m0-10v2h2V7h-2m0 6h2v-2h-2v2z"/></svg>
        </span>
      </label>
    
    <a href="./" title="Qemistree" class="md-nav__link md-nav__link--active">
      Qemistree
    </a>
    
      
<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
      </span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#page-contributors" class="md-nav__link">
    Page Contributors
  </a>
  
</li>
      
    </ul>
  
</nav>
    
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../mmvec/" title="MMVEC" class="md-nav__link">
      MMVEC
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../mergepolarity/" title="Merge Network Polarity" class="md-nav__link">
      Merge Network Polarity
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../chemdir/" title="Chemical Directionality" class="md-nav__link">
      Chemical Directionality
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-nav__toggle md-toggle" data-md-toggle="nav-12" type="checkbox" id="nav-12">
    
    <label class="md-nav__link" for="nav-12">
      Spectral Libraries
      <span class="md-nav__icon md-icon">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M8.59 16.58L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.42z"/></svg>
      </span>
    </label>
    <nav class="md-nav" aria-label="Spectral Libraries" data-md-level="1">
      <label class="md-nav__title" for="nav-12">
        <span class="md-nav__icon md-icon">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
        </span>
        Spectral Libraries
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../browselibraries/" title="Browsing Reference Spectral Libraries" class="md-nav__link">
      Browsing Reference Spectral Libraries
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../spectrumcuration/" title="Add/Curate Reference Spectrum in the Library" class="md-nav__link">
      Add/Curate Reference Spectrum in the Library
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../batchupload/" title="Reference Spectral Library Batch Addition" class="md-nav__link">
      Reference Spectral Library Batch Addition
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../downloadlibraries/" title="Downloading Reference Spectral Libraries" class="md-nav__link">
      Downloading Reference Spectral Libraries
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-nav__toggle md-toggle" data-md-toggle="nav-12-5" type="checkbox" id="nav-12-5">
    
    <label class="md-nav__link" for="nav-12-5">
      Open-source Method for Reference Library Generation
      <span class="md-nav__icon md-icon">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M8.59 16.58L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.42z"/></svg>
      </span>
    </label>
    <nav class="md-nav" aria-label="Open-source Method for Reference Library Generation" data-md-level="2">
      <label class="md-nav__title" for="nav-12-5">
        <span class="md-nav__icon md-icon">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
        </span>
        Open-source Method for Reference Library Generation
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../msmschooser/" title="MSMS-Chooser Workflow" class="md-nav__link">
      MSMS-Chooser Workflow
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../msmschooserdataacquisitioncomputational/" title="Computational Data Acquisition Prep" class="md-nav__link">
      Computational Data Acquisition Prep
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../msmschooserdataacquisitionQE/" title="Data Acqusition - Thermo QExactive (v1.0)" class="md-nav__link">
      Data Acqusition - Thermo QExactive (v1.0)
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../tag_info/" title="GNPS Tag Contribution and Analyses" class="md-nav__link">
      GNPS Tag Contribution and Analyses
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-nav__toggle md-toggle" data-md-toggle="nav-13" type="checkbox" id="nav-13">
    
    <label class="md-nav__link" for="nav-13">
      Create a Public Datasets
      <span class="md-nav__icon md-icon">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M8.59 16.58L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.42z"/></svg>
      </span>
    </label>
    <nav class="md-nav" aria-label="Create a Public Datasets" data-md-level="1">
      <label class="md-nav__title" for="nav-13">
        <span class="md-nav__icon md-icon">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
        </span>
        Create a Public Datasets
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../datasets/" title="Dataset Creation/Sharing" class="md-nav__link">
      Dataset Creation/Sharing
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../continuousid/" title="Continuous Identification" class="md-nav__link">
      Continuous Identification
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../moleculeexplorer/" title="Molecule Explorer" class="md-nav__link">
      Molecule Explorer
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-nav__toggle md-toggle" data-md-toggle="nav-14" type="checkbox" id="nav-14">
    
    <label class="md-nav__link" for="nav-14">
      Community/Social Features
      <span class="md-nav__icon md-icon">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M8.59 16.58L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.42z"/></svg>
      </span>
    </label>
    <nav class="md-nav" aria-label="Community/Social Features" data-md-level="1">
      <label class="md-nav__title" for="nav-14">
        <span class="md-nav__icon md-icon">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
        </span>
        Community/Social Features
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../user/" title="User Page" class="md-nav__link">
      User Page
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../troubleshooting/" title="Troubleshooting" class="md-nav__link">
      Troubleshooting
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-nav__toggle md-toggle" data-md-toggle="nav-16" type="checkbox" id="nav-16">
    
    <label class="md-nav__link" for="nav-16">
      Tutorial Guides
      <span class="md-nav__icon md-icon">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M8.59 16.58L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.42z"/></svg>
      </span>
    </label>
    <nav class="md-nav" aria-label="Tutorial Guides" data-md-level="1">
      <label class="md-nav__title" for="nav-16">
        <span class="md-nav__icon md-icon">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
        </span>
        Tutorial Guides
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorials/bacterianetworking/" title="Molecular Networking - Analog Discovery in Microbial Extracts" class="md-nav__link">
      Molecular Networking - Analog Discovery in Microbial Extracts
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorials/americangutmzmine/" title="Feature-Based Molecular Networking (FBMN) with MZmine2 - Microbiome Molecules in Fecal Samples in American Gut Project" class="md-nav__link">
      Feature-Based Molecular Networking (FBMN) with MZmine2 - Microbiome Molecules in Fecal Samples in American Gut Project
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorials/americangut-ms-dial/" title="FBMN with MS-DIAL - Microbiome Molecules in Fecal Samples in American Gut Project" class="md-nav__link">
      FBMN with MS-DIAL - Microbiome Molecules in Fecal Samples in American Gut Project
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorials/coffee-tutorial-metaboscape/" title="FBMN with MetaboScape - Coffee Analysis" class="md-nav__link">
      FBMN with MetaboScape - Coffee Analysis
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorials/featurebasedgnps/" title="FBMN analysis at GNPS" class="md-nav__link">
      FBMN analysis at GNPS
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../changelog/" title="Change Log" class="md-nav__link">
      Change Log
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../gnps_community/" title="Join the GNPS community" class="md-nav__link">
      Join the GNPS community
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../workshops/" title="GNPS Workshops/Training" class="md-nav__link">
      GNPS Workshops/Training
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../logos/" title="GNPS Logos" class="md-nav__link">
      GNPS Logos
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../contact/" title="Contact/Office Hours" class="md-nav__link">
      Contact/Office Hours
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../toolindex/" title="GNPS Tool Index" class="md-nav__link">
      GNPS Tool Index
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../api/" title="API" class="md-nav__link">
      API
    </a>
  </li>

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              <div class="md-sidebar md-sidebar--secondary" data-md-component="toc">
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    
<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
      </span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#page-contributors" class="md-nav__link">
    Page Contributors
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          <div class="md-content">
            <article class="md-content__inner md-typeset">
              
                
                  <a href="https://github.com/CCMS-UCSD/GNPSDocumentation/blob/master/docs/qemistree.md" title="Edit this page" class="md-content__button md-icon">
                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25z"/></svg>
                  </a>
                
                
                  
                
                
                  <h1>Qemistree</h1>
                
                <p><strong>QEMISTREE</strong> - a tool to represnt metabolomics data as trees to explore chemical diversity</p>
<p><em>Introduction</em>: Qemistree is a computation tool to build a tree of mass-spectrometry (LC-MS/MS) features to perform which enables the use of phylogeny-based tools to study chemical composition of samples. 
This documentation aims to provide a user-guide on how to run a Qemistree workflow on GNPS.</p>
<hr />
<p><strong>STEP 1: Collecting the right input files</strong></p>
<p>Users can perform QEMISTREE analysis in GNPS after processing LC-MS/MS data through <a href="https://ccms-ucsd.github.io/GNPSDocumentation/featurebasedmolecularnetworking/">Feature-Based Molecular Networking or FBMN</a> using <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-11-395">MZmine2</a>. The input files for Qemistree are:</p>
<p><strong>1. Required</strong>: A SIRIUS MGF file generated from the MZmine workflow</p>
<p><strong>2. Required</strong>: A quant table called <code>qiime2_table.qza</code> with all the identified features (an output of <a href="https://ccms-ucsd.github.io/GNPSDocumentation/featurebasedmolecularnetworking/">FBMN</a>, and found  within the folder <code>qiime2_output</code>)</p>
<p><strong>3. Optional</strong>: A metadata file called <code>qiime2_metadata.tsv</code> (an output of <a href="https://ccms-ucsd.github.io/GNPSDocumentation/featurebasedmolecularnetworking/">FBMN</a>, and also found in folder <code>qiime2_output</code>)</p>
<p><strong>4. Optional</strong>: A library identification file (TSV) (an output of <a href="https://ccms-ucsd.github.io/GNPSDocumentation/featurebasedmolecularnetworking/">FBMN</a>, and found in the folder <code>clusterinfo_summary</code>)</p>
<p>Follow the steps below to generate these files:</p>
<p>a. Follow the documentation for <a href="https://ccms-ucsd.github.io/GNPSDocumentation/featurebasedmolecularnetworking-with-mzmine2/">Feature-Based Molecular Networking using MZmine2</a> to generate an aligned feature list for your LC-MS/MS data and export the necessary MGF and CSV files for FBMN GNPS workflow.</p>
<p>b. While still in MZmine2, select your aligned feature list, then click on the tab for feature list methods and select Export/Import, followed by Export for SIRIUS.</p>
<p>c. Choose the Mass list that you used to generate your feature list, and specify a path and filename for your SIRIUS file. Click OK.</p>
<p>d. Go to GNPS server and run a <a href="https://gnps.ucsd.edu/ProteoSAFe/index.jsp?params=%7B%22workflow%22:%22FEATURE-BASED-MOLECULAR-NETWORKING%22,%22library_on_server%22:%22d.speclibs;%22%7D">FBMN</a> job using the GNPS quant CSV, GNPS MGF (both generated in step a, above) and <a href="https://mwang87.github.io/ReDU-MS2-Documentation/HowtoContribute/">ReDU</a> metadata file.</p>
<p>e. The Qemistree task utilizes the output files from FBMN. These can be downloaded by clicking on <code>Download qiime2 Emperor qzv</code> (<code>qiime2_table.qza</code> and <code>qiime2_metadata.tsv</code> from folder <code>qiime2_output</code> and  <code>clusterinfo_summary</code> for the tsv file within). You can start a Qemistree task by clicking on <code>Visualize with Qemistree</code> link on your FBMN status page </p>
<p><strong>Note</strong> <code>Visualize with Qemistree</code> option is available for FBMN release version &gt;= 20. If your FBMN job was run using an older version, please re-run the job by clicking on <code>Clone to Latest Version</code> to follow the rest of the tutorial.</p>
<hr />
<p><strong>STEP 2: Running a Qemistree job</strong></p>
<p><strong>1.</strong> Go to the status page of your FBMN job:</p>
<p><img alt="alt text" src="../img/qemistree/FBMN_statuspage.png" title="FBMN Status Page" /></p>
<p>Click on <code>Visualize with Qemistree</code> under the <code>Advanced Views - Experimental Views</code> to analyze your data using Qemistree. This method auto-populates the input fields <code>Quantification Table</code>, <code>Metadata Table</code> and <code>Library Identifications</code> from your corresponding FBMN job. Additionally, you need to upload your SIRIUS MGF file exported from Mzmine2 as describes in Step 1.</p>
<p><img alt="alt text" src="../img/qemistree/workflowpage_gnps.png" title="workflow" /></p>
<p><strong>2.</strong> Make sure the correct files are included as described in the beginning:</p>
<ol>
<li>Sirius MGF Spectrum file</li>
<li><code>qiime2_table</code> for Quantification Table</li>
<li><code>qiime2_metadata</code> for Metadata Table</li>
<li><code>clusterinfo_summary</code> file for Library Identifications</li>
</ol>
<p><strong>3.</strong> Under the header: <em>Advanced options</em> select the following:
1. Instrument Type: <code>orbitrap</code> or <code>qtof</code>
2. Ionization Mode: <code>positive</code>, <code>negative</code> or <code>auto</code></p>
<p>If you are signed in to the server the email address will auto-populate. If not, add your email address and click on submit. The runtime depends on the number of features in your dataset -- a typical dataset (few thousand features) can take a few hours. You will get an email once the job finishes and then you are ready to explore your molecular trees!</p>
<hr />
<p><strong>STEP 3: Analysing the results from a Qemistree job from the status page</strong></p>
<p>Once the job is finished successfully, you will see the status page as below.</p>
<p><img alt="alt text" src="../img/qemistree/Qemistree_statuspage.png" title="status page" /></p>
<p><strong>1. View Summary</strong> gives a list of the molecules, their structures and the chemical taxonomy assigned by <a href="https://jcheminf.biomedcentral.com/articles/10.1186/s13321-016-0174-y">ClassyFire</a> taxonomic levels. It tabulates the annotations for all MS/MS spectra for which we could predict molecular fingerprint using Sirius and CSI:FingerID <a href="https://www.pnas.org/content/112/41/12580">Dührkop et. al</a>.</p>
<p><img alt="alt text" src="../img/qemistree/itoloutput.png" title="iTol Files" /></p>
<p><strong>2. View Qemistree iTOL Tree</strong> to visualize the chemical tree based on similarity of molecular fingerprints predicted by Sirius and CSI:FingerID. Follow the <code>click here</code> button for Qemistree visualization in <a href="https://academic.oup.com/nar/article/47/W1/W256/5424068">iTOL</a>. Qemistree classifies the features/molecules based on <a href="https://jcheminf.biomedcentral.com/articles/10.1186/s13321-016-0174-y">ClassyFire</a> chemical taxonomy into chemical kingdom, superclass, class, subclass, and direct_parent which can be visualized as tip labels in the tree. </p>
<p><strong>Note.</strong> The tree visualized above is the default visualization which includes all the molecules that had structural annotations from either spectral library matches or CSI:FingerID structural predictions. This tree is decorated with clade color and tip labels based on ClassyFire <code>class</code> assignment. This visualization can be interactively modified using <code>View Qemistree Dashboard</code> link (described in Step 4 below).</p>
<p>Shown below is an example default tree.</p>
<p><img alt="alt text" src="../img/qemistree/treeexample.png" title="Qemistree file" /></p>
<p>We recommend you to download the tree and tree decoration files by clicking on the boxes: <strong>Qemistree, Labels, Colors, and Abundance</strong>. These provide the tree file of the features that have smiles (<code>qemistree.tree</code>), the label for each tip of the tree (<code>labels.txt</code>), the color of each clade (<code>colors.txt</code>), and the relative abundance of the feature in the metadata category chosen for the job (<code>barplots.txt</code>). This is because the auto-generated tree is only available for 30 days &amp; therefore it is recommended that you download the associated files and upload it on iTOL using your login credential for permanent storage.</p>
<p><strong>3.</strong> Under <strong>Advanced Views</strong>, click on <code>View qiime2 Emperor Plots</code> to visualize the Principal Coordinate Analysis result using weighted <a href="https://www.ncbi.nlm.nih.gov/pubmed/16332807">UniFrac distances</a> based on the chemical relationships based on predicted molecular fingerprints.  </p>
<p><strong>4</strong> Visualizing chemical trees using the <strong>View Qemistree dashboard</strong>. You can further explore and modify the Qemistree visualization interactively using the <a href="https://qemistree.ucsd.edu/">Qemistree dashboard</a>. This dashboard is available as a link on your Qemistree job status page for direct access.</p>
<p>The example Qemistree task on the dashboard provides is based on a subset of a global foodomics dataset. </p>
<p><img alt="alt text" src="../img/qemistree/qemistree_dashboard.png" title="Qemistree Dashbaord" /></p>
<p>For your own data visualization, enter the following information on the dashboard:</p>
<ol>
<li>Qemistree task ID: This is the unique ID of Qemistree job on GNPS. It can be found in your Jobs page in the <code>Description</code> column. Alternatively, click on <code>View Qemistree Dashboard</code> on the job status page, and this field will be auto-populated.</li>
<li>Column to filter qemistree features: Enter the feature metadata column to prune the tree. This can be a chemical taxonomy level or structural annotation type (all structures, MS2 structures, CSI:FingerID structures).</li>
<li>Column to label tree tips: Enter chemical taxonomic level that should be used to label each feature on the tree (tree tips).</li>
<li>You can include additional options for labelling features by MS2 library match or parent m/z values, when a feature cannot be assigned a chemical taxonomy by ClassyFire.</li>
<li>Choose metadata column by which to visualize the abundance of the features, and whether to normalize the abundance.</li>
<li>Click on Submit. You can quickly generate many Qemistree-iTOL visualizations by changing the values entered in step 2-5 to explore your data.</li>
<li>Click on <code>Datasets</code> tab in iTOL to visualize the relative abundance of each feature in the sample metadata category you chose in Step 5.</li>
</ol>
<p>You can interactively modify the aethetics of this visualization (such as colors, fonts, sizes etc) interactively by using the control panel in iTOL. </p>
<hr />
<p><strong>STEP 4. Advanced analysis using QIIME2</strong> 
For advanced users, we encourage you to download all the files generated from this analysis by clicking on <code>Download Qiime2 data</code> on the status page of the job and explore your data using tools available in <a href="https://qiime2.org/">QIIME2</a>. Qemistree workflow formats all resulting files such that they are compatible with statistical and visualization tools in QIIME2 and we encourage users to leverage QIIME2 tools for data exploration.  </p>
<p>Among the folders downloaded, the <code>output folder</code> contains <code>qemistree.qza</code> and <code>qemistree-pruned-smiles.qza</code> which are two tree files. The <code>qemistree.qza</code> file contains all the features for which molecular fingerprints could be predicted using Sirius+CSI:FingerID; <code>qemistree-pruned-smiles.qza</code> which contains the features that were annotated with molecular structures using spectral matching or CSI:FingerID. Moreover, <code>output_folder</code> also contains: 1. <code>merged-feature-table.qza</code> which has the abundances of features per sample and 2. <code>classified-feature-data.qza</code> which has feature metadata (parent mass, retention time, spectral library match, molecular structures and Classyfire chemical taxonomy for each feature). </p>
<p>You can use these files to perform additional analyses in the QIIME2 which can be installed on your computer using the documentation <a href="https://docs.qiime2.org/2019.10/install/">here</a>. Some suggested QIIME2 analytical tools are linked below:
1. <a href="https://docs.qiime2.org/2020.2/plugins/available/feature-table/heatmap/">Generate heatmap of features per sample</a>
2. <a href="https://docs.qiime2.org/2020.2/plugins/available/diversity/alpha-phylogenetic/">Alpha-diversity using Faith's PD</a> for within sample chemical diversity 
3. <a href="https://docs.qiime2.org/2020.2/plugins/available/diversity/beta-phylogenetic/">Beta-diversity using UniFrac</a> for between sample chemical comparison
4. <a href="https://github.com/biocore/songbird">Songbird analysis</a> for identifying differentially abundant molecules between sample groupings
5. <a href="https://github.com/biocore/mmvec">mmvec analyses</a> for microbe-metabolite interactions</p>
<p>Additionally, we encourage you to explore the plethora of tools available <a href="https://docs.qiime2.org/2020.2/plugins/">here</a> to guide further analyses of your metabolomics data.</p>
<p>For detailed information on all the steps performed during Qemistree processing, please check out our source code and documentation <a href="https://github.com/biocore/q2-qemistree">here</a></p>
<p><strong>CITATIONS</strong>:</p>
<p><a href="https://www.pnas.org/content/112/41/12580">Dührkop, K., Shen, H., Meusel, M., Rousu, J. &amp; Böcker, S. Searching molecular structure databases with tandem mass spectra using CSI:FingerID. Proc. Natl. Acad. Sci. U. S. A. 112, 12580–5 (2015)</a></p>
<p><a href="https://jcheminf.biomedcentral.com/articles/10.1186/s13321-016-0174-y">Djoumbou Feunang, Y. et al. ClassyFire: automated chemical classification with a comprehensive, computable taxonomy. J. Cheminform. 8, 61 (2016)</a></p>
<p><a href="https://academic.oup.com/nar/article/47/W1/W256/5424068">Letunic, I. &amp; Bork, P. Interactive Tree Of Life (iTOL) v4: recent updates and new developments. Nucleic Acids Research vol. 47 W256–W259 (2019)</a></p>
<h2 id="page-contributors">Page Contributors<a class="headerlink" href="#page-contributors" title="Permanent link">&para;</a></h2>
<p><span class='git-page-authors git-authors'><a href='mailto:mwang87@gmail.com'>Ming Wang</a> (7.69%), <a href='mailto:dacharya@wisc.edu'>Deepa Acharya</a> (51.54%), <a href='mailto:anupriya.tripathi92@gmail.com'>Anupriya Tripathi</a> (40.77%)</span></p>
                
                  
                    

<hr>
<div class="md-source-date">
  <small>
    
      Last update: June 19, 2020 17:46:45
    
  </small>
</div>
                  
                
              
              
                


              
            </article>
          </div>
        </div>
      </main>
      
        
<footer class="md-footer">
  
    <div class="md-footer-nav">
      <nav class="md-footer-nav__inner md-grid" aria-label="Footer">
        
          <a href="../ReDU/" title="Reanalysis of Data User Interface (ReDU)" class="md-footer-nav__link md-footer-nav__link--prev" rel="prev">
            <div class="md-footer-nav__button md-icon">
              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
            </div>
            <div class="md-footer-nav__title">
              <div class="md-ellipsis">
                <span class="md-footer-nav__direction">
                  Previous
                </span>
                Reanalysis of Data User Interface (ReDU)
              </div>
            </div>
          </a>
        
        
          <a href="../mmvec/" title="MMVEC" class="md-footer-nav__link md-footer-nav__link--next" rel="next">
            <div class="md-footer-nav__title">
              <div class="md-ellipsis">
                <span class="md-footer-nav__direction">
                  Next
                </span>
                MMVEC
              </div>
            </div>
            <div class="md-footer-nav__button md-icon">
              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4z"/></svg>
            </div>
          </a>
        
      </nav>
    </div>
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-footer-copyright">
        
        Made with
        <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
          Material for MkDocs
        </a>
      </div>
      
    </div>
  </div>
</footer>
      
    </div>
    
      <script src="../assets/javascripts/vendor.d710d30a.min.js"></script>
      <script src="../assets/javascripts/bundle.4a5ba8d6.min.js"></script><script id="__lang" type="application/json">{"clipboard.copy": "Copy to clipboard", "clipboard.copied": "Copied to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.result.placeholder": "Type to start searching", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents"}</script>
      
      <script>
        app = initialize({
          base: "..",
          features: [],
          search: Object.assign({
            worker: "../assets/javascripts/worker/search.9b3611bd.min.js"
          }, typeof search !== "undefined" && search)
        })
      </script>
      
    
  </body>
</html>