
<!DOCTYPE html>

<html class="no-js" lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width,initial-scale=1" name="viewport"/>
<link href="https://ccms-ucsd.github.io/GNPSDocumentation/qemistree/" rel="canonical"/>
<link href="../assets/images/favicon.png" rel="icon"/>
<meta content="mkdocs-1.1.2, mkdocs-material-7.1.6" name="generator"/>
<title>Qemistree - GNPS Documentation</title>
<link href="../assets/stylesheets/main.875de78c.min.css" rel="stylesheet"/>
<link href="../assets/stylesheets/palette.f1a3b89f.min.css" rel="stylesheet"/>
<link crossorigin="" href="https://fonts.gstatic.com" rel="preconnect"/>
<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700%7CRoboto+Mono&amp;display=fallback" rel="stylesheet"/>
<style>:root{--md-text-font-family:"Roboto";--md-code-font-family:"Roboto Mono"}</style>
<script>window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)},ga.l=+new Date,ga("create","UA-8412213-5","ccms-ucsd.github.io"),ga("set","anonymizeIp",!0),ga("send","pageview"),document.addEventListener("DOMContentLoaded",function(){document.forms.search&&document.forms.search.query.addEventListener("blur",function(){var e;this.value&&(e=document.location.pathname,ga("send","pageview",e+"?q="+this.value))}),"undefined"!=typeof location$&&location$.subscribe(function(e){ga("send","pageview",e.pathname)})})</script>
<script async="" src="https://www.google-analytics.com/analytics.js"></script>
</head>
<body data-md-color-accent="none" data-md-color-primary="none" data-md-color-scheme="" dir="ltr">
<script>function __prefix(e){return new URL("..",location).pathname+"."+e}function __get(e,t=localStorage){return JSON.parse(t.getItem(__prefix(e)))}</script>
<input autocomplete="off" class="md-toggle" data-md-toggle="drawer" id="__drawer" type="checkbox"/>
<input autocomplete="off" class="md-toggle" data-md-toggle="search" id="__search" type="checkbox"/>
<label class="md-overlay" for="__drawer"></label>
<div data-md-component="skip">
<a class="md-skip" href="#page-contributors">
          Skip to content
        </a>
</div>
<div data-md-component="announce">
</div>
<header class="md-header" data-md-component="header">
<nav aria-label="Header" class="md-header__inner md-grid">
<a aria-label="GNPS Documentation" class="md-header__button md-logo" data-md-component="logo" href=".." title="GNPS Documentation">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"></path></svg>
</a>
<label class="md-header__button md-icon" for="__drawer">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2z"></path></svg>
</label>
<div class="md-header__title" data-md-component="header-title">
<div class="md-header__ellipsis">
<div class="md-header__topic">
<span class="md-ellipsis">
            GNPS Documentation
          </span>
</div>
<div class="md-header__topic" data-md-component="header-topic">
<span class="md-ellipsis">
            
              Qemistree
            
          </span>
</div>
</div>
</div>
<label class="md-header__button md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"></path></svg>
</label>
<div class="md-search" data-md-component="search" role="dialog">
<label class="md-search__overlay" for="__search"></label>
<div class="md-search__inner" role="search">
<form class="md-search__form" name="search">
<input aria-label="Search" autocapitalize="off" autocomplete="off" autocorrect="off" class="md-search__input" data-md-component="search-query" data-md-state="active" name="query" placeholder="Search" required="" spellcheck="false" type="text"/>
<label class="md-search__icon md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"></path></svg>
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"></path></svg>
</label>
<button aria-label="Clear" class="md-search__icon md-icon" tabindex="-1" type="reset">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"></path></svg>
</button>
</form>
<div class="md-search__output">
<div class="md-search__scrollwrap" data-md-scrollfix="">
<div class="md-search-result" data-md-component="search-result">
<div class="md-search-result__meta">
            Initializing search
          </div>
<ol class="md-search-result__list"></ol>
</div>
</div>
</div>
</div>
</div>
<div class="md-header__source">
<a class="md-source" data-md-component="source" href="https://github.com/CCMS-UCSD/GNPSDocumentation/" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    CCMS-UCSD/GNPSDocumentation
  </div>
</a>
</div>
</nav>
</header>
<div class="md-container" data-md-component="container">
<main class="md-main" data-md-component="main">
<div class="md-main__inner md-grid">
<div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Navigation" class="md-nav md-nav--primary" data-md-level="0">
<label class="md-nav__title" for="__drawer">
<a aria-label="GNPS Documentation" class="md-nav__button md-logo" data-md-component="logo" href=".." title="GNPS Documentation">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"></path></svg>
</a>
    GNPS Documentation
  </label>
<div class="md-nav__source">
<a class="md-source" data-md-component="source" href="https://github.com/CCMS-UCSD/GNPSDocumentation/" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    CCMS-UCSD/GNPSDocumentation
  </div>
</a>
</div>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="..">
        GNPS Introduction
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../isgnpsright/">
        Is my data compatible with GNPS?
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../faq/">
        Frequently Asked Questions
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../superquickstart/">
        Superquick Start Guide
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../quickstart/">
        Quick Start Guide
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gnpsanalysisoverview/">
        GNPS Analysis Tools Overview
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../lexicon/">
        GNPS Lexicon
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_8" id="__nav_8" type="checkbox"/>
<label class="md-nav__link" for="__nav_8">
        Mass Spec/Computational Background
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Mass Spec/Computational Background" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_8">
<span class="md-nav__icon md-icon"></span>
          Mass Spec/Computational Background
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../massspecbackground/networkingtheory/">
        Molecular Networking
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_9" id="__nav_9" type="checkbox"/>
<label class="md-nav__link" for="__nav_9">
        Data Preparation/Upload
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Data Preparation/Upload" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_9">
<span class="md-nav__icon md-icon"></span>
          Data Preparation/Upload
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../fileconversion/">
        Mass Spectrometry File Conversion
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fileupload/">
        GNPS File Upload
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../metadata/">
        Metadata
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_10" id="__nav_10" type="checkbox"/>
<label class="md-nav__link" for="__nav_10">
        Recommended Data Analysis
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Recommended Data Analysis" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_10">
<span class="md-nav__icon md-icon"></span>
          Recommended Data Analysis
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../networking/">
        Molecular Networking
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_10_2" id="__nav_10_2" type="checkbox"/>
<label class="md-nav__link" for="__nav_10_2">
        GC-MS Molecular Networking
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="GC-MS Molecular Networking" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_10_2">
<span class="md-nav__icon md-icon"></span>
          GC-MS Molecular Networking
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../gcanalysis/">
        GC-MS on GNPS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gc-ms-deconvolution/">
        GC-MS Deconvolution
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gc-ms-library-molecular-network/">
        GC-MS Library Search and Molecular Networking
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../networkingviews/">
        Molecular Networking Views
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../librarysearch/">
        Spectral Library Search
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../masst/">
        MASST Search (Mass Search Tool)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../resultviews/">
        Result Views Navigation
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_11" id="__nav_11" type="checkbox"/>
<label class="md-nav__link" for="__nav_11">
        Mass Spec Data Visualization
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Mass Spec Data Visualization" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_11">
<span class="md-nav__icon md-icon"></span>
          Mass Spec Data Visualization
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../lcms-dashboard/">
        GNPS Dashboard
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gnpsdatasetexplorer/">
        GNPS Dataset Explorer
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../usi/">
        Metabolomics Spectrum Resolver
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--active md-nav__item--nested">
<input checked="" class="md-nav__toggle md-toggle" data-md-toggle="__nav_12" id="__nav_12" type="checkbox"/>
<label class="md-nav__link" for="__nav_12">
        Advanced Data Analysis
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Advanced Data Analysis" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_12">
<span class="md-nav__icon md-icon"></span>
          Advanced Data Analysis
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../networkingfiles/">
        Molecular Networking Files
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_12_2" id="__nav_12_2" type="checkbox"/>
<label class="md-nav__link" for="__nav_12_2">
        Feature Based Molecular Networking
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Feature Based Molecular Networking" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_12_2">
<span class="md-nav__icon md-icon"></span>
          Feature Based Molecular Networking
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking/">
        FBMN Workflow
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-mzmine2/">
        FBMN with MZmine
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-ms-dial/">
        FBMN with MS-DIAL
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-xcms3/">
        FBMN with XCMS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-openms/">
        FBMN with OpenMS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-metaboscape/">
        FBMN with MetaboScape
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-progenesisQI/">
        FBMN with Progenesis QI
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-mztab-m/">
        FBMN with mzTab-M
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-cytoscape/">
        FBMN export to Cytoscape
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_12_3" id="__nav_12_3" type="checkbox"/>
<label class="md-nav__link" for="__nav_12_3">
        FBMN - Ion Identity Networking
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="FBMN - Ion Identity Networking" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_12_3">
<span class="md-nav__icon md-icon"></span>
          FBMN - Ion Identity Networking
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin/">
        IIN Workflow
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin-mzmine/">
        IIN with MZmine2
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin-xcms/">
        IIN with XCMS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin-msdial/">
        IIN with MS-DIAL
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../ms2lda/">
        MS2LDA and MotifDB Substructure Discovery
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../cytoscape/">
        Molecular Networking in Cytoscape
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../dereplicator/">
        DEREPLICATOR
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../dereplicator%2B/">
        DEREPLICATOR+
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../molDiscovery/">
        MolDiscovery
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../nap/">
        Network Annotation Propagation (NAP)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../ms2lda/">
        MS2LDA and MotifDB Substructure Discovery
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../molnetenhancer/">
        MolNetEnhancer
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../metaminer/">
        MetaMiner
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../sirius/">
        Sirius
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../ReDU/">
        Reanalysis of Data User Interface (ReDU)
      </a>
</li>
<li class="md-nav__item md-nav__item--active">
<input class="md-nav__toggle md-toggle" data-md-toggle="toc" id="__toc" type="checkbox"/>
<label class="md-nav__link md-nav__link--active" for="__toc">
          Qemistree
          <span class="md-nav__icon md-icon"></span>
</label>
<a class="md-nav__link md-nav__link--active" href="./">
        Qemistree
      </a>
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#page-contributors">
    Page Contributors
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../mmvec/">
        MMVEC
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../mergepolarity/">
        Merge Network Polarity
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../chemdir/">
        Chemical Directionality
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_13" id="__nav_13" type="checkbox"/>
<label class="md-nav__link" for="__nav_13">
        Spectral Libraries
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Spectral Libraries" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_13">
<span class="md-nav__icon md-icon"></span>
          Spectral Libraries
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../browselibraries/">
        Browsing Reference Spectral Libraries
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../spectrumcuration/">
        Add/Curate Reference Spectrum in the Library
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../batchupload/">
        Reference Spectral Library Batch Addition
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../batchuploadmzvault/">
        Reference Spectral Library mzVault Addition
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../downloadlibraries/">
        Downloading Reference Spectral Libraries
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_13_6" id="__nav_13_6" type="checkbox"/>
<label class="md-nav__link" for="__nav_13_6">
        Open-source Method for Reference Library Generation
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Open-source Method for Reference Library Generation" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_13_6">
<span class="md-nav__icon md-icon"></span>
          Open-source Method for Reference Library Generation
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../msmschooser/">
        MSMS-Chooser Workflow
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../msmschooserdataacquisitioncomputational/">
        Computational Data Acquisition Prep
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../msmschooserdataacquisitionQE/">
        Data Acqusition - Thermo QExactive (v1.0)
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tag_info/">
        GNPS Tag Contribution and Analyses
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_14" id="__nav_14" type="checkbox"/>
<label class="md-nav__link" for="__nav_14">
        Create a Public Datasets
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Create a Public Datasets" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_14">
<span class="md-nav__icon md-icon"></span>
          Create a Public Datasets
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../datasets/">
        Dataset Creation/Sharing
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../continuousid/">
        Continuous Identification
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../moleculeexplorer/">
        Molecule Explorer
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_15" id="__nav_15" type="checkbox"/>
<label class="md-nav__link" for="__nav_15">
        Community/Social Features
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Community/Social Features" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_15">
<span class="md-nav__icon md-icon"></span>
          Community/Social Features
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../user/">
        User Page
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../troubleshooting/">
        Troubleshooting
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_17" id="__nav_17" type="checkbox"/>
<label class="md-nav__link" for="__nav_17">
        Tutorial Guides
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Tutorial Guides" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_17">
<span class="md-nav__icon md-icon"></span>
          Tutorial Guides
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/bacterianetworking/">
        Molecular Networking - Analog Discovery in Microbial Extracts
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/americangutmzmine/">
        Feature-Based Molecular Networking (FBMN) with MZmine2 - Microbiome Molecules in Fecal Samples in American Gut Project
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/americangut-ms-dial/">
        FBMN with MS-DIAL - Microbiome Molecules in Fecal Samples in American Gut Project
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/coffee-tutorial-metaboscape/">
        FBMN with MetaboScape - Coffee Analysis
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/featurebasedgnps/">
        FBMN analysis at GNPS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/rdd/">
        Reference Data Driven Analysis
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../changelog/">
        Change Log
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gnps_community/">
        Join the GNPS community
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../workshops/">
        GNPS Workshops/Training
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../logos/">
        GNPS Logos
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../contact/">
        Contact/Office Hours
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../toolindex/">
        GNPS Tool Index
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../api/">
        API
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../developers/">
        For Developers
      </a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#page-contributors">
    Page Contributors
  </a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-content" data-md-component="content">
<article class="md-content__inner md-typeset">
<a class="md-content__button md-icon" href="https://github.com/CCMS-UCSD/GNPSDocumentation/edit/master/docs/qemistree.md" title="Edit this page">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25z"></path></svg>
</a>
<h1>Qemistree</h1>
<p><strong>QEMISTREE</strong> - a tool to represnt metabolomics data as trees to explore chemical diversity</p>
<p><em>Introduction</em>: Qemistree is a computation tool to build a tree of mass-spectrometry (LC-MS/MS) features to perform which enables the use of phylogeny-based tools to study chemical composition of samples. The article is available in <a href="https://www.nature.com/articles/s41589-020-00677-3">Nat. Chem. Biol. (2020)</a>.
This documentation aims to provide a user-guide on how to run a Qemistree workflow on GNPS. </p>
<p><strong>Please use this interim <a href="https://proteomics2.ucsd.edu/ProteoSAFe/index.jsp?params=%7B%22workflow%22:%22QEMISTREE%22%7D">link to run Qemitree</a> on your data after FBMN. This will be updated in the next GNPS release.</strong></p>
<hr/>
<p><strong>STEP 1: Collecting the right input files</strong></p>
<p>Users can perform QEMISTREE analysis in GNPS after processing LC-MS/MS data through <a href="../featurebasedmolecularnetworking/">Feature-Based Molecular Networking or FBMN</a> using <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-11-395">MZmine2</a>. The input files for Qemistree are:</p>
<p><strong>1. Required</strong>: A SIRIUS MGF file generated from the MZmine workflow. See the <a href="../featurebasedmolecularnetworking-with-mzmine2/">FBMN with MZmine2</a>.</p>
<p><strong>2. Required</strong>: A quant table called <code>qiime2_table.qza</code> with all the identified features (an output of <a href="../featurebasedmolecularnetworking/">FBMN</a>, and found  within the folder <code>qiime2_output</code>)</p>
<p><strong>3. Optional</strong>: A metadata file called <code>qiime2_metadata.tsv</code> (an output of <a href="../featurebasedmolecularnetworking/">FBMN</a>, and also found in folder <code>qiime2_output</code>)</p>
<p><strong>4. Optional</strong>: A library identification file (TSV) (an output of <a href="../featurebasedmolecularnetworking/">FBMN</a>, and found in the folder <code>clusterinfo_summary</code>)</p>
<p>Follow the steps below to generate these files:</p>
<p>a. Follow the documentation for <a href="../featurebasedmolecularnetworking-with-mzmine2/">Feature-Based Molecular Networking using MZmine2</a> to generate an aligned feature list for your LC-MS/MS data and export the necessary MGF and CSV files for FBMN GNPS workflow.</p>
<p>b. While still in MZmine2, select your aligned feature list, then click on the tab for feature list methods and select Export/Import, followed by Export for SIRIUS.</p>
<p>c. Choose the Mass list that you used to generate your feature list, and specify a path and filename for your SIRIUS file. Click OK.</p>
<p>d. Go to GNPS server and run a <a href="https://gnps.ucsd.edu/ProteoSAFe/index.jsp?params=%7B%22workflow%22:%22FEATURE-BASED-MOLECULAR-NETWORKING%22,%22library_on_server%22:%22d.speclibs;%22%7D">FBMN</a> job using the GNPS quant CSV, GNPS MGF (both generated in step a, above) and <a href="https://mwang87.github.io/ReDU-MS2-Documentation/HowtoContribute/">ReDU</a> metadata file.</p>
<p>e. The Qemistree task utilizes the output files from FBMN. These can be downloaded by clicking on <code>Download qiime2 Emperor qzv</code> (<code>qiime2_table.qza</code> and <code>qiime2_metadata.tsv</code> from folder <code>qiime2_output</code> and  <code>clusterinfo_summary</code> for the tsv file within). You can start a Qemistree task by clicking on <code>Visualize with Qemistree</code> link on your FBMN status page </p>
<p><strong>Note</strong> <code>Visualize with Qemistree</code> option is available for FBMN release version &gt;= 20. If your FBMN job was run using an older version, please re-run the job by clicking on <code>Clone to Latest Version</code> to follow the rest of the tutorial.</p>
<hr/>
<p><strong>STEP 2: Running a Qemistree job</strong></p>
<p><strong>1.</strong> Go to the status page of your FBMN job:</p>
<p><img alt="alt text" src="../img/qemistree/FBMN_statuspage.png" title="FBMN Status Page"/></p>
<p>Click on <code>Visualize with Qemistree</code> under the <code>Advanced Views - Experimental Views</code> to analyze your data using Qemistree. This method auto-populates the input fields <code>Quantification Table</code>, <code>Metadata Table</code> and <code>Library Identifications</code> from your corresponding FBMN job. Additionally, you need to upload your SIRIUS MGF file exported from Mzmine2 as describes in Step 1.</p>
<p><img alt="alt text" src="../img/qemistree/workflowpage_gnps.png" title="workflow"/></p>
<p><strong>2.</strong> Make sure the correct files are included as described in the beginning:</p>
<ol>
<li>Sirius MGF Spectrum file</li>
<li><code>qiime2_table</code> for Quantification Table</li>
<li><code>qiime2_metadata</code> for Metadata Table</li>
<li><code>clusterinfo_summary</code> file for Library Identifications</li>
</ol>
<p><strong>3.</strong> Under the header: <em>Advanced options</em> select the following:
1. Instrument Type: <code>orbitrap</code> or <code>qtof</code>
2. Ionization Mode: <code>positive</code>, <code>negative</code> or <code>auto</code></p>
<p>If you are signed in to the server the email address will auto-populate. If not, add your email address and click on submit. The runtime depends on the number of features in your dataset -- a typical dataset (few thousand features) can take a few hours. You will get an email once the job finishes and then you are ready to explore your molecular trees!</p>
<hr/>
<p><strong>STEP 3: Analysing the results from a Qemistree job from the status page</strong></p>
<p>Once the job is finished successfully, you will see the status page as below.</p>
<p><img alt="alt text" src="../img/qemistree/Qemistree_statuspage.png" title="status page"/></p>
<p><strong>1. View Summary</strong> gives a list of the molecules, their structures and the chemical taxonomy assigned by <a href="https://jcheminf.biomedcentral.com/articles/10.1186/s13321-016-0174-y">ClassyFire</a> taxonomic levels. It tabulates the annotations for all MS/MS spectra for which we could predict molecular fingerprint using Sirius and CSI:FingerID <a href="https://www.pnas.org/content/112/41/12580">Dührkop et. al</a>.</p>
<p><img alt="alt text" src="../img/qemistree/itoloutput.png" title="iTol Files"/></p>
<p><strong>2. View Qemistree iTOL Tree</strong> to visualize the chemical tree based on similarity of molecular fingerprints predicted by Sirius and CSI:FingerID. Follow the <code>click here</code> button for Qemistree visualization in <a href="https://academic.oup.com/nar/article/47/W1/W256/5424068">iTOL</a>. Qemistree classifies the features/molecules based on <a href="https://jcheminf.biomedcentral.com/articles/10.1186/s13321-016-0174-y">ClassyFire</a> chemical taxonomy into chemical kingdom, superclass, class, subclass, and direct_parent which can be visualized as tip labels in the tree. </p>
<p><strong>Note.</strong> The tree visualized above is the default visualization which includes all the molecules that had structural annotations from either spectral library matches or CSI:FingerID structural predictions. This tree is decorated with clade color and tip labels based on ClassyFire <code>class</code> assignment. This visualization can be interactively modified using <code>View Qemistree Dashboard</code> link (described in Step 4 below).</p>
<p>Shown below is an example default tree.</p>
<p><img alt="alt text" src="../img/qemistree/treeexample.png" title="Qemistree file"/></p>
<p>We recommend you to download the tree and tree decoration files by clicking on the boxes: <strong>Qemistree, Labels, Colors, and Abundance</strong>. These provide the tree file of the features that have smiles (<code>qemistree.tree</code>), the label for each tip of the tree (<code>labels.txt</code>), the color of each clade (<code>colors.txt</code>), and the relative abundance of the feature in the metadata category chosen for the job (<code>barplots.txt</code>). This is because the auto-generated tree is only available for 30 days &amp; therefore it is recommended that you download the associated files and upload it on iTOL using your login credential for permanent storage.</p>
<p><strong>3.</strong> Under <strong>Advanced Views</strong>, click on <code>View qiime2 Emperor Plots</code> to visualize the Principal Coordinate Analysis result using weighted <a href="https://www.ncbi.nlm.nih.gov/pubmed/16332807">UniFrac distances</a> based on the chemical relationships based on predicted molecular fingerprints.  </p>
<p><strong>4.</strong> Visualizing chemical trees using the <strong>View Qemistree dashboard</strong>. You can further explore and modify the Qemistree visualization interactively using the <a href="https://qemistree.ucsd.edu/">Qemistree dashboard</a>. This dashboard is available as a link on your Qemistree job status page for direct access.</p>
<p>The example Qemistree task on the dashboard provides is based on a subset of a global foodomics dataset. </p>
<p><img alt="alt text" src="../img/qemistree/qemistree_dashboard.png" title="Qemistree Dashbaord"/></p>
<p>For your own data visualization, enter the following information on the dashboard:</p>
<ol>
<li>Qemistree task ID: This is the unique ID of Qemistree job on GNPS. It can be found in your Jobs page in the <code>Description</code> column. Alternatively, click on <code>View Qemistree Dashboard</code> on the job status page, and this field will be auto-populated.</li>
<li>Column to filter qemistree features: Enter the feature metadata column to prune the tree. This can be a chemical taxonomy level or structural annotation type (all structures, MS2 structures, CSI:FingerID structures).</li>
<li>Column to label tree tips: Enter chemical taxonomic level that should be used to label each feature on the tree (tree tips).</li>
<li>You can include additional options for labelling features by MS2 library match or parent m/z values, when a feature cannot be assigned a chemical taxonomy by ClassyFire.</li>
<li>Choose metadata column by which to visualize the abundance of the features, and whether to normalize the abundance.</li>
<li>Click on Submit. You can quickly generate many Qemistree-iTOL visualizations by changing the values entered in step 2-5 to explore your data.</li>
<li>Click on <code>Datasets</code> tab in iTOL to visualize the relative abundance of each feature in the sample metadata category you chose in Step 5.</li>
</ol>
<p>You can interactively modify the aethetics of this visualization (such as colors, fonts, sizes etc) interactively by using the control panel in iTOL. </p>
<p><strong>5.</strong> You can download the .qzv file generated in the dashboard and drag and drop to https://view.qiime2.org/. Clicking on “View this tree using iTol” will build the tree with all the information contained in the <strong>Qemistree, Labels, Colors, and Abundance</strong> files. In this way, it will not be possible to trace the branches back to the fingerprints (and, consequently, to the Feature IDs) since the information you can get putting the cursor on the nodes is its branch length and the classification information. </p>
<p>If you want to trace the branches back to the feature IDs, you can download the Qemistree.tree file, and upload it in iTol directly in your account. In this way, you will get a tree without any other information added, just the fingerprint information for each node.</p>
<p><img alt="alt text" src="../img/qemistree/Qemistree_Fingerprints.png" title="Qemistree fingerprints iTol"/></p>
<p>On the job status window, you can click on “view summary” and download this entire table to check which feature ID corresponds to the fingerprint you are interested in (seen on the tree).</p>
<p><img alt="alt text" src="../img/qemistree/Table_fingerprints.png" title="Qemistree SummaryView"/></p>
<p>You can also just drag and drop to iTol the <strong>Labels, Colors, and Abundance</strong> files to have all this information in the tree again.</p>
<hr/>
<p><strong>STEP 4. Advanced analysis using QIIME2</strong> 
For advanced users, we encourage you to download all the files generated from this analysis by clicking on <code>Download Qiime2 data</code> on the status page of the job and explore your data using tools available in <a href="https://qiime2.org/">QIIME2</a>. Qemistree workflow formats all resulting files such that they are compatible with statistical and visualization tools in QIIME2 and we encourage users to leverage QIIME2 tools for data exploration.  </p>
<p>Among the folders downloaded, the <code>output folder</code> contains <code>qemistree.qza</code> and <code>qemistree-pruned-smiles.qza</code> which are two tree files. The <code>qemistree.qza</code> file contains all the features for which molecular fingerprints could be predicted using Sirius+CSI:FingerID; <code>qemistree-pruned-smiles.qza</code> which contains the features that were annotated with molecular structures using spectral matching or CSI:FingerID. Moreover, <code>output_folder</code> also contains: 1. <code>merged-feature-table.qza</code> which has the abundances of features per sample and 2. <code>classified-feature-data.qza</code> which has feature metadata (parent mass, retention time, spectral library match, molecular structures and Classyfire chemical taxonomy for each feature). </p>
<p>You can use these files to perform additional analyses in the QIIME2 which can be installed on your computer using the documentation <a href="https://docs.qiime2.org/2019.10/install/">here</a>. Some suggested QIIME2 analytical tools are linked below:
1. <a href="https://docs.qiime2.org/2020.2/plugins/available/feature-table/heatmap/">Generate heatmap of features per sample</a>
2. <a href="https://docs.qiime2.org/2020.2/plugins/available/diversity/alpha-phylogenetic/">Alpha-diversity using Faith's PD</a> for within sample chemical diversity 
3. <a href="https://docs.qiime2.org/2020.2/plugins/available/diversity/beta-phylogenetic/">Beta-diversity using UniFrac</a> for between sample chemical comparison
4. <a href="https://github.com/biocore/songbird">Songbird analysis</a> for identifying differentially abundant molecules between sample groupings
5. <a href="https://github.com/biocore/mmvec">mmvec analyses</a> for microbe-metabolite interactions</p>
<p>Additionally, we encourage you to explore the plethora of tools available <a href="https://docs.qiime2.org/2020.2/plugins/">here</a> to guide further analyses of your metabolomics data.</p>
<p>For detailed information on all the steps performed during Qemistree processing, please check out our source code and documentation <a href="https://github.com/biocore/q2-qemistree">here</a></p>
<p><strong>CITATIONS</strong>:</p>
<p><a href="https://www.nature.com/articles/s41589-020-00677-3">Tripathi, A., Vázquez-Baeza, Y., Gauglitz, J.M. et al. Chemically informed analyses of metabolomics mass spectrometry data with Qemistree. Nat Chem Biol (2020)</a></p>
<p><a href="https://www.pnas.org/content/112/41/12580">Dührkop, K., Shen, H., Meusel, M., Rousu, J. &amp; Böcker, S. Searching molecular structure databases with tandem mass spectra using CSI:FingerID. Proc. Natl. Acad. Sci. U. S. A. 112, 12580–5 (2015)</a></p>
<p><a href="https://jcheminf.biomedcentral.com/articles/10.1186/s13321-016-0174-y">Djoumbou Feunang, Y. et al. ClassyFire: automated chemical classification with a comprehensive, computable taxonomy. J. Cheminform. 8, 61 (2016)</a></p>
<p><a href="https://academic.oup.com/nar/article/47/W1/W256/5424068">Letunic, I. &amp; Bork, P. Interactive Tree Of Life (iTOL) v4: recent updates and new developments. Nucleic Acids Research vol. 47 W256–W259 (2019)</a></p>
<h2 id="page-contributors">Page Contributors<a class="headerlink" href="#page-contributors" title="Permanent link">¶</a></h2>
<p><span class="git-page-authors git-authors"><a href="mailto:mwang87@gmail.com">Ming Wang</a> (6.85%), <a href="mailto:louisfelix.nothias@gmail.com">lfnothias</a> (7.53%), <a href="mailto:dacharya@wisc.edu">Deepa Acharya</a> (45.89%), <a href="mailto:anupriya.tripathi92@gmail.com">Anupriya Tripathi</a> (30.82%), <a href="mailto:55901470+helenamrusso@users.noreply.github.com">helenamrusso</a> (8.9%)</span></p>
<hr/>
<div class="md-source-date">
<small>
    
      Last update: <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-datetime">March 2, 2021 16:20:18</span>
</small>
</div>
</article>
</div>
</div>
</main>
<footer class="md-footer">
<nav aria-label="Footer" class="md-footer__inner md-grid">
<a class="md-footer__link md-footer__link--prev" href="../ReDU/" rel="prev">
<div class="md-footer__button md-icon">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"></path></svg>
</div>
<div class="md-footer__title">
<div class="md-ellipsis">
<span class="md-footer__direction">
                Previous
              </span>
              Reanalysis of Data User Interface (ReDU)
            </div>
</div>
</a>
<a class="md-footer__link md-footer__link--next" href="../mmvec/" rel="next">
<div class="md-footer__title">
<div class="md-ellipsis">
<span class="md-footer__direction">
                Next
              </span>
              MMVEC
            </div>
</div>
<div class="md-footer__button md-icon">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4z"></path></svg>
</div>
</a>
</nav>
<div class="md-footer-meta md-typeset">
<div class="md-footer-meta__inner md-grid">
<div class="md-footer-copyright">
        
        Made with
        <a href="https://squidfunk.github.io/mkdocs-material/" rel="noopener" target="_blank">
          Material for MkDocs
        </a>
</div>
</div>
</div>
</footer>
</div>
<div class="md-dialog" data-md-component="dialog">
<div class="md-dialog__inner md-typeset"></div>
</div>
<script id="__config" type="application/json">{"base": "..", "features": [], "translations": {"clipboard.copy": "Copy to clipboard", "clipboard.copied": "Copied to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.placeholder": "Type to start searching", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.term.missing": "Missing"}, "search": "../assets/javascripts/workers/search.d351de03.min.js", "version": null}</script>
<script src="../assets/javascripts/bundle.34eae1b6.min.js"></script>
</body>
</html>