
<!DOCTYPE html>

<html class="no-js" lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width,initial-scale=1" name="viewport"/>
<link href="https://ccms-ucsd.github.io/GNPSDocumentation/dereplicator/" rel="canonical"/>
<link href="../assets/images/favicon.png" rel="shortcut icon"/>
<meta content="mkdocs-1.1.2, mkdocs-material-7.0.2" name="generator"/>
<title>DEREPLICATOR - GNPS Documentation</title>
<link href="../assets/stylesheets/main.1655a90d.min.css" rel="stylesheet"/>
<link href="../assets/stylesheets/palette.7fa14f5b.min.css" rel="stylesheet"/>
<link crossorigin="" href="https://fonts.gstatic.com" rel="preconnect"/>
<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700%7CRoboto+Mono&amp;display=fallback" rel="stylesheet"/>
<style>:root{--md-text-font-family:"Roboto";--md-code-font-family:"Roboto Mono"}</style>
<script>window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)},ga.l=+new Date,ga("create","UA-8412213-5","ccms-ucsd.github.io"),ga("set","anonymizeIp",!0),ga("send","pageview"),document.addEventListener("DOMContentLoaded",function(){document.forms.search&&document.forms.search.query.addEventListener("blur",function(){var e;this.value&&(e=document.location.pathname,ga("send","pageview",e+"?q="+this.value))}),"undefined"!=typeof location$&&location$.subscribe(function(e){ga("send","pageview",e.pathname)})})</script>
<script async="" src="https://www.google-analytics.com/analytics.js"></script>
</head>
<body data-md-color-accent="none" data-md-color-primary="none" data-md-color-scheme="" dir="ltr">
<input autocomplete="off" class="md-toggle" data-md-toggle="drawer" id="__drawer" type="checkbox"/>
<input autocomplete="off" class="md-toggle" data-md-toggle="search" id="__search" type="checkbox"/>
<label class="md-overlay" for="__drawer"></label>
<div data-md-component="skip">
<a class="md-skip" href="#dereplicator-insilico-peptidic-natural-products-tools">
          Skip to content
        </a>
</div>
<div data-md-component="announce">
</div>
<header class="md-header" data-md-component="header">
<nav aria-label="Header" class="md-header__inner md-grid">
<a aria-label="GNPS Documentation" class="md-header__button md-logo" href="https://ccms-ucsd.github.io/GNPSDocumentation/" title="GNPS Documentation">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"></path></svg>
</a>
<label class="md-header__button md-icon" for="__drawer">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2z"></path></svg>
</label>
<div class="md-header__title" data-md-component="header-title">
<div class="md-header__ellipsis">
<div class="md-header__topic">
<span class="md-ellipsis">
            GNPS Documentation
          </span>
</div>
<div class="md-header__topic" data-md-component="header-topic">
<span class="md-ellipsis">
            
              DEREPLICATOR
            
          </span>
</div>
</div>
</div>
<div class="md-header__options">
</div>
<label class="md-header__button md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"></path></svg>
</label>
<div class="md-search" data-md-component="search" role="dialog">
<label class="md-search__overlay" for="__search"></label>
<div class="md-search__inner" role="search">
<form class="md-search__form" name="search">
<input aria-label="Search" autocapitalize="off" autocomplete="off" autocorrect="off" class="md-search__input" data-md-component="search-query" data-md-state="active" name="query" placeholder="Search" required="" spellcheck="false" type="text"/>
<label class="md-search__icon md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"></path></svg>
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"></path></svg>
</label>
<button aria-label="Clear" class="md-search__icon md-icon" tabindex="-1" type="reset">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"></path></svg>
</button>
</form>
<div class="md-search__output">
<div class="md-search__scrollwrap" data-md-scrollfix="">
<div class="md-search-result" data-md-component="search-result">
<div class="md-search-result__meta">
            Initializing search
          </div>
<ol class="md-search-result__list"></ol>
</div>
</div>
</div>
</div>
</div>
<div class="md-header__source">
<a class="md-source" data-md-component="source" href="https://github.com/CCMS-UCSD/GNPSDocumentation/" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><path d="M439.55 236.05L244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    CCMS-UCSD/GNPSDocumentation
  </div>
</a>
</div>
</nav>
</header>
<div class="md-container" data-md-component="container">
<main class="md-main" data-md-component="main">
<div class="md-main__inner md-grid">
<div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Navigation" class="md-nav md-nav--primary" data-md-level="0">
<label class="md-nav__title" for="__drawer">
<a aria-label="GNPS Documentation" class="md-nav__button md-logo" href="https://ccms-ucsd.github.io/GNPSDocumentation/" title="GNPS Documentation">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"></path></svg>
</a>
    GNPS Documentation
  </label>
<div class="md-nav__source">
<a class="md-source" data-md-component="source" href="https://github.com/CCMS-UCSD/GNPSDocumentation/" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><path d="M439.55 236.05L244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    CCMS-UCSD/GNPSDocumentation
  </div>
</a>
</div>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="..">
        GNPS Introduction
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../isgnpsright/">
        Is my data compatible with GNPS?
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../faq/">
        Frequently Asked Questions
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../superquickstart/">
        Superquick Start Guide
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../quickstart/">
        Quick Start Guide
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gnpsanalysisoverview/">
        GNPS Analysis Tools Overview
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../lexicon/">
        GNPS Lexicon
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_8" id="__nav_8" type="checkbox"/>
<label class="md-nav__link" for="__nav_8">
        Mass Spec/Computational Background
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Mass Spec/Computational Background" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_8">
<span class="md-nav__icon md-icon"></span>
          Mass Spec/Computational Background
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../massspecbackground/networkingtheory/">
        Molecular Networking
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_9" id="__nav_9" type="checkbox"/>
<label class="md-nav__link" for="__nav_9">
        Data Preparation/Upload
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Data Preparation/Upload" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_9">
<span class="md-nav__icon md-icon"></span>
          Data Preparation/Upload
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../fileconversion/">
        Mass Spectrometry File Conversion
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fileupload/">
        GNPS File Upload
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../metadata/">
        Metadata
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_10" id="__nav_10" type="checkbox"/>
<label class="md-nav__link" for="__nav_10">
        Recommended Data Analysis
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Recommended Data Analysis" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_10">
<span class="md-nav__icon md-icon"></span>
          Recommended Data Analysis
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../networking/">
        Molecular Networking
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_10_2" id="__nav_10_2" type="checkbox"/>
<label class="md-nav__link" for="__nav_10_2">
        GC-MS Molecular Networking
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="GC-MS Molecular Networking" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_10_2">
<span class="md-nav__icon md-icon"></span>
          GC-MS Molecular Networking
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../gcanalysis/">
        GC-MS on GNPS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gc-ms-deconvolution/">
        GC-MS Deconvolution
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gc-ms-library-molecular-network/">
        GC-MS Library Search and Molecular Networking
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../networkingviews/">
        Molecular Networking Views
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../librarysearch/">
        Spectral Library Search
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../masst/">
        MASST Search (Mass Search Tool)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../resultviews/">
        Result Views Navigation
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_11" id="__nav_11" type="checkbox"/>
<label class="md-nav__link" for="__nav_11">
        Mass Spec Data Visualization
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Mass Spec Data Visualization" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_11">
<span class="md-nav__icon md-icon"></span>
          Mass Spec Data Visualization
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../lcms-dashboard/">
        GNPS Dashboard
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gnpsdatasetexplorer/">
        GNPS Dataset Explorer
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--active md-nav__item--nested">
<input checked="" class="md-nav__toggle md-toggle" data-md-toggle="__nav_12" id="__nav_12" type="checkbox"/>
<label class="md-nav__link" for="__nav_12">
        Advanced Data Analysis
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Advanced Data Analysis" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_12">
<span class="md-nav__icon md-icon"></span>
          Advanced Data Analysis
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../networkingfiles/">
        Molecular Networking Files
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_12_2" id="__nav_12_2" type="checkbox"/>
<label class="md-nav__link" for="__nav_12_2">
        Feature Based Molecular Networking
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Feature Based Molecular Networking" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_12_2">
<span class="md-nav__icon md-icon"></span>
          Feature Based Molecular Networking
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking/">
        FBMN Workflow
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-mzmine2/">
        FBMN with MZmine
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-ms-dial/">
        FBMN with MS-DIAL
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-xcms3/">
        FBMN with XCMS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-openms/">
        FBMN with OpenMS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-metaboscape/">
        FBMN with MetaboScape
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-progenesisQI/">
        FBMN with Progenesis QI
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-mztab-m/">
        FBMN with mzTab-M
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-cytoscape/">
        FBMN export to Cytoscape
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_12_3" id="__nav_12_3" type="checkbox"/>
<label class="md-nav__link" for="__nav_12_3">
        FBMN - Ion Identity Networking
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="FBMN - Ion Identity Networking" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_12_3">
<span class="md-nav__icon md-icon"></span>
          FBMN - Ion Identity Networking
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin/">
        IIN Workflow
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin-mzmine/">
        IIN with MZmine2
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin-xcms/">
        IIN with XCMS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin-msdial/">
        IIN with MS-DIAL
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../ms2lda/">
        MS2LDA and MotifDB Substructure Discovery
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../cytoscape/">
        Molecular Networking in Cytoscape
      </a>
</li>
<li class="md-nav__item md-nav__item--active">
<input class="md-nav__toggle md-toggle" data-md-toggle="toc" id="__toc" type="checkbox"/>
<label class="md-nav__link md-nav__link--active" for="__toc">
          DEREPLICATOR
          <span class="md-nav__icon md-icon"></span>
</label>
<a class="md-nav__link md-nav__link--active" href="./">
        DEREPLICATOR
      </a>
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#dereplicator-insilico-peptidic-natural-products-tools">
    DEREPLICATOR - Insilico Peptidic Natural Products Tools
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#manuscripts-and-citations">
    Manuscripts and Citations
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#development">
    Development
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#accessing-the-dereplicator-tools-on-gnps">
    Accessing the DEREPLICATOR tools on GNPS
  </a>
<nav aria-label="Accessing the DEREPLICATOR tools on GNPS" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#step-1-go-to-gnps">
    STEP 1: Go to GNPS
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#step-2-access-dereplicator">
    STEP 2: Access DEREPLICATOR
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#step-3-select-your-files">
    STEP 3: SELECT YOUR FILES
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#step-4-configure">
    STEP 4: CONFIGURE
  </a>
<nav aria-label="STEP 4: CONFIGURE" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#basic-options">
    BASIC OPTIONS:
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#advanced-options">
    ADVANCED OPTIONS:
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#advanced-dereplicator-options">
    ADVANCED DEREPLICATOR OPTIONS:
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#advanced-varquest-options">
    ADVANCED VARQUEST OPTIONS:
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#step-5-view-results">
    STEP 5: VIEW RESULTS
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#step-6-analyse">
    STEP 6: ANALYSE
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#step-7-map-annotations-on-molecular-networks">
    STEP 7: MAP ANNOTATIONS ON MOLECULAR NETWORKS
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#step-8-validate">
    STEP 8: VALIDATE
  </a>
<nav aria-label="STEP 8: VALIDATE" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#validating-a-dereplicator-annotation">
    Validating a DEREPLICATOR annotation:
  </a>
</li>
</ul>
</nav>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#dereplicator">
    DEREPLICATOR+
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#contribute-to-the-documentation">
    Contribute to the Documentation
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#page-contributions">
    Page Contributions
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../nap/">
        Network Annotation Propagation (NAP)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../ms2lda/">
        MS2LDA and MotifDB Substructure Discovery
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../molnetenhancer/">
        MolNetEnhancer
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../metaminer/">
        MetaMiner
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../sirius/">
        Sirius
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../ReDU/">
        Reanalysis of Data User Interface (ReDU)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../qemistree/">
        Qemistree
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../mmvec/">
        MMVEC
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../mergepolarity/">
        Merge Network Polarity
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../chemdir/">
        Chemical Directionality
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../usi/">
        Spectrum Resolover
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_13" id="__nav_13" type="checkbox"/>
<label class="md-nav__link" for="__nav_13">
        Spectral Libraries
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Spectral Libraries" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_13">
<span class="md-nav__icon md-icon"></span>
          Spectral Libraries
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../browselibraries/">
        Browsing Reference Spectral Libraries
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../spectrumcuration/">
        Add/Curate Reference Spectrum in the Library
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../batchupload/">
        Reference Spectral Library Batch Addition
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../downloadlibraries/">
        Downloading Reference Spectral Libraries
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_13_5" id="__nav_13_5" type="checkbox"/>
<label class="md-nav__link" for="__nav_13_5">
        Open-source Method for Reference Library Generation
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Open-source Method for Reference Library Generation" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_13_5">
<span class="md-nav__icon md-icon"></span>
          Open-source Method for Reference Library Generation
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../msmschooser/">
        MSMS-Chooser Workflow
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../msmschooserdataacquisitioncomputational/">
        Computational Data Acquisition Prep
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../msmschooserdataacquisitionQE/">
        Data Acqusition - Thermo QExactive (v1.0)
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tag_info/">
        GNPS Tag Contribution and Analyses
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_14" id="__nav_14" type="checkbox"/>
<label class="md-nav__link" for="__nav_14">
        Create a Public Datasets
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Create a Public Datasets" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_14">
<span class="md-nav__icon md-icon"></span>
          Create a Public Datasets
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../datasets/">
        Dataset Creation/Sharing
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../continuousid/">
        Continuous Identification
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../moleculeexplorer/">
        Molecule Explorer
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_15" id="__nav_15" type="checkbox"/>
<label class="md-nav__link" for="__nav_15">
        Community/Social Features
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Community/Social Features" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_15">
<span class="md-nav__icon md-icon"></span>
          Community/Social Features
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../user/">
        User Page
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../troubleshooting/">
        Troubleshooting
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_17" id="__nav_17" type="checkbox"/>
<label class="md-nav__link" for="__nav_17">
        Tutorial Guides
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Tutorial Guides" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_17">
<span class="md-nav__icon md-icon"></span>
          Tutorial Guides
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/bacterianetworking/">
        Molecular Networking - Analog Discovery in Microbial Extracts
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/americangutmzmine/">
        Feature-Based Molecular Networking (FBMN) with MZmine2 - Microbiome Molecules in Fecal Samples in American Gut Project
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/americangut-ms-dial/">
        FBMN with MS-DIAL - Microbiome Molecules in Fecal Samples in American Gut Project
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/coffee-tutorial-metaboscape/">
        FBMN with MetaboScape - Coffee Analysis
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/featurebasedgnps/">
        FBMN analysis at GNPS
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../changelog/">
        Change Log
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gnps_community/">
        Join the GNPS community
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../workshops/">
        GNPS Workshops/Training
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../logos/">
        GNPS Logos
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../contact/">
        Contact/Office Hours
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../toolindex/">
        GNPS Tool Index
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../api/">
        API
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../developers/">
        For Developers
      </a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#dereplicator-insilico-peptidic-natural-products-tools">
    DEREPLICATOR - Insilico Peptidic Natural Products Tools
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#manuscripts-and-citations">
    Manuscripts and Citations
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#development">
    Development
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#accessing-the-dereplicator-tools-on-gnps">
    Accessing the DEREPLICATOR tools on GNPS
  </a>
<nav aria-label="Accessing the DEREPLICATOR tools on GNPS" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#step-1-go-to-gnps">
    STEP 1: Go to GNPS
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#step-2-access-dereplicator">
    STEP 2: Access DEREPLICATOR
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#step-3-select-your-files">
    STEP 3: SELECT YOUR FILES
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#step-4-configure">
    STEP 4: CONFIGURE
  </a>
<nav aria-label="STEP 4: CONFIGURE" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#basic-options">
    BASIC OPTIONS:
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#advanced-options">
    ADVANCED OPTIONS:
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#advanced-dereplicator-options">
    ADVANCED DEREPLICATOR OPTIONS:
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#advanced-varquest-options">
    ADVANCED VARQUEST OPTIONS:
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#step-5-view-results">
    STEP 5: VIEW RESULTS
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#step-6-analyse">
    STEP 6: ANALYSE
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#step-7-map-annotations-on-molecular-networks">
    STEP 7: MAP ANNOTATIONS ON MOLECULAR NETWORKS
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#step-8-validate">
    STEP 8: VALIDATE
  </a>
<nav aria-label="STEP 8: VALIDATE" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#validating-a-dereplicator-annotation">
    Validating a DEREPLICATOR annotation:
  </a>
</li>
</ul>
</nav>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#dereplicator">
    DEREPLICATOR+
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#contribute-to-the-documentation">
    Contribute to the Documentation
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#page-contributions">
    Page Contributions
  </a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-content" data-md-component="content">
<article class="md-content__inner md-typeset">
<a class="md-content__button md-icon" href="https://github.com/CCMS-UCSD/GNPSDocumentation/edit/master/docs/dereplicator.md" title="Edit this page">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25z"></path></svg>
</a>
<h1>DEREPLICATOR</h1>
<h2 id="dereplicator-insilico-peptidic-natural-products-tools">DEREPLICATOR - Insilico Peptidic Natural Products Tools<a class="headerlink" href="#dereplicator-insilico-peptidic-natural-products-tools" title="Permanent link">¶</a></h2>
<ul>
<li><a href="#dereplicator---insilico-peptidic-natural-products-tools">DEREPLICATOR - Insilico Peptidic Natural Products Tools</a></li>
<li><a href="#manuscripts-and-citations">Manuscripts and Citations</a></li>
<li><a href="#development">Development</a></li>
<li><a href="#accessing-the-dereplicator-tools-on-gnps">Accessing the DEREPLICATOR tools on GNPS</a><ul>
<li><a href="#step-1--go-to-gnps">STEP 1: Go to GNPS</a></li>
<li><a href="#step-2--access-dereplicator">STEP 2: Access DEREPLICATOR</a></li>
<li><a href="#step-3--select-your-files">STEP 3: SELECT YOUR FILES</a></li>
<li><a href="#step-4--configure">STEP 4: CONFIGURE</a></li>
<li><a href="#basic-options-">BASIC OPTIONS:</a></li>
<li><a href="#advanced-options-">ADVANCED OPTIONS:</a></li>
<li><a href="#advanced-dereplicator-options-">ADVANCED DEREPLICATOR OPTIONS:</a></li>
<li><a href="#advanced-varquest-options-">ADVANCED VARQUEST OPTIONS:</a></li>
<li><a href="#step-5--view-results">STEP 5: VIEW RESULTS</a></li>
<li><a href="#step-6--analyse">STEP 6: ANALYSE</a></li>
<li><a href="#step-7--map-annotations-on-molecular-networks">STEP 7: MAP ANNOTATIONS ON MOLECULAR NETWORKS</a></li>
<li><a href="#step-8--validate">STEP 8: VALIDATE</a></li>
</ul>
</li>
<li><a href="#dereplicator-">DEREPLICATOR+</a></li>
</ul>
<p>The Insilico Peptidic Natural Products <strong>DEREPLICATOR</strong> is a bioinformatic tool available on GNPS that allows the annotation of known peptidic natural products in MS/MS data using in silico fragmentation tree.</p>
<p>The introduction of <strong>DEREPLICATOR VarQuest</strong> made possible to search variants of known peptidic natural products on GNPS.</p>
<p>The introduction of <strong>DEREPLICATOR+ (PLUS)</strong> improved the confidence in the annotation from the DEREPLICATOR, and expanded annotations to non-peptidic natural products on GNPS. </p>
<p>These algorithms and software are developed in collaboration of <a href="http://cab.spbu.ru">Saint Petersburg State University</a> (Russia), 
<a href="http://cseweb.ucsd.edu/~ppevzner/">University of California San Diego</a> (CA, USA) 
and <a href="http://mohimanilab.cbd.cmu.edu">Carnegie Mellon University</a> (PA, USA).</p>
<h2 id="manuscripts-and-citations">Manuscripts and Citations<a class="headerlink" href="#manuscripts-and-citations" title="Permanent link">¶</a></h2>
<p>If you used DEREPLICATOR tools, please cite the corresponding manuscripts:</p>
<p><strong>DEREPLICATOR:</strong>
Mohimani et al., Dereplication of Peptidic Natural Products Through Database Search of Mass Spectra, <em>Nature Chemical Biology</em>, 13, 30–37 (2017). <a href="https://doi.org/10.1038/nchembio.2219">https://doi.org/10.1038/nchembio.2219</a>.</p>
<p><strong>DEREPLICATOR VarQuest</strong>: Gurevich, A. et al. Increased diversity of peptidic natural products revealed by modification-tolerant database search of mass spectra. <em>Nature Microbiology</em> 3, 319–327 (2018). <a href="https://doi.org/10.1038/s41564-017-0094-2">https://doi.org/10.1038/s41564-017-0094-2</a></p>
<p><strong>DEREPLICATOR+</strong>: Mohimani, H. et al. Dereplication of microbial metabolites through database search of mass spectra. <em>Nature Communication</em> 9, 4035 (2018). <a href="https://doi.org/10.1038/s41467-018-06082-8">https://doi.org/10.1038/s41467-018-06082-8</a></p>
<h2 id="development">Development<a class="headerlink" href="#development" title="Permanent link">¶</a></h2>
<p>The <a href="https://github.com/ablab/npdtools">GitHub repository</a> provides latest source code and release. </p>
<h2 id="accessing-the-dereplicator-tools-on-gnps">Accessing the DEREPLICATOR tools on GNPS<a class="headerlink" href="#accessing-the-dereplicator-tools-on-gnps" title="Permanent link">¶</a></h2>
<p>The <strong>DEREPLICATOR (VarQuest)</strong>, and <strong>DEREPLICATOR+</strong> are available on GNPS on the <a href="https://gnps.ucsd.edu/ProteoSAFe/static/gnps-theoretical.jsp">in silico tools page</a>.</p>
<h4 id="step-1-go-to-gnps">STEP 1: Go to GNPS<a class="headerlink" href="#step-1-go-to-gnps" title="Permanent link">¶</a></h4>
<p>Go to <a href="https://gnps.ucsd.edu/ProteoSAFe/static/gnps-splash.jsp">http://gnps.ucsd.edu</a>, and create/login to your account.</p>
<h4 id="step-2-access-dereplicator">STEP 2: Access DEREPLICATOR<a class="headerlink" href="#step-2-access-dereplicator" title="Permanent link">¶</a></h4>
<p>Find the "In Silico Tools" box and click on "Browse Tools".</p>
<p><img alt="img" src="../img/dereplicator/Figure1.png"/></p>
<p>Choose either DEREPLICATOR (VarQuest) or DEREPLICATOR+.</p>
<p><img alt="img" src="../img/dereplicator/Figure2.png"/></p>
<h4 id="step-3-select-your-files">STEP 3: SELECT YOUR FILES<a class="headerlink" href="#step-3-select-your-files" title="Permanent link">¶</a></h4>
<p><strong>Select/Import a mass spectrometry file</strong>:
The input format accepted are mzML/mzXML/MGF. Note that have the option to import an existing GNPS dataset (by clicking on "Share Files") or upload your own data (by clicking on "Upload Files"). See the corresponding GNPS documentation for FTP upload.</p>
<p>After getting done with selecting files, click on Finish Selection.</p>
<h4 id="step-4-configure">STEP 4: CONFIGURE<a class="headerlink" href="#step-4-configure" title="Permanent link">¶</a></h4>
<p><strong>Configure the parameters for the DEREPLICATOR job</strong>:
Select a title for your job, and adjust parameters depending on your data and the mass spectrometer used.</p>
<p><img alt="img" src="../img/dereplicator/Figure3.png"/></p>
<h5 id="basic-options">BASIC OPTIONS:<a class="headerlink" href="#basic-options" title="Permanent link">¶</a></h5>
<p>-- <strong>Precursor Ion Mass Tolerance</strong>:  This value specifies how much fragment ions can be shifted from their expected m/z values. Default value is ± 0.02 Da for high-resolution instruments (q-TOF, q-Orbitrap) and ± 0.5 Da for low-resolution instruments (ion traps, QqQ).</p>
<p>-- <strong>Fragment ion Mass Tolerance</strong>: This value specifies how much fragment ions can be shifted from their expected m/z values. Default value is ± 0.02 Da for high-resolution instruments (q-TOF, q-Orbitrap) and ± 0.5 Da for low-resolution instruments (ion traps, QqQ).</p>
<p>-- <strong>Search analog (VarQuest)</strong> [RECOMMENDED TO USE]: The VarQuest algorithm can search for analogs of known natural products in MS/MS data using Dereplicator. VarQuest is a modification-tolerant database search tool that discovers unexpected modifications in a blind mode. Since the computational space for variable PNP identification is computationally expensive, VarQuest first constructs a set of feasible PSM using a simple scoring approach and further filters out this set using a rigorous scoring method. Using VarQuest will increase the job time. The use of VarQuest analogs option is recommended as it expands the search space, and it reduces the overall run time.</p>
<h5 id="advanced-options">ADVANCED OPTIONS:<a class="headerlink" href="#advanced-options" title="Permanent link">¶</a></h5>
<p>-- <strong>PNP database</strong> : a choice of two databases Regular/Extended (Extended one is more informative but 2 times larger, i.e. two times slower processing)</p>
<p>-- <strong>Mimimum number of AA</strong>: Dereplicator is able to search for short peptides. This parameter determines what is the minimum number of amino acids, we consider for the compounds that are considered in Dereplicator search.</p>
<p>-- <strong>Accurate P-values</strong> : accurate (but slow) algorithm for P-value computation (MS-DPR), by default raw estimation is used.</p>
<p>-- <strong>Max charge</strong>: Maximum charge allowed.</p>
<h5 id="advanced-dereplicator-options">ADVANCED DEREPLICATOR OPTIONS:<a class="headerlink" href="#advanced-dereplicator-options" title="Permanent link">¶</a></h5>
<p>-- <strong>Max Isotopic Shift</strong>: When working with molecules with 1000 Da  or higher masses, mass spectrometers commonly predict the precursor mass 1 or 2 Da higher than the monoisotopic mass due to the isotopic pattern. Setting Maximum isotopic shift to higher than 0 allows Dereplicator to consider these shifts in the precursor mass.</p>
<p>-- <strong>Adducts</strong> : Dereplicator can search in addition to protonated adducts, sodiated and potassium adducts.</p>
<h5 id="advanced-varquest-options">ADVANCED VARQUEST OPTIONS:<a class="headerlink" href="#advanced-varquest-options" title="Permanent link">¶</a></h5>
<p>-- <strong>Max Allowed Modification Mass</strong> : Maximum difference allowed between the precursor ion of an unknown modified peptide and the (dereplicated) known peptide.</p>
<p>-- <strong>Min Matched Peaks with Known Compound</strong> : Minimum number of common peaks between the MS/MS spectrum of an unknown modified peptide and the (dereplicated) known peptide.</p>
<p>Select an email. Then click on submit.</p>
<p>You will soon get an email with the link to the results when your job is finished. You can also check the status of your job in "Jobs" section.</p>
<h4 id="step-5-view-results">STEP 5: VIEW RESULTS<a class="headerlink" href="#step-5-view-results" title="Permanent link">¶</a></h4>
<p><strong>See the DEREPLICATOR job status</strong>:</p>
<p>The status of the job can be vizualized in your account job list, and through the link in provided in email you will received when the job is done.</p>
<p>The job can be <em>Cloned</em>, for further parameters modifications. Summary Statistics and Workflow Parameters can be consulted on the respective links.</p>
<p>Clic on <em>View Unique Peptides</em> (recommended), to have get the list of annotated molecules.</p>
<p>Clic on the <em>View All PSM</em> (Peptide-Spectrum Match) to have detailed view of the peptides-to-spectrum matches.</p>
<p><img alt="img" src="../img/dereplicator/Figure4.png"/></p>
<h4 id="step-6-analyse">STEP 6: ANALYSE<a class="headerlink" href="#step-6-analyse" title="Permanent link">¶</a></h4>
<p><strong>Explore the DEREPLICATOR annotations:</strong>
Annotations can be sorted using various column (usually the score or the p-value), or a compound name can be searched with the filter function.</p>
<p><img alt="img" src="../img/dereplicator/Figure5.png"/></p>
<p>Clic on <em>Show Annotation</em> to see the experimental spectrum and the matching with the fragmentation tree of the Dereplicator database.</p>
<p><img alt="img" src="../img/dereplicator/Figure6.png"/></p>
<p>The experimental MS/MS spectrum can be viewed in the left panel. The blue peaks are fragment ions that matched the fragmentation tree of peptidic natural products in the database.</p>
<p>The structure of the molecule is displayed on the right.</p>
<p>In the Annotated fragments table, lists the ion fragments that matched with the fragmentation tree, along with their mass error, charge and intensities.</p>
<p>Click on a blue peak to highlighted the corresponding fragment on the molecular structure, along with its properties.</p>
<p><img alt="img" src="../img/dereplicator/Figure7.png"/></p>
<h4 id="step-7-map-annotations-on-molecular-networks">STEP 7: MAP ANNOTATIONS ON MOLECULAR NETWORKS<a class="headerlink" href="#step-7-map-annotations-on-molecular-networks" title="Permanent link">¶</a></h4>
<p><strong>Map and Explore the DEREPLICATOR annotations in the networks:</strong>
1. Run a classical or feature-based molecular networks with the spectral data (Files format can be mzML/mzXML/MGF). <a href="../networking/">See the following documentation</a>. Note that the <a href="../featurebasedmolecularnetworking/">Feature-Based Molecular Network workflow</a> can run the <strong>DEREPLICATOR VarQuest</strong> automatically (with default parameters only).</p>
<ol>
<li>If you run a classical molecular networking job, download the clustered spectra (.MGF file). Click on <em>Download Clustered Spectra as MGF</em>. If you run feature-based molecular network, uses the same MS/MS spectral summary file (.MGF file).</li>
</ol>
<p><img alt="img" src="../img/dereplicator/Figure8.png"/></p>
<ol>
<li>
<p>Upload the MGF file for the clustered spectra or the MS/MS spectral summary, and run a DEREPLICATOR job (see present documentation).</p>
</li>
<li>
<p>When the job is completed click on <em>View Unique Peptides</em>.</p>
</li>
<li>
<p>Click on <em>View unique Peptide</em>, or <em>View all PSM (Peptide-Spectrum Matches)</em>. On the annotation results page, click on download with Download button and unzip the file on your computer.</p>
</li>
</ol>
<p><img alt="img" src="../img/dereplicator/Figure9.png"/></p>
<ol>
<li>
<p>Open the corresponding molecular networks with Cytoscape. <a href="../cytoscape/">See documentation here</a>, and here for <a href="../featurebasedmolecularnetworking/">Feature-Based Molecular Networking</a></p>
</li>
<li>
<p>In Cytoscape, import the .TSV table as <em>attribute table</em>. In Cytoscape go to <em>File / Import / Table / File</em> and select the .TSV annotation file for the dereplicator (see above).</p>
</li>
<li>
<p>In the preview window, click on the <em>Scan</em> header to set the meaning to <em>Key</em>. The value in that column corresponds to the "shared name" or "ClusterIdx" in the molecular network.</p>
</li>
</ol>
<p><img alt="img" src="../img/dereplicator/Figure10.png"/></p>
<ol>
<li>
<p>Select/Unselect column as needed, and eventually change the name for table header if they are already in the molecular networks (otherwise, they will be replaced with the new table imported).</p>
</li>
<li>
<p>Change the column header name from <em>SMILES</em> into <em>SMILES_derepl</em></p>
</li>
<li>
<p>Click on import. Now the attribute table is mapped into the networks.</p>
</li>
<li>
<p>The structures can be viewed directly into Cytoscape with the ChemViz2 plugin. See the <a href="../nap/">Network Annotation Propagation</a> documentation for more details.</p>
</li>
<li>
<p>Explore Dereplicator annotations in the network with the <a href="https://ccms-ucsd.github.io/GNPSDocumentation/featurebasedmolecularnetworking-cytoscape/#the-select-function">select function of Cytoscape</a>.</p>
</li>
</ol>
<h4 id="step-8-validate">STEP 8: VALIDATE<a class="headerlink" href="#step-8-validate" title="Permanent link">¶</a></h4>
<p><strong>Validate a DEREPLICATOR annotation</strong>:
More than 60% of Dereplicator annotations on GNPS Massive datasets (April 2016) were manually curated.</p>
<h5 id="validating-a-dereplicator-annotation">Validating a DEREPLICATOR annotation:<a class="headerlink" href="#validating-a-dereplicator-annotation" title="Permanent link">¶</a></h5>
<p>Ideally the annotation should be validated by comparing the fragmentation spectra and other properties (retention time, cross-colision section, ...) of an authentic reference compound analyzed in the same LC-MS/MS conditions. The MS/MS spectrum and retention time should be identical between the observed and reference MS/MS spectra (co-injection can be done to verify retention time). \
In practice, the confidence in the annotation can be increased by:
1) comparing with a fragmentation spectra obtained in other experimental condition (i.e. other mass spectrometry plateform).\
2) using other computational annotations tools. \
3) examining if the main fragment ions are consistent with the structure proposed by the DEREPLICATOR.
4) if genome sequence(s) are available, observing a biosynthetic gene cluster associated with the molecule.</p>
<p>In any cases the following are recommended:
- The MS/MS spectrum from the raw file must be inspected (consistency, noise level, ...)</p>
<ul>
<li>
<p>The adducts detected must be confirmed using the MS1 spectrum (mono-charged, di-charged ?, protonated adduct ?, ...). This can be done MZmine2.</p>
</li>
<li>
<p>The molecular formula should be verified. <a href="https://bio.informatik.uni-jena.de/software/sirius/">Sirius</a> can be used for that (only for single-charged compounds).</p>
</li>
<li>
<p>Look at the biological source(s) (species) for the peptidic natural product (Google scholar, Dictionnary of Natural Products, AntiMarin, MarinLit, ...), to see if it is consistent with the sample.</p>
</li>
</ul>
<h2 id="dereplicator">DEREPLICATOR+<a class="headerlink" href="#dereplicator" title="Permanent link">¶</a></h2>
<ul>
<li>The DEREPLICATOR+ is very similar to the DEREPLICATOR workflow, and currently no documentation is available.</li>
</ul>
<h2 id="contribute-to-the-documentation">Contribute to the Documentation<a class="headerlink" href="#contribute-to-the-documentation" title="Permanent link">¶</a></h2>
<ul>
<li>For informations/feature request, please open an "Issue" on the <a href="https://github.com/CCMS-UCSD/GNPSDocumentation"><em>CCMS-UCSD/GNPSDocumentation</em></a> GitHub repository.</li>
<li>To contribute directly to the GNPS documentation, fork the <a href="https://github.com/CCMS-UCSD/GNPSDocumentation"><em>CCMS-UCSD/GNPSDocumentation</em></a> repository, and make a "Pull Request".</li>
</ul>
<h2 id="page-contributions">Page Contributions<a class="headerlink" href="#page-contributions" title="Permanent link">¶</a></h2>
<p><span class="git-page-authors git-authors"><a href="mailto:mwang87@gmail.com">Ming Wang</a> (6.19%), <a href="mailto:louisfelix.nothias@gmail.com">lfnothias</a> (93.81%)</span></p>
<hr/>
<div class="md-source-date">
<small>
    
      Last update: <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-datetime">August 12, 2020 04:16:23</span>
</small>
</div>
</article>
</div>
</div>
</main>
<footer class="md-footer">
<nav aria-label="Footer" class="md-footer__inner md-grid">
<a class="md-footer__link md-footer__link--prev" href="../cytoscape/" rel="prev">
<div class="md-footer__button md-icon">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"></path></svg>
</div>
<div class="md-footer__title">
<div class="md-ellipsis">
<span class="md-footer__direction">
                Previous
              </span>
              Molecular Networking in Cytoscape
            </div>
</div>
</a>
<a class="md-footer__link md-footer__link--next" href="../nap/" rel="next">
<div class="md-footer__title">
<div class="md-ellipsis">
<span class="md-footer__direction">
                Next
              </span>
              Network Annotation Propagation (NAP)
            </div>
</div>
<div class="md-footer__button md-icon">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4z"></path></svg>
</div>
</a>
</nav>
<div class="md-footer-meta md-typeset">
<div class="md-footer-meta__inner md-grid">
<div class="md-footer-copyright">
        
        Made with
        <a href="https://squidfunk.github.io/mkdocs-material/" rel="noopener" target="_blank">
          Material for MkDocs
        </a>
</div>
</div>
</div>
</footer>
</div>
<div class="md-dialog" data-md-component="dialog">
<div class="md-dialog__inner md-typeset"></div>
</div>
<script id="__config" type="application/json">{"base": "..", "features": [], "translations": {"clipboard.copy": "Copy to clipboard", "clipboard.copied": "Copied to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.placeholder": "Type to start searching", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.term.missing": "Missing"}, "search": "../assets/javascripts/workers/search.f5903571.min.js", "version": null}</script>
<script src="../assets/javascripts/bundle.3981fc68.min.js"></script>
</body>
</html>