



<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      <meta http-equiv="x-ua-compatible" content="ie=edge">
      
      
        <link rel="canonical" href="https://ccms-ucsd.github.io/GNPSDocumentation/gc-ms-library-molecular-network/">
      
      
      
        <meta name="lang:clipboard.copy" content="Copy to clipboard">
      
        <meta name="lang:clipboard.copied" content="Copied to clipboard">
      
        <meta name="lang:search.language" content="en">
      
        <meta name="lang:search.pipeline.stopwords" content="True">
      
        <meta name="lang:search.pipeline.trimmer" content="True">
      
        <meta name="lang:search.result.none" content="No matching documents">
      
        <meta name="lang:search.result.one" content="1 matching document">
      
        <meta name="lang:search.result.other" content="# matching documents">
      
        <meta name="lang:search.tokenizer" content="[\s\-]+">
      
      <link rel="shortcut icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.0.4, mkdocs-material-4.6.0">
    
    
      
        <title>GC-MS Library Search and Molecular Networking - GNPS Documentation</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/application.1b62728e.css">
      
      
    
    
      <script src="../assets/javascripts/modernizr.268332fc.js"></script>
    
    
      
        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700%7CRoboto+Mono&display=fallback">
        <style>body,input{font-family:"Roboto","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
      
    
    <link rel="stylesheet" href="../assets/fonts/material-icons.css">
    
    
    
      
        
<script>
  window.ga = window.ga || function() {
    (ga.q = ga.q || []).push(arguments)
  }
  ga.l = +new Date
  /* Setup integration and send page view */
  ga("create", "UA-8412213-5", "ccms-ucsd.github.io")
  ga("set", "anonymizeIp", true)
  ga("send", "pageview")
  /* Register handler to log search on blur */
  document.addEventListener("DOMContentLoaded", () => {
    if (document.forms.search) {
      var query = document.forms.search.query
      query.addEventListener("blur", function() {
        if (this.value) {
          var path = document.location.pathname;
          ga("send", "pageview", path + "?q=" + this.value)
        }
      })
    }
  })
</script>
<script async src="https://www.google-analytics.com/analytics.js"></script>
      
    
    
  </head>
  
    <body dir="ltr">
  
    <svg class="md-svg">
      <defs>
        
        
      </defs>
    </svg>
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" data-md-component="overlay" for="__drawer"></label>
    
      <a href="#gc-ms-spectral-library-search-and-molecular-networking" tabindex="1" class="md-skip">
        Skip to content
      </a>
    
    
      <header class="md-header" data-md-component="header">
  <nav class="md-header-nav md-grid">
    <div class="md-flex">
      <div class="md-flex__cell md-flex__cell--shrink">
        <a href="https://ccms-ucsd.github.io/GNPSDocumentation/" title="GNPS Documentation" class="md-header-nav__button md-logo">
          
            <i class="md-icon"></i>
          
        </a>
      </div>
      <div class="md-flex__cell md-flex__cell--shrink">
        <label class="md-icon md-icon--menu md-header-nav__button" for="__drawer"></label>
      </div>
      <div class="md-flex__cell md-flex__cell--stretch">
        <div class="md-flex__ellipsis md-header-nav__title" data-md-component="title">
          
            <span class="md-header-nav__topic">
              GNPS Documentation
            </span>
            <span class="md-header-nav__topic">
              
                GC-MS Library Search and Molecular Networking
              
            </span>
          
        </div>
      </div>
      <div class="md-flex__cell md-flex__cell--shrink">
        
          <label class="md-icon md-icon--search md-header-nav__button" for="__search"></label>
          
<div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="query" data-md-state="active">
      <label class="md-icon md-search__icon" for="__search"></label>
      <button type="reset" class="md-icon md-search__icon" data-md-component="reset" tabindex="-1">
        &#xE5CD;
      </button>
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="result">
          <div class="md-search-result__meta">
            Type to start searching
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
        
      </div>
      
        <div class="md-flex__cell md-flex__cell--shrink">
          <div class="md-header-nav__source">
            


  

<a href="/" title="Go to repository" class="md-source" data-md-source="">
  
  <div class="md-source__repository">
    
  </div>
</a>
          </div>
        </div>
      
    </div>
  </nav>
</header>
    
    <div class="md-container">
      
        
      
      
      <main class="md-main" role="main">
        <div class="md-main__inner md-grid" data-md-component="container">
          
            
              <div class="md-sidebar md-sidebar--primary" data-md-component="navigation">
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    <nav class="md-nav md-nav--primary" data-md-level="0">
  <label class="md-nav__title md-nav__title--site" for="__drawer">
    <a href="https://ccms-ucsd.github.io/GNPSDocumentation/" title="GNPS Documentation" class="md-nav__button md-logo">
      
        <i class="md-icon"></i>
      
    </a>
    GNPS Documentation
  </label>
  
    <div class="md-nav__source">
      


  

<a href="/" title="Go to repository" class="md-source" data-md-source="">
  
  <div class="md-source__repository">
    
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      


  <li class="md-nav__item">
    <a href=".." title="GNPS Introduction" class="md-nav__link">
      GNPS Introduction
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../isgnpsright/" title="Is my data compatible with GNPS?" class="md-nav__link">
      Is my data compatible with GNPS?
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../faq/" title="Frequently Asked Questions" class="md-nav__link">
      Frequently Asked Questions
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../superquickstart/" title="Superquick Start Guide" class="md-nav__link">
      Superquick Start Guide
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../quickstart/" title="Quick Start Guide" class="md-nav__link">
      Quick Start Guide
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../gnpsanalysisoverview/" title="GNPS Analysis Tools Overview" class="md-nav__link">
      GNPS Analysis Tools Overview
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-7" type="checkbox" id="nav-7">
    
    <label class="md-nav__link" for="nav-7">
      Mass Spec/Computational Background
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-7">
        Mass Spec/Computational Background
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../massspecbackground/networkingtheory/" title="Molecular Networking" class="md-nav__link">
      Molecular Networking
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-8" type="checkbox" id="nav-8">
    
    <label class="md-nav__link" for="nav-8">
      Data Preparation/Upload
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-8">
        Data Preparation/Upload
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../fileconversion/" title="Mass Spectrometry File Conversion" class="md-nav__link">
      Mass Spectrometry File Conversion
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../fileupload/" title="GNPS File Upload" class="md-nav__link">
      GNPS File Upload
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../metadata/" title="Metadata" class="md-nav__link">
      Metadata
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      

  


  <li class="md-nav__item md-nav__item--active md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-9" type="checkbox" id="nav-9" checked>
    
    <label class="md-nav__link" for="nav-9">
      Recommended Data Analysis
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-9">
        Recommended Data Analysis
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../networking/" title="Molecular Networking" class="md-nav__link">
      Molecular Networking
    </a>
  </li>

        
          
          
          

  


  <li class="md-nav__item md-nav__item--active md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-9-2" type="checkbox" id="nav-9-2" checked>
    
    <label class="md-nav__link" for="nav-9-2">
      GC-MS Molecular Networking
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="2">
      <label class="md-nav__title" for="nav-9-2">
        GC-MS Molecular Networking
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../gcanalysis/" title="GC-MS on GNPS" class="md-nav__link">
      GC-MS on GNPS
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../gc-ms-deconvolution/" title="GC-MS Deconvolution" class="md-nav__link">
      GC-MS Deconvolution
    </a>
  </li>

        
          
          
          

  


  <li class="md-nav__item md-nav__item--active">
    
    <input class="md-toggle md-nav__toggle" data-md-toggle="toc" type="checkbox" id="__toc">
    
      
    
    
      <label class="md-nav__link md-nav__link--active" for="__toc">
        GC-MS Library Search and Molecular Networking
      </label>
    
    <a href="./" title="GC-MS Library Search and Molecular Networking" class="md-nav__link md-nav__link--active">
      GC-MS Library Search and Molecular Networking
    </a>
    
      
<nav class="md-nav md-nav--secondary">
  
  
    
  
  
    <label class="md-nav__title" for="__toc">Table of contents</label>
    <ul class="md-nav__list" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#configure-the-workflow" class="md-nav__link">
    Configure the workflow
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#inspect-the-results" class="md-nav__link">
    Inspect the Results
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#cloning-a-job" class="md-nav__link">
    Cloning a job
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sharing-data" class="md-nav__link">
    Sharing data
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#addingcurating-reference-spectra" class="md-nav__link">
    Adding/Curating reference spectra
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#visualization" class="md-nav__link">
    Visualization
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#network-visualization-in-cytoscape" class="md-nav__link">
    Network Visualization in Cytoscape
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#molecular-mapping-in-ili" class="md-nav__link">
    Molecular mapping in ili
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#page-contributors" class="md-nav__link">
    Page contributors
  </a>
  
</li>
      
      
      
      
      
    </ul>
  
</nav>
    
  </li>

        
      </ul>
    </nav>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../networkingviews/" title="Molecular Networking Views" class="md-nav__link">
      Molecular Networking Views
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../librarysearch/" title="Spectral Library Search" class="md-nav__link">
      Spectral Library Search
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../masst/" title="MASST Search (Mass Search Tool)" class="md-nav__link">
      MASST Search (Mass Search Tool)
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../resultviews/" title="Result Views Navigation" class="md-nav__link">
      Result Views Navigation
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-10" type="checkbox" id="nav-10">
    
    <label class="md-nav__link" for="nav-10">
      Advanced Data Analysis
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-10">
        Advanced Data Analysis
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../dereplicator/" title="DEREPLICATOR" class="md-nav__link">
      DEREPLICATOR
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-10-2" type="checkbox" id="nav-10-2">
    
    <label class="md-nav__link" for="nav-10-2">
      Feature Based Molecular Networking
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="2">
      <label class="md-nav__title" for="nav-10-2">
        Feature Based Molecular Networking
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking/" title="FBMN Workflow" class="md-nav__link">
      FBMN Workflow
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-mzmine2/" title="FBMN with MZmine" class="md-nav__link">
      FBMN with MZmine
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-ms-dial/" title="FBMN with MS-DIAL" class="md-nav__link">
      FBMN with MS-DIAL
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-xcms3/" title="FBMN with XCMS3" class="md-nav__link">
      FBMN with XCMS3
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-openms/" title="FBMN with OpenMS" class="md-nav__link">
      FBMN with OpenMS
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-metaboscape/" title="FBMN with MetaboScape" class="md-nav__link">
      FBMN with MetaboScape
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-progenesisQI/" title="FBMN with Progenesis QI" class="md-nav__link">
      FBMN with Progenesis QI
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-mztab-m/" title="FBMN with mzTab-M" class="md-nav__link">
      FBMN with mzTab-M
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-cytoscape/" title="FBMN export to Cytoscape" class="md-nav__link">
      FBMN export to Cytoscape
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../ms2lda/" title="MS2LDA and MotifDB Substructure Discovery" class="md-nav__link">
      MS2LDA and MotifDB Substructure Discovery
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../cytoscape/" title="Molecular Networking in Cytoscape" class="md-nav__link">
      Molecular Networking in Cytoscape
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../molnetenhancer/" title="MolNetEnhancer" class="md-nav__link">
      MolNetEnhancer
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../nap/" title="Network Annotation Propagation (NAP)" class="md-nav__link">
      Network Annotation Propagation (NAP)
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../ReDU/" title="Reanalysis of Data User Interface (ReDU)" class="md-nav__link">
      Reanalysis of Data User Interface (ReDU)
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-11" type="checkbox" id="nav-11">
    
    <label class="md-nav__link" for="nav-11">
      Spectral Libraries
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-11">
        Spectral Libraries
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../browselibraries/" title="Browsing Reference Spectral Libraries" class="md-nav__link">
      Browsing Reference Spectral Libraries
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../spectrumcuration/" title="Add/Curate Reference Spectrum in the Library" class="md-nav__link">
      Add/Curate Reference Spectrum in the Library
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../batchupload/" title="Reference Spectral Library Batch Addition" class="md-nav__link">
      Reference Spectral Library Batch Addition
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../downloadlibraries/" title="Downloading Reference Spectral Libraries" class="md-nav__link">
      Downloading Reference Spectral Libraries
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-11-5" type="checkbox" id="nav-11-5">
    
    <label class="md-nav__link" for="nav-11-5">
      Open-source Method for Reference Library Generation
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="2">
      <label class="md-nav__title" for="nav-11-5">
        Open-source Method for Reference Library Generation
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../msmschooserdataacquisitionQE/" title="Data Acqusition - Thermo QExactive (v1.0)" class="md-nav__link">
      Data Acqusition - Thermo QExactive (v1.0)
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../msmschooser/" title="Computational Selection of MSMS to Include in Library (MSMS-Chooser)" class="md-nav__link">
      Computational Selection of MSMS to Include in Library (MSMS-Chooser)
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-12" type="checkbox" id="nav-12">
    
    <label class="md-nav__link" for="nav-12">
      Create a Public Datasets
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-12">
        Create a Public Datasets
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../datasets/" title="Dataset Creation/Sharing" class="md-nav__link">
      Dataset Creation/Sharing
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../continuousid/" title="Continuous Identification" class="md-nav__link">
      Continuous Identification
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../moleculeexplorer/" title="Molecule Explorer" class="md-nav__link">
      Molecule Explorer
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-13" type="checkbox" id="nav-13">
    
    <label class="md-nav__link" for="nav-13">
      Community/Social Features
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-13">
        Community/Social Features
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../user/" title="User Page" class="md-nav__link">
      User Page
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../troubleshooting/" title="Troubleshooting" class="md-nav__link">
      Troubleshooting
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-15" type="checkbox" id="nav-15">
    
    <label class="md-nav__link" for="nav-15">
      Tutorial Guides
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-15">
        Tutorial Guides
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorials/bacterianetworking/" title="Molecular Networking - Analog Discovery in Microbial Extracts" class="md-nav__link">
      Molecular Networking - Analog Discovery in Microbial Extracts
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorials/americangutmzmine/" title="Feature-Based Molecular Networking (FBMN) with MZmine2 - Microbiome Molecules in Fecal Samples in American Gut Project" class="md-nav__link">
      Feature-Based Molecular Networking (FBMN) with MZmine2 - Microbiome Molecules in Fecal Samples in American Gut Project
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorials/americangut-ms-dial/" title="FBMN with MS-DIAL - Microbiome Molecules in Fecal Samples in American Gut Project" class="md-nav__link">
      FBMN with MS-DIAL - Microbiome Molecules in Fecal Samples in American Gut Project
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorials/coffee-tutorial-metaboscape/" title="FBMN with MetaboScape - Coffee Analysis" class="md-nav__link">
      FBMN with MetaboScape - Coffee Analysis
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorials/featurebasedgnps/" title="FBMN analysis at GNPS" class="md-nav__link">
      FBMN analysis at GNPS
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../changelog/" title="Change Log" class="md-nav__link">
      Change Log
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../gnps_community/" title="Join the GNPS community" class="md-nav__link">
      Join the GNPS community
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../workshops/" title="GNPS Workshops" class="md-nav__link">
      GNPS Workshops
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../logos/" title="GNPS Logos" class="md-nav__link">
      GNPS Logos
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../contact/" title="Contact" class="md-nav__link">
      Contact
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../api/" title="API" class="md-nav__link">
      API
    </a>
  </li>

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              <div class="md-sidebar md-sidebar--secondary" data-md-component="toc">
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    
<nav class="md-nav md-nav--secondary">
  
  
    
  
  
    <label class="md-nav__title" for="__toc">Table of contents</label>
    <ul class="md-nav__list" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#configure-the-workflow" class="md-nav__link">
    Configure the workflow
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#inspect-the-results" class="md-nav__link">
    Inspect the Results
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#cloning-a-job" class="md-nav__link">
    Cloning a job
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sharing-data" class="md-nav__link">
    Sharing data
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#addingcurating-reference-spectra" class="md-nav__link">
    Adding/Curating reference spectra
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#visualization" class="md-nav__link">
    Visualization
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#network-visualization-in-cytoscape" class="md-nav__link">
    Network Visualization in Cytoscape
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#molecular-mapping-in-ili" class="md-nav__link">
    Molecular mapping in ili
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#page-contributors" class="md-nav__link">
    Page contributors
  </a>
  
</li>
      
      
      
      
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          <div class="md-content">
            <article class="md-content__inner md-typeset">
              
                
                  <a href="https://github.com/CCMS-UCSD/GNPSDocumentation/blob/master/docs/gc-ms-library-molecular-network.md" title="Edit this page" class="md-icon md-content__icon">&#xE3C9;</a>
                
                
                <h1 id="gc-ms-spectral-library-search-and-molecular-networking">GC-MS Spectral Library Search and Molecular Networking</h1>
<h2 id="configure-the-workflow">Configure the workflow</h2>
<p>There are two options to open the job page for conducting a spectral library search and create a molecular network.</p>
<ul>
<li>By clicking on the link “Search Spectral Library and Molecular Network” from the deconvolution job page status. See <em>GC-MS Deconvolution for GNPS</em> documentation. </li>
<li>By clicking on the “Analyze” icon from the GNPS home page.</li>
</ul>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Fig_1B.png" /></strong></p>
<p>The <strong>"Search Option" section</strong> in the job window allows importing files needed for the spectral library search and the molecular networking generation. By clicking on "select Input Files", a pop-up window appears, allowing to select every needed file.</p>
<p>(1), (2) The .mgf file containing the deconvolved EI spectra and the .csv feature quantification table automatically (with balance score) adds if the spectral library search is launched from the deconvolution job performed with MS-Hub. Otherwise, they can be added by the user from others processing software. The quantification table is formatted as the MZmine2 output table with a "raw ID", "row m/z", "row retention time" columns, and all the peak area values for all .mzML files.</p>
<p>(3) Select your metadata table formatted as a tab-separated .txt file. To create your metadata table properly, see the documentation <a href="../metadata/">here</a>. </p>
<p>(4) Public Libraries files are selected by default. However, user could add their libraries on GNPS by following the instructions <a href="../batchupload/">here</a>, or by uploading them in .mgf format (drag and drop on “Upload File” section). </p>
<p>(5) The carbon marker table should be added. This table must be formatted as a comma separated values (.csv) file in the following headers: Compound_Name - name of the compound (string), Carbon_Number - number of carbons (integer), RT - retention time in seconds (float). <a href="../static/gc_kovats_skin.csv">Example File</a>. The file can also be created from the raw experimental chromatograms of the RI marker compounds using MZMine2 as described in this <a href="https://youtu.be/XodHMJcuwnk">video tutorial</a>. </p>
<p><img alt="img" src="../img/GC-MS_documentation/Fig_7.png" /></p>
<p>The <strong>job window</strong> proposed four fields of advanced search, filtering, network, and Kovats index calculation options. Click on the “Show Fields” button to show advanced options. The default parameters recommended for obtaining good results have been determined and pre-populated as described in the table Parameters for Library Search. However, users are encouraged to adjust the settings as appropriate for their analysis. </p>
<p>(7) Enter your email address.</p>
<p>(8) The job can be launched by clicking on the “Submit” button.</p>
<p><img alt="img" src="../img/GC-MS_documentation/Fig_7B.png" /></p>
<p><strong>Table: Parameters for Library Search</strong></p>
<table>
<thead>
<tr>
<th><strong><em>Search Options</em></strong></th>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>Fillable Field</strong></td>
<td><strong>Definition</strong></td>
<td><strong>Recommended User Input</strong></td>
</tr>
<tr>
<td>Spectrum Files</td>
<td>Selection of files needed to analyze.</td>
<td>The deconvoluted files usually are directly included into the Library search/Networking job. Allowed file formats are .mzML or .CDF.</td>
</tr>
<tr>
<td>FORCE_EXACT_MATCH</td>
<td></td>
<td>Keep default value of 1.</td>
</tr>
<tr>
<td>Precursor Ion Mass Tolerance</td>
<td>Maximum mass deviation between the observed precursor ion and the ions selected within the libraries included in the search.</td>
<td>The choice of the precursor mass tolerance setting usually is influenced directly by the accuracy of the mass analyzer being used to measure the precursor spectra in LC-MS analysis, but modification of this parameter are not meaningful for EI data because there is no precursor ion. 20000 is the default value to encompass all ions.</td>
</tr>
<tr>
<td>Min Matched Peaks</td>
<td>Minimum number of common fragment ions that should share a spectra to be considered as a spectral library annotation.</td>
<td>Default value is 6, but note that this parameter should be tuned depending on the molecules of interest and experimental conditions. It is recommended to perform several jobs modifying this parameter for testing possible outcomes variations in annotations. Typically high quality features contain more than 15 ions in EI data.</td>
</tr>
<tr>
<td>Fragment Ion Mass Tolerance</td>
<td>Maximum mass deviation between the observed fragments and the ions selected within the libraries included in the search.</td>
<td>The choice of the precursor mass tolerance setting is influenced directly by the accuracy of the mass analyzer being used to measure the precursor spectra. Default value is ± 0.025 Da for high-resolution mass spectrometers (q-TOF, q-Orbitrap). For low-resolution MS/MS instruments (e.g. , ion traps, triple-quadrupole/QqQ), a PIMT ion mass tolerance of ± 2.0 Da is recommended. Note that this parameters should be tuned depending on data.</td>
</tr>
<tr>
<td>Score Threshold</td>
<td>Minimum cosine score to be considered as an annotation in spectral library search.</td>
<td>0.85 value is set as default. This value has been determined to effectively delineate subclasses and Levels 6 and 7 and usually performs well with data obtained through experiments without derivatization, but that will depend on the quality of the mass spectra obtained. For derivatized compounds the cosine value for good matches is generally lower by ~0.1 due to an inferior quality of the EI spectra.</td>
</tr>
</tbody>
</table>
<table>
<thead>
<tr>
<th><strong><em>Advanced Search Options</em></strong></th>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>Fillable Field</strong></td>
<td><strong>Definition</strong></td>
<td><strong>Recommended User Input</strong></td>
</tr>
<tr>
<td>Library Class</td>
<td>The level of library to include hits from.</td>
<td>Keep default of “Bronze” to include all possible hits from libraries.</td>
</tr>
<tr>
<td>Top Hits Per Spectrum</td>
<td>How many top hits ranked by the cosine score to be included into the results.</td>
<td>The default of 1. It is recommended to keep this number relatively high (for example, 10 to include top 10 matches) to allow for correct annotation selection if it is not a top hit.</td>
</tr>
<tr>
<td>Spectral Library</td>
<td>Selection of spectral libraries to include in the search.</td>
<td>Select all appropriate EI libraries by selecting them and clicking “Library Files”. Click “Finish Selection” when completed. Any of the user’s own libraries could be included into the search as long as they are converted into .mgf format. If the libraries are curated and could be shared with the community, it is encouraged to upload them to GNPS as described <a href="../batchupload/">here</a>. Users have access to open and public libraries uploaded to GNPS, these can be selected at: CCMS_SpectralLibraries -&gt; GNPS_GC_Libraries.</td>
</tr>
<tr>
<td>Search Analogs</td>
<td>Used for tandem MS data.</td>
<td>Keep default of “Don’t Search”. This option is used for LC-MS data.</td>
</tr>
<tr>
<td>Maximum Analog Search Mass Difference</td>
<td>Used for tandem MS data.</td>
<td>Disabled when above option is set at “Don’t Search”.</td>
</tr>
</tbody>
</table>
<table>
<thead>
<tr>
<th><strong><em>Advanced Filtering Options</em></strong></th>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>Fillable Field</strong></td>
<td><strong>Definition</strong></td>
<td><strong>Recommended User Input</strong></td>
</tr>
<tr>
<td>Filter StdDev Intensity</td>
<td>Used for tandem MS data.</td>
<td>It is highly recommended to use a default value of 0 so that no filter is applied.</td>
</tr>
<tr>
<td>Filter precursor window</td>
<td>Used for tandem MS data.</td>
<td>“Don’t Filter” is set as default for the EI data because there is no precursor ion.</td>
</tr>
<tr>
<td>Filter peaks in 50 Da window</td>
<td>Removes peaks that are not one of the top 6 most intense within a +/- 50 Da window.</td>
<td>Depending on the deconvolution results, this could be turned on or off: if fragmentation patterns contain a large number of small peaks, this filtering should be turned off, as it may filter out relevant peaks that could be signal.</td>
</tr>
<tr>
<td>Filter SNR Intensity</td>
<td>All ions with a signal to noise ratio (SNR) in the EI spectrum below this value raw intensity will be deleted.</td>
<td>Use a default value of 0 so that no filter is applied, especially if the raw intensities of your data are very low. This value can be modified by the user for excluding background ions and just include the mass spectra of certain quality.</td>
</tr>
<tr>
<td>Filter library</td>
<td>Used for tandem MS data.</td>
<td>Don’t Filter is set as default for the EI data because there is no precursor ion.</td>
</tr>
<tr>
<td>Min Peak Int</td>
<td>All ions with a signal in the EI spectrum below this raw intensity will be deleted.</td>
<td>Use a default value of 0 so that no filter is applied, especially if the raw intensities of your data are very low.</td>
</tr>
</tbody>
</table>
<table>
<thead>
<tr>
<th><strong><em>Advanced Network Options</em></strong></th>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>Fillable Field</strong></td>
<td><strong>Definition</strong></td>
<td><strong>Recommended User Input</strong></td>
</tr>
<tr>
<td>Meta-table (Optional)</td>
<td>Selection of metadata to include in the search.</td>
<td>The metadata file describes the samples properties and allows more flexibility for data analysis and visualization. It is an alternative way to assign groups when selecting data input files within the workflow of GNPS, see an example <a href="../metadata/">here</a>.</td>
</tr>
<tr>
<td>Min pairs cos</td>
<td>Minimum cosine score required for an edge to be formed between nodes. Lower value will increase the size of the clusters by inducing the clustering of less related spectra, higher value will limit do the opposite.</td>
<td>0.7 value is set as default. This value has been determined to effectively delineate subclasses and Levels 6 and 7 and usually performs well with data obtained through experiments without derivatization, but that will depend on the quality of the mass spectra obtained. For derivatized compounds the cosine value for good matches is generally lower due to an inferior quality of the EI spectra.</td>
</tr>
<tr>
<td>Network TopK</td>
<td>Pairs of nodes are reported only if they are found to be best matches in the topK in both directions that are above the minimum cosine score.</td>
<td>Default is set to 10. This value enables the network to be more or less complex.</td>
</tr>
<tr>
<td>Quant-table (Optional)</td>
<td>Table including feature ID, retention time (in min.) with balance score, Rel. Max Integral and Sample\Best order.</td>
<td>The quantification table usually is directly included into the Library search/Networking job or can be imported externally. External deconvolution tools do not provide balance score and it will not be available in the search results.</td>
</tr>
<tr>
<td>Maximum shift between precursors</td>
<td>The maximum allowed difference in precursor m/z differences.</td>
<td>Must be set to the higher m/z expected in the analysis for EI data since no precursor information is available (default value of 500)</td>
</tr>
<tr>
<td>Maximum Connected Component Size</td>
<td>Maximum number of nodes that can be connected in a single sub-network of a molecular network. This process iteratively breaks up large ‘hairball’ networks (of false positives) by removing the lowest scoring alignments (by cosine score) first until the resulting pieces fall below the maximum size.</td>
<td>Default setting is 100 – this value can be set to 0 to allow for an unlimited number of nodes or a higher setting can be used for larger data sets or for data sets containing many structurally-related molecules. Low value encourages breaking up the “hairball” clusters but increases the chance that some nodes become disconnected and fall out of the network.</td>
</tr>
</tbody>
</table>
<table>
<thead>
<tr>
<th><strong><em>Advanced Kovats Index Calculation Options</em></strong></th>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>Fillable Field</strong></td>
<td><strong>Definition</strong></td>
<td><strong>Recommended User Input</strong></td>
</tr>
<tr>
<td>Perform Kovats Calculation</td>
<td>Whether to perform calculation of Kovats retention index.</td>
<td>If the user need to calculate Kovats RIs for the compounds during library search this parameter should be selected. (i) The Input Carbon Marker File could be provided for Kovats markers, but it must be conducted with identical protocol as the rest of the experiment.</td>
</tr>
<tr>
<td>Input Carbon Marker File (Optional)</td>
<td>If provided, this information will be used for retention index (RI_ estimation.</td>
<td>The RI markers information should be submitted as a comma separated values (.csv) file in the following headers: Compound_Name - name of the compound (string), Carbon_Number - number of carbons (integer), RT - retention time in seconds (float). <a href="../static/gc_kovats_skin.csv">Example File</a>.</td>
</tr>
<tr>
<td>MSHub Balance Score</td>
<td>The file that contains information for the balance score.</td>
<td>The balance score is an estimation of mass spectrum patterns reproducibility across the entire dataset. This information is only available if deconvolution is conducted using MSHub and the file is appended automatically when launching library search job from completed MSHub deconvolution job. This file is optional and can be omitted when using deconvolution results from other deconvolution software.</td>
</tr>
<tr>
<td><strong><em>Advanced Deprecated Options</em></strong></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td>These options are only relevant if/when the parent mass information is available and thus are strongly recommended to be left default.</td>
</tr>
</tbody>
</table>
<h2 id="inspect-the-results">Inspect the Results</h2>
<p>After the job has been completed, the <strong>job status page</strong> gives access to several molecular networking results views:</p>
<p>(i) “ View All Library Hits” opens the interface that allows visualization of spectral matches to the reference database from selected libraries. It will display all of the top matches for each feature to different libraries and relevant information including retention time. </p>
<ul>
<li>
<p>The displayed columns could be edited by clicking the “Select columns” link in the upper left corner. The possible annotations could be sorted by their cosine value to aid in selecting the best annotation using the researcher's judgment. The “Scan” field corresponds to the feature number in the .mgf file generated at the feature detection step and corresponds to the “No” field listed in the feature table. </p>
</li>
<li>
<p>Clicking on the spectrum icon at the left side of the screen will show a mirror plot comparison between the query spectrum and the reference spectrum. This allows assessing the quality of matches between the deconvolution and the reference spectra. </p>
</li>
</ul>
<p>(ii) The “View Top Hits” option accesses the list of: top hits (Compound_Name) with the corresponding Scan (feature number), matched library class, cosine, number of shared peaks between library entry and the spectral pattern, abundance as TIC, retention time given in units used in input raw files, and information from the reference library entry, for all annotated spectra.</p>
<p>(iii) The "View All Spectra With Annotations" option gives access to all annotated spectra with the same information listed. In addition, the balance score and Kovats RI values are also listed (if available) to allow including this information into making decisions for feature annotations.</p>
<p>(vi) The link “View Kovats Calculation Result” shows Kovats calculation results for each spectrum if selected when launching the job.</p>
<p>The displayed columns could be edited by clicking the "Select columns" link and selecting/deselecting columns as needed. </p>
<p><strong>NOTE</strong>: Clicking the "Download" link in ether view would allow downloading the results as a table will all the associated information for each library hit (including InChi, SMILES and CAS, whenever available); the file with the name "MOLECULAR-LIBRARYSEARCH-GC*.tsv" is in the root of the downloadable compressed folder.</p>
<p>The button at the top of the page “Back to status page” allows going back. To visualize molecular networks generated, the user has to download the input network files from the “Download Cytoscape Data” link, save the folder, and unzip it.</p>
<p>In the job status page, some advanced views are available to visualized qiime2 emperor plots or bi-plots, and to download the associated tables. For more information, follow this <a href="https://ccms-ucsd.github.io/GNPSDocumentation/molnetenhancer/#view-qiime2-emperor-plots">link</a>.</p>
<p><img alt="img" src="../img/GC-MS_documentation/Fig_10.png" /></p>
<h3 id="cloning-a-job">Cloning a job</h3>
<p>If the same analysis needs to be repeated, it is possible to clone the job by clicking “clone” on the job status page. Cloning a job allows users to view all parameters and files that were used and rerun the job with the same, or adjusted parameters and files. Note that if data were imported from private user workspace and not from within MassIVE, other users will not have access to the data and consequently will not be able to rerun or reproduce the GNPS job.  </p>
<h3 id="sharing-data">Sharing data</h3>
<p>The analysis conducted on GNPS as well as the deposited raw data could be shared as a hyperlink to the GNPS job(s) and the MassIVE accession number, correspondingly. GNPS stores data inputs and setting used for the analyses of the data, so the analysis could be shared and reproduced. <a href="../datasets/">See the documentation</a>.</p>
<h3 id="addingcurating-reference-spectra">Adding/Curating reference spectra</h3>
<p>High confidence spectra from experimental data or pure standards could be added as a reference spectrum to GNPS for future reference. <a href="../spectrumcuration/">See documentation</a>. If the user wishes to upload &gt;50 reference spectra to GNPS, a batch upload should be used to as detailed in the <a href="../batchupload/">online help</a>. The compliance of the file format can be verified using an online workflow.</p>
<h2 id="visualization">Visualization</h2>
<h3 id="network-visualization-in-cytoscape">Network Visualization in Cytoscape</h3>
<p>The network can be visualized in an external software such as Cytoscape [1], an open-source software platform used to visualize, analyze, and annotate molecular networks from GNPS. Cytoscape is available for download from<a href="http://www.cytoscape.org/"> here</a>. This software allowed to encode any property of the network (i.e. node shape or color, edge thickness) with a metadata category (i.e. cohort, cosine score, compound family). An online tutorial, created with Cytoscape 3.7, used for molecular networking generated with LC-MS/MS data, can be accessed <a href="../featurebasedmolecularnetworking-cytoscape/">here</a>. Many of these steps could be reproduced in the same way for molecular networking from GC-MS data. </p>
<p>[1] Shannon, P., et al. (2003). Cytoscape: a software environment for integrated models of biomolecular interaction networks. <em>Genome Res, 13</em>(11), 2498-2504. doi:10.1101/gr.1239303</p>
<p><strong>(A)</strong> Here we show an example of molecular networks visualizing in Cytoscape, generated from a study of volatiles related to aging of IPA beer (<a href="https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=1b300f711b404a3a8a9325352974adcd">GNPS link</a>). In Cytoscape, the different aging times (0, 1, 2, 3, 4, 5, 6, 8, 12, and 20 days) are represented by different colors, and the metabolites present in the beer (each node) can be mapped on the molecular networks. The size of each node represents the total abundance of the feature and the colors in the pie chart shows the proportions of the compound at different aging duration times (gradient from green to yellow to red). </p>
<p><strong>(B)</strong> Zoom in a cluster of terpenes annotated with compounds name is shown as an example. Networking can facilitate the interpretation of data by visualizing chemical changes. In this example, the annotated compounds E-nerolidol and alpha-limonene-diepoxide, framed in pink, appear to be present only in fresh beer (0 days) and disappear as the beer ages. </p>
<p><strong>(C)</strong> Other information can be mapped onto the network, for example, the same zoom with cosine values between nodes is shown instead of compound name annotations.</p>
<p><strong>NOTE</strong>: The name of the node corresponds to the best match of the spectral pattern to the library across the entire dataset. Consequently, for some nodes the top match is not selected as the node's name as there exists other spectral pattern in the data that has better match to the same compound in the library. In such cases, the next best matching annotation is used. </p>
<p><img alt="img" src="../img/GC-MS_documentation/Fig_11.png" /></p>
<h3 id="molecular-mapping-in-ili">Molecular mapping in ili</h3>
<p>Spatial distribution of different chemical features could be visualized in 2D or 3D using ili, a visualization app for 3D molecular cartography [2]. ‘ili could be used either on the <a href="http://ili.embl.de/">website</a> or as a <a href="https://goo.gl/3KAA8U">Chrome extension</a>. For mapping, two files need to be uploaded (DRAG AND DROP): an image for 2D (PNG or JPG format) or model for 3D (STL format), and a .cvs file with the coordinates of the spots on the model that corresponds to the sampled locations. Examples of mapping applications are available on ‘ili <a href="https://ili.embl.de/">website</a>. Here it is demonstrated how to map the volatile data on the same model as shown in the “Human skin metabolome” example. </p>
<p>[2] Protsyuk, I., et al. (2018). 3D molecular cartography using LC–MS facilitated by Optimus and'ili software. <em>Nature protocols</em>, <em>13</em>(1), 134.https://doi.org/10.1038/nprot.2017.122</p>
<p><img alt="img" src="../img/GC-MS_documentation/Fig_12.png" /></p>
<p>To create the .csv table, download the quantification table from GNPS deconvolution (example <a href="https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=9127bd48c09b4087ab0fd9a1407b3da1">GNPS job</a>) output and add four columns with the x, y, z, and radius coordinates from the different sampling points as described [2].  For ease of navigation, the rows with feature descriptors could be deleted. The detailed instructions on creating the correctly formatted table with coordinates are given <a href="https://github.com/alexandrovteam/Optimus/blob/master/Obtaining coordinates of sampling spots.md">here</a>. </p>
<p><img alt="img" src="../img/GC-MS_documentation/Fig_13.png" /></p>
<p>To create chemical maps, drag and drop the example <a href="https://github.com/aaksenov1/Human-volatilome-3D-mapping-">volatilome table</a> human 3D model. Different parameters can be set such as the spots, the mapping color, or the angles of visualization. To display different features, click on the feature identifier value listed above the color scale on the bottom right corner of the screen and select features to display. To annotate molecular features of interest, refer to the GNPS library search job for the matching features.</p>
<p><img alt="img" src="../img/GC-MS_documentation/Fig_14.png" /></p>
<h2 id="page-contributors">Page contributors</h2>
<p>Alexander Aksenov (UCSD), Melissa Nothias-Esposito (UCSD), Mabel C. Gonzalez (Universidad de los Andes, UCSD), Louis Felix Nothias (UCSD).</p>
                
                  
                
                
              
              
                


              
            </article>
          </div>
        </div>
      </main>
      
        
<footer class="md-footer">
  
    <div class="md-footer-nav">
      <nav class="md-footer-nav__inner md-grid">
        
          <a href="../gc-ms-deconvolution/" title="GC-MS Deconvolution" class="md-flex md-footer-nav__link md-footer-nav__link--prev" rel="prev">
            <div class="md-flex__cell md-flex__cell--shrink">
              <i class="md-icon md-icon--arrow-back md-footer-nav__button"></i>
            </div>
            <div class="md-flex__cell md-flex__cell--stretch md-footer-nav__title">
              <span class="md-flex__ellipsis">
                <span class="md-footer-nav__direction">
                  Previous
                </span>
                GC-MS Deconvolution
              </span>
            </div>
          </a>
        
        
          <a href="../networkingviews/" title="Molecular Networking Views" class="md-flex md-footer-nav__link md-footer-nav__link--next" rel="next">
            <div class="md-flex__cell md-flex__cell--stretch md-footer-nav__title">
              <span class="md-flex__ellipsis">
                <span class="md-footer-nav__direction">
                  Next
                </span>
                Molecular Networking Views
              </span>
            </div>
            <div class="md-flex__cell md-flex__cell--shrink">
              <i class="md-icon md-icon--arrow-forward md-footer-nav__button"></i>
            </div>
          </a>
        
      </nav>
    </div>
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-footer-copyright">
        
        powered by
        <a href="https://www.mkdocs.org">MkDocs</a>
        and
        <a href="https://squidfunk.github.io/mkdocs-material/">
          Material for MkDocs</a>
      </div>
      
    </div>
  </div>
</footer>
      
    </div>
    
      <script src="../assets/javascripts/application.808e90bb.js"></script>
      
      <script>app.initialize({version:"1.0.4",url:{base:".."}})</script>
      
    
  </body>
</html>