
<!DOCTYPE html>

<html class="no-js" lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width,initial-scale=1" name="viewport"/>
<link href="https://ccms-ucsd.github.io/GNPSDocumentation/networking/" rel="canonical"/>
<link href="../assets/images/favicon.png" rel="icon"/>
<meta content="mkdocs-1.1.2, mkdocs-material-7.1.3" name="generator"/>
<title>Molecular Networking - GNPS Documentation</title>
<link href="../assets/stylesheets/main.e35208c4.min.css" rel="stylesheet"/>
<link href="../assets/stylesheets/palette.ef6f36e2.min.css" rel="stylesheet"/>
<link crossorigin="" href="https://fonts.gstatic.com" rel="preconnect"/>
<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700%7CRoboto+Mono&amp;display=fallback" rel="stylesheet"/>
<style>:root{--md-text-font-family:"Roboto";--md-code-font-family:"Roboto Mono"}</style>
<script>window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)},ga.l=+new Date,ga("create","UA-8412213-5","ccms-ucsd.github.io"),ga("set","anonymizeIp",!0),ga("send","pageview"),document.addEventListener("DOMContentLoaded",function(){document.forms.search&&document.forms.search.query.addEventListener("blur",function(){var e;this.value&&(e=document.location.pathname,ga("send","pageview",e+"?q="+this.value))}),"undefined"!=typeof location$&&location$.subscribe(function(e){ga("send","pageview",e.pathname)})})</script>
<script async="" src="https://www.google-analytics.com/analytics.js"></script>
</head>
<body data-md-color-accent="none" data-md-color-primary="none" data-md-color-scheme="" dir="ltr">
<script>function __prefix(e){return new URL("..",location).pathname+"."+e}function __get(e,t=localStorage){return JSON.parse(t.getItem(__prefix(e)))}</script>
<input autocomplete="off" class="md-toggle" data-md-toggle="drawer" id="__drawer" type="checkbox"/>
<input autocomplete="off" class="md-toggle" data-md-toggle="search" id="__search" type="checkbox"/>
<label class="md-overlay" for="__drawer"></label>
<div data-md-component="skip">
<a class="md-skip" href="#molecular-networking-overview">
          Skip to content
        </a>
</div>
<div data-md-component="announce">
</div>
<header class="md-header" data-md-component="header">
<nav aria-label="Header" class="md-header__inner md-grid">
<a aria-label="GNPS Documentation" class="md-header__button md-logo" data-md-component="logo" href=".." title="GNPS Documentation">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"></path></svg>
</a>
<label class="md-header__button md-icon" for="__drawer">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2z"></path></svg>
</label>
<div class="md-header__title" data-md-component="header-title">
<div class="md-header__ellipsis">
<div class="md-header__topic">
<span class="md-ellipsis">
            GNPS Documentation
          </span>
</div>
<div class="md-header__topic" data-md-component="header-topic">
<span class="md-ellipsis">
            
              Molecular Networking
            
          </span>
</div>
</div>
</div>
<label class="md-header__button md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"></path></svg>
</label>
<div class="md-search" data-md-component="search" role="dialog">
<label class="md-search__overlay" for="__search"></label>
<div class="md-search__inner" role="search">
<form class="md-search__form" name="search">
<input aria-label="Search" autocapitalize="off" autocomplete="off" autocorrect="off" class="md-search__input" data-md-component="search-query" data-md-state="active" name="query" placeholder="Search" required="" spellcheck="false" type="text"/>
<label class="md-search__icon md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"></path></svg>
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"></path></svg>
</label>
<button aria-label="Clear" class="md-search__icon md-icon" tabindex="-1" type="reset">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"></path></svg>
</button>
</form>
<div class="md-search__output">
<div class="md-search__scrollwrap" data-md-scrollfix="">
<div class="md-search-result" data-md-component="search-result">
<div class="md-search-result__meta">
            Initializing search
          </div>
<ol class="md-search-result__list"></ol>
</div>
</div>
</div>
</div>
</div>
<div class="md-header__source">
<a class="md-source" data-md-component="source" href="https://github.com/CCMS-UCSD/GNPSDocumentation/" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    CCMS-UCSD/GNPSDocumentation
  </div>
</a>
</div>
</nav>
</header>
<div class="md-container" data-md-component="container">
<main class="md-main" data-md-component="main">
<div class="md-main__inner md-grid">
<div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Navigation" class="md-nav md-nav--primary" data-md-level="0">
<label class="md-nav__title" for="__drawer">
<a aria-label="GNPS Documentation" class="md-nav__button md-logo" data-md-component="logo" href=".." title="GNPS Documentation">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"></path></svg>
</a>
    GNPS Documentation
  </label>
<div class="md-nav__source">
<a class="md-source" data-md-component="source" href="https://github.com/CCMS-UCSD/GNPSDocumentation/" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    CCMS-UCSD/GNPSDocumentation
  </div>
</a>
</div>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="..">
        GNPS Introduction
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../isgnpsright/">
        Is my data compatible with GNPS?
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../faq/">
        Frequently Asked Questions
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../superquickstart/">
        Superquick Start Guide
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../quickstart/">
        Quick Start Guide
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gnpsanalysisoverview/">
        GNPS Analysis Tools Overview
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../lexicon/">
        GNPS Lexicon
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_8" id="__nav_8" type="checkbox"/>
<label class="md-nav__link" for="__nav_8">
        Mass Spec/Computational Background
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Mass Spec/Computational Background" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_8">
<span class="md-nav__icon md-icon"></span>
          Mass Spec/Computational Background
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../massspecbackground/networkingtheory/">
        Molecular Networking
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_9" id="__nav_9" type="checkbox"/>
<label class="md-nav__link" for="__nav_9">
        Data Preparation/Upload
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Data Preparation/Upload" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_9">
<span class="md-nav__icon md-icon"></span>
          Data Preparation/Upload
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../fileconversion/">
        Mass Spectrometry File Conversion
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fileupload/">
        GNPS File Upload
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../metadata/">
        Metadata
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--active md-nav__item--nested">
<input checked="" class="md-nav__toggle md-toggle" data-md-toggle="__nav_10" id="__nav_10" type="checkbox"/>
<label class="md-nav__link" for="__nav_10">
        Recommended Data Analysis
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Recommended Data Analysis" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_10">
<span class="md-nav__icon md-icon"></span>
          Recommended Data Analysis
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item md-nav__item--active">
<input class="md-nav__toggle md-toggle" data-md-toggle="toc" id="__toc" type="checkbox"/>
<label class="md-nav__link md-nav__link--active" for="__toc">
          Molecular Networking
          <span class="md-nav__icon md-icon"></span>
</label>
<a class="md-nav__link md-nav__link--active" href="./">
        Molecular Networking
      </a>
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#molecular-networking-overview">
    Molecular Networking Overview
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#data-input-preparation">
    Data Input Preparation
  </a>
<nav aria-label="Data Input Preparation" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#mass-spectrometry-data-files">
    Mass Spectrometry data files
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#metadata">
    Metadata
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#running-molecular-networking">
    Running Molecular Networking
  </a>
<nav aria-label="Running Molecular Networking" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#molecular-networking-workflow-selection">
    Molecular Networking Workflow Selection
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#selecting-files-to-analyze">
    Selecting Files to Analyze
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#organizing-selected-files-into-groups">
    Organizing Selected Files into Groups
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#parameter-walkthrough">
    Parameter Walkthrough
  </a>
<nav aria-label="Parameter Walkthrough" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#basic-options">
    Basic Options
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#advanced-network-options">
    Advanced Network Options
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#advanced-library-search-options">
    Advanced Library Search Options
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#advanced-filtering-options">
    Advanced Filtering Options
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#parameter-presets">
    Parameter Presets
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#status-page">
    Status Page
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#online-exploration-of-molecular-networks">
    Online Exploration of Molecular Networks
  </a>
<nav aria-label="Online Exploration of Molecular Networks" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#networking-results-views-description">
    Networking Results Views Description
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#citation">
    Citation
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#documentation-for-ili">
    Documentation for Ìli
  </a>
<nav aria-label="Documentation for Ìli" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#references">
    References
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#page-contributors">
    Page Contributors
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_10_2" id="__nav_10_2" type="checkbox"/>
<label class="md-nav__link" for="__nav_10_2">
        GC-MS Molecular Networking
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="GC-MS Molecular Networking" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_10_2">
<span class="md-nav__icon md-icon"></span>
          GC-MS Molecular Networking
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../gcanalysis/">
        GC-MS on GNPS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gc-ms-deconvolution/">
        GC-MS Deconvolution
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gc-ms-library-molecular-network/">
        GC-MS Library Search and Molecular Networking
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../networkingviews/">
        Molecular Networking Views
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../librarysearch/">
        Spectral Library Search
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../masst/">
        MASST Search (Mass Search Tool)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../resultviews/">
        Result Views Navigation
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_11" id="__nav_11" type="checkbox"/>
<label class="md-nav__link" for="__nav_11">
        Mass Spec Data Visualization
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Mass Spec Data Visualization" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_11">
<span class="md-nav__icon md-icon"></span>
          Mass Spec Data Visualization
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../lcms-dashboard/">
        GNPS Dashboard
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gnpsdatasetexplorer/">
        GNPS Dataset Explorer
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../usi/">
        Metabolomics Spectrum Resolver
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_12" id="__nav_12" type="checkbox"/>
<label class="md-nav__link" for="__nav_12">
        Advanced Data Analysis
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Advanced Data Analysis" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_12">
<span class="md-nav__icon md-icon"></span>
          Advanced Data Analysis
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../networkingfiles/">
        Molecular Networking Files
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_12_2" id="__nav_12_2" type="checkbox"/>
<label class="md-nav__link" for="__nav_12_2">
        Feature Based Molecular Networking
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Feature Based Molecular Networking" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_12_2">
<span class="md-nav__icon md-icon"></span>
          Feature Based Molecular Networking
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking/">
        FBMN Workflow
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-mzmine2/">
        FBMN with MZmine
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-ms-dial/">
        FBMN with MS-DIAL
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-xcms3/">
        FBMN with XCMS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-openms/">
        FBMN with OpenMS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-metaboscape/">
        FBMN with MetaboScape
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-progenesisQI/">
        FBMN with Progenesis QI
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-mztab-m/">
        FBMN with mzTab-M
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-cytoscape/">
        FBMN export to Cytoscape
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_12_3" id="__nav_12_3" type="checkbox"/>
<label class="md-nav__link" for="__nav_12_3">
        FBMN - Ion Identity Networking
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="FBMN - Ion Identity Networking" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_12_3">
<span class="md-nav__icon md-icon"></span>
          FBMN - Ion Identity Networking
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin/">
        IIN Workflow
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin-mzmine/">
        IIN with MZmine2
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin-xcms/">
        IIN with XCMS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin-msdial/">
        IIN with MS-DIAL
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../ms2lda/">
        MS2LDA and MotifDB Substructure Discovery
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../cytoscape/">
        Molecular Networking in Cytoscape
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../dereplicator/">
        DEREPLICATOR
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../dereplicator%2B/">
        DEREPLICATOR+
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../molDiscovery/">
        MolDiscovery
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../nap/">
        Network Annotation Propagation (NAP)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../ms2lda/">
        MS2LDA and MotifDB Substructure Discovery
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../molnetenhancer/">
        MolNetEnhancer
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../metaminer/">
        MetaMiner
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../sirius/">
        Sirius
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../ReDU/">
        Reanalysis of Data User Interface (ReDU)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../qemistree/">
        Qemistree
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../mmvec/">
        MMVEC
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../mergepolarity/">
        Merge Network Polarity
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../chemdir/">
        Chemical Directionality
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_13" id="__nav_13" type="checkbox"/>
<label class="md-nav__link" for="__nav_13">
        Spectral Libraries
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Spectral Libraries" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_13">
<span class="md-nav__icon md-icon"></span>
          Spectral Libraries
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../browselibraries/">
        Browsing Reference Spectral Libraries
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../spectrumcuration/">
        Add/Curate Reference Spectrum in the Library
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../batchupload/">
        Reference Spectral Library Batch Addition
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../batchuploadmzvault/">
        Reference Spectral Library mzVault Addition
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../downloadlibraries/">
        Downloading Reference Spectral Libraries
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_13_6" id="__nav_13_6" type="checkbox"/>
<label class="md-nav__link" for="__nav_13_6">
        Open-source Method for Reference Library Generation
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Open-source Method for Reference Library Generation" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_13_6">
<span class="md-nav__icon md-icon"></span>
          Open-source Method for Reference Library Generation
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../msmschooser/">
        MSMS-Chooser Workflow
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../msmschooserdataacquisitioncomputational/">
        Computational Data Acquisition Prep
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../msmschooserdataacquisitionQE/">
        Data Acqusition - Thermo QExactive (v1.0)
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tag_info/">
        GNPS Tag Contribution and Analyses
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_14" id="__nav_14" type="checkbox"/>
<label class="md-nav__link" for="__nav_14">
        Create a Public Datasets
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Create a Public Datasets" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_14">
<span class="md-nav__icon md-icon"></span>
          Create a Public Datasets
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../datasets/">
        Dataset Creation/Sharing
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../continuousid/">
        Continuous Identification
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../moleculeexplorer/">
        Molecule Explorer
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_15" id="__nav_15" type="checkbox"/>
<label class="md-nav__link" for="__nav_15">
        Community/Social Features
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Community/Social Features" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_15">
<span class="md-nav__icon md-icon"></span>
          Community/Social Features
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../user/">
        User Page
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../troubleshooting/">
        Troubleshooting
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_17" id="__nav_17" type="checkbox"/>
<label class="md-nav__link" for="__nav_17">
        Tutorial Guides
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Tutorial Guides" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_17">
<span class="md-nav__icon md-icon"></span>
          Tutorial Guides
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/bacterianetworking/">
        Molecular Networking - Analog Discovery in Microbial Extracts
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/americangutmzmine/">
        Feature-Based Molecular Networking (FBMN) with MZmine2 - Microbiome Molecules in Fecal Samples in American Gut Project
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/americangut-ms-dial/">
        FBMN with MS-DIAL - Microbiome Molecules in Fecal Samples in American Gut Project
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/coffee-tutorial-metaboscape/">
        FBMN with MetaboScape - Coffee Analysis
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/featurebasedgnps/">
        FBMN analysis at GNPS
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../changelog/">
        Change Log
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gnps_community/">
        Join the GNPS community
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../workshops/">
        GNPS Workshops/Training
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../logos/">
        GNPS Logos
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../contact/">
        Contact/Office Hours
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../toolindex/">
        GNPS Tool Index
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../api/">
        API
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../developers/">
        For Developers
      </a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#molecular-networking-overview">
    Molecular Networking Overview
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#data-input-preparation">
    Data Input Preparation
  </a>
<nav aria-label="Data Input Preparation" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#mass-spectrometry-data-files">
    Mass Spectrometry data files
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#metadata">
    Metadata
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#running-molecular-networking">
    Running Molecular Networking
  </a>
<nav aria-label="Running Molecular Networking" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#molecular-networking-workflow-selection">
    Molecular Networking Workflow Selection
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#selecting-files-to-analyze">
    Selecting Files to Analyze
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#organizing-selected-files-into-groups">
    Organizing Selected Files into Groups
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#parameter-walkthrough">
    Parameter Walkthrough
  </a>
<nav aria-label="Parameter Walkthrough" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#basic-options">
    Basic Options
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#advanced-network-options">
    Advanced Network Options
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#advanced-library-search-options">
    Advanced Library Search Options
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#advanced-filtering-options">
    Advanced Filtering Options
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#parameter-presets">
    Parameter Presets
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#status-page">
    Status Page
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#online-exploration-of-molecular-networks">
    Online Exploration of Molecular Networks
  </a>
<nav aria-label="Online Exploration of Molecular Networks" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#networking-results-views-description">
    Networking Results Views Description
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#citation">
    Citation
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#documentation-for-ili">
    Documentation for Ìli
  </a>
<nav aria-label="Documentation for Ìli" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#references">
    References
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#page-contributors">
    Page Contributors
  </a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-content" data-md-component="content">
<article class="md-content__inner md-typeset">
<a class="md-content__button md-icon" href="https://github.com/CCMS-UCSD/GNPSDocumentation/edit/master/docs/networking.md" title="Edit this page">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25z"></path></svg>
</a>
<h1>Molecular Networking</h1>
<h2 id="molecular-networking-overview">Molecular Networking Overview<a class="headerlink" href="#molecular-networking-overview" title="Permanent link">¶</a></h2>
<p>Molecular networks are visual displays of the chemical space present in tandem mass spectrometry (MS/MS) experiments. This visualization approach can detect sets of spectra from related molecules (molecular networks), even when the spectra themselves are not matched to any known compounds.</p>
<p>The visualization of molecular networks in GNPS represents each spectrum as a node, and spectrum-to-spectrum alignments as edges (connections) between nodes. Nodes can be supplemented with metadata, including dereplication matches or information that is provided by the user, e.g. as abundance, origin of product, biochemical activity, or hydrophobicity, which can be reflected in a node’s size or color. This map of all related molecules is visualized as a global molecular network.</p>
<p>For more information about Molecular Networking check out the <a href="https://www.nature.com/articles/nbt.3597">GNPS publication</a> from 2016.</p>
<h2 id="data-input-preparation">Data Input Preparation<a class="headerlink" href="#data-input-preparation" title="Permanent link">¶</a></h2>
<p>The follow data is can be put into molecular networking.</p>
<ol>
<li>Mass Spectrometry data files (required)</li>
<li>Metadata/Group Mapping (optional)</li>
<li>`ili 3D mapping files (optional)</li>
</ol>
<h3 id="mass-spectrometry-data-files">Mass Spectrometry data files<a class="headerlink" href="#mass-spectrometry-data-files" title="Permanent link">¶</a></h3>
<p>Molecular networking supports mzXML, mzML, and mgf file formats. To convert your file to the appropriate formats, check out our <a href="../fileconversion/">documentation</a>.</p>
<h3 id="metadata">Metadata<a class="headerlink" href="#metadata" title="Permanent link">¶</a></h3>
<p>More information provided <a href="../metadata/">here</a>.</p>
<h2 id="running-molecular-networking">Running Molecular Networking<a class="headerlink" href="#running-molecular-networking" title="Permanent link">¶</a></h2>
<h3 id="molecular-networking-workflow-selection">Molecular Networking Workflow Selection<a class="headerlink" href="#molecular-networking-workflow-selection" title="Permanent link">¶</a></h3>
<p>From the main <a href="https://gnps.ucsd.edu/ProteoSAFe/static/gnps-splash.jsp">GNPS page</a>, click the <a href="https://gnps.ucsd.edu/ProteoSAFe/index.jsp?params=%7B%22workflow%22:%22METABOLOMICS-SNETS-V2%22,%22library_on_server%22:%22d.speclibs;%22%7D">"Create Molecular Network"</a> button.</p>
<p><img alt="analysis" src="../img/networking/network_selection.png"/></p>
<p>This will bring you to the workflow input to start networking.</p>
<p><img alt="networking_input_form" src="../img/networking/networking_input_form.png"/></p>
<p>Provide a detailed title for your molecular network. This title will be helpful when you retrieve your data after the workflow is completed.</p>
<p><img alt="set title" src="../img/quickstart/networking_title.gif"/></p>
<h3 id="selecting-files-to-analyze">Selecting Files to Analyze<a class="headerlink" href="#selecting-files-to-analyze" title="Permanent link">¶</a></h3>
<p>Select files for analysis by first clicking "Select Input Files".</p>
<p><img alt="set title" src="../img/networking/file_selection.png"/></p>
<p>This bring a popup window. We can select our own files to analyze.</p>
<p><img alt="set title" src="../img/networking/file_selection.gif"/></p>
<p>Alternatively, we can import an existing public dataset if we know the dataset accession (<a href="../datasets/">Browse Datasets</a>). From the Select Input Files popup, you will be a shares files tab. There you will find a box called Import Data Share. In this box you may enter an accession of a MassIVE dataset. Upon clicking import the dataset will appear in your workspace to select input files to analyze from.</p>
<p><img alt="set title" src="../img/networking/dataset_import.gif"/></p>
<p><img alt="set title" src="../img/networking/dataset_selection.gif"/></p>
<h3 id="organizing-selected-files-into-groups">Organizing Selected Files into Groups<a class="headerlink" href="#organizing-selected-files-into-groups" title="Permanent link">¶</a></h3>
<p>By default, files can be categorized into separate groups (G1, G2, etc.). For example, case and control or two different microbes can be separate groups. Using the basic options, only six groups can be created. Individual files or entire folders can be selected.</p>
<p>Click Finish Selection which will close the pop-up window.</p>
<h3 id="parameter-walkthrough">Parameter Walkthrough<a class="headerlink" href="#parameter-walkthrough" title="Permanent link">¶</a></h3>
<h4 id="basic-options">Basic Options<a class="headerlink" href="#basic-options" title="Permanent link">¶</a></h4>
<table>
<thead>
<tr>
<th>Parameter</th>
<th>Description</th>
<th>Default</th>
</tr>
</thead>
<tbody>
<tr>
<td>Precursor ion mass tolerance (PIMT)</td>
<td>Parameter used for MS-Cluster and spectral library search. Specify the precursor ions mass tolerance, in Daltons. This value influences the aforementioned clustering of nearly-identical MS/MS spectra via MS-Cluster. Note that the value of this parameters should be consistent with the capabilities of the mass spectrometer and the specific instrument method used to generated the MS/MS data. <a href="../lexicon/">See the ion mass accuracy/tolerance table for help choosing the values</a>. Recommended Values value is ± 0.02 Da for high-resolution instruments (q-TOF, q-Orbitrap) and ± 2.0 Da for low-resolution instruments (ion traps, QqQ).</td>
<td>2.0</td>
</tr>
<tr>
<td>Fragment Ion Mass Tolerance (FIMT)</td>
<td>Parameters used for MS-Cluster, molecular networking, and MS/MS spectral library searches. For every group of MS/MS spectra being considered for clustering (consensus spectrum creation), this value specifies how much fragment ions can be shifted from their expected m/z values. <a href="../lexicon/">See the ion mass accuracy/tolerance table for help choosing the values</a>. Recommended Values value is ± 0.02 Da for high-resolution instruments (q-TOF, q-Orbitrap) and ± 0.5 Da for low-resolution instruments (ion traps, QqQ).</td>
<td>0.5</td>
</tr>
</tbody>
</table>
<h4 id="advanced-network-options">Advanced Network Options<a class="headerlink" href="#advanced-network-options" title="Permanent link">¶</a></h4>
<p><img alt="set title" src="../img/networking/advanced_networking_options.png"/></p>
<table>
<thead>
<tr>
<th>Parameter</th>
<th>Description</th>
<th>Default</th>
<th>Notes</th>
</tr>
</thead>
<tbody>
<tr>
<td>Min Pairs Cos</td>
<td>Minimum cosine score that must occur between a pair of consensus MS/MS spectra in order for an edge to be formed in the molecular network.</td>
<td>0.7</td>
<td>Lower value will increase the size of the clusters by inducing the clustering of less related MS/MS spectra, higher value will limit do the opposite.</td>
</tr>
<tr>
<td>Minimum Matched Fragment Ion (Min Matched Peaks)</td>
<td>Parameters used for molecular networking. Is the minimum number of common fragment ions that are shared by two separate consensus MS/MS spectra in order to be connected by an edge in the molecular network</td>
<td>6</td>
<td>A low value will permit linkages between spectra of molecules with few similar fragment ions, but it will result in many more less-related spectra being connected to the network. An higher value will do the opposite. Default value is 6, but note that this parameters should be adjusted depending on the experimental conditions for mass spectra acquisition (such as mode of ionisation, fragmentation conditions, and the mobile phase,  ...), and the collision-induced fragmentation behavior of the molecules of interest within the samples. High molecular weight (MW) compounds, and compounds with more hetero-atoms will generally tend to produce more fragment ions. However, this rule cannot be systematized. For example, some lipids with high MW generate only few fragment ions. Note that spectra with fewer than 5 fragment ions will by default not be considered for networking and result in single nodes.</td>
</tr>
<tr>
<td>Node TopK</td>
<td>Maximum number of neighbor nodes for one single node</td>
<td>10</td>
<td>The edge between two nodes are kept only if both nodes are within each other's ‘TopK’ most similar nodes.  For example, if this value is set at 20, then a single node may be connected to up to 20 other nodes.  Keeping this value low makes very large networks (many nodes) much easier to visualize.</td>
</tr>
<tr>
<td>Minimum Cluster Size</td>
<td>Minimum number of MS/MS spectra in a consensus MS/MS spectra to be considered for molecular networking</td>
<td>2</td>
<td>Requires MS-Cluster to be on. This parameter should be tuned based on the dataset size. See presets below</td>
</tr>
<tr>
<td>Run MSCluster</td>
<td>Cluster MS/MS spectra before networking</td>
<td>Yes</td>
<td>MSCluster will analyze every MS/MS spectra resulting from ions that fall within the defined precursor ion mass tolerance, and will merge the nearly-identical MS/MS spectra (above the cosine score) into a single consensus MS/MS spectrum. Each consensus MS/MS spectrum usually consist of multiple MS/MS spectra from across multiple LC-MS runs (or data files)</td>
</tr>
<tr>
<td>Maximum Connected Component Size</td>
<td>Maximum size of nodes allowed in a single connected network</td>
<td>100</td>
<td>Maximum size of nodes allowed in a single connected network. Nodes within a single connected molecular network will be separated by increasing cosine threshold for that specific connected molecular network. Default value is 100. Use 0 to allow an unlimited number of nodes in a single network. Note that with large datasets, or when a great number of related molecules are in the dataset, this value should be higher (or turn to 0) to retain as much information as possible. Downstream, these larger networks can be visualized using Cytoscape layout algorithms that can increase the intra-network clustering, allowing to visualize spectral groups in the network despite the number of nodes in the network.</td>
</tr>
<tr>
<td>Metadata File</td>
<td>Text input to describe experimental setup</td>
<td></td>
<td>Replaces both Group mapping and Attribute Mapping</td>
</tr>
<tr>
<td>Group Mapping</td>
<td>Input text file organizing input files into groups</td>
<td></td>
<td>Input text file organizing input files into groups. used as a more flexible alternative to assigning groups during data input selection.</td>
</tr>
<tr>
<td>Attribute Mapping</td>
<td>Attribute mapping eases visualization of different groups within cytoscape</td>
<td></td>
<td>Input text file organizing groups into attributes</td>
</tr>
</tbody>
</table>
<h4 id="advanced-library-search-options">Advanced Library Search Options<a class="headerlink" href="#advanced-library-search-options" title="Permanent link">¶</a></h4>
<p><img alt="set title" src="../img/networking/advanced_library_search_options.png"/></p>
<table>
<thead>
<tr>
<th>Parameter</th>
<th>Description</th>
<th>Default</th>
</tr>
</thead>
<tbody>
<tr>
<td>Library Search Min Matched Peaks</td>
<td>Minimum number of common fragment ions that MS/MS spectra should contain in order to be considered for spectral library annotation. Default value is 6, but note that this parameters should be tuned depending of the molecule of interest, and the experimental conditions (such as the ionisation mode, and the fragmentation conditions, ...). For example, collision-induced fragmentation of some lipids produce only few fragment ions. A lower value will allow clustering of MS/MS spectra containing less  fragment ions, however it will also induce clustering of  MS/MS spectra from different molecular-type to be connected in one network. An higher value will do the opposite</td>
<td>6</td>
</tr>
<tr>
<td>Score Threshold</td>
<td>Minimum cosine score that MS/MS spectra should get in spectral matching with MS/MS spectral libraries in order to be considered an annotation.</td>
<td>0.7</td>
</tr>
<tr>
<td>Search Analogs</td>
<td>Will search data for analogs to library spectra</td>
<td>Don't Search</td>
</tr>
<tr>
<td>Maximum Analog Search Mass Difference</td>
<td>Maximum mass shift between library and putative analog found</td>
<td>100 (Da)</td>
</tr>
</tbody>
</table>
<h4 id="advanced-filtering-options">Advanced Filtering Options<a class="headerlink" href="#advanced-filtering-options" title="Permanent link">¶</a></h4>
<p><img alt="set title" src="../img/networking/advanced_filtering_options.png"/></p>
<table>
<thead>
<tr>
<th>Parameter</th>
<th>Description</th>
<th>Default</th>
<th>Notes</th>
</tr>
</thead>
<tbody>
<tr>
<td>Filter stdDev Intensity</td>
<td>Deprecated</td>
<td>0</td>
<td>Not recommended to change</td>
</tr>
<tr>
<td>Minimum Fragment Ion Intensity</td>
<td>All fragment ions in the MS/MS spectrum below this raw intensity will be deleted.  By default, no filter.</td>
<td>0</td>
<td>Reduce to 0 if your data's raw intensities are very low.</td>
</tr>
<tr>
<td>Filter Precursor Ion Window</td>
<td>All peaks in a +/- 17 Da around precursor ion mass are deleted. By default, yes filter. This removes the residual precursor ion, which is frequently observed in MS/MS spectra acquired on qTOFs.</td>
<td>Filter</td>
<td></td>
</tr>
<tr>
<td>Filter library</td>
<td>Apply peak filters to library</td>
<td>Filter</td>
<td></td>
</tr>
<tr>
<td>Filter peaks in 50Da Window</td>
<td>Filter out peaks that are not top 6 most intense peaks in a +/- 50Da window</td>
<td>Filter</td>
<td>Turn off if your data is very small molecules as it might filter out a lot peaks in the lower mass ranges that might be signal.</td>
</tr>
<tr>
<td>Filter Spectra from G6 as Blanks Before Networking</td>
<td>Enables the filtering out of consensus clusters for consideration in the molecular network if it appears in any file in G6</td>
<td>Don't Filter</td>
<td></td>
</tr>
</tbody>
</table>
<h3 id="parameter-presets">Parameter Presets<a class="headerlink" href="#parameter-presets" title="Permanent link">¶</a></h3>
<p>We have several parameter presets that seem to work well depending on the dataset size.</p>
<ol>
<li>Small datasets - up to 5 LC/MS files</li>
<li>Medium Datasets - 5 to 400 LC/MS files</li>
<li>Large Datasets - 400+ LC/MS files</li>
<li>Big Data(sets) - Lets talk!</li>
</ol>
<h2 id="status-page">Status Page<a class="headerlink" href="#status-page" title="Permanent link">¶</a></h2>
<p>Upon submission of the workflow, you will be brought to the status page. This will show you the progress of your molecular networking job.</p>
<p><img alt="img" src="../img/networking/status.png"/></p>
<p>Molecular networks usually take</p>
<ul>
<li>10 min for small datasets</li>
<li>1 hour for medium datasets</li>
<li>Several hours for large datasets</li>
<li>FOREVER for big datasets</li>
</ul>
<p>If you find your molecular network is taking longer than this, please <a href="../contact/">contact us</a>.</p>
<h2 id="online-exploration-of-molecular-networks">Online Exploration of Molecular Networks<a class="headerlink" href="#online-exploration-of-molecular-networks" title="Permanent link">¶</a></h2>
<p>After completing a Molecular Networking workflow utilizing GNPS, analysis can be done within the web interface. The GNPS web interface provides a quick and easy way to perform initial analysis of your data particularly if you want to view the MS2 spectra of the nodes/clusters/networks generated by the Molecular Networking workflow.</p>
<h3 id="networking-results-views-description">Networking Results Views Description<a class="headerlink" href="#networking-results-views-description" title="Permanent link">¶</a></h3>
<p><strong>Default Views to Explore Molecular Networks</strong></p>
<table>
<thead>
<tr>
<th>View</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../networkingviews/#view-all-library-hits">View All Library Hits</a></td>
<td>All spectral library matches between consensus MS/MS spectra and the selected libraries</td>
</tr>
<tr>
<td>View Unique Library Compounds</td>
<td>All unique library spectrum matches</td>
</tr>
<tr>
<td><a href="../networkingviews/#view-all-clusters-with-ids">View All Clusters With IDs</a></td>
<td>All consensus MS/MS spectra created by MSCluster that were considered for library search and molecular networking. This does not include consensus MS/MS spectra that fell below the minimum cluster size threshold</td>
</tr>
</tbody>
</table>
<p><strong>Network Visualizations</strong></p>
<table>
<thead>
<tr>
<th>View</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../networkingviews/#view-spectral-families">View Spectral Families (In Browser Network Visualizer)</a></td>
<td>List of all Spectral Families (i.e. connected components in the network) as well as view the network visualized in the browser</td>
</tr>
<tr>
<td>Network Summarizing Graphs</td>
<td>Brief statistics about the network such as number of spectra, number of nodes, number of spectral families, identification rates, etc. (legacy output)</td>
</tr>
</tbody>
</table>
<p><strong>Methods and Citation for Manuscripts</strong></p>
<table>
<thead>
<tr>
<th>View</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td>Networking Parameters and Written Network Description</td>
<td>Automatic description of molecular networking generation. This can be used in the methods section of manuscripts</td>
</tr>
</tbody>
</table>
<p><strong>Export/Download Network Files</strong></p>
<table>
<thead>
<tr>
<th>View</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td>Download Clustered Spectra as MGF</td>
<td>Download consensus MS/MS spectra as an MGF file</td>
</tr>
<tr>
<td>Download GraphML for Cytoscape</td>
<td>Download network files (graphML) for import into Cytoscape</td>
</tr>
<tr>
<td>Download Bucket Table</td>
<td>Download spectral abundance for all consensus spectra across all files (calculated within each file as the sum precursor abundance)</td>
</tr>
<tr>
<td>Download BioM for Qiime/Qiita</td>
<td>Download BioM for spectral counts for analysis in Qiime/Qiita multivariate statistics tools</td>
</tr>
<tr>
<td>Download Metadata for Qiime</td>
<td>Download Metadata file for analysis in Qiime</td>
</tr>
<tr>
<td>Download ili Data</td>
<td>Download the abundance information formatted for ili</td>
</tr>
</tbody>
</table>
<p><strong>Advanced Views - Global Public Dataset Matches</strong></p>
<table>
<thead>
<tr>
<th>View</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td>View Matches to All Public Datasets</td>
<td>View matches of consensus MS/MS spectra in the network to all public datasets</td>
</tr>
</tbody>
</table>
<p><strong>Advanced Views - Third Party Visualization</strong></p>
<table>
<thead>
<tr>
<th>View</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../networkingviews/#view-principle-coordinate-analysis-pcoa-plots">View Emperor PCoA Plot in GNPS</a></td>
<td>In browser visualization of all analyzed samples with PCoA</td>
</tr>
<tr>
<td>View ili in GNPS</td>
<td>Visualization of 3D ili mapping in GNPS</td>
</tr>
</tbody>
</table>
<p><strong>Advanced Views - Experimental Views</strong></p>
<table>
<thead>
<tr>
<th>View</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td>Direct Cytoscape Preview/Download</td>
<td>Preview entire molecular network and download direct cys file to open in Cytoscape</td>
</tr>
</tbody>
</table>
<!-- ### Publishing Data

To increase transparency in data analysis, please include the URL of your specific GNPS analysis job.

Additionally, we recommend that you deposit the data you used for analysis into the MassIVE dataset repository in order to make your raw data public. To do this please click the "Publish Data in MassIVE" button:

TODO: image and implement -->
<h2 id="citation">Citation<a class="headerlink" href="#citation" title="Permanent link">¶</a></h2>
<p>If you are using GNPS for analysis, please cite the manuscript</p>
<p><a href="https://www.nature.com/articles/nbt.3597">Mingxun Wang, Jeremy J Carver, Vanessa V Phelan, Laura M Sanchez, Neha Garg, Yao Peng, et al. "Sharing and community curation of mass spectrometry data with Global Natural Products Social Molecular Networking" Nature Biotechnology 34.8 (2016): 828-837. PMID: 27504778</a></p>
<p>If you use MS-Cluster in your molecular networks, cite the following as well:</p>
<p><a href="https://pubs.acs.org/doi/abs/10.1021/pr070361e">Frank, Ari M., et al. "Clustering millions of tandem mass spectra." Journal of proteome research 7.01 (2007): 113-122.</a></p>
<h2 id="documentation-for-ili">Documentation for Ìli<a class="headerlink" href="#documentation-for-ili" title="Permanent link">¶</a></h2>
<p>Spatial distribution of molecular features can be visualized in 2D or 3D throughout Ìli in GNPS, a Molecular cartography approach <a href="https://www.nature.com/articles/nprot.2017.122">Protsyuk et al., 2017</a>. For mapping, it is necessary to upload the metadata (.txt) containing the ìli coordinates (x, y, z and radio) and the 2D or 3D model (STL format), along with the data files (mzXML format) for Classic Molecular Network.</p>
<p>First, it is necessary to obtain the 2D/3D model. The 2D model may be a picture, a map or other, and the 3D model can be obtained by scanning the sample. The coordinates should be pointed in the 2D/3D model by Meshlab software before uploading in GNPS. A free Meshlab version can be downloaded from <a href="http://www.meshlab.net/">here</a>. In Meshlab, use the PickPoint tool to point coordinates in the model as figure below.</p>
<p><img alt="Fig. 1" src="../img/%C3%ACli/Fig_1_meshlab.png"/></p>
<p>A list with the coordinates will appear in the new box, and you can rename them according to your data. After that, a <strong>.csv table containing the coordinates</strong> and the <strong>.stl pointed model</strong> should be exported from Meshlab.</p>
<p><img alt="Fig. 2" src="../img/%C3%ACli/Fig_2_meshlab.png"/></p>
<p>More details about how to create a .csv table with the coordinates can be found <a href="https://github.com/MolecularCartography/Optimus/blob/master/Obtaining%20coordinates%20of%20sampling%20spots.md">here</a>. The coordinates must be added in the metadata (.txt format) according to <a href="https://ccms-ucsd.github.io/GNPSDocumentation/metadata/">ReDU format</a>.</p>
<p><img alt="Fig. 3" src="../img/%C3%ACli/Fig_3_metadata.png"/></p>
<p>All files should be selected according to indicated below:</p>
<p><img alt="Fig. 4" src="../img/%C3%ACli/Fig_4_data-upload.png"/></p>
<p>Finish selection and set the parameters to create the Molecular Network. More details regarding Classic Molecular Network can be found <a href="https://ccms-ucsd.github.io/GNPSDocumentation/networking/">here</a>.</p>
<p>To create ìli visualization you must go in the <strong>Advanced Output Options</strong> and change the ìli <strong>Mapping Output</strong> to <strong>“Create”</strong>. Submit the job.</p>
<p><img alt="Fig. 5" src="../img/%C3%ACli/Fig_5_create_ili.png"/></p>
<p>Once the job was done, click on “View ìli in GNPS” to visualize the feature mapping.</p>
<p><img alt="Fig. 6" src="../img/%C3%ACli/Fig_6_Vizualization.png"/></p>
<p>A window for visualization will open. You can explore your data changing the feature in the bottom right. You can also set different parameters in the right window, such as the size of the spot or the color of mapping. There, you will also find some more examples to explore.</p>
<p><img alt="Fig. 7" src="../img/%C3%ACli/Fig_7_Vizualization.png"/></p>
<p>More details about ìli can be found <a href="https://github.com/MolecularCartography/ili">here</a>.</p>
<h3 id="references">References<a class="headerlink" href="#references" title="Permanent link">¶</a></h3>
<p>Protsyuk, I., et al. (2018). 3D molecular cartography using LC–MS facilitated by Optimus and'ili software. Nature protocols, 13(1), 134. <a href="https://doi.org/10.1038/nprot.2017.122">https://doi.org/10.1038/nprot.2017.122</a></p>
<h2 id="page-contributors">Page Contributors<a class="headerlink" href="#page-contributors" title="Permanent link">¶</a></h2>
<p><span class="git-page-authors git-authors"><a href="mailto:mwang87@gmail.com">Ming Wang</a> (81.32%), <a href="mailto:louisfelix.nothias@gmail.com">lfnothias</a> (0.78%), <a href="mailto:41308046+alan-jarmusch@users.noreply.github.com">alan-jarmusch</a> (0.39%), <a href="mailto:enmeilin@gmail.com">Madeleine Ernst</a> (0.39%), <a href="mailto:55205645+abauermeister@users.noreply.github.com">abauermeister</a> (17.12%)</span></p>
<hr/>
<div class="md-source-date">
<small>
    
      Last update: <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-datetime">January 16, 2021 00:22:29</span>
</small>
</div>
</article>
</div>
</div>
</main>
<footer class="md-footer">
<nav aria-label="Footer" class="md-footer__inner md-grid">
<a class="md-footer__link md-footer__link--prev" href="../metadata/" rel="prev">
<div class="md-footer__button md-icon">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"></path></svg>
</div>
<div class="md-footer__title">
<div class="md-ellipsis">
<span class="md-footer__direction">
                Previous
              </span>
              Metadata
            </div>
</div>
</a>
<a class="md-footer__link md-footer__link--next" href="../gcanalysis/" rel="next">
<div class="md-footer__title">
<div class="md-ellipsis">
<span class="md-footer__direction">
                Next
              </span>
              GC-MS on GNPS
            </div>
</div>
<div class="md-footer__button md-icon">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4z"></path></svg>
</div>
</a>
</nav>
<div class="md-footer-meta md-typeset">
<div class="md-footer-meta__inner md-grid">
<div class="md-footer-copyright">
        
        Made with
        <a href="https://squidfunk.github.io/mkdocs-material/" rel="noopener" target="_blank">
          Material for MkDocs
        </a>
</div>
</div>
</div>
</footer>
</div>
<div class="md-dialog" data-md-component="dialog">
<div class="md-dialog__inner md-typeset"></div>
</div>
<script id="__config" type="application/json">{"base": "..", "features": [], "translations": {"clipboard.copy": "Copy to clipboard", "clipboard.copied": "Copied to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.placeholder": "Type to start searching", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.term.missing": "Missing"}, "search": "../assets/javascripts/workers/search.fe42c31b.min.js", "version": null}</script>
<script src="../assets/javascripts/bundle.4ea5477f.min.js"></script>
</body>
</html>