
<!DOCTYPE html>

<html class="no-js" lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width,initial-scale=1" name="viewport"/>
<link href="https://ccms-ucsd.github.io/GNPSDocumentation/ms2lda/" rel="canonical"/>
<link href="../assets/images/favicon.png" rel="icon"/>
<meta content="mkdocs-1.2.2, mkdocs-material-7.2.2" name="generator"/>
<title>MS2LDA and MotifDB Substructure Discovery - GNPS Documentation</title>
<link href="../assets/stylesheets/main.1118c9be.min.css" rel="stylesheet"/>
<link href="../assets/stylesheets/palette.ba0d045b.min.css" rel="stylesheet"/>
<link crossorigin="" href="https://fonts.gstatic.com" rel="preconnect"/>
<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700%7CRoboto+Mono&amp;display=fallback" rel="stylesheet"/>
<style>:root{--md-text-font-family:"Roboto";--md-code-font-family:"Roboto Mono"}</style>
<script>window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)},ga.l=+new Date,ga("create","UA-8412213-5","auto"),ga("set","anonymizeIp",!0),ga("send","pageview"),document.addEventListener("DOMContentLoaded",function(){document.forms.search&&document.forms.search.query.addEventListener("blur",function(){var e;this.value&&(e=document.location.pathname,ga("send","pageview",e+"?q="+this.value))}),"undefined"!=typeof location$&&location$.subscribe(function(e){ga("send","pageview",e.pathname)})})</script>
<script async="" src="https://www.google-analytics.com/analytics.js"></script>
</head>
<body data-md-color-accent="none" data-md-color-primary="none" data-md-color-scheme="" dir="ltr">
<script>function __prefix(e){return new URL("..",location).pathname+"."+e}function __get(e,t=localStorage){return JSON.parse(t.getItem(__prefix(e)))}</script>
<input autocomplete="off" class="md-toggle" data-md-toggle="drawer" id="__drawer" type="checkbox"/>
<input autocomplete="off" class="md-toggle" data-md-toggle="search" id="__search" type="checkbox"/>
<label class="md-overlay" for="__drawer"></label>
<div data-md-component="skip">
<a class="md-skip" href="#ms2lda-substructure-discovery">
          Skip to content
        </a>
</div>
<div data-md-component="announce">
</div>
<header class="md-header" data-md-component="header">
<nav aria-label="Header" class="md-header__inner md-grid">
<a aria-label="GNPS Documentation" class="md-header__button md-logo" data-md-component="logo" href=".." title="GNPS Documentation">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"></path></svg>
</a>
<label class="md-header__button md-icon" for="__drawer">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2z"></path></svg>
</label>
<div class="md-header__title" data-md-component="header-title">
<div class="md-header__ellipsis">
<div class="md-header__topic">
<span class="md-ellipsis">
            GNPS Documentation
          </span>
</div>
<div class="md-header__topic" data-md-component="header-topic">
<span class="md-ellipsis">
            
              MS2LDA and MotifDB Substructure Discovery
            
          </span>
</div>
</div>
</div>
<label class="md-header__button md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"></path></svg>
</label>
<div class="md-search" data-md-component="search" role="dialog">
<label class="md-search__overlay" for="__search"></label>
<div class="md-search__inner" role="search">
<form class="md-search__form" name="search">
<input aria-label="Search" autocapitalize="off" autocomplete="off" autocorrect="off" class="md-search__input" data-md-component="search-query" name="query" placeholder="Search" required="" spellcheck="false" type="text"/>
<label class="md-search__icon md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"></path></svg>
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"></path></svg>
</label>
<nav aria-label="Search" class="md-search__options">
<button aria-label="Clear" class="md-search__icon md-icon" tabindex="-1" type="reset">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"></path></svg>
</button>
</nav>
</form>
<div class="md-search__output">
<div class="md-search__scrollwrap" data-md-scrollfix="">
<div class="md-search-result" data-md-component="search-result">
<div class="md-search-result__meta">
            Initializing search
          </div>
<ol class="md-search-result__list"></ol>
</div>
</div>
</div>
</div>
</div>
<div class="md-header__source">
<a class="md-source" data-md-component="source" href="https://github.com/CCMS-UCSD/GNPSDocumentation/" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    CCMS-UCSD/GNPSDocumentation
  </div>
</a>
</div>
</nav>
</header>
<div class="md-container" data-md-component="container">
<main class="md-main" data-md-component="main">
<div class="md-main__inner md-grid">
<div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Navigation" class="md-nav md-nav--primary" data-md-level="0">
<label class="md-nav__title" for="__drawer">
<a aria-label="GNPS Documentation" class="md-nav__button md-logo" data-md-component="logo" href=".." title="GNPS Documentation">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"></path></svg>
</a>
    GNPS Documentation
  </label>
<div class="md-nav__source">
<a class="md-source" data-md-component="source" href="https://github.com/CCMS-UCSD/GNPSDocumentation/" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    CCMS-UCSD/GNPSDocumentation
  </div>
</a>
</div>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="..">
        GNPS Introduction
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../isgnpsright/">
        Is my data compatible with GNPS?
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../faq/">
        Frequently Asked Questions
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../superquickstart/">
        Superquick Start Guide
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../quickstart/">
        Quick Start Guide
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gnpsanalysisoverview/">
        GNPS Analysis Tools Overview
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../lexicon/">
        GNPS Lexicon
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_8" id="__nav_8" type="checkbox"/>
<label class="md-nav__link" for="__nav_8">
        Mass Spec/Computational Background
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Mass Spec/Computational Background" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_8">
<span class="md-nav__icon md-icon"></span>
          Mass Spec/Computational Background
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../massspecbackground/networkingtheory/">
        Molecular Networking
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_9" id="__nav_9" type="checkbox"/>
<label class="md-nav__link" for="__nav_9">
        Data Preparation/Upload
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Data Preparation/Upload" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_9">
<span class="md-nav__icon md-icon"></span>
          Data Preparation/Upload
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../fileconversion/">
        Mass Spectrometry File Conversion
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fileconversion_waters/">
        File Conversion (Waters)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fileupload/">
        GNPS File Upload
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../metadata/">
        Metadata
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_10" id="__nav_10" type="checkbox"/>
<label class="md-nav__link" for="__nav_10">
        Recommended Data Analysis
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Recommended Data Analysis" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_10">
<span class="md-nav__icon md-icon"></span>
          Recommended Data Analysis
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../networking/">
        Molecular Networking
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_10_2" id="__nav_10_2" type="checkbox"/>
<label class="md-nav__link" for="__nav_10_2">
        GC-MS Molecular Networking
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="GC-MS Molecular Networking" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_10_2">
<span class="md-nav__icon md-icon"></span>
          GC-MS Molecular Networking
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../gcanalysis/">
        GC-MS on GNPS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gc-ms-deconvolution/">
        GC-MS Deconvolution
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gc-ms-library-molecular-network/">
        GC-MS Library Search and Molecular Networking
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../networkingviews/">
        Molecular Networking Views
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../librarysearch/">
        Spectral Library Search
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../masst/">
        MASST Search (Mass Search Tool)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../resultviews/">
        Result Views Navigation
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_11" id="__nav_11" type="checkbox"/>
<label class="md-nav__link" for="__nav_11">
        Mass Spec Data Visualization
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Mass Spec Data Visualization" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_11">
<span class="md-nav__icon md-icon"></span>
          Mass Spec Data Visualization
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../lcms-dashboard/">
        GNPS Dashboard
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gnpsdatasetexplorer/">
        GNPS Dataset Explorer
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../usi/">
        Metabolomics Spectrum Resolver
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--active md-nav__item--nested">
<input checked="" class="md-nav__toggle md-toggle" data-md-toggle="__nav_12" id="__nav_12" type="checkbox"/>
<label class="md-nav__link" for="__nav_12">
        Advanced Data Analysis
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Advanced Data Analysis" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_12">
<span class="md-nav__icon md-icon"></span>
          Advanced Data Analysis
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../networkingfiles/">
        Molecular Networking Files
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_12_2" id="__nav_12_2" type="checkbox"/>
<label class="md-nav__link" for="__nav_12_2">
        Feature Based Molecular Networking
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Feature Based Molecular Networking" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_12_2">
<span class="md-nav__icon md-icon"></span>
          Feature Based Molecular Networking
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking/">
        FBMN Workflow
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-mzmine2/">
        FBMN with MZmine
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-ms-dial/">
        FBMN with MS-DIAL
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-xcms3/">
        FBMN with XCMS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-openms/">
        FBMN with OpenMS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-metaboscape/">
        FBMN with MetaboScape
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-progenesisQI/">
        FBMN with Progenesis QI
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-mztab-m/">
        FBMN with mzTab-M
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-cytoscape/">
        FBMN export to Cytoscape
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_12_3" id="__nav_12_3" type="checkbox"/>
<label class="md-nav__link" for="__nav_12_3">
        FBMN - Ion Identity Networking
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="FBMN - Ion Identity Networking" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_12_3">
<span class="md-nav__icon md-icon"></span>
          FBMN - Ion Identity Networking
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin/">
        IIN Workflow
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin-mzmine/">
        IIN with MZmine2
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin-xcms/">
        IIN with XCMS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin-msdial/">
        IIN with MS-DIAL
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<input class="md-nav__toggle md-toggle" data-md-toggle="toc" id="__toc" type="checkbox"/>
<label class="md-nav__link md-nav__link--active" for="__toc">
          MS2LDA and MotifDB Substructure Discovery
          <span class="md-nav__icon md-icon"></span>
</label>
<a class="md-nav__link md-nav__link--active" href="./">
        MS2LDA and MotifDB Substructure Discovery
      </a>
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#introduction">
    Introduction
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#collecting-the-right-input-files">
    Collecting the Right Input Files
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#quickstart-from-processed-molecular-network">
    Quickstart from Processed Molecular Network
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#setting-the-ms2lda-parameters">
    Setting the MS2LDA Parameters
  </a>
<nav aria-label="Setting the MS2LDA Parameters" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#lda-parameters">
    LDA Parameters
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#selecting-motifdb-motif-sets">
    Selecting MotifDB Motif Sets
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#output-networking-file-thresholds">
    Output Networking File Thresholds
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#ms2lda-output-files">
    MS2LDA Output Files
  </a>
<nav aria-label="MS2LDA Output Files" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#view-motif-labels-for-spectra">
    View Motif Labels for Spectra
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#individual-motif-details">
    Individual Motif Details
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#view-pairs-with-motif-edges">
    View Pairs with Motif Edges
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#view-motif-pdf">
    View Motif PDF
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#download-motif-pdf">
    Download Motif PDF
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#download-pairs">
    Download Pairs
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#download-ms2lda-dict">
    Download MS2LDA Dict
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#analysing-the-ms2lda-results">
    Analysing the MS2LDA results
  </a>
<nav aria-label="Analysing the MS2LDA results" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#analysing-the-results-in-gnps">
    Analysing the results in GNPS
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#taking-the-results-to-ms2lda-web-app">
    Taking the results to MS2LDA web app
  </a>
<nav aria-label="Taking the results to MS2LDA web app" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#uploading-the-dictionary-to-ms2lda-web-app">
    Uploading the dictionary to MS2LDA web app
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#viewing-the-experiment-results">
    Viewing the Experiment Results
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#annotating-motifs">
    Annotating Motifs
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#ms1-differential-analyses-within-the-ms2lda-web-app">
    MS1 Differential Analyses within the MS2LDA web app
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#uploading-annotated-motif-sets-into-motifdb">
    Uploading annotated motif sets into MotifDB
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#future-directions">
    Future Directions
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#citations">
    Citations
  </a>
<nav aria-label="Citations" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#introduction-of-ms2lda-substructure-discovery-algorithm">
    Introduction of MS2LDA Substructure Discovery algorithm
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#publication-of-the-ms2ldaorg-web-app-to-discover-and-analyse-mass2motifs">
    Publication of the MS2LDA.org web app to discover and analyse Mass2Motifs
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#extended-version-of-ms2lda-algorithm-to-handle-multiple-files-at-once">
    Extended version of MS2LDA algorithm to handle multiple files at once
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#page-contributors">
    Page Contributors
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../cytoscape/">
        Molecular Networking in Cytoscape
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../dereplicator/">
        DEREPLICATOR
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../dereplicator%2B/">
        DEREPLICATOR+
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../molDiscovery/">
        MolDiscovery
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../nap/">
        Network Annotation Propagation (NAP)
      </a>
</li>
<li class="md-nav__item md-nav__item--active">
<input class="md-nav__toggle md-toggle" data-md-toggle="toc" id="__toc" type="checkbox"/>
<label class="md-nav__link md-nav__link--active" for="__toc">
          MS2LDA and MotifDB Substructure Discovery
          <span class="md-nav__icon md-icon"></span>
</label>
<a class="md-nav__link md-nav__link--active" href="./">
        MS2LDA and MotifDB Substructure Discovery
      </a>
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#introduction">
    Introduction
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#collecting-the-right-input-files">
    Collecting the Right Input Files
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#quickstart-from-processed-molecular-network">
    Quickstart from Processed Molecular Network
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#setting-the-ms2lda-parameters">
    Setting the MS2LDA Parameters
  </a>
<nav aria-label="Setting the MS2LDA Parameters" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#lda-parameters">
    LDA Parameters
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#selecting-motifdb-motif-sets">
    Selecting MotifDB Motif Sets
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#output-networking-file-thresholds">
    Output Networking File Thresholds
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#ms2lda-output-files">
    MS2LDA Output Files
  </a>
<nav aria-label="MS2LDA Output Files" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#view-motif-labels-for-spectra">
    View Motif Labels for Spectra
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#individual-motif-details">
    Individual Motif Details
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#view-pairs-with-motif-edges">
    View Pairs with Motif Edges
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#view-motif-pdf">
    View Motif PDF
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#download-motif-pdf">
    Download Motif PDF
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#download-pairs">
    Download Pairs
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#download-ms2lda-dict">
    Download MS2LDA Dict
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#analysing-the-ms2lda-results">
    Analysing the MS2LDA results
  </a>
<nav aria-label="Analysing the MS2LDA results" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#analysing-the-results-in-gnps">
    Analysing the results in GNPS
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#taking-the-results-to-ms2lda-web-app">
    Taking the results to MS2LDA web app
  </a>
<nav aria-label="Taking the results to MS2LDA web app" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#uploading-the-dictionary-to-ms2lda-web-app">
    Uploading the dictionary to MS2LDA web app
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#viewing-the-experiment-results">
    Viewing the Experiment Results
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#annotating-motifs">
    Annotating Motifs
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#ms1-differential-analyses-within-the-ms2lda-web-app">
    MS1 Differential Analyses within the MS2LDA web app
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#uploading-annotated-motif-sets-into-motifdb">
    Uploading annotated motif sets into MotifDB
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#future-directions">
    Future Directions
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#citations">
    Citations
  </a>
<nav aria-label="Citations" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#introduction-of-ms2lda-substructure-discovery-algorithm">
    Introduction of MS2LDA Substructure Discovery algorithm
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#publication-of-the-ms2ldaorg-web-app-to-discover-and-analyse-mass2motifs">
    Publication of the MS2LDA.org web app to discover and analyse Mass2Motifs
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#extended-version-of-ms2lda-algorithm-to-handle-multiple-files-at-once">
    Extended version of MS2LDA algorithm to handle multiple files at once
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#page-contributors">
    Page Contributors
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../molnetenhancer/">
        MolNetEnhancer
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../metaminer/">
        MetaMiner
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../sirius/">
        Sirius
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../ReDU/">
        Reanalysis of Data User Interface (ReDU)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../qemistree/">
        Qemistree
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../mmvec/">
        MMVEC
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../mergepolarity/">
        Merge Network Polarity
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../chemdir/">
        Chemical Directionality
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_13" id="__nav_13" type="checkbox"/>
<label class="md-nav__link" for="__nav_13">
        Spectral Libraries
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Spectral Libraries" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_13">
<span class="md-nav__icon md-icon"></span>
          Spectral Libraries
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../browselibraries/">
        Browsing Reference Spectral Libraries
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../spectrumcuration/">
        Add/Curate Reference Spectrum in the Library
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../batchupload/">
        Reference Spectral Library Batch Addition
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../batchuploadmzvault/">
        Reference Spectral Library mzVault Addition
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../downloadlibraries/">
        Downloading Reference Spectral Libraries
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_13_6" id="__nav_13_6" type="checkbox"/>
<label class="md-nav__link" for="__nav_13_6">
        Open-source Method for Reference Library Generation
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Open-source Method for Reference Library Generation" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_13_6">
<span class="md-nav__icon md-icon"></span>
          Open-source Method for Reference Library Generation
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../msmschooser/">
        MSMS-Chooser Workflow
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../msmschooserdataacquisitioncomputational/">
        Computational Data Acquisition Prep
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../msmschooserdataacquisitionQE/">
        Data Acqusition - Thermo QExactive (v1.0)
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tag_info/">
        GNPS Tag Contribution and Analyses
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_14" id="__nav_14" type="checkbox"/>
<label class="md-nav__link" for="__nav_14">
        Create a Public Datasets
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Create a Public Datasets" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_14">
<span class="md-nav__icon md-icon"></span>
          Create a Public Datasets
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../datasets/">
        Dataset Creation/Sharing
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../continuousid/">
        Continuous Identification
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../moleculeexplorer/">
        Molecule Explorer
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_15" id="__nav_15" type="checkbox"/>
<label class="md-nav__link" for="__nav_15">
        Community/Social Features
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Community/Social Features" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_15">
<span class="md-nav__icon md-icon"></span>
          Community/Social Features
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../user/">
        User Page
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../troubleshooting/">
        Troubleshooting
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_17" id="__nav_17" type="checkbox"/>
<label class="md-nav__link" for="__nav_17">
        Tutorial Guides
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Tutorial Guides" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_17">
<span class="md-nav__icon md-icon"></span>
          Tutorial Guides
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/bacterianetworking/">
        Molecular Networking - Analog Discovery in Microbial Extracts
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/americangutmzmine/">
        Feature-Based Molecular Networking (FBMN) with MZmine2 - Microbiome Molecules in Fecal Samples in American Gut Project
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/americangut-ms-dial/">
        FBMN with MS-DIAL - Microbiome Molecules in Fecal Samples in American Gut Project
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/coffee-tutorial-metaboscape/">
        FBMN with MetaboScape - Coffee Analysis
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/featurebasedgnps/">
        FBMN analysis at GNPS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/rdd/">
        Reference Data Driven Analysis
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../changelog/">
        Change Log
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gnps_community/">
        Join the GNPS community
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../workshops/">
        GNPS Workshops/Training
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../logos/">
        GNPS Logos
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../contact/">
        Contact/Office Hours
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../toolindex/">
        GNPS Tool Index
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../api/">
        API
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../developers/">
        For Developers
      </a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#introduction">
    Introduction
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#collecting-the-right-input-files">
    Collecting the Right Input Files
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#quickstart-from-processed-molecular-network">
    Quickstart from Processed Molecular Network
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#setting-the-ms2lda-parameters">
    Setting the MS2LDA Parameters
  </a>
<nav aria-label="Setting the MS2LDA Parameters" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#lda-parameters">
    LDA Parameters
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#selecting-motifdb-motif-sets">
    Selecting MotifDB Motif Sets
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#output-networking-file-thresholds">
    Output Networking File Thresholds
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#ms2lda-output-files">
    MS2LDA Output Files
  </a>
<nav aria-label="MS2LDA Output Files" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#view-motif-labels-for-spectra">
    View Motif Labels for Spectra
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#individual-motif-details">
    Individual Motif Details
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#view-pairs-with-motif-edges">
    View Pairs with Motif Edges
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#view-motif-pdf">
    View Motif PDF
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#download-motif-pdf">
    Download Motif PDF
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#download-pairs">
    Download Pairs
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#download-ms2lda-dict">
    Download MS2LDA Dict
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#analysing-the-ms2lda-results">
    Analysing the MS2LDA results
  </a>
<nav aria-label="Analysing the MS2LDA results" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#analysing-the-results-in-gnps">
    Analysing the results in GNPS
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#taking-the-results-to-ms2lda-web-app">
    Taking the results to MS2LDA web app
  </a>
<nav aria-label="Taking the results to MS2LDA web app" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#uploading-the-dictionary-to-ms2lda-web-app">
    Uploading the dictionary to MS2LDA web app
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#viewing-the-experiment-results">
    Viewing the Experiment Results
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#annotating-motifs">
    Annotating Motifs
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#ms1-differential-analyses-within-the-ms2lda-web-app">
    MS1 Differential Analyses within the MS2LDA web app
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#uploading-annotated-motif-sets-into-motifdb">
    Uploading annotated motif sets into MotifDB
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#future-directions">
    Future Directions
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#citations">
    Citations
  </a>
<nav aria-label="Citations" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#introduction-of-ms2lda-substructure-discovery-algorithm">
    Introduction of MS2LDA Substructure Discovery algorithm
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#publication-of-the-ms2ldaorg-web-app-to-discover-and-analyse-mass2motifs">
    Publication of the MS2LDA.org web app to discover and analyse Mass2Motifs
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#extended-version-of-ms2lda-algorithm-to-handle-multiple-files-at-once">
    Extended version of MS2LDA algorithm to handle multiple files at once
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#page-contributors">
    Page Contributors
  </a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-content" data-md-component="content">
<article class="md-content__inner md-typeset">
<a class="md-content__button md-icon" href="https://github.com/CCMS-UCSD/GNPSDocumentation/edit/master/docs/ms2lda.md" title="Edit this page">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25z"></path></svg>
</a>
<h1 id="ms2lda-substructure-discovery">MS2LDA Substructure Discovery<a class="headerlink" href="#ms2lda-substructure-discovery" title="Permanent link">¶</a></h1>
<h2 id="introduction">Introduction<a class="headerlink" href="#introduction" title="Permanent link">¶</a></h2>
<p><img alt="img" src="../img/ms2lda/banner.png"/></p>
<p>MS2LDA is a tool that decomposes molecular fragmentation data derived from large metabolomics experiments into annotated Mass2Motifs or discovers Mass2Motifs from experimental data. Mass2Motifs are fragmentation patterns of often co-occurring mass fragment peaks and/or neutral losses that often represent molecular substructures. Check out the <a href="http://ms2lda.org/">MS2LDA website</a> here where you can find more information, browse through data sets, and sign up for an account to run the Mass2Motif discovery on your own data. At GNPS, we are working with the MS2LDA team to integrate both workflows which allows users to map Mass2Motif occurrences in their Molecular Families.</p>
<p>It is now also possible to run MS2LDA analyses through the GNPS interface <a href="https://gnps.ucsd.edu/ProteoSAFe/index.jsp?params=%7B%22workflow%22:%22MS2LDA_MOTIFDB%22%7D">here</a> which will lead to the below displayed job entry page.</p>
<p><img alt="img" src="../img/ms2lda/overview.png"/></p>
<p>In this section, it is explained what input files are needed, how the parameters can be chosen, and what kind of output you get.
Finally, it is explained how the obtained MS2LDA model can be further explored on the <a href="http://www.ms2lda.org">MS2LDA</a> website by uploading the generated dict file.</p>
<h2 id="collecting-the-right-input-files">Collecting the Right Input Files<a class="headerlink" href="#collecting-the-right-input-files" title="Permanent link">¶</a></h2>
<p>After providing the GNPS analysis with a title, the right input files are needed to start an MS2LDA analysis. They can be obtained with the classical molecular network (MS-Cluster based, <a href="../networking/">see documentation here</a>) or from the Feature-Based Molecular Networking workflow (using for example MZmine2, <a href="../featurebasedmolecularnetworking/">see documentation here</a>. These files are:</p>
<ul>
<li>Required: Spectral data (MGF file) which can be downloaded from a processed Molecular Networking job</li>
<li>Required: Edges file including selfloops which can be downloaded from a processed Molecular Networking job (i.e., typically found in the networkedges_selfloop folder within the downloaded GNPS job folder)</li>
<li>Optional: MS1 peak abundance information (CSV file generated by MZmine2). The format must be similar to the MZmine2 format used with the <a href="../featurebasedmolecularnetworking/">Feature based Molecular Networking</a>. This will allow you to perform MS1 differential analysis within the MS2LDA web app to find discriminate Mass2Motifs between groups.</li>
</ul>
<p><img alt="img" src="../img/ms2lda/InputData.png"/></p>
<p>After selecting the input files, the right MGF Source needs to be selected: "MSCluster" for a classical Molecular Networking derived clustered MGF file, and "MZmine2" for a Feature based Molecular Networking derived MGF file.</p>
<h2 id="quickstart-from-processed-molecular-network">Quickstart from Processed Molecular Network<a class="headerlink" href="#quickstart-from-processed-molecular-network" title="Permanent link">¶</a></h2>
<p>To facilitate MS2LDA analysis of processed Molecular Networking jobs, in the newest (V2) version and the Feature Based Molecular Networking job, under "Advanced Views - Experimental Views" there is a button <u>Analyze with MS2LDA</u> that will bring you directly to the MS2LDA analysis page prefilling the relevant input files (spectral MGF file and edge pair file including selfloops). See also the animated GIF below.
Please note that you can adapt the title and you must check the relevant MS2LDA parameter settings for your data set.</p>
<p><img alt="img" src="../img/ms2lda/MS2LDAfromMolNetw.gif"/></p>
<h2 id="setting-the-ms2lda-parameters">Setting the MS2LDA Parameters<a class="headerlink" href="#setting-the-ms2lda-parameters" title="Permanent link">¶</a></h2>
<p>There are three sections of parameters to fill out of which the first is the most important. Default values are provided for all settings, but each of them is dependent on the input data. We provide some guidelines here to ease your choice.</p>
<h3 id="lda-parameters">LDA Parameters<a class="headerlink" href="#lda-parameters" title="Permanent link">¶</a></h3>
<p><img alt="img" src="../img/ms2lda/LDAparameters.png"/></p>
<p>The "Advanced MS2LDA parameters" displays four parameter values to set. The first is the bin width used to bin the MS2 fragments and losses. Defaults for Q-Exactive (0.005 Da) and ToF (0.01 Da) Data are provided. For instruments like Ion Traps a larger bin width of 0.1 Da is recommended. The appropriate bin width to choose is dependent on the expected mass variation of mass fragments across mass spectra in your data.</p>
<p><img alt="img" src="../img/ms2lda/LDAparameterBinning.png"/></p>
<p>The number of LDA iterations is set to 1000 by default which should work in most cases.</p>
<p>The Minimum MS2 Intensity is meant to filter out the basic noise from the MS/MS spectra. It is recommended to open the inputted MGF file and check the noise levels. Typically they range from 25-200 (ToF Data) to 5000 (Q-Exactive Data). Inclusion of noise will unnecessarily lengten the analysis time and create non-informative Mass2Motifs.</p>
<p>The LDA Free Motifs correspond to the number of "free Mass2Motifs" that are added on top of the prefixed annotated Mass2Motifs from MotifDB. This number depends on your data: if you input a larger number of fragmented molecules (i.e., &gt; 4,000) that includes lots of novel chemistry not present in MotifDB it is best to put the LDA Free Motifs on 300 - 500. If the data set is relatively small with a considerable amount of known chemistry, you best put this number to 50 - 150.</p>
<h3 id="selecting-motifdb-motif-sets">Selecting MotifDB Motif Sets<a class="headerlink" href="#selecting-motifdb-motif-sets" title="Permanent link">¶</a></h3>
<p>By default all Motif Sets will be imported from <a href="http://www.ms2lda.org/motifdb">MotifDB</a>. However, it is wise to exclude annotated motif sets of sources that are very distant from the source of your input data. Note that the GNPS and MassBank are annotated Mass2Motifs discovered in reference molecules.</p>
<p><img alt="img" src="../img/ms2lda/LDAparameterMotifDB.png"/></p>
<h3 id="output-networking-file-thresholds">Output Networking File Thresholds<a class="headerlink" href="#output-networking-file-thresholds" title="Permanent link">¶</a></h3>
<p>After running MS2LDA, output files are created that include a new networking (edge) file that can be used to create an enriched Molecular Network including Mass2Motif distributions through the Molecular Families. To do so, cut-off values (thresholds) are needed for LDA scores to consider a Mass2Motif to be linked to a molecule or not and how many you want to show in the network. Default values are provided that were found to work well so far.
In <a href="https://pubs.acs.org/doi/abs/10.1021/acs.analchem.7b01391">this</a> publication the two scores are explained: probability and overlap score.</p>
<p><img alt="img" src="../img/ms2lda/LDAoutput_thresholds.png"/></p>
<p>The Figure (from <a href="https://pubs.acs.org/doi/abs/10.1021/acs.analchem">van der Hooft, Anal. Chem. 2017 89147569-7577</a>) below illustrates the overlap score (O) and probability threshold (P) with two hypothetical molecular spectra and three hypothetical Mass2Motifs. In example A, molecule 1 has an overlap score of 1 with Mass2Motif 1 as the motif is in the molecule in its entirety. The probability of this connection is 0.5 as it accounts for half of the molecule’s total intensity. In example B, the same molecule has an overlap score of
(approximately) 0.5 with Mass2Motif 2 as it includes roughly half of the Mass2Motif. </p>
<p><img alt="img" src="../img/ms2lda/overlapprobabliltyscore.png"/></p>
<p>Note that you will have to provide your email address in the appropriate field in order to be notified upon completion of your GNPS-MS2LDA job.</p>
<h2 id="ms2lda-output-files">MS2LDA Output Files<a class="headerlink" href="#ms2lda-output-files" title="Permanent link">¶</a></h2>
<p>Upon succesful completion, you will be able to see the output job page as displayed below:</p>
<p><img alt="img" src="../img/ms2lda/output.png"/></p>
<p>There are different views and files for download available:</p>
<h3 id="view-motif-labels-for-spectra">View Motif Labels for Spectra<a class="headerlink" href="#view-motif-labels-for-spectra" title="Permanent link">¶</a></h3>
<p>This view displays all the fragmented molecules (scans) and each of the motifs linked to them [above the set thresholds as specified in the Output Networking file thresholds].
It can be downloaded as a tab separated file.</p>
<p><img alt="img" src="../img/ms2lda/MotifLabels.png"/></p>
<h3 id="individual-motif-details">Individual Motif Details<a class="headerlink" href="#individual-motif-details" title="Permanent link">¶</a></h3>
<p>This view displays all molecule-motif links [some molecules have more than one motif linked and will appear multiple times in this table] and also shows the annotation if available from MotifDB.
It can be downloaded as a tab separated file.</p>
<p><img alt="img" src="../img/ms2lda/MotifDetails.png"/></p>
<h3 id="view-pairs-with-motif-edges">View Pairs with Motif Edges<a class="headerlink" href="#view-pairs-with-motif-edges" title="Permanent link">¶</a></h3>
<p>This view is the new edge file enriched with Motif interactions: note that in the "Interact" column apart from cosine now different Mass2Motifs are listed [above the set thresholds as specified in the Output Networking file thresholds].
It can be downloaded as a tab separated file - this can be used to create a new enriched Molecular Network in CytoScape.</p>
<p><img alt="img" src="../img/ms2lda/PairsMotifEdges.png"/></p>
<h3 id="view-motif-pdf">View Motif PDF<a class="headerlink" href="#view-motif-pdf" title="Permanent link">¶</a></h3>
<p>This displays a PDF report of all the Mass2Motifs. Each Mass2Motif spans two pages with the first page a graphical overview of the relevant mass fragments (red, on top, pointing up) and neutral losses (green, bottom, pointing down) with the y-axis being the intensity which equals the features' probability to be found in the Mass2Motif, and the x-axis being the m/z (rounded up to nominal values in the labels) in Daltons.
The second page displays the top mass fragments and neutral losses with their probabilities.</p>
<h3 id="download-motif-pdf">Download Motif PDF<a class="headerlink" href="#download-motif-pdf" title="Permanent link">¶</a></h3>
<p>This downloads the PDF report as explained in "View Motif PDF".</p>
<h3 id="download-pairs">Download Pairs<a class="headerlink" href="#download-pairs" title="Permanent link">¶</a></h3>
<p>This downloads the new edge file as explained in "View Pairs with Motif Edges".</p>
<h3 id="download-ms2lda-dict">Download MS2LDA Dict<a class="headerlink" href="#download-ms2lda-dict" title="Permanent link">¶</a></h3>
<p>This downloads the MS2LDA Dict file that houses all the relevant information on the MS2LDA experiment.</p>
<h2 id="analysing-the-ms2lda-results">Analysing the MS2LDA results<a class="headerlink" href="#analysing-the-ms2lda-results" title="Permanent link">¶</a></h2>
<h3 id="analysing-the-results-in-gnps">Analysing the results in GNPS<a class="headerlink" href="#analysing-the-results-in-gnps" title="Permanent link">¶</a></h3>
<p>The different available views enable a quick overview of the MS2LDA experiment. Especially when annotated Mass2Motifs are found, this could help in further downstream analysis.
However, when no annotation is provided, the Mass2Motif is one of the "free Motifs" that was discovered by LDA. With the MS2LDA Dict file through the MS2LDA web app there are ways to analyse these Mass2Motifs further and also annotate them.</p>
<p>In general, when analysing annotating or novel Mass2Motifs, it is important to keep <a href="http://ms2lda.org/disclaimer/">this</a> in mind!</p>
<h3 id="taking-the-results-to-ms2lda-web-app">Taking the results to MS2LDA web app<a class="headerlink" href="#taking-the-results-to-ms2lda-web-app" title="Permanent link">¶</a></h3>
<h4 id="uploading-the-dictionary-to-ms2lda-web-app">Uploading the dictionary to MS2LDA web app<a class="headerlink" href="#uploading-the-dictionary-to-ms2lda-web-app" title="Permanent link">¶</a></h4>
<p>After downloading the MS2LSA Dict file from the finished MS2LDA experiment, this can be uploaded into the <a href="http://www.ms2lda.org">MS2LDA web app</a>. There is a <a href="http://www.ms2lda.org/user_guide/">User Guide</a> available that covers the main functionalities of the MS2LDA web app.</p>
<p>If it is your first time at the MS2LDA web app, make sure you get a login account before continuing. Once logged in, go to the experiments section:</p>
<p><img alt="img" src="../img/ms2lda/AppCreateExperiments.png"/></p>
<p>By default, you arrive on the Create tab which lets you create an MS2LDA experiment from mass spectrometry data.</p>
<p><img alt="img" src="../img/ms2lda/AppAddExp.png"/></p>
<p>Click on the Upload tab, to continue:</p>
<p><img alt="img" src="../img/ms2lda/AppAddExpUpload.png"/></p>
<p>Now, you will have to provide a unique name for the MS2LDA experiment and you can also provide relevant details.
Most importantly, you will need to select the location of the downloaded MS2LDA Dict file under "LDA output file".
Also, you will need to select the bin width as used in the GNPS-MS2LDA workflow ("Bin Width") from the available options.</p>
<p><img alt="img" src="../img/ms2lda/AppUploadDictFile.png"/></p>
<p>After you are ready, you can click "Upload Your Experiment" and the upload will start.</p>
<h4 id="viewing-the-experiment-results">Viewing the Experiment Results<a class="headerlink" href="#viewing-the-experiment-results" title="Permanent link">¶</a></h4>
<p>Once the MS2LDA dict file finished uploading, you can browse the MS2LDA experiment in the web app like a regular MS2LDa experiment. If you click on the experiment name, multiple tabs will appear:</p>
<p><img alt="img" src="../img/ms2lda/AppExperimentTabsOverview.png"/></p>
<p>Here, you can click on the Summary Page tab which leads you to the summary of the experiment that includes all relevant Tables. For example, the Motif Table can be browsed, sorted, searched, and downloaded as CSV:</p>
<p><img alt="img" src="../img/ms2lda/AppExperimentSummaryPageMotifs.png"/></p>
<h4 id="annotating-motifs">Annotating Motifs<a class="headerlink" href="#annotating-motifs" title="Permanent link">¶</a></h4>
<p>When clicking on the relevant "free Motif" name a Motif Page will appear where the relevant mass fragment and neutral losses are listed including their scores and a frequency plot. Also, all the spectra the Mass2Motif appears in can be browsed to analyse the nature of the fragmentation pattern.
Importantly, if you can annote the motif with any structural information, this can be provided on top of the page with the Annotation field for the full annotation and the Short Annotation field for a label to display in graphics - make sure to save the Motif Page afterwards! After reloading the Summary Page or the Show Mass2Motif tabs, the annotation(s) will be displayed and are searchable.</p>
<p><img alt="img" src="../img/ms2lda/AppExampleFreeMotifPage.png"/></p>
<p>Finally, on top of the summary page there is a toggle where you can make the entire MS2LDA experiment public. See for example the public <a href="http://ms2lda.org/basicviz/summary/190/">MassBank Experiment</a>.</p>
<h4 id="ms1-differential-analyses-within-the-ms2lda-web-app">MS1 Differential Analyses within the MS2LDA web app<a class="headerlink" href="#ms1-differential-analyses-within-the-ms2lda-web-app" title="Permanent link">¶</a></h4>
<p>If the Feature based Molecular Networking workflow was used, the MS1 peak information is available in the MS2LDA Dict file. This can be used to create MS1 analysis to find Mass2Motif that are discriminative between two sample groups.
More information can be found in <a href="http://www.pnas.org/content/113/48/13738.short">this</a> publication and the <a href="http://www.ms2lda.org/user_guide/#ms1_analysis">MS2LDA web app User Guide</a> explains how to set up such an analysis within your experiment.</p>
<h3 id="uploading-annotated-motif-sets-into-motifdb">Uploading annotated motif sets into MotifDB<a class="headerlink" href="#uploading-annotated-motif-sets-into-motifdb" title="Permanent link">¶</a></h3>
<p>This is a future feature that will become available in the course of this year. It will allow you to make your new annotations searchable and usable by the community.</p>
<h2 id="future-directions">Future Directions<a class="headerlink" href="#future-directions" title="Permanent link">¶</a></h2>
<p>At GNPS, we are working with the MS2LDA team to promote publication of curated sets of Mass2Motifs into <a href="http://www.ms2lda.org/motifdb">MotifDB</a> that can help to annotate your MS/MS spectra.</p>
<h2 id="citations">Citations<a class="headerlink" href="#citations" title="Permanent link">¶</a></h2>
<p>Have you found MS2LDA to be useful for your data analysis? Please cite the relevant literature from below.</p>
<h3 id="introduction-of-ms2lda-substructure-discovery-algorithm">Introduction of MS2LDA Substructure Discovery algorithm<a class="headerlink" href="#introduction-of-ms2lda-substructure-discovery-algorithm" title="Permanent link">¶</a></h3>
<p><a href="http://www.pnas.org/content/113/48/13738.short">van der Hooft, Justin J. J., Joe Wandy, Michael P. Barrett, Karl E.V. Burgess, and Simon Rogers. "Topic modeling for untargeted substructure exploration in metabolomics." Proceedings of the National Academy of Sciences 113, no. 48 (2016): 13738-13743.</a></p>
<h3 id="publication-of-the-ms2ldaorg-web-app-to-discover-and-analyse-mass2motifs">Publication of the MS2LDA.org web app to discover and analyse Mass2Motifs<a class="headerlink" href="#publication-of-the-ms2ldaorg-web-app-to-discover-and-analyse-mass2motifs" title="Permanent link">¶</a></h3>
<p><a href="https://academic.oup.com/bioinformatics/article/34/2/317/4158166">Wandy, Joe, Yunfeng Zhu, Justin J.J. van der Hooft, Ronan Daly, Michael P. Barrett, and Simon Rogers. "Ms2lda.org: web-based topic modelling for substructure discovery in mass spectrometry." Bioinformatics 34, no. 2 (2018): 317-318.</a></p>
<h3 id="extended-version-of-ms2lda-algorithm-to-handle-multiple-files-at-once">Extended version of MS2LDA algorithm to handle multiple files at once<a class="headerlink" href="#extended-version-of-ms2lda-algorithm-to-handle-multiple-files-at-once" title="Permanent link">¶</a></h3>
<p><a href="https://pubs.acs.org/doi/abs/10.1021/acs.analchem.7b01391">van der Hooft, Justin J. J., Joe Wandy, Francesca Young, Sandosh Padmanabhan, Konstantinos Gerasidimis, Karl E.V. Burgess, Michael P. Barrett, and Simon Rogers. "Unsupervised Discovery and Comparison of Structural Families Across Multiple Samples in Untargeted Metabolomics." Analytical Chemistry 89, no. 14 (2017): 7569-7577.</a></p>
<p>Latest developments in MS2LDA.org: introduction of <a href="http://ms2lda.org/motifdb/">MotifDB</a> with annotated Mass2Motif sets and <a href="http://www.emetabolomics.org">MAGMa</a> integration for Mass2Motif annotation of reference compounds</p>
<p><a href="https://pubs.rsc.org/en/Content/ArticleLanding/2019/FD/C8FD00235E#!divAbstract">Rogers, Simon, Cher Wei Ong, Joe Wandy, Madeleine Ernst, Lars Ridder, and Justin J.J. van der Hooft. "Deciphering complex metabolite mixtures by unsupervised and supervised substructure discovery and semi-automated annotation from MS/MS spectra." Faraday Discussions themed collection  Challenges in analysis of complex natural mixtures (2019).</a></p>
<h2 id="page-contributors">Page Contributors<a class="headerlink" href="#page-contributors" title="Permanent link">¶</a></h2>
<p><span class="git-page-authors git-authors"><a href="mailto:mwang87@gmail.com">Ming Wang</a> (7.21%), <a href="mailto:louisfelix.nothias@gmail.com">lfnothias</a> (16.35%), <a href="mailto:jjjvanderhooft@gmail.com">Justin van der Hooft</a> (76.44%)</span></p>
<hr/>
<div class="md-source-date">
<small>
    
      Last update: <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-datetime">June 13, 2020 23:18:29</span>
</small>
</div>
</article>
</div>
</div>
</main>
<footer class="md-footer">
<nav aria-label="Footer" class="md-footer__inner md-grid">
<a aria-label="Previous: Network Annotation Propagation (NAP)" class="md-footer__link md-footer__link--prev" href="../nap/" rel="prev">
<div class="md-footer__button md-icon">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"></path></svg>
</div>
<div class="md-footer__title">
<div class="md-ellipsis">
<span class="md-footer__direction">
                Previous
              </span>
              Network Annotation Propagation (NAP)
            </div>
</div>
</a>
<a aria-label="Next: MolNetEnhancer" class="md-footer__link md-footer__link--next" href="../molnetenhancer/" rel="next">
<div class="md-footer__title">
<div class="md-ellipsis">
<span class="md-footer__direction">
                Next
              </span>
              MolNetEnhancer
            </div>
</div>
<div class="md-footer__button md-icon">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4z"></path></svg>
</div>
</a>
</nav>
<div class="md-footer-meta md-typeset">
<div class="md-footer-meta__inner md-grid">
<div class="md-footer-copyright">
        
        Made with
        <a href="https://squidfunk.github.io/mkdocs-material/" rel="noopener" target="_blank">
          Material for MkDocs
        </a>
</div>
</div>
</div>
</footer>
</div>
<div class="md-dialog" data-md-component="dialog">
<div class="md-dialog__inner md-typeset"></div>
</div>
<script id="__config" type="application/json">{"base": "..", "features": [], "translations": {"clipboard.copy": "Copy to clipboard", "clipboard.copied": "Copied to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.placeholder": "Type to start searching", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.term.missing": "Missing", "select.version.title": "Select version"}, "search": "../assets/javascripts/workers/search.709b4209.min.js", "version": null}</script>
<script src="../assets/javascripts/bundle.2b46852b.min.js"></script>
</body>
</html>