



<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      <meta http-equiv="x-ua-compatible" content="ie=edge">
      
      
      
      
        <meta name="lang:clipboard.copy" content="Copy to clipboard">
      
        <meta name="lang:clipboard.copied" content="Copied to clipboard">
      
        <meta name="lang:search.language" content="en">
      
        <meta name="lang:search.pipeline.stopwords" content="True">
      
        <meta name="lang:search.pipeline.trimmer" content="True">
      
        <meta name="lang:search.result.none" content="No matching documents">
      
        <meta name="lang:search.result.one" content="1 matching document">
      
        <meta name="lang:search.result.other" content="# matching documents">
      
        <meta name="lang:search.tokenizer" content="[\s\-]+">
      
      <link rel="shortcut icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.0.4, mkdocs-material-4.4.2">
    
    
      
        <title>FBMN with MZmine2 - GNPS Documentation</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/application.30686662.css">
      
      
    
    
      <script src="../assets/javascripts/modernizr.74668098.js"></script>
    
    
      
        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700|Roboto+Mono&display=fallback">
        <style>body,input{font-family:"Roboto","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
      
    
    <link rel="stylesheet" href="../assets/fonts/material-icons.css">
    
    
    
      
        
<script>
  window.ga = window.ga || function() {
    (ga.q = ga.q || []).push(arguments)
  }
  ga.l = +new Date
  /* Setup integration and send page view */
  ga("create", "UA-8412213-5", "ccms-ucsd.github.io")
  ga("set", "anonymizeIp", true)
  ga("send", "pageview")
  /* Register handler to log search on blur */
  document.addEventListener("DOMContentLoaded", () => {
    if (document.forms.search) {
      var query = document.forms.search.query
      query.addEventListener("blur", function() {
        if (this.value) {
          var path = document.location.pathname;
          ga("send", "pageview", path + "?q=" + this.value)
        }
      })
    }
  })
</script>
<script async src="https://www.google-analytics.com/analytics.js"></script>
      
    
    
  </head>
  
    <body dir="ltr">
  
    <svg class="md-svg">
      <defs>
        
        
      </defs>
    </svg>
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" data-md-component="overlay" for="__drawer"></label>
    
      <a href="#introduction" tabindex="1" class="md-skip">
        Skip to content
      </a>
    
    
      <header class="md-header" data-md-component="header">
  <nav class="md-header-nav md-grid">
    <div class="md-flex">
      <div class="md-flex__cell md-flex__cell--shrink">
        <a href=".." title="GNPS Documentation" class="md-header-nav__button md-logo">
          
            <i class="md-icon"></i>
          
        </a>
      </div>
      <div class="md-flex__cell md-flex__cell--shrink">
        <label class="md-icon md-icon--menu md-header-nav__button" for="__drawer"></label>
      </div>
      <div class="md-flex__cell md-flex__cell--stretch">
        <div class="md-flex__ellipsis md-header-nav__title" data-md-component="title">
          
            <span class="md-header-nav__topic">
              GNPS Documentation
            </span>
            <span class="md-header-nav__topic">
              
                FBMN with MZmine2
              
            </span>
          
        </div>
      </div>
      <div class="md-flex__cell md-flex__cell--shrink">
        
          <label class="md-icon md-icon--search md-header-nav__button" for="__search"></label>
          
<div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="query" data-md-state="active">
      <label class="md-icon md-search__icon" for="__search"></label>
      <button type="reset" class="md-icon md-search__icon" data-md-component="reset" tabindex="-1">
        &#xE5CD;
      </button>
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="result">
          <div class="md-search-result__meta">
            Type to start searching
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
        
      </div>
      
        <div class="md-flex__cell md-flex__cell--shrink">
          <div class="md-header-nav__source">
            


  

<a href="/" title="Go to repository" class="md-source" data-md-source="">
  
  <div class="md-source__repository">
    
  </div>
</a>
          </div>
        </div>
      
    </div>
  </nav>
</header>
    
    <div class="md-container">
      
        
      
      
      <main class="md-main" role="main">
        <div class="md-main__inner md-grid" data-md-component="container">
          
            
              <div class="md-sidebar md-sidebar--primary" data-md-component="navigation">
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    <nav class="md-nav md-nav--primary" data-md-level="0">
  <label class="md-nav__title md-nav__title--site" for="__drawer">
    <a href=".." title="GNPS Documentation" class="md-nav__button md-logo">
      
        <i class="md-icon"></i>
      
    </a>
    GNPS Documentation
  </label>
  
    <div class="md-nav__source">
      


  

<a href="/" title="Go to repository" class="md-source" data-md-source="">
  
  <div class="md-source__repository">
    
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      


  <li class="md-nav__item">
    <a href=".." title="GNPS Introduction" class="md-nav__link">
      GNPS Introduction
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../isgnpsright/" title="Is my data compatible with GNPS?" class="md-nav__link">
      Is my data compatible with GNPS?
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../faq/" title="Frequently Asked Questions" class="md-nav__link">
      Frequently Asked Questions
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../superquickstart/" title="Superquick Start Guide" class="md-nav__link">
      Superquick Start Guide
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../quickstart/" title="Quick Start Guide" class="md-nav__link">
      Quick Start Guide
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../gnpsanalysisoverview/" title="GNPS Analysis Tools Overview" class="md-nav__link">
      GNPS Analysis Tools Overview
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-7" type="checkbox" id="nav-7">
    
    <label class="md-nav__link" for="nav-7">
      Mass Spec/Computational Background
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-7">
        Mass Spec/Computational Background
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../massspecbackground/networkingtheory/" title="Molecular Networking" class="md-nav__link">
      Molecular Networking
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-8" type="checkbox" id="nav-8">
    
    <label class="md-nav__link" for="nav-8">
      Data Preparation/Upload
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-8">
        Data Preparation/Upload
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../fileconversion/" title="Mass Spectrometry File Conversion" class="md-nav__link">
      Mass Spectrometry File Conversion
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../fileupload/" title="GNPS File Upload" class="md-nav__link">
      GNPS File Upload
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../metadata/" title="Metadata" class="md-nav__link">
      Metadata
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-9" type="checkbox" id="nav-9">
    
    <label class="md-nav__link" for="nav-9">
      Recommended Data Analysis
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-9">
        Recommended Data Analysis
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../networking/" title="Molecular Networking" class="md-nav__link">
      Molecular Networking
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../networkingviews/" title="Molecular Networking Views" class="md-nav__link">
      Molecular Networking Views
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../librarysearch/" title="Spectral Library Search" class="md-nav__link">
      Spectral Library Search
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../masst/" title="MASST Search (Mass Search Tool)" class="md-nav__link">
      MASST Search (Mass Search Tool)
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../resultviews/" title="Result Views Navigation" class="md-nav__link">
      Result Views Navigation
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      

  


  <li class="md-nav__item md-nav__item--active md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-10" type="checkbox" id="nav-10" checked>
    
    <label class="md-nav__link" for="nav-10">
      Advanced Data Analysis
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-10">
        Advanced Data Analysis
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../dereplicator/" title="DEREPLICATOR" class="md-nav__link">
      DEREPLICATOR
    </a>
  </li>

        
          
          
          

  


  <li class="md-nav__item md-nav__item--active md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-10-2" type="checkbox" id="nav-10-2" checked>
    
    <label class="md-nav__link" for="nav-10-2">
      Feature Based Molecular Networking
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="2">
      <label class="md-nav__title" for="nav-10-2">
        Feature Based Molecular Networking
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking/" title="FBMN Workflow" class="md-nav__link">
      FBMN Workflow
    </a>
  </li>

        
          
          
          

  


  <li class="md-nav__item md-nav__item--active">
    
    <input class="md-toggle md-nav__toggle" data-md-toggle="toc" type="checkbox" id="__toc">
    
    
      <label class="md-nav__link md-nav__link--active" for="__toc">
        FBMN with MZmine2
      </label>
    
    <a href="./" title="FBMN with MZmine2" class="md-nav__link md-nav__link--active">
      FBMN with MZmine2
    </a>
    
      
<nav class="md-nav md-nav--secondary">
  
  
  
    <label class="md-nav__title" for="__toc">Table of contents</label>
    <ul class="md-nav__list" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#introduction" class="md-nav__link">
    Introduction
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mass-spectrometry-processing-with-mzmine2" class="md-nav__link">
    Mass spectrometry processing with MZmine2
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#citations-and-development" class="md-nav__link">
    Citations and development
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mass-spectrometry-data-processing-with-mzmine2-step-by-step-documentation" class="md-nav__link">
    Mass Spectrometry Data Processing with MZmine2: step-by-step documentation
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#download-the-mzmine2-software" class="md-nav__link">
    Download the MZmine2 software
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#convert-your-lc-msms-data-to-an-open-format" class="md-nav__link">
    Convert your LC-MS/MS Data to an Open Format
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#processing-steps" class="md-nav__link">
    Processing Steps
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#below-is-the-schematic-representation-of-the-lc-msms-data-processing-steps-with-mzmine2" class="md-nav__link">
    Below is the schematic representation of the LC-MS/MS data processing steps with MZmine2:
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#below-is-the-overview-of-the-lc-msms-data-processing-steps-in-the-mzmine2-batch-mode" class="md-nav__link">
    Below is the overview of the LC-MS/MS data processing steps in the MZmine2 batch mode:
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#batch-import" class="md-nav__link">
    Batch Import
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#processing-steps_1" class="md-nav__link">
    Processing Steps
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#1-import-files" class="md-nav__link">
    1. Import Files
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#2-mass-detection" class="md-nav__link">
    2. Mass Detection
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#3-build-chromatogram-lc-ms-feature-detection-part-1" class="md-nav__link">
    3. Build Chromatogram (LC-MS feature detection part 1)
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#4-deconvolve-the-chromatogram-lc-ms-feature-detection-part-2" class="md-nav__link">
    4. Deconvolve the Chromatogram (LC-MS feature detection part 2)
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#5-group-isotopes-and-co-eluting-ions" class="md-nav__link">
    5. Group isotopes and co-eluting ions
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#6-order-the-peaklists" class="md-nav__link">
    6. Order the peaklists
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#7-lc-ms-feature-alignement-peaklist-alignement" class="md-nav__link">
    7. LC-MS feature alignement (Peaklist alignement)
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#8-detect-missing-peaks-gap-filling-optional" class="md-nav__link">
    8. Detect Missing Peaks / Gap Filling (Optional)
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#9-filter-the-peaklist-to-msms-peaks-optional" class="md-nav__link">
    9. Filter the Peaklist to MS/MS Peaks (Optional)
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#10-use-the-gnps-export-module" class="md-nav__link">
    10. Use the GNPS Export module
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#use-the-dedicated-module-submit-toexport-for-gnps-in-mzmine-to-export-the-needed-file" class="md-nav__link">
    Use the dedicated module "Submit to/Export for GNPS" in MZmine to export the needed file:
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#the-files-can-be-uploaded-to-the-gnps-web-platform-and-feature-based-molecular-networking-job-can-be-directly-launched" class="md-nav__link">
    The files can be uploaded to the GNPS web-platform and Feature-Based Molecular Networking job can be directly launched
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#video-tutorial-quick-mzmine2-export-to-gnps-for-fbmn" class="md-nav__link">
    Video Tutorial - Quick MZMine2 Export to GNPS for FBMN.
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#feature-based-molecular-networking-in-gnps" class="md-nav__link">
    Feature Based Molecular Networking in GNPS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#video-tutorial-analyze-feature-based-molecular-networking-in-gnps" class="md-nav__link">
    Video Tutorial - Analyze Feature Based Molecular Networking in GNPS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#feature-based-molecular-networking-in-cytoscape" class="md-nav__link">
    Feature-Based Molecular Networking in Cytoscape
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#demo-gnps-job-of-feature-based-molecular-networking" class="md-nav__link">
    Demo GNPS job of Feature Based Molecular Networking
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tutorials" class="md-nav__link">
    Tutorials
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#page-contributors" class="md-nav__link">
    Page contributors
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#contribute-to-the-documentation" class="md-nav__link">
    Contribute to the Documentation
  </a>
  
</li>
      
      
      
      
      
    </ul>
  
</nav>
    
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-ms-dial/" title="FBMN with MS-DIAL" class="md-nav__link">
      FBMN with MS-DIAL
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-xcms3/" title="FBMN with XCMS3" class="md-nav__link">
      FBMN with XCMS3
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-openms/" title="FBMN with OpenMS" class="md-nav__link">
      FBMN with OpenMS
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-metaboscape/" title="FBMN with MetaboScape" class="md-nav__link">
      FBMN with MetaboScape
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-progenesisQI/" title="FBMN with Progenesis QI" class="md-nav__link">
      FBMN with Progenesis QI
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-cytoscape/" title="FBMW with Cytoscape" class="md-nav__link">
      FBMW with Cytoscape
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../ms2lda/" title="MS2LDA and MotifDB Substructure Discovery" class="md-nav__link">
      MS2LDA and MotifDB Substructure Discovery
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../cytoscape/" title="Molecular Networking in Cytoscape" class="md-nav__link">
      Molecular Networking in Cytoscape
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../molnetenhancer/" title="MolNetEnhancer" class="md-nav__link">
      MolNetEnhancer
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../nap/" title="Network Annotation Propagation (NAP)" class="md-nav__link">
      Network Annotation Propagation (NAP)
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../ReDU/" title="Reanalysis of Data User Interface (ReDU)" class="md-nav__link">
      Reanalysis of Data User Interface (ReDU)
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-11" type="checkbox" id="nav-11">
    
    <label class="md-nav__link" for="nav-11">
      Spectral Libraries
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-11">
        Spectral Libraries
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../browselibraries/" title="Browsing Reference Spectral Libraries" class="md-nav__link">
      Browsing Reference Spectral Libraries
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../spectrumcuration/" title="Reference Spectral Library Curation" class="md-nav__link">
      Reference Spectral Library Curation
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../batchupload/" title="Reference Spectral Library Batch Addition" class="md-nav__link">
      Reference Spectral Library Batch Addition
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../downloadlibraries/" title="Downloading Reference Spectral Libraries" class="md-nav__link">
      Downloading Reference Spectral Libraries
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-11-5" type="checkbox" id="nav-11-5">
    
    <label class="md-nav__link" for="nav-11-5">
      Open-source Method for Reference Library Generation
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="2">
      <label class="md-nav__title" for="nav-11-5">
        Open-source Method for Reference Library Generation
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../msmschooserdataacquisitionQE/" title="Data Acqusition - Thermo QExactive (v1.0)" class="md-nav__link">
      Data Acqusition - Thermo QExactive (v1.0)
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../msmschooser/" title="Computational Selection of MSMS to Include in Library (MSMS-Chooser)" class="md-nav__link">
      Computational Selection of MSMS to Include in Library (MSMS-Chooser)
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-12" type="checkbox" id="nav-12">
    
    <label class="md-nav__link" for="nav-12">
      Public Datasets
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-12">
        Public Datasets
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../datasets/" title="Dataset Creation/Sharing" class="md-nav__link">
      Dataset Creation/Sharing
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../continuousid/" title="Continuous Identification" class="md-nav__link">
      Continuous Identification
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../moleculeexplorer/" title="Molecule Explorer" class="md-nav__link">
      Molecule Explorer
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-13" type="checkbox" id="nav-13">
    
    <label class="md-nav__link" for="nav-13">
      Community/Social Features
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-13">
        Community/Social Features
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../user/" title="User Page" class="md-nav__link">
      User Page
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../troubleshooting/" title="Troubleshooting" class="md-nav__link">
      Troubleshooting
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-15" type="checkbox" id="nav-15">
    
    <label class="md-nav__link" for="nav-15">
      Tutorial Guides
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-15">
        Tutorial Guides
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorials/bacterianetworking/" title="Molecular Networking - Analog Discovery in Microbial Extracts" class="md-nav__link">
      Molecular Networking - Analog Discovery in Microbial Extracts
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorials/americangutmzmine/" title="Feature Based Molecular Networking with MZmine2 - Microbiome Molecules in Fecal Samples in American Gut Project" class="md-nav__link">
      Feature Based Molecular Networking with MZmine2 - Microbiome Molecules in Fecal Samples in American Gut Project
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorials/americangut-ms-dial/" title="Feature Based Molecular Networking with MSDial - Microbiome Molecules in Fecal Samples in American Gut Project" class="md-nav__link">
      Feature Based Molecular Networking with MSDial - Microbiome Molecules in Fecal Samples in American Gut Project
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorials/coffee-tutorial-metaboscape/" title="Feature Based Molecular Networking with Metaboscape - Coffee Analysis" class="md-nav__link">
      Feature Based Molecular Networking with Metaboscape - Coffee Analysis
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorials/featurebasedgnps/" title="Feature Based Molecular Networking analysis at GNPS" class="md-nav__link">
      Feature Based Molecular Networking analysis at GNPS
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-16" type="checkbox" id="nav-16">
    
    <label class="md-nav__link" for="nav-16">
      Presentation Materials
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-16">
        Presentation Materials
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../logos/" title="Logos" class="md-nav__link">
      Logos
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../changelog/" title="Change Log" class="md-nav__link">
      Change Log
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../contact/" title="Contact" class="md-nav__link">
      Contact
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../workshops/" title="GNPS Workshops" class="md-nav__link">
      GNPS Workshops
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../api/" title="API" class="md-nav__link">
      API
    </a>
  </li>

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              <div class="md-sidebar md-sidebar--secondary" data-md-component="toc">
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    
<nav class="md-nav md-nav--secondary">
  
  
  
    <label class="md-nav__title" for="__toc">Table of contents</label>
    <ul class="md-nav__list" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#introduction" class="md-nav__link">
    Introduction
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mass-spectrometry-processing-with-mzmine2" class="md-nav__link">
    Mass spectrometry processing with MZmine2
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#citations-and-development" class="md-nav__link">
    Citations and development
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mass-spectrometry-data-processing-with-mzmine2-step-by-step-documentation" class="md-nav__link">
    Mass Spectrometry Data Processing with MZmine2: step-by-step documentation
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#download-the-mzmine2-software" class="md-nav__link">
    Download the MZmine2 software
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#convert-your-lc-msms-data-to-an-open-format" class="md-nav__link">
    Convert your LC-MS/MS Data to an Open Format
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#processing-steps" class="md-nav__link">
    Processing Steps
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#below-is-the-schematic-representation-of-the-lc-msms-data-processing-steps-with-mzmine2" class="md-nav__link">
    Below is the schematic representation of the LC-MS/MS data processing steps with MZmine2:
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#below-is-the-overview-of-the-lc-msms-data-processing-steps-in-the-mzmine2-batch-mode" class="md-nav__link">
    Below is the overview of the LC-MS/MS data processing steps in the MZmine2 batch mode:
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#batch-import" class="md-nav__link">
    Batch Import
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#processing-steps_1" class="md-nav__link">
    Processing Steps
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#1-import-files" class="md-nav__link">
    1. Import Files
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#2-mass-detection" class="md-nav__link">
    2. Mass Detection
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#3-build-chromatogram-lc-ms-feature-detection-part-1" class="md-nav__link">
    3. Build Chromatogram (LC-MS feature detection part 1)
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#4-deconvolve-the-chromatogram-lc-ms-feature-detection-part-2" class="md-nav__link">
    4. Deconvolve the Chromatogram (LC-MS feature detection part 2)
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#5-group-isotopes-and-co-eluting-ions" class="md-nav__link">
    5. Group isotopes and co-eluting ions
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#6-order-the-peaklists" class="md-nav__link">
    6. Order the peaklists
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#7-lc-ms-feature-alignement-peaklist-alignement" class="md-nav__link">
    7. LC-MS feature alignement (Peaklist alignement)
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#8-detect-missing-peaks-gap-filling-optional" class="md-nav__link">
    8. Detect Missing Peaks / Gap Filling (Optional)
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#9-filter-the-peaklist-to-msms-peaks-optional" class="md-nav__link">
    9. Filter the Peaklist to MS/MS Peaks (Optional)
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#10-use-the-gnps-export-module" class="md-nav__link">
    10. Use the GNPS Export module
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#use-the-dedicated-module-submit-toexport-for-gnps-in-mzmine-to-export-the-needed-file" class="md-nav__link">
    Use the dedicated module "Submit to/Export for GNPS" in MZmine to export the needed file:
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#the-files-can-be-uploaded-to-the-gnps-web-platform-and-feature-based-molecular-networking-job-can-be-directly-launched" class="md-nav__link">
    The files can be uploaded to the GNPS web-platform and Feature-Based Molecular Networking job can be directly launched
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#video-tutorial-quick-mzmine2-export-to-gnps-for-fbmn" class="md-nav__link">
    Video Tutorial - Quick MZMine2 Export to GNPS for FBMN.
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#feature-based-molecular-networking-in-gnps" class="md-nav__link">
    Feature Based Molecular Networking in GNPS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#video-tutorial-analyze-feature-based-molecular-networking-in-gnps" class="md-nav__link">
    Video Tutorial - Analyze Feature Based Molecular Networking in GNPS
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#feature-based-molecular-networking-in-cytoscape" class="md-nav__link">
    Feature-Based Molecular Networking in Cytoscape
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#demo-gnps-job-of-feature-based-molecular-networking" class="md-nav__link">
    Demo GNPS job of Feature Based Molecular Networking
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tutorials" class="md-nav__link">
    Tutorials
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#page-contributors" class="md-nav__link">
    Page contributors
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#contribute-to-the-documentation" class="md-nav__link">
    Contribute to the Documentation
  </a>
  
</li>
      
      
      
      
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          <div class="md-content">
            <article class="md-content__inner md-typeset">
              
                
                  <a href="https://github.com/CCMS-UCSD/GNPSDocumentation/blob/master/docs/featurebasedmolecularnetworking-with-mzmine2.md" title="Edit this page" class="md-icon md-content__icon">&#xE3C9;</a>
                
                
                  <h1>FBMN with MZmine2</h1>
                
                <h2 id="introduction">Introduction</h2>
<p><strong>Feature-Based Molecular Networking</strong> (FBMN) is a computational method that bridges popular mass spectrometry data processing tools for LC-MS/MS and molecular networking analysis on <a href="http://gnps.ucsd.edu">GNPS</a>. The supported tools are: <a href="./">MZmine2</a>, <a href="../featurebasedmolecularnetworking-with-openms/">OpenMS</a>, <a href="../featurebasedmolecularnetworking-with-ms-dial/">MS-DIAL</a>, <a href="../featurebasedmolecularnetworking-with-metaboscape/">MetaboScape</a>, <a href="../featurebasedmolecularnetworking-with-xcms3/">XCMS</a>, and <a href="../featurebasedmolecularnetworking-with-progenesisQI/">Progenesis QI</a>.</p>
<p>The main documentation for Feature-Based Molecular Networking <a href="../featurebasedmolecularnetworking/">can be accessed here:</a></p>
<p>Below we describe how to use MZmine2 with the FBMN workflow on GNPS.</p>
<h2 id="mass-spectrometry-processing-with-mzmine2">Mass spectrometry processing with MZmine2</h2>
<p>Download the latest version of MZmine2 software (version MZmine v2.33 minimum) at <a href="https://github.com/mzmine/mzmine2/releases">https://github.com/mzmine/mzmine2/releases</a>.</p>
<h3 id="citations-and-development">Citations and development</h3>
<p>This work builds on the efforts of our many colleagues, please cite their work:</p>
<p>Katajamaa, M., Miettinen, J. &amp; Oresic, M. <a href="https://doi.org/10.1093/bioinformatics/btk039">MZmine: toolbox for processing and visualization of mass spectrometry based molecular profile data.</a> Bioinformatics 22, 634–636 (2006).</p>
<p>Pluskal, T., Castillo, S., Villar-Briones, A. &amp; Oresic, <a href="https://doi.org/10.1186/1471-2105-11-395">M. MZmine 2: modular framework for processing, visualizing, and analyzing mass spectrometry-based molecular profile data.</a> BMC Bioinformatics 11, 395 (2010).</p>
<p>Wang, M. et al. <a href="https://doi.org/10.1038/nbt.3597">Sharing and community curation of mass spectrometry data with Global Natural Products Social Molecular Networking</a>. Nat. Biotechnol. 34, 828–837 (2016)</p>
<p>The development of the features used in the pipeline is <a href="https://github.com/mzmine/mzmine2">publicly accessible here</a>.</p>
<h3 id="mass-spectrometry-data-processing-with-mzmine2-step-by-step-documentation">Mass Spectrometry Data Processing with MZmine2: step-by-step documentation</h3>
<p>In MZmine2, a sequence of steps are performed to process the mass spectrometry data. Here we will present key steps required to process LC-MS/MS data acquired in non-targeted mode (data dependent acquisition). For convenience we also provide a batch file (XML format) that can be imported directly in MZmine2.</p>
<p><strong>IMPORTANT:</strong> MZmine2 parameters will vary depending on the instrument used, the acquisition parameters, and samples studied. The following documentation serves a basic guideline for using MZmine2 with the Feature-Based Molecular Networking workflow.</p>
<p>Please consult the resources below for more details on MZmine2 processing:</p>
<ul>
<li>The official documentation <a href="http://mzmine.github.io/documentation.html">http://mzmine.github.io/documentation.html</a>,</li>
<li>The <a href="http://www.pharmacognosie-parisdescartes.fr/pdf/150420_MZmine_Tutorial_UNIGE.pdf">MZmine tutorial</a> by Pierre-Marie Allard and Joelle Houriet from the University of Geneva.</li>
<li>The video tutorial about <a href="../tutorials/americangutmzmine/">MZmine2 processing for Feature Based Molecular Networking</a>. <strong>Note that this video is slighlty outdatted</strong>, so please refer to the steps described in this documentation.</li>
</ul>
<iframe width="700" height="400" src="https://www.youtube.com/embed/5jjMllbwD-U"> </iframe>

<ul>
<li>The video tutorial about [Quick MZMine2 Export to GNPS for Feature Based Molecular Networking]</li>
</ul>
<iframe width="800" height="400" src="https://www.youtube.com/embed/vFcGG7T_44E"> </iframe>

<h3 id="download-the-mzmine2-software">Download the MZmine2 software</h3>
<p>Download the latest version of MZmine2 software (version MZmine v2.33 minimum) at <a href="https://github.com/mzmine/mzmine2/releases">https://github.com/mzmine/mzmine2/releases</a>.</p>
<h3 id="convert-your-lc-msms-data-to-an-open-format">Convert your LC-MS/MS Data to an Open Format</h3>
<p>MZmine2 accepts different input formats. Note that we recommand to first convert your files to mzML format before doing MZmine2 processing. <a href="../fileconversion/">See the documentation here</a>.</p>
<h4 id="processing-steps">Processing Steps</h4>
<h5 id="below-is-the-schematic-representation-of-the-lc-msms-data-processing-steps-with-mzmine2">Below is the schematic representation of the LC-MS/MS data processing steps with MZmine2:</h5>
<p><img alt="complete workflow view" src="../img/mzmine/Workflow_mzmine.png" /></p>
<h5 id="below-is-the-overview-of-the-lc-msms-data-processing-steps-in-the-mzmine2-batch-mode">Below is the overview of the LC-MS/MS data processing steps in the MZmine2 batch mode:</h5>
<p><img alt="img" src="../img/mzmine/batch_overview.png" /></p>
<h4 id="batch-import">Batch Import</h4>
<p>Here are some MZmine2 batch that are compatible with the Feature-Based Molecular Networking workflow. These batch files can be imported into MZMine2 (Batch mode):</p>
<table>
<thead>
<tr>
<th>Instrument</th>
<th>Gradient Length</th>
<th>Matrix Type</th>
<th>Sample Size</th>
<th>Download</th>
</tr>
</thead>
<tbody>
<tr>
<td>Bruker Maxis HD qTof</td>
<td>10 Min</td>
<td>Stool</td>
<td>20</td>
<td><a href="../static/maxis_12min_stool_20.xml">Batch</a></td>
</tr>
</tbody>
</table>
<h4 id="processing-steps_1">Processing Steps</h4>
<p>Below is a walk-through of all the steps</p>
<h4 id="1-import-files">1. Import Files</h4>
<p>Go to Menu: Raw data methods &gt; Raw data import &gt; Select the files</p>
<p><img alt="img" src="../img/mzmine/2_import-raw.png" /></p>
<h4 id="2-mass-detection">2. Mass Detection</h4>
<p>This step creates mass lists from your raw data.</p>
<p>Perform mass detection on MS level 1: Menu: Raw data methods &gt; Mass detection &gt; Set filter : MS level 1</p>
<p><strong>IMPORTANT</strong> Set an appropriate intensity threshold. You can use the preview window to assess the right threshold on your data. As a rule of thumb, the value should at least correspond to the minimum value set for the triggering of the MS2 scan event. (Example: MAXIS-QTOF: 1E3, Q-Exactive 1E4)</p>
<p>Perform mass detection on MS level 2. The same mass list name must be used.</p>
<p>Go to Menu: Raw data methods &gt; Mass detection &gt; Set filter : MS level 2.</p>
<p><strong>IMPORTANT:</strong> Make sure to set an intensity threshold representative of noise level in the MS2 spectra. This is typically lower than for MS1. (Example: maXis QTOF: 1E2; LTQ-XL Orbitrap 1E4, Q-Exactive: 0). If you have any doubt, set it to 0.</p>
<p><img alt="img" src="../img/mzmine/mass_detection_ms2.png" /></p>
<h4 id="3-build-chromatogram-lc-ms-feature-detection-part-1">3. Build Chromatogram (LC-MS feature detection part 1)</h4>
<p>Go to Menu: Raw data methods / Chromatogram builder</p>
<h4 id="4-deconvolve-the-chromatogram-lc-ms-feature-detection-part-2">4. Deconvolve the Chromatogram (LC-MS feature detection part 2)</h4>
<p>Go to Menu: Peak list methods &gt; Peak detection &gt; Chromatogram deconvolution</p>
<p><strong>IMPORTANT:</strong> tick both options "m/z range for MS2 scan pairing (Da)" and "RT range for MS2 scan pairing (min)". The values have to be defined according to your experimental setup (expected MS mass accuracy and chromatographic peak width).</p>
<p>Example for a UHPLC colum (1.7 µm C18, 50 × 2.1 mm, flow rate of 0.5 mL/min):</p>
<ul>
<li>
<p>maXis-QTOF: 12 min gradient, 0.02 Da and 0.15 min</p>
</li>
<li>
<p>Q-Exactive: 5 min gradient, 0.01 Da and 0.1 min</p>
</li>
</ul>
<p><img alt="img" src="../img/mzmine/deconvolve.png" /></p>
<h4 id="5-group-isotopes-and-co-eluting-ions">5. Group isotopes and co-eluting ions</h4>
<p>Use the "Isotopic peaks grouper" [recommended] or other alternative (such as the CAMERA module).</p>
<p>Go to Menu: Peak list methods / Isotopes / Isotopic peaks grouper.</p>
<p><strong>IMPORTANT:</strong>  This depends on your expected peak shapes, duty cycle time and the MS mass accuracy. (Example: MAXIS-QTOF, 10 min gradient, 0.1 min, 0.02 m/z; Q-Exactive, 5 min gradient, 0.05 min, 0.01 m/z)</p>
<h4 id="6-order-the-peaklists">6. Order the peaklists</h4>
<p>Go to Menu: Peak list methods &gt; Order peak lists.</p>
<p><strong>IMPORTANT:</strong> This is to ensure the reproducibility of the processing Indeed, the aligned peak list will change slighlty if that step is not performed. </p>
<h4 id="7-lc-ms-feature-alignement-peaklist-alignement">7. LC-MS feature alignement (Peaklist alignement)</h4>
<p>In this step, the peak lists from each sample will be aligned in one aligned peak list. The alignement is performed iteratively using the first peak list selected (see MZmine documentation). For that reason, make sure the first sample is adapted (not a negative control) or to manually put an representative peaklist in the first position.</p>
<p>Go to Menu: Peak list methods &gt; Alignment &gt; Join aligner</p>
<h4 id="8-detect-missing-peaks-gap-filling-optional">8. Detect Missing Peaks / Gap Filling (Optional)</h4>
<p>Gap filling enables to retrieve the intensity of a peak in all the samples, even if it was not detected in a previous processing step.
Go to Menu: Peak list methods / Gap filling / Peak finder (multi-threaded).</p>
<p><strong>IMPORTANT:</strong> This step is optional. Use the multi-threaded peak finder for fast processing.</p>
<h4 id="9-filter-the-peaklist-to-msms-peaks-optional">9. Filter the Peaklist to MS/MS Peaks (Optional)</h4>
<p>Depending on the number of features in the aligned peaklist, it is possible to filter the peaklist to keep only features with minimum number of occurences ("Minimum peaks in a row") or a mininum number of isotopic peaks for the feature ("Minimum peaks in an isotope pattern"), or to "Keep only peaks with MS2 scan (GNPS)".</p>
<p>Go to Menu: Peak list methods &gt; Filtering &gt; Peak list row filter &gt; Select the filters</p>
<p><strong>IMPORTANT:</strong> if you use a filter, we recommend using the filter "Reset the peak number ID"</p>
<p><strong>IMPORTANT</strong> Note that this step was mandatory in the previous version of FBMN with MZmine2, now the filter "Keep only peaks with MS2 scan (GNPS)" is optional.</p>
<p><img alt="img" src="../img/mzmine/ms2_filtering.png" /></p>
<h4 id="10-use-the-gnps-export-module">10. Use the GNPS Export module</h4>
<h5 id="use-the-dedicated-module-submit-toexport-for-gnps-in-mzmine-to-export-the-needed-file">Use the dedicated module "Submit to/Export for GNPS" in MZmine to export the needed file:</h5>
<ul>
<li>
<p>a <em>feature quantification table</em> with LC-MS feature intensities (.CSV file format)</p>
</li>
<li>
<p>a <em>MS/MS spectral summary file</em>, with a representative MS/MS spectrum per LC-MS feature. List of the MS/MS spectra (.MGF file format) (the most intense MS/MS per feature is selected).</p>
</li>
</ul>
<p>Select the last "filtered aligned peaklist" and Go to Menu: Peak list methods &gt; Export &gt; Export for/Submit to GNPS</p>
<p><img alt="img" src="/img/mzmine/gnps_export_module_V2.png" /></p>
<p>See an example of files generated by the export module using the workflow:
<a href="https://github.com/CCMS-UCSD/GNPSDocumentation/tree/master/docs/tutorials/AG_tutorial_files">here</a>.</p>
<h5 id="the-files-can-be-uploaded-to-the-gnps-web-platform-and-feature-based-molecular-networking-job-can-be-directly-launched">The files can be uploaded to the GNPS web-platform and Feature-Based Molecular Networking job can be directly launched</h5>
<p><strong>IMPORTANT:</strong> While the possibility to submit the files directly to GNPS and launch a FBMN job on the fly is really convenient for quick data analysis, the job and files will not be saved in your personal account on GNPS if you do not provide username/password, and you are limited to basic presets of parameters. For that reason, we recommend to upload your files with the FTP uploader <a href="../fileupload/">(see documentation)</a> and prepare your job <a href="https://gnps.ucsd.edu/ProteoSAFe/index.jsp?params=%7B%22workflow%22:%22FEATURE-BASED-MOLECULAR-NETWORKING%22,%22library_on_server%22:%22d.speclibs;%22%7D">directly on GNPS</a> (you must be logged in first), or alternatively to clone the submitted job.</p>
<p><img alt="img" src="/img/mzmine/gnps_export_module_V2_submission.png" /></p>
<p>In the "Export for/Submit to GNPS" module, select the option: "Submit to GNPS"</p>
<ul>
<li>
<p>[Optional] Metadata file: specify the path to the metadata table in GNPS format. [See documentation here] (networking.md/#metadata)</p>
</li>
<li>
<p>Select the parameters presets for the GNPS job.</p>
</li>
<li>
<p>Email: specify the email to forward the job link</p>
</li>
<li>
<p>Annotation edges (Experimental): that option can be ignored.</p>
</li>
<li>
<p>Open website: if ticked, will open the job webpage.</p>
</li>
</ul>
<p>ADDITIONAL NOTES: The feature table must contain at least the row ID, the row m/z, and row retention time, along with the sample columns. It is currently mandatory for the sample name headers to have the following format: "filename Peak area". Depending on the steps used in MZmine the sample name header can be "filename baseline-corrected Peak area", but this has to be changed back to "filename Peak area".</p>
<h3 id="video-tutorial-quick-mzmine2-export-to-gnps-for-fbmn">Video Tutorial - Quick MZMine2 Export to GNPS for FBMN.</h3>
<p>The workflow for Feature Based Molecular Networking in GNPS is different from the classic molecular networking workflow. <a href="https://gnps.ucsd.edu/ProteoSAFe/index.jsp?params=%7B%22workflow%22:%22FEATURE-BASED-MOLECULAR-NETWORKING%22,%22library_on_server%22:%22d.speclibs;%22%7D">Access the FBMN workflow here</a> (You need to be logged in first !)</p>
<iframe width="800" height="500" src="https://www.youtube.com/embed/vFcGG7T_44E"> </iframe>

<h3 id="feature-based-molecular-networking-in-gnps">Feature Based Molecular Networking in GNPS</h3>
<p>The main documentation of the Feature Based Molecular Networking workflow on GNPS <a href="../featurebasedmolecularnetworking/">can be consulted on that page</a>. The workflow for Feature Based Molecular Networking in GNPS is different from the "classic" molecular networking workflow. <a href="https://gnps.ucsd.edu/ProteoSAFe/index.jsp?params=%7B%22workflow%22:%22METABOLOMICS-SNETS-MZMINE%22,%22library_on_server%22:%22d.speclibs;%22%7D">Access the FBMN workflow here</a> (You need to be logged in first !).</p>
<p>Basically, you will need to upload the files produced by MZmine2 (test files are accessible <a href="https://github.com/CCMS-UCSD/GNPSDocumentation/tree/master/docs/tutorials/AG_tutorial_files">here</a>):</p>
<ol>
<li>A <em>feature quantification table</em> (.CSV file format).</li>
<li>A <em>MS/MS spectral summary</em> (.MGF file format)</li>
<li>A <em>metadata table</em> - described <a href="../networking/#metadata">here</a></li>
</ol>
<p>There are several additional normalization options specifically for feature detection. We can normalize the features per LC/MS run and aggregate by groups with either a sum or average (recommended).</p>
<p><img alt="img" src="../img/mzmine/quant_options.png" /></p>
<h3 id="video-tutorial-analyze-feature-based-molecular-networking-in-gnps">Video Tutorial - Analyze Feature Based Molecular Networking in GNPS</h3>
<p>This video presents 
<iframe width="800" height="500" src="https://www.youtube.com/embed/NTkQ0fS1aug"> </iframe></p>
<h3 id="feature-based-molecular-networking-in-cytoscape">Feature-Based Molecular Networking in Cytoscape</h3>
<p>Cytoscape is an open source software platform used to visualize, analyze and annotate molecular networks from GNPS. <a href="../featurebasedmolecularnetworking-cytoscape/">See the documentation here</a></p>
<h3 id="demo-gnps-job-of-feature-based-molecular-networking">Demo GNPS job of Feature Based Molecular Networking</h3>
<p><a href="https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=52a390c8eb654b7fa8d61a1c7a4aaab5">Here is an example FBMN</a> job with files resulting from MZmine2 processing of a subset of the [American Gut Project] (http://humanfoodproject.com/americangut/).</p>
<h2 id="tutorials">Tutorials</h2>
<p>See our <a href="../tutorials/americangutmzmine/">tutorial on using MZmine2</a> for FBMN analysis of a cohort from the [American Gut Project] (http://humanfoodproject.com/americangut/), and our <a href="../tutorials/featurebasedgnps/">tutorial on running a FBMN analysis on GNP</a>.</p>
<h2 id="page-contributors">Page contributors</h2>
<p>Louis Felix Nothias (UCSD), Daniel Petras (UCSD), Ming Wang (UCSD), Ivan Protsyuk (EMBL, Heidelberg, Germany).</p>
<h2 id="contribute-to-the-documentation">Contribute to the Documentation</h2>
<ul>
<li>For informations/feature request, please open an "Issue" on the <a href="(https://github.com/CCMS-UCSD/GNPSDocumentation)"><em>CCMS-UCSD/GNPSDocumentation</em></a> GitHub repository.</li>
<li>To contribute directly to the GNPS documentation, fork the <a href="(https://github.com/CCMS-UCSD/GNPSDocumentation)"><em>CCMS-UCSD/GNPSDocumentation</em></a> repository, and make a "Pull Request".</li>
</ul>
                
                  
                
              
              
                


              
            </article>
          </div>
        </div>
      </main>
      
        
<footer class="md-footer">
  
    <div class="md-footer-nav">
      <nav class="md-footer-nav__inner md-grid">
        
          <a href="../featurebasedmolecularnetworking/" title="FBMN Workflow" class="md-flex md-footer-nav__link md-footer-nav__link--prev" rel="prev">
            <div class="md-flex__cell md-flex__cell--shrink">
              <i class="md-icon md-icon--arrow-back md-footer-nav__button"></i>
            </div>
            <div class="md-flex__cell md-flex__cell--stretch md-footer-nav__title">
              <span class="md-flex__ellipsis">
                <span class="md-footer-nav__direction">
                  Previous
                </span>
                FBMN Workflow
              </span>
            </div>
          </a>
        
        
          <a href="../featurebasedmolecularnetworking-with-ms-dial/" title="FBMN with MS-DIAL" class="md-flex md-footer-nav__link md-footer-nav__link--next" rel="next">
            <div class="md-flex__cell md-flex__cell--stretch md-footer-nav__title">
              <span class="md-flex__ellipsis">
                <span class="md-footer-nav__direction">
                  Next
                </span>
                FBMN with MS-DIAL
              </span>
            </div>
            <div class="md-flex__cell md-flex__cell--shrink">
              <i class="md-icon md-icon--arrow-forward md-footer-nav__button"></i>
            </div>
          </a>
        
      </nav>
    </div>
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-footer-copyright">
        
        powered by
        <a href="https://www.mkdocs.org">MkDocs</a>
        and
        <a href="https://squidfunk.github.io/mkdocs-material/">
          Material for MkDocs</a>
      </div>
      
    </div>
  </div>
</footer>
      
    </div>
    
      <script src="../assets/javascripts/application.c648116f.js"></script>
      
      <script>app.initialize({version:"1.0.4",url:{base:".."}})</script>
      
    
  </body>
</html>