
<!DOCTYPE html>

<html class="no-js" lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width,initial-scale=1" name="viewport"/>
<link href="https://ccms-ucsd.github.io/GNPSDocumentation/featurebasedmolecularnetworking-with-mzmine2/" rel="canonical"/>
<link href="../assets/images/favicon.png" rel="icon"/>
<meta content="mkdocs-1.2.3, mkdocs-material-8.2.1" name="generator"/>
<title>FBMN with MZmine - GNPS Documentation</title>
<link href="../assets/stylesheets/main.e8d9bf0c.min.css" rel="stylesheet"/>
<link href="../assets/stylesheets/palette.e6a45f82.min.css" rel="stylesheet"/>
<link crossorigin="" href="https://fonts.gstatic.com" rel="preconnect"/>
<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700%7CRoboto+Mono&amp;display=fallback" rel="stylesheet"/>
<style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
<script>__md_scope=new URL("..",location),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
<script>window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)},ga.l=+new Date,ga("create","UA-8412213-5","auto"),ga("set","anonymizeIp",!0),ga("send","pageview"),document.addEventListener("DOMContentLoaded",function(){document.forms.search&&document.forms.search.query.addEventListener("blur",function(){var e;this.value&&(e=document.location.pathname,ga("send","pageview",e+"?q="+this.value))}),"undefined"!=typeof location$&&location$.subscribe(function(e){ga("send","pageview",e.pathname)})})</script>
<script async="" src="https://www.google-analytics.com/analytics.js"></script>
</head>
<body data-md-color-accent="none" data-md-color-primary="none" data-md-color-scheme="" dir="ltr">
<input autocomplete="off" class="md-toggle" data-md-toggle="drawer" id="__drawer" type="checkbox"/>
<input autocomplete="off" class="md-toggle" data-md-toggle="search" id="__search" type="checkbox"/>
<label class="md-overlay" for="__drawer"></label>
<div data-md-component="skip">
<a class="md-skip" href="#introduction">
          Skip to content
        </a>
</div>
<div data-md-component="announce">
</div>
<header class="md-header" data-md-component="header">
<nav aria-label="Header" class="md-header__inner md-grid">
<a aria-label="GNPS Documentation" class="md-header__button md-logo" data-md-component="logo" href=".." title="GNPS Documentation">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"></path></svg>
</a>
<label class="md-header__button md-icon" for="__drawer">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2z"></path></svg>
</label>
<div class="md-header__title" data-md-component="header-title">
<div class="md-header__ellipsis">
<div class="md-header__topic">
<span class="md-ellipsis">
            GNPS Documentation
          </span>
</div>
<div class="md-header__topic" data-md-component="header-topic">
<span class="md-ellipsis">
            
              FBMN with MZmine
            
          </span>
</div>
</div>
</div>
<label class="md-header__button md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"></path></svg>
</label>
<div class="md-search" data-md-component="search" role="dialog">
<label class="md-search__overlay" for="__search"></label>
<div class="md-search__inner" role="search">
<form class="md-search__form" name="search">
<input aria-label="Search" autocapitalize="off" autocomplete="off" autocorrect="off" class="md-search__input" data-md-component="search-query" name="query" placeholder="Search" required="" spellcheck="false" type="text"/>
<label class="md-search__icon md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"></path></svg>
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"></path></svg>
</label>
<nav aria-label="Search" class="md-search__options">
<button aria-label="Clear" class="md-search__icon md-icon" tabindex="-1" type="reset">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"></path></svg>
</button>
</nav>
</form>
<div class="md-search__output">
<div class="md-search__scrollwrap" data-md-scrollfix="">
<div class="md-search-result" data-md-component="search-result">
<div class="md-search-result__meta">
            Initializing search
          </div>
<ol class="md-search-result__list"></ol>
</div>
</div>
</div>
</div>
</div>
<div class="md-header__source">
<a class="md-source" data-md-component="source" href="https://github.com/CCMS-UCSD/GNPSDocumentation/" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><!--! Font Awesome Free 6.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    CCMS-UCSD/GNPSDocumentation
  </div>
</a>
</div>
</nav>
</header>
<div class="md-container" data-md-component="container">
<main class="md-main" data-md-component="main">
<div class="md-main__inner md-grid">
<div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Navigation" class="md-nav md-nav--primary" data-md-level="0">
<label class="md-nav__title" for="__drawer">
<a aria-label="GNPS Documentation" class="md-nav__button md-logo" data-md-component="logo" href=".." title="GNPS Documentation">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"></path></svg>
</a>
    GNPS Documentation
  </label>
<div class="md-nav__source">
<a class="md-source" data-md-component="source" href="https://github.com/CCMS-UCSD/GNPSDocumentation/" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><!--! Font Awesome Free 6.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    CCMS-UCSD/GNPSDocumentation
  </div>
</a>
</div>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="..">
        GNPS Introduction
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../isgnpsright/">
        Is my data compatible with GNPS?
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../faq/">
        Frequently Asked Questions
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../superquickstart/">
        Superquick Start Guide
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../quickstart/">
        Quick Start Guide
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gnpsanalysisoverview/">
        GNPS Analysis Tools Overview
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../lexicon/">
        GNPS Lexicon
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_8" id="__nav_8" type="checkbox"/>
<label class="md-nav__link" for="__nav_8">
          Mass Spec/Computational Background
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Mass Spec/Computational Background" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_8">
<span class="md-nav__icon md-icon"></span>
          Mass Spec/Computational Background
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../massspecbackground/networkingtheory/">
        Molecular Networking
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_9" id="__nav_9" type="checkbox"/>
<label class="md-nav__link" for="__nav_9">
          Data Preparation/Upload
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Data Preparation/Upload" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_9">
<span class="md-nav__icon md-icon"></span>
          Data Preparation/Upload
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../fileconversion/">
        Mass Spectrometry File Conversion
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fileconversion_waters/">
        File Conversion (Waters)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fileupload/">
        GNPS File Upload
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../metadata/">
        Metadata
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_10" id="__nav_10" type="checkbox"/>
<label class="md-nav__link" for="__nav_10">
          Recommended Data Analysis
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Recommended Data Analysis" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_10">
<span class="md-nav__icon md-icon"></span>
          Recommended Data Analysis
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../networking/">
        Molecular Networking
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_10_2" id="__nav_10_2" type="checkbox"/>
<label class="md-nav__link" for="__nav_10_2">
          GC-MS Molecular Networking
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="GC-MS Molecular Networking" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_10_2">
<span class="md-nav__icon md-icon"></span>
          GC-MS Molecular Networking
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../gcanalysis/">
        GC-MS on GNPS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gc-ms-deconvolution/">
        GC-MS Deconvolution
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gc-ms-library-molecular-network/">
        GC-MS Library Search and Molecular Networking
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../networkingviews/">
        Molecular Networking Views
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../librarysearch/">
        Spectral Library Search
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../masst/">
        MASST Search (Mass Search Tool)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../resultviews/">
        Result Views Navigation
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_11" id="__nav_11" type="checkbox"/>
<label class="md-nav__link" for="__nav_11">
          Mass Spec Data Visualization
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Mass Spec Data Visualization" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_11">
<span class="md-nav__icon md-icon"></span>
          Mass Spec Data Visualization
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../lcms-dashboard/">
        GNPS Dashboard
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gnpsdatasetexplorer/">
        GNPS Dataset Explorer
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../usi/">
        Metabolomics Spectrum Resolver
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../dashboard_use_cases/">
        GNPS Dashboard Use Cases
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--active md-nav__item--nested">
<input checked="" class="md-nav__toggle md-toggle" data-md-toggle="__nav_12" id="__nav_12" type="checkbox"/>
<label class="md-nav__link" for="__nav_12">
          Advanced Data Analysis
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Advanced Data Analysis" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_12">
<span class="md-nav__icon md-icon"></span>
          Advanced Data Analysis
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../networkingfiles/">
        Molecular Networking Files
      </a>
</li>
<li class="md-nav__item md-nav__item--active md-nav__item--nested">
<input checked="" class="md-nav__toggle md-toggle" data-md-toggle="__nav_12_2" id="__nav_12_2" type="checkbox"/>
<label class="md-nav__link" for="__nav_12_2">
          Feature Based Molecular Networking
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Feature Based Molecular Networking" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_12_2">
<span class="md-nav__icon md-icon"></span>
          Feature Based Molecular Networking
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking/">
        FBMN Workflow
      </a>
</li>
<li class="md-nav__item md-nav__item--active">
<input class="md-nav__toggle md-toggle" data-md-toggle="toc" id="__toc" type="checkbox"/>
<label class="md-nav__link md-nav__link--active" for="__toc">
          FBMN with MZmine
          <span class="md-nav__icon md-icon"></span>
</label>
<a class="md-nav__link md-nav__link--active" href="./">
        FBMN with MZmine
      </a>
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#introduction">
    Introduction
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#mass-spectrometry-processing-with-mzmine">
    Mass spectrometry processing with MZmine
  </a>
<nav aria-label="Mass spectrometry processing with MZmine" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#citations-and-development">
    Citations and development
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#installation">
    Installation
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#data-processing-with-mzmine-for-fbmn">
    Data Processing with MZmine for FBMN
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#convert-your-lc-msms-data-to-an-open-format">
    Convert your LC-MS/MS Data to an Open Format
  </a>
<nav aria-label="Convert your LC-MS/MS Data to an Open Format" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#processing-steps">
    Processing Steps
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#batch-import">
    Batch Import
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#processing-steps_1">
    Processing Steps
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#1-import-files">
    1. Import Files
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#2-mass-detection">
    2. Mass Detection
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#3-build-chromatogram-lc-ms-feature-detection-part-1">
    3. Build Chromatogram (LC-MS feature detection part 1)
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#4-deconvolve-the-chromatogram-lc-ms-feature-detection-part-2">
    4. Deconvolve the Chromatogram (LC-MS feature detection part 2)
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#5-group-isotopes-and-co-eluting-ions">
    5. Group isotopes and co-eluting ions
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#6-order-the-peaklists">
    6. Order the peaklists
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#7-lc-ms-feature-alignement-peaklist-alignement">
    7. LC-MS feature alignement (Peaklist alignement)
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#8-optional-detect-missing-peaks-gap-filling">
    8. (Optional) Detect Missing Peaks / Gap Filling
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#9-optional-filter-the-peaklist-to-msms-peaks">
    9. (Optional) Filter the Peaklist to MS/MS Peaks
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#10-use-the-gnps-export-module">
    10. Use the GNPS Export module
  </a>
<nav aria-label="10. Use the GNPS Export module" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#the-files-can-be-uploaded-to-the-gnps-web-platform-and-feature-based-molecular-networking-job-can-be-directly-launched">
    The files can be uploaded to the GNPS web-platform and Feature-Based Molecular Networking job can be directly launched
  </a>
</li>
</ul>
</nav>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#video-tutorial-quick-mzmine-export-to-gnps-for-fbmn">
    Video Tutorial - Quick MZMine Export to GNPS for FBMN.
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#fbmn-in-gnps">
    FBMN in GNPS
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#video-tutorial-analyze-fbmn-jobs-in-gnps">
    Video Tutorial - Analyze FBMN jobs in GNPS
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#fbmn-in-cytoscape">
    FBMN in Cytoscape
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#tutorials">
    Tutorials
  </a>
<nav aria-label="Tutorials" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#join-the-gnps-community">
    Join the GNPS Community !
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#page-contributors">
    Page Contributors
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-ms-dial/">
        FBMN with MS-DIAL
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-xcms3/">
        FBMN with XCMS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-openms/">
        FBMN with OpenMS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-metaboscape/">
        FBMN with MetaboScape
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-progenesisQI/">
        FBMN with Progenesis QI
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-mztab-m/">
        FBMN with mzTab-M
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-cytoscape/">
        FBMN export to Cytoscape
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_12_3" id="__nav_12_3" type="checkbox"/>
<label class="md-nav__link" for="__nav_12_3">
          FBMN - Ion Identity Networking
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="FBMN - Ion Identity Networking" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_12_3">
<span class="md-nav__icon md-icon"></span>
          FBMN - Ion Identity Networking
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin/">
        IIN Workflow
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin-mzmine/">
        IIN with MZmine2
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin-xcms/">
        IIN with XCMS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin-msdial/">
        IIN with MS-DIAL
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../ms2lda/">
        MS2LDA and MotifDB Substructure Discovery
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../cytoscape/">
        Molecular Networking in Cytoscape
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../dereplicator/">
        DEREPLICATOR
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../dereplicator%2B/">
        DEREPLICATOR+
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../molDiscovery/">
        MolDiscovery
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../nap/">
        Network Annotation Propagation (NAP)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../ms2lda/">
        MS2LDA and MotifDB Substructure Discovery
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../molnetenhancer/">
        MolNetEnhancer
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../metaminer/">
        MetaMiner
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../sirius/">
        Sirius
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../ReDU/">
        Reanalysis of Data User Interface (ReDU)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../qemistree/">
        Qemistree
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../mmvec/">
        MMVEC
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../mergepolarity/">
        Merge Network Polarity
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../chemdir/">
        Chemical Directionality
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_13" id="__nav_13" type="checkbox"/>
<label class="md-nav__link" for="__nav_13">
          Spectral Libraries
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Spectral Libraries" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_13">
<span class="md-nav__icon md-icon"></span>
          Spectral Libraries
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../browselibraries/">
        Browsing Reference Spectral Libraries
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../spectrumcuration/">
        Add/Curate Reference Spectrum in the Library
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../batchupload/">
        Reference Spectral Library Batch Addition
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../batchuploadmzvault/">
        Reference Spectral Library mzVault Addition
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../downloadlibraries/">
        Downloading Reference Spectral Libraries
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_13_6" id="__nav_13_6" type="checkbox"/>
<label class="md-nav__link" for="__nav_13_6">
          Open-source Method for Reference Library Generation
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Open-source Method for Reference Library Generation" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_13_6">
<span class="md-nav__icon md-icon"></span>
          Open-source Method for Reference Library Generation
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../msmschooser/">
        MSMS-Chooser Workflow
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../msmschooserdataacquisitioncomputational/">
        Computational Data Acquisition Prep
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../msmschooserdataacquisitionQE/">
        Data Acqusition - Thermo QExactive (v1.0)
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tag_info/">
        GNPS Tag Contribution and Analyses
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_14" id="__nav_14" type="checkbox"/>
<label class="md-nav__link" for="__nav_14">
          Create a Public Datasets
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Create a Public Datasets" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_14">
<span class="md-nav__icon md-icon"></span>
          Create a Public Datasets
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../datasets/">
        Dataset Creation/Sharing
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../continuousid/">
        Continuous Identification
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../moleculeexplorer/">
        Molecule Explorer
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_15" id="__nav_15" type="checkbox"/>
<label class="md-nav__link" for="__nav_15">
          Community/Social Features
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Community/Social Features" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_15">
<span class="md-nav__icon md-icon"></span>
          Community/Social Features
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../user/">
        User Page
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../troubleshooting/">
        Troubleshooting
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_17" id="__nav_17" type="checkbox"/>
<label class="md-nav__link" for="__nav_17">
          Tutorial Guides
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Tutorial Guides" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_17">
<span class="md-nav__icon md-icon"></span>
          Tutorial Guides
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/bacterianetworking/">
        Molecular Networking - Analog Discovery in Microbial Extracts
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/americangutmzmine/">
        Feature-Based Molecular Networking (FBMN) with MZmine2 - Microbiome Molecules in Fecal Samples in American Gut Project
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/americangut-ms-dial/">
        FBMN with MS-DIAL - Microbiome Molecules in Fecal Samples in American Gut Project
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/coffee-tutorial-metaboscape/">
        FBMN with MetaboScape - Coffee Analysis
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/featurebasedgnps/">
        FBMN analysis at GNPS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/rdd/">
        Reference Data Driven Analysis
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../changelog/">
        Change Log
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gnps_community/">
        Join the GNPS community
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../workshops/">
        GNPS Workshops/Training
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../logos/">
        GNPS Logos
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../contact/">
        Contact/Office Hours
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../toolindex/">
        GNPS Tool Index
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../api/">
        API
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../developers/">
        For Developers
      </a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#introduction">
    Introduction
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#mass-spectrometry-processing-with-mzmine">
    Mass spectrometry processing with MZmine
  </a>
<nav aria-label="Mass spectrometry processing with MZmine" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#citations-and-development">
    Citations and development
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#installation">
    Installation
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#data-processing-with-mzmine-for-fbmn">
    Data Processing with MZmine for FBMN
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#convert-your-lc-msms-data-to-an-open-format">
    Convert your LC-MS/MS Data to an Open Format
  </a>
<nav aria-label="Convert your LC-MS/MS Data to an Open Format" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#processing-steps">
    Processing Steps
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#batch-import">
    Batch Import
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#processing-steps_1">
    Processing Steps
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#1-import-files">
    1. Import Files
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#2-mass-detection">
    2. Mass Detection
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#3-build-chromatogram-lc-ms-feature-detection-part-1">
    3. Build Chromatogram (LC-MS feature detection part 1)
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#4-deconvolve-the-chromatogram-lc-ms-feature-detection-part-2">
    4. Deconvolve the Chromatogram (LC-MS feature detection part 2)
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#5-group-isotopes-and-co-eluting-ions">
    5. Group isotopes and co-eluting ions
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#6-order-the-peaklists">
    6. Order the peaklists
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#7-lc-ms-feature-alignement-peaklist-alignement">
    7. LC-MS feature alignement (Peaklist alignement)
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#8-optional-detect-missing-peaks-gap-filling">
    8. (Optional) Detect Missing Peaks / Gap Filling
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#9-optional-filter-the-peaklist-to-msms-peaks">
    9. (Optional) Filter the Peaklist to MS/MS Peaks
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#10-use-the-gnps-export-module">
    10. Use the GNPS Export module
  </a>
<nav aria-label="10. Use the GNPS Export module" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#the-files-can-be-uploaded-to-the-gnps-web-platform-and-feature-based-molecular-networking-job-can-be-directly-launched">
    The files can be uploaded to the GNPS web-platform and Feature-Based Molecular Networking job can be directly launched
  </a>
</li>
</ul>
</nav>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#video-tutorial-quick-mzmine-export-to-gnps-for-fbmn">
    Video Tutorial - Quick MZMine Export to GNPS for FBMN.
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#fbmn-in-gnps">
    FBMN in GNPS
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#video-tutorial-analyze-fbmn-jobs-in-gnps">
    Video Tutorial - Analyze FBMN jobs in GNPS
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#fbmn-in-cytoscape">
    FBMN in Cytoscape
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#tutorials">
    Tutorials
  </a>
<nav aria-label="Tutorials" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#join-the-gnps-community">
    Join the GNPS Community !
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#page-contributors">
    Page Contributors
  </a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-content" data-md-component="content">
<article class="md-content__inner md-typeset">
<a class="md-content__button md-icon" href="https://github.com/CCMS-UCSD/GNPSDocumentation/edit/master/docs/featurebasedmolecularnetworking-with-mzmine2.md" title="Edit this page">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25z"></path></svg>
</a>
<h1>FBMN with MZmine</h1>
<h2 id="introduction">Introduction<a class="headerlink" href="#introduction" title="Permanent link">¶</a></h2>
<p>The main documentation for <strong>Feature-Based Molecular Networking</strong> <a href="../featurebasedmolecularnetworking/">can be accessed here</a>. See <a href="https://www.nature.com/articles/s41592-020-0933-6">our article</a>.</p>
<p>Below we describe how to use <strong>MZmine2 v2.51</strong> with the FBMN workflow on GNPS. We have previously written the documentation for v2.33 and have noted differences in the software versions. </p>
<h2 id="mass-spectrometry-processing-with-mzmine">Mass spectrometry processing with MZmine<a class="headerlink" href="#mass-spectrometry-processing-with-mzmine" title="Permanent link">¶</a></h2>
<h3 id="citations-and-development">Citations and development<a class="headerlink" href="#citations-and-development" title="Permanent link">¶</a></h3>
<div class="admonition quote">
<p class="admonition-title">Recommended Citations</p>
<p>This work builds on the efforts and tools from our many colleagues, please cite their work:</p>
<p>Nothias, L.-F., Petras, D., Schmid, R. et al. <a href="https://www.nature.com/articles/s41592-020-0933-6">Feature-based molecular networking in the GNPS analysis environment</a>. Nat. Methods 17, 905–908 (2020).</p>
<p>Wang, M. et al. <a href="https://doi.org/10.1038/nbt.3597">Sharing and community curation of mass spectrometry data with Global Natural Products Social Molecular Networking</a>. Nat. Biotechnol. 34, 828–837 (2016).</p>
<p>Katajamaa, M., Miettinen, J. &amp; Oresic, M. <a href="https://doi.org/10.1093/bioinformatics/btk039">MZmine: toolbox for processing and visualization of mass spectrometry based molecular profile data.</a> Bioinformatics 22, 634–636 (2006).</p>
<p>Pluskal, T., Castillo, S., Villar-Briones, A. &amp; Oresic, M. <a href="https://doi.org/10.1186/1471-2105-11-395">MZmine 2: modular framework for processing, visualizing, and analyzing mass spectrometry-based molecular profile data.</a> BMC Bioinformatics 11, 395 (2010).</p>
<p>The development of the features used in the pipeline is <a href="https://github.com/mzmine/mzmine2">publicly accessible here</a>.</p>
</div>
<h3 id="installation">Installation<a class="headerlink" href="#installation" title="Permanent link">¶</a></h3>
<p>Download the latest version of MZmine software (version MZmine v2.33 minimum) at <a href="https://github.com/mzmine/mzmine2/releases">https://github.com/mzmine/mzmine2/releases</a>.</p>
<h3 id="data-processing-with-mzmine-for-fbmn">Data Processing with MZmine for FBMN<a class="headerlink" href="#data-processing-with-mzmine-for-fbmn" title="Permanent link">¶</a></h3>
<p>In MZmine, a sequence of steps are performed to process the mass spectrometry data. Here we will present key steps required to process LC-MS/MS data acquired in non-targeted mode (data dependent acquisition). For convenience we also provide a batch file (XML format) that can be imported directly in MZmine.</p>
<div class="admonition warning">
<p class="admonition-title">Warning</p>
<p>MZmine parameters will vary depending on the instrument used, the acquisition parameters, and samples studied. The following documentation serves a basic guideline for using MZmine with the FBMN workflow.</p>
</div>
<p>Please consult the resources below for more details on MZmine processing:</p>
<ul>
<li>The official documentation <a href="http://mzmine.github.io/documentation.html">http://mzmine.github.io/documentation.html</a>,</li>
<li>The <a href="http://www.pharmacognosie-parisdescartes.fr/pdf/150420_MZmine_Tutorial_UNIGE.pdf">MZmine tutorial</a> by Pierre-Marie Allard and Joelle Houriet from the University of Geneva.</li>
<li><a href="https://mzmine.github.io/ADAP_user_manual.pdf">ADAP User Manual</a></li>
<li>The video tutorial below about <a href="../tutorials/americangutmzmine/">MZmine2 processing for Feature Based Molecular Networking</a>. <strong>Note that this video is slighlty outdatted</strong>, so please refer to the steps described in this documentation.</li>
</ul>
<iframe height="350" src="https://www.youtube.com/embed/5jjMllbwD-U" width="600"> </iframe>
<ul>
<li>The video tutorial below about Quick MZMine2 Export to GNPS for FBMN</li>
</ul>
<iframe height="350" src="https://www.youtube.com/embed/vFcGG7T_44E" width="600"> </iframe>
<h3 id="convert-your-lc-msms-data-to-an-open-format">Convert your LC-MS/MS Data to an Open Format<a class="headerlink" href="#convert-your-lc-msms-data-to-an-open-format" title="Permanent link">¶</a></h3>
<p>MZmine accepts different input formats. Note that we recommand to first convert your files to mzML format before doing MZmine2 processing. <a href="../fileconversion/">See the documentation here</a>.</p>
<h4 id="processing-steps">Processing Steps<a class="headerlink" href="#processing-steps" title="Permanent link">¶</a></h4>
<p>Below is the schematic representation of the LC-MS/MS data processing steps with MZmine (thanks Daniel Petras !):</p>
<p><img alt="complete workflow view" src="../img/mzmine/Workflow_mzmine.png"/></p>
<p>Below is the overview of the LC-MS/MS data processing steps in the MZmine batch mode:</p>
<p><img alt="img" src="../img/mzmine/batch_overview.png"/></p>
<h4 id="batch-import">Batch Import<a class="headerlink" href="#batch-import" title="Permanent link">¶</a></h4>
<p>Here are some MZmine batch that are compatible with the FBMN workflow. These batch files can be imported into MZMine (Batch mode):</p>
<table>
<thead>
<tr>
<th>Instrument</th>
<th>Gradient Length</th>
<th>Matrix Type</th>
<th>Sample Size</th>
<th>Download</th>
</tr>
</thead>
<tbody>
<tr>
<td>Bruker Maxis HD qTof</td>
<td>10 Min</td>
<td>Stool</td>
<td>20</td>
<td><a href="../static/maxis_12min_stool_20.xml">Batch</a></td>
</tr>
<tr>
<td>Please contribute a send batch file for various instruments</td>
<td></td>
<td></td>
<td></td>
<td><a href="../static/maxis_12min_stool_20.xml">Batch</a></td>
</tr>
</tbody>
</table>
<h4 id="processing-steps_1">Processing Steps<a class="headerlink" href="#processing-steps_1" title="Permanent link">¶</a></h4>
<p>Below is a walk-through of all the steps</p>
<h4 id="1-import-files">1. Import Files<a class="headerlink" href="#1-import-files" title="Permanent link">¶</a></h4>
<p>Go to Menu: <em>Raw data methods &gt; Raw data import &gt; Select the files</em></p>
<p><img alt="img" src="../img/mzmine/2_import-raw.png"/></p>
<h4 id="2-mass-detection">2. Mass Detection<a class="headerlink" href="#2-mass-detection" title="Permanent link">¶</a></h4>
<p>This step creates mass lists from your raw LC-MS/MS data (non-targeted mode).</p>
<p>Perform mass detection on MS level 1: Menu: <em>Raw data methods &gt; Feature Detection &gt; Mass detection &gt; Set filters: MS level 1.</em></p>
<p>(version 2.33)
Menu: Raw data methods &gt; Mass detection &gt; Set filter : MS level 1</p>
<div class="admonition important">
<p class="admonition-title">Important</p>
<p>Set an appropriate intensity threshold. You can use the preview window to assess the right threshold on your data. As a rule of thumb, the value should at least correspond to the minimum value set for the triggering of the MS2 scan event. (Example: MAXIS-QTOF: 1E3, Q-Exactive 1E4)</p>
</div>
<p>Perform mass detection on MS level 2. The same mass list name must be used.</p>
<p>Go to Menu: <em>Raw data methods &gt; Feature Detection &gt; Mass detection &gt; Set filters: MS level 2.</em></p>
<p>(version 2.33)
Go to: <em>Raw data methods &gt; Mass detection &gt; Set filter : MS level 2.</em></p>
<div class="admonition important">
<p class="admonition-title">Important</p>
<p>Make sure to set an intensity threshold representative of noise level in the MS2 spectra. This is typically lower than for MS1. (Example: maXis QTOF: 1E2; LTQ-XL Orbitrap 1E4, Q-Exactive: 0). If you have any doubt, set it to 0.</p>
</div>
<p><img alt="img" src="../img/mzmine/mass_detection_ms2.png"/></p>
<h4 id="3-build-chromatogram-lc-ms-feature-detection-part-1">3. Build Chromatogram (LC-MS feature detection part 1)<a class="headerlink" href="#3-build-chromatogram-lc-ms-feature-detection-part-1" title="Permanent link">¶</a></h4>
<p>Starting with MZmine 2.39, the original Chromatogram builder is considered deprecated. It has been replaced with the ADAP Chromatogram Builder. The ADAP Module includes parameters for the Min group size # of scans and Group intensity threshold. Further explanation for these parameters can be found in the <a href="http://mzmine.github.io/documentation.html">ADAP Tutorial</a>. If you use the ADAP Chromatogram Builder, please cite the publication below.</p>
<p>Myers, O.D. et al, <a href="https://pubs.acs.org/doi/abs/10.1021/acs.analchem.7b00947">One Step Forward for Reducing False Positive and False Negative Compound Identifications from Mass Spectrometry Metabolomics Data: New Algorithms for Constructing Extracted Ion Chromatograms and Detecting Chromatographic Peaks</a>. Anal. Chem. 89, 17, 8696-8703 (2017).</p>
<p>Go to: <em>Raw data methods &gt; Feature Detection &gt; Chromatogram builder OR ADAP Chromatogram builder</em></p>
<p>(version 2.33)
Go to: <em>Raw data methods &gt; Chromatogram builder</em></p>
<h4 id="4-deconvolve-the-chromatogram-lc-ms-feature-detection-part-2">4. Deconvolve the Chromatogram (LC-MS feature detection part 2)<a class="headerlink" href="#4-deconvolve-the-chromatogram-lc-ms-feature-detection-part-2" title="Permanent link">¶</a></h4>
<p>Go to Menu: <em>Feature list methods &gt; Feature detection &gt; Chromatogram deconvolution</em></p>
<p>(version 2.33)
Go to Menu: <em>Peak list methods &gt; Peak detection &gt; Chromatogram deconvolution</em></p>
<div class="admonition important">
<p class="admonition-title">Important</p>
<p>tick both options "m/z range for MS2 scan pairing (Da)" and "RT range for MS2 scan pairing (min)". The values have to be defined according to your experimental setup (expected MS mass accuracy and chromatographic peak width).</p>
</div>
<p>Example for a UHPLC colum (1.7 µm C18, 50 × 2.1 mm, flow rate of 0.5 mL/min):</p>
<ul>
<li>
<p>maXis-QTOF: 12 min gradient, 0.02 Da and 0.15 min</p>
</li>
<li>
<p>Q-Exactive: 5 min gradient, 0.01 Da and 0.1 min</p>
</li>
</ul>
<p><img alt="img" src="../img/mzmine/deconvolve.png"/></p>
<h4 id="5-group-isotopes-and-co-eluting-ions">5. Group isotopes and co-eluting ions<a class="headerlink" href="#5-group-isotopes-and-co-eluting-ions" title="Permanent link">¶</a></h4>
<p>Use the "Isotopic peaks grouper" [recommended] or other alternative (such as the CAMERA module).</p>
<p>Go to Menu: <em>Feature list methods &gt; Isotopes &gt; Isotopic peaks grouper</em></p>
<p>(version 2.33)
Go to Menu: <em>Peak list methods &gt; Isotopes &gt; Isotopic peaks grouper.</em></p>
<div class="admonition important">
<p class="admonition-title">Important</p>
<p>This depends on your expected peak shapes, duty cycle time and the MS mass accuracy. (Example: MAXIS-QTOF, 10 min gradient, 0.1 min, 0.02 m/z; Q-Exactive, 5 min gradient, 0.05 min, 0.01 m/z)</p>
</div>
<h4 id="6-order-the-peaklists">6. Order the peaklists<a class="headerlink" href="#6-order-the-peaklists" title="Permanent link">¶</a></h4>
<p>Go to Menu: <em>Peak list methods &gt; Order peak lists</em>.</p>
<div class="admonition important">
<p class="admonition-title">Important</p>
<p>This is to ensure the reproducibility of the processing. Indeed, the aligned peak list will change slighlty if that step is not performed. </p>
</div>
<h4 id="7-lc-ms-feature-alignement-peaklist-alignement">7. LC-MS feature alignement (Peaklist alignement)<a class="headerlink" href="#7-lc-ms-feature-alignement-peaklist-alignement" title="Permanent link">¶</a></h4>
<p>In this step, the peak lists from each sample will be aligned in one aligned peak list. The alignement is performed iteratively using the first peak list selected (see MZmine documentation). For that reason, make sure the first sample is adapted (not a negative control) or to manually put an representative peaklist in the first position.</p>
<p>Go to Menu: <em>Feature list methods &gt; Alignment &gt; Join aligner</em></p>
<p>(version 2.33)
Go to Menu: <em>Peak list methods &gt; Alignment &gt; Join aligner</em></p>
<h4 id="8-optional-detect-missing-peaks-gap-filling">8. (Optional) Detect Missing Peaks / Gap Filling<a class="headerlink" href="#8-optional-detect-missing-peaks-gap-filling" title="Permanent link">¶</a></h4>
<p>Gap filling enables to retrieve the intensity of a peak in all the samples, even if it was not detected in a previous processing step.
Go to Menu: <em>Feature list methods &gt; Gap filling &gt; Peak finder (multi-threaded)</em>.</p>
<p>(version 2.33)
Go to Menu: <em>Peak list methods &gt; Gap filling &gt; Peak finder (multi-threaded)</em>.</p>
<div class="admonition important">
<p class="admonition-title">Important</p>
<p>This step is optional. Use the multi-threaded peak finder for fast processing.</p>
</div>
<h4 id="9-optional-filter-the-peaklist-to-msms-peaks">9. (Optional) Filter the Peaklist to MS/MS Peaks<a class="headerlink" href="#9-optional-filter-the-peaklist-to-msms-peaks" title="Permanent link">¶</a></h4>
<p>Depending on the number of features in the aligned peaklist, it is possible to filter the peaklist to keep only features with minimum number of occurences ("Minimum peaks in a row") or a mininum number of isotopic peaks for the feature ("Minimum peaks in an isotope pattern"), or to "Keep only peaks with MS2 scan (GNPS)".</p>
<p>Go to Menu: <em>Feature list methods &gt; Filtering &gt; Feature list rows filter &gt; Select the filters</em></p>
<p>(version 2.33)
Go to Menu: <em>Peak list methods &gt; Filtering &gt; Peak list row filter &gt; Select the filters</em></p>
<div class="admonition important">
<p class="admonition-title">Important</p>
<p>if you use a filter, we recommend using the filter "Reset the peak number ID"</p>
</div>
<div class="admonition important">
<p class="admonition-title">Important</p>
<p>Note that this step was mandatory in the prototype versions of FBMN with MZmine, now the filter "Keep only peaks with MS2 scan (GNPS)" is optional.</p>
</div>
<h4 id="10-use-the-gnps-export-module">10. Use the GNPS Export module<a class="headerlink" href="#10-use-the-gnps-export-module" title="Permanent link">¶</a></h4>
<p>Use the dedicated module "<em>Submit to/Export for GNPS</em>" in MZmine under <em>Feature list methods &gt; Export/Import</em>  to export the needed file:</p>
<ul>
<li>the <strong>feature quantification table</strong> (.CSV file format) with LC-MS feature intensities.</li>
<li>the <strong>MS/MS spectral summary</strong> (.MGF file), with a representative MS/MS spectrum per LC-MS feature. The MS/MS spectrum correspond either to the most intense MS/MS found for the feature, or to the merged spectrum (new feature !)</li>
</ul>
<p>Select the lastest "filtered aligned peaklist" generated and Go to Menu: <em>Peak list methods &gt; Export &gt; Export for/Submit to GNPS</em></p>
<p><img alt="img" src="../img/mzmine/gnps_export_module_V2.png"/></p>
<p>See an example of files generated by the export module using the workflow:
<a href="https://github.com/CCMS-UCSD/GNPSDocumentation/tree/master/docs/tutorials/AG_tutorial_files">here</a>.</p>
<h5 id="the-files-can-be-uploaded-to-the-gnps-web-platform-and-feature-based-molecular-networking-job-can-be-directly-launched">The files can be uploaded to the GNPS web-platform and Feature-Based Molecular Networking job can be directly launched<a class="headerlink" href="#the-files-can-be-uploaded-to-the-gnps-web-platform-and-feature-based-molecular-networking-job-can-be-directly-launched" title="Permanent link">¶</a></h5>
<p><strong>IMPORTANT:</strong> While the possibility to submit the files directly to GNPS and launch a FBMN job on the fly is really convenient for quick data analysis, the files will not be saved in your personal account on GNPS and are periodically deleted, which will prevent future cloning of the jobs. If you do not provide username/password, and you are limited to basic presets of parameters. For that reason, we recommend to upload your files with the FTP uploader <a href="../fileupload/">(see documentation)</a> and prepare your job <a href="https://gnps.ucsd.edu/ProteoSAFe/index.jsp?params=%7B%22workflow%22:%22FEATURE-BASED-MOLECULAR-NETWORKING%22,%22library_on_server%22:%22d.speclibs;%22%7D">directly on GNPS</a> (you must be logged in first).</p>
<p><img alt="img" src="../img/mzmine/gnps_export_module_V2_submission.png"/></p>
<p>In the "<em>Export for/Submit to GNPS</em>" module, select the option: "<em>Submit to GNPS</em>"</p>
<ul>
<li>
<p>[Optional] Metadata file: specify the path to the metadata table in GNPS format. <a href="../metadata/">See documentation here</a></p>
</li>
<li>
<p>Select the parameters presets for the GNPS job.</p>
</li>
<li>
<p>[Optional] Email: specify the email to forward the job link</p>
</li>
<li>
<p>[Optional] Annotation edges (Experimental feature that will described later).</p>
</li>
<li>
<p>[Optional] Open website: if ticked, will open the job webpage.</p>
</li>
</ul>
<p>ADDITIONAL NOTES: The feature table must contain at least the row ID, the row m/z, and row retention time, along with the sample columns. It is currently mandatory for the sample name headers to have the following format: "filename Peak area". Depending on the steps used in MZmine the sample name header can be "filename baseline-corrected Peak area", but this has to be changed back to "filename Peak area".</p>
<h3 id="video-tutorial-quick-mzmine-export-to-gnps-for-fbmn">Video Tutorial - Quick MZMine Export to GNPS for FBMN.<a class="headerlink" href="#video-tutorial-quick-mzmine-export-to-gnps-for-fbmn" title="Permanent link">¶</a></h3>
<p>The workflow for Feature Based Molecular Networking in GNPS is different from the classic molecular networking workflow. <a href="https://gnps.ucsd.edu/ProteoSAFe/index.jsp?params=%7B%22workflow%22:%22FEATURE-BASED-MOLECULAR-NETWORKING%22,%22library_on_server%22:%22d.speclibs;%22%7D">Access the FBMN workflow here</a> (You need to be logged in first !)</p>
<iframe height="350" src="https://www.youtube.com/embed/vFcGG7T_44E" width="600"> </iframe>
<h3 id="fbmn-in-gnps">FBMN in GNPS<a class="headerlink" href="#fbmn-in-gnps" title="Permanent link">¶</a></h3>
<p>The main documentation of the Feature Based Molecular Networking workflow on GNPS <a href="../featurebasedmolecularnetworking/">can be consulted on that page</a>. The workflow for Feature Based Molecular Networking in GNPS is different from the "classic" molecular networking workflow. <a href="https://gnps.ucsd.edu/ProteoSAFe/index.jsp?params=%7B%22workflow%22:%22METABOLOMICS-SNETS-MZMINE%22,%22library_on_server%22:%22d.speclibs;%22%7D">Access the FBMN workflow here</a> (You need to be logged in first !).</p>
<p>Basically, you will need to upload the files produced by MZmine (test files are accessible <a href="https://github.com/CCMS-UCSD/GNPSDocumentation/tree/master/docs/tutorials/AG_tutorial_files">here</a>):</p>
<ul>
<li>The <em>feature quantification table</em> (.CSV file format).</li>
<li>The <em>MS/MS spectral summary</em> (.MGF file format)</li>
<li>[Optional] The <em>metadata table</em> - described <a href="../metadata/">here</a></li>
</ul>
<p>There are several additional normalization options specifically for feature detection. We can normalize the features per LC/MS run and aggregate by groups with either a sum or average (recommended).</p>
<p><img alt="img" src="../img/mzmine/quant_options.png"/></p>
<p><a href="https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=52a390c8eb654b7fa8d61a1c7a4aaab5">Here is an example FBMN</a> job with files resulting from MZmine2 processing of a subset of the <a href="http://humanfoodproject.com/americangut/">American Gut Project</a>.</p>
<p>ADDITIONAL NOTES: In case you want to run the Feature-Based Molecular networking with just part of the samples you processed in MZmine, it is possible to filter the exported files (.csv and .mgf) to keep only the features of interest. In the case of the .csv file, the rows of specific features can just be deleted. Within FBMN in GNPS, the workflow will automatically only consider these set to of features in the csv file. </p>
<p>For other purposes, if you would like to filter down the .mgf file, there are two options of Python scripts to filter the file. One of them can be run in the <a href="https://github.com/Zquinlan/Python_scripts/blob/master/Metabalomics/parse_mgf_by_feature_ID.py">Terminal</a>, while the other can be run in a <a href="https://github.com/helenamrusso/Python_scripts">Jupyter notebook</a>.</p>
<h3 id="video-tutorial-analyze-fbmn-jobs-in-gnps">Video Tutorial - Analyze FBMN jobs in GNPS<a class="headerlink" href="#video-tutorial-analyze-fbmn-jobs-in-gnps" title="Permanent link">¶</a></h3>
<p>This video presents </p>
<iframe height="500" src="https://www.youtube.com/embed/NTkQ0fS1aug" width="800"> </iframe>
<h3 id="fbmn-in-cytoscape">FBMN in Cytoscape<a class="headerlink" href="#fbmn-in-cytoscape" title="Permanent link">¶</a></h3>
<p>Cytoscape is an open source software platform used to visualize, analyze and annotate molecular networks from GNPS. <a href="../featurebasedmolecularnetworking-cytoscape/">See the documentation here</a></p>
<h2 id="tutorials">Tutorials<a class="headerlink" href="#tutorials" title="Permanent link">¶</a></h2>
<p>See our <a href="../tutorials/americangutmzmine/">tutorial on using MZmine2</a> for FBMN analysis of a cohort from the <a href="http://humanfoodproject.com/americangut/">American Gut Project</a>, and our <a href="../tutorials/featurebasedgnps/">tutorial on running a FBMN analysis on GNP</a>.</p>
<h3 id="join-the-gnps-community">Join the GNPS Community !<a class="headerlink" href="#join-the-gnps-community" title="Permanent link">¶</a></h3>
<ul>
<li>For feature request, or to report bugs, please open an "Issue" on the <a href="https://github.com/CCMS-UCSD/GNPS_Workflows"><em>CCMS-UCSD/GNPS_Workflows</em> GitHub repository</a>.</li>
<li>To contribute to the GNPS documentation, please use GitHub by forking the <a href="https://github.com/CCMS-UCSD/GNPSDocumentation"><em>CCMS-UCSD/GNPSDocumentation</em></a> repository, and make a "Pull Request" with the changes.</li>
</ul>
<h2 id="page-contributors">Page Contributors<a class="headerlink" href="#page-contributors" title="Permanent link">¶</a></h2>
<p><span class="git-page-authors git-authors"><a href="mailto:mwang87@gmail.com">Ming Wang</a> (12.22%), <a href="mailto:louisfelix.nothias@gmail.com">lfnothias</a> (71.48%), <a href="mailto:enmeilin@gmail.com">Madeleine Ernst</a> (0.37%), <a href="mailto:iprotsyuk@google.com">Ivan Protsyuk</a> (1.85%), <a href="mailto:50462434+grimcynthia@users.noreply.github.com">Cynthia Grim</a> (13.33%), <a href="mailto:55901470+helenamrusso@users.noreply.github.com">helenamrusso</a> (0.74%)</span></p>
<hr/>
<div class="md-source-file">
<small>
    
      Last update:
      <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-datetime">April 2, 2021 16:56:16</span>
</small>
</div>
</article>
</div>
</div>
</main>
<footer class="md-footer">
<nav aria-label="Footer" class="md-footer__inner md-grid">
<a aria-label="Previous: FBMN Workflow" class="md-footer__link md-footer__link--prev" href="../featurebasedmolecularnetworking/" rel="prev">
<div class="md-footer__button md-icon">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"></path></svg>
</div>
<div class="md-footer__title">
<div class="md-ellipsis">
<span class="md-footer__direction">
                Previous
              </span>
              FBMN Workflow
            </div>
</div>
</a>
<a aria-label="Next: FBMN with MS-DIAL" class="md-footer__link md-footer__link--next" href="../featurebasedmolecularnetworking-with-ms-dial/" rel="next">
<div class="md-footer__title">
<div class="md-ellipsis">
<span class="md-footer__direction">
                Next
              </span>
              FBMN with MS-DIAL
            </div>
</div>
<div class="md-footer__button md-icon">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4z"></path></svg>
</div>
</a>
</nav>
<div class="md-footer-meta md-typeset">
<div class="md-footer-meta__inner md-grid">
<div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" rel="noopener" target="_blank">
      Material for MkDocs
    </a>
</div>
</div>
</div>
</footer>
</div>
<div class="md-dialog" data-md-component="dialog">
<div class="md-dialog__inner md-typeset"></div>
</div>
<script id="__config" type="application/json">{"base": "..", "features": [], "translations": {"clipboard.copy": "Copy to clipboard", "clipboard.copied": "Copied to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.placeholder": "Type to start searching", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.term.missing": "Missing", "select.version.title": "Select version"}, "search": "../assets/javascripts/workers/search.bd0b6b67.min.js"}</script>
<script src="../assets/javascripts/bundle.8aa65030.min.js"></script>
</body>
</html>