



<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      <meta http-equiv="x-ua-compatible" content="ie=edge">
      
      
      
      
        <meta name="lang:clipboard.copy" content="Copy to clipboard">
      
        <meta name="lang:clipboard.copied" content="Copied to clipboard">
      
        <meta name="lang:search.language" content="en">
      
        <meta name="lang:search.pipeline.stopwords" content="True">
      
        <meta name="lang:search.pipeline.trimmer" content="True">
      
        <meta name="lang:search.result.none" content="No matching documents">
      
        <meta name="lang:search.result.one" content="1 matching document">
      
        <meta name="lang:search.result.other" content="# matching documents">
      
        <meta name="lang:search.tokenizer" content="[\s\-]+">
      
      <link rel="shortcut icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.0.4, mkdocs-material-3.3.0">
    
    
      
        <title>Feature Based Molecular Networking - GNPS Documentation</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/application.572ca0f0.css">
      
      
    
    
      <script src="../assets/javascripts/modernizr.962652e9.js"></script>
    
    
      
        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700|Roboto+Mono">
        <style>body,input{font-family:"Roboto","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
      
    
    <link rel="stylesheet" href="../assets/fonts/material-icons.css">
    
    
    
      
        <script async src="https://www.googletagmanager.com/gtag/js?id=UA-8412213-5"></script>
<script>function gtag(){dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],gtag("js",new Date),gtag("config","UA-8412213-5")</script>
      
    
    
  </head>
  
    <body dir="ltr">
  
    <svg class="md-svg">
      <defs>
        
        
      </defs>
    </svg>
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" data-md-component="overlay" for="__drawer"></label>
    
      <a href="#what-is-feature-based-molecular-networking" tabindex="1" class="md-skip">
        Skip to content
      </a>
    
    
      <header class="md-header" data-md-component="header">
  <nav class="md-header-nav md-grid">
    <div class="md-flex">
      <div class="md-flex__cell md-flex__cell--shrink">
        <a href=".." title="GNPS Documentation" class="md-header-nav__button md-logo">
          
            <i class="md-icon"></i>
          
        </a>
      </div>
      <div class="md-flex__cell md-flex__cell--shrink">
        <label class="md-icon md-icon--menu md-header-nav__button" for="__drawer"></label>
      </div>
      <div class="md-flex__cell md-flex__cell--stretch">
        <div class="md-flex__ellipsis md-header-nav__title" data-md-component="title">
          
            
              <span class="md-header-nav__topic">
                GNPS Documentation
              </span>
              <span class="md-header-nav__topic">
                Feature Based Molecular Networking
              </span>
            
          
        </div>
      </div>
      <div class="md-flex__cell md-flex__cell--shrink">
        
          
            <label class="md-icon md-icon--search md-header-nav__button" for="__search"></label>
            
<div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="query" data-md-state="active">
      <label class="md-icon md-search__icon" for="__search"></label>
      <button type="reset" class="md-icon md-search__icon" data-md-component="reset" tabindex="-1">
        &#xE5CD;
      </button>
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="result">
          <div class="md-search-result__meta">
            Type to start searching
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
          
        
      </div>
      
    </div>
  </nav>
</header>
    
    <div class="md-container">
      
        
      
      
      <main class="md-main">
        <div class="md-main__inner md-grid" data-md-component="container">
          
            
              <div class="md-sidebar md-sidebar--primary" data-md-component="navigation">
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    <nav class="md-nav md-nav--primary" data-md-level="0">
  <label class="md-nav__title md-nav__title--site" for="__drawer">
    <a href=".." title="GNPS Documentation" class="md-nav__button md-logo">
      
        <i class="md-icon"></i>
      
    </a>
    GNPS Documentation
  </label>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      


  <li class="md-nav__item">
    <a href=".." title="GNPS Introduction" class="md-nav__link">
      GNPS Introduction
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../isgnpsright/" title="Is my data compatible with GNPS?" class="md-nav__link">
      Is my data compatible with GNPS?
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../faq/" title="Frequently Asked Questions" class="md-nav__link">
      Frequently Asked Questions
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../superquickstart/" title="Superquick Start Guide" class="md-nav__link">
      Superquick Start Guide
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../quickstart/" title="Quick Start Guide" class="md-nav__link">
      Quick Start Guide
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../gnpsanalysisoverview/" title="GNPS Analysis Tools Overview" class="md-nav__link">
      GNPS Analysis Tools Overview
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../changelog/" title="Change Log" class="md-nav__link">
      Change Log
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../contact/" title="Contact" class="md-nav__link">
      Contact
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../workshops/" title="GNPS Workshops" class="md-nav__link">
      GNPS Workshops
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-10" type="checkbox" id="nav-10">
    
    <label class="md-nav__link" for="nav-10">
      Mass Spec/Computational Background
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-10">
        Mass Spec/Computational Background
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../massspecbackground/networkingtheory/" title="Molecular Networking" class="md-nav__link">
      Molecular Networking
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-11" type="checkbox" id="nav-11">
    
    <label class="md-nav__link" for="nav-11">
      Data Preparation/Upload
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-11">
        Data Preparation/Upload
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../fileconversion/" title="Mass Spectrometry File Conversion" class="md-nav__link">
      Mass Spectrometry File Conversion
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../fileupload/" title="GNPS File Upload" class="md-nav__link">
      GNPS File Upload
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-12" type="checkbox" id="nav-12">
    
    <label class="md-nav__link" for="nav-12">
      Recommended Data Analysis
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-12">
        Recommended Data Analysis
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../networking/" title="Molecular Networking" class="md-nav__link">
      Molecular Networking
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../networkingviews/" title="Molecular Networking Views" class="md-nav__link">
      Molecular Networking Views
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../librarysearch/" title="Spectral Library Search" class="md-nav__link">
      Spectral Library Search
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../masst/" title="MASST Search (Mass Search Tool)" class="md-nav__link">
      MASST Search (Mass Search Tool)
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../resultviews/" title="Result Views Navigation" class="md-nav__link">
      Result Views Navigation
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      

  


  <li class="md-nav__item md-nav__item--active md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-13" type="checkbox" id="nav-13" checked>
    
    <label class="md-nav__link" for="nav-13">
      Advanced Data Analysis
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-13">
        Advanced Data Analysis
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../cytoscape/" title="Molecular Networking in Cytoscape" class="md-nav__link">
      Molecular Networking in Cytoscape
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../dereplicator/" title="Dereplicator" class="md-nav__link">
      Dereplicator
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../nap/" title="Network Annotation Propagation (NAP)" class="md-nav__link">
      Network Annotation Propagation (NAP)
    </a>
  </li>

        
          
          
          

  


  <li class="md-nav__item md-nav__item--active">
    
    <input class="md-toggle md-nav__toggle" data-md-toggle="toc" type="checkbox" id="__toc">
    
    
      <label class="md-nav__link md-nav__link--active" for="__toc">
        Feature Based Molecular Networking
      </label>
    
    <a href="./" title="Feature Based Molecular Networking" class="md-nav__link md-nav__link--active">
      Feature Based Molecular Networking
    </a>
    
      
<nav class="md-nav md-nav--secondary">
  
  
  
    <label class="md-nav__title" for="__toc">Table of contents</label>
    <ul class="md-nav__list" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#what-is-feature-based-molecular-networking" title="What is Feature Based Molecular Networking" class="md-nav__link">
    What is Feature Based Molecular Networking
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#why-feature-based-molecular-networking" title="Why Feature Based Molecular Networking" class="md-nav__link">
    Why Feature Based Molecular Networking
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#feature-detection" title="Feature Detection" class="md-nav__link">
    Feature Detection
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#mass-spectrometry-data-feature-detection-with-mzmine2" title="Mass spectrometry data Feature Detection with MZmine2" class="md-nav__link">
    Mass spectrometry data Feature Detection with MZmine2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mzmine-batch-steps" title="MZMine Batch Steps" class="md-nav__link">
    MZMine Batch Steps
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1-import-files" title="1. Import Files" class="md-nav__link">
    1. Import Files
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#2-mass-detection" title="2. Mass Detection" class="md-nav__link">
    2. Mass Detection
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#3-build-chromatogram" title="3. Build Chromatogram" class="md-nav__link">
    3. Build Chromatogram
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#4-deconvolve-the-chromatogram" title="4. Deconvolve the Chromatogram" class="md-nav__link">
    4. Deconvolve the Chromatogram
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#5-group-isotopes-and-co-eluting-ions" title="5. Group isotopes and co-eluting ions" class="md-nav__link">
    5. Group isotopes and co-eluting ions
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#6-order-the-peaklists-optional" title="6. Order the peaklists (Optional)" class="md-nav__link">
    6. Order the peaklists (Optional)
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#7-align-features" title="7. Align Features" class="md-nav__link">
    7. Align Features
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#8-detect-missing-peaks-gap-filling-optional" title="8. Detect Missing Peaks / Gap filling (Optional)" class="md-nav__link">
    8. Detect Missing Peaks / Gap filling (Optional)
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#9-filter-to-msms-peaks" title="9. Filter to MS/MS Peaks" class="md-nav__link">
    9. Filter to MS/MS Peaks
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#10-export-feature-abundances-filtered" title="10. Export Feature Abundances (filtered)" class="md-nav__link">
    10. Export Feature Abundances (filtered)
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#11-export-msms-as-mgf-for-gnps" title="11. Export MS/MS as MGF for GNPS" class="md-nav__link">
    11. Export MS/MS as MGF for GNPS
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#feature-based-molecular-networking-in-gnps" title="Feature Based Molecular Networking in GNPS" class="md-nav__link">
    Feature Based Molecular Networking in GNPS
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#video-tutorial" title="Video Tutorial" class="md-nav__link">
    Video Tutorial
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#results-tables" title="Results Tables" class="md-nav__link">
    Results Tables
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#visualizing-abundances-in-cytoscape-as-pie-charts" title="Visualizing Abundances in Cytoscape as Pie Charts" class="md-nav__link">
    Visualizing Abundances in Cytoscape as Pie Charts
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tutorials" title="Tutorials" class="md-nav__link">
    Tutorials
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#citation" title="Citation" class="md-nav__link">
    Citation
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
      
      
      
      
    </ul>
  
</nav>
    
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../ms2lda/" title="MS2LDA and MotifDB Substructure Discovery" class="md-nav__link">
      MS2LDA and MotifDB Substructure Discovery
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-14" type="checkbox" id="nav-14">
    
    <label class="md-nav__link" for="nav-14">
      Community/Social Features
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-14">
        Community/Social Features
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-14-1" type="checkbox" id="nav-14-1">
    
    <label class="md-nav__link" for="nav-14-1">
      Spectral Libraries
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="2">
      <label class="md-nav__title" for="nav-14-1">
        Spectral Libraries
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../browselibraries/" title="Browsing Reference Spectral Libraries" class="md-nav__link">
      Browsing Reference Spectral Libraries
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../spectrumcuration/" title="Reference Spectral Library Curation" class="md-nav__link">
      Reference Spectral Library Curation
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../batchupload/" title="Reference Spectral Library Batch Addition" class="md-nav__link">
      Reference Spectral Library Batch Addition
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../downloadlibraries/" title="Downloading Reference Spectral Libraries" class="md-nav__link">
      Downloading Reference Spectral Libraries
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

        
          
          
          


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-14-2" type="checkbox" id="nav-14-2">
    
    <label class="md-nav__link" for="nav-14-2">
      Datasets
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="2">
      <label class="md-nav__title" for="nav-14-2">
        Datasets
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../datasets/" title="Dataset Creation/Sharing" class="md-nav__link">
      Dataset Creation/Sharing
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../continuousid/" title="Continuous Identification" class="md-nav__link">
      Continuous Identification
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../moleculeexplorer/" title="Molecule Explorer" class="md-nav__link">
      Molecule Explorer
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../user/" title="User Page" class="md-nav__link">
      User Page
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../troubleshooting/" title="Troubleshooting" class="md-nav__link">
      Troubleshooting
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-16" type="checkbox" id="nav-16">
    
    <label class="md-nav__link" for="nav-16">
      Tutorial Guides
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-16">
        Tutorial Guides
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorials/bacterianetworking/" title="Molecular Networking - Analog Discovery in Microbial Extracts" class="md-nav__link">
      Molecular Networking - Analog Discovery in Microbial Extracts
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorials/americangutmzmine/" title="Feature Based Molecular Networking - MZmine2 Feature Detection - Microbiome Molecules in Fecal Samples in American Gut Project" class="md-nav__link">
      Feature Based Molecular Networking - MZmine2 Feature Detection - Microbiome Molecules in Fecal Samples in American Gut Project
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorials/featurebasedgnps/" title="Feature Based Molecular Networking - GNPS" class="md-nav__link">
      Feature Based Molecular Networking - GNPS
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-17" type="checkbox" id="nav-17">
    
    <label class="md-nav__link" for="nav-17">
      Presentation Materials
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-17">
        Presentation Materials
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../logos/" title="Logos" class="md-nav__link">
      Logos
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              <div class="md-sidebar md-sidebar--secondary" data-md-component="toc">
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    
<nav class="md-nav md-nav--secondary">
  
  
  
    <label class="md-nav__title" for="__toc">Table of contents</label>
    <ul class="md-nav__list" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#what-is-feature-based-molecular-networking" title="What is Feature Based Molecular Networking" class="md-nav__link">
    What is Feature Based Molecular Networking
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#why-feature-based-molecular-networking" title="Why Feature Based Molecular Networking" class="md-nav__link">
    Why Feature Based Molecular Networking
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#feature-detection" title="Feature Detection" class="md-nav__link">
    Feature Detection
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#mass-spectrometry-data-feature-detection-with-mzmine2" title="Mass spectrometry data Feature Detection with MZmine2" class="md-nav__link">
    Mass spectrometry data Feature Detection with MZmine2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mzmine-batch-steps" title="MZMine Batch Steps" class="md-nav__link">
    MZMine Batch Steps
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1-import-files" title="1. Import Files" class="md-nav__link">
    1. Import Files
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#2-mass-detection" title="2. Mass Detection" class="md-nav__link">
    2. Mass Detection
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#3-build-chromatogram" title="3. Build Chromatogram" class="md-nav__link">
    3. Build Chromatogram
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#4-deconvolve-the-chromatogram" title="4. Deconvolve the Chromatogram" class="md-nav__link">
    4. Deconvolve the Chromatogram
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#5-group-isotopes-and-co-eluting-ions" title="5. Group isotopes and co-eluting ions" class="md-nav__link">
    5. Group isotopes and co-eluting ions
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#6-order-the-peaklists-optional" title="6. Order the peaklists (Optional)" class="md-nav__link">
    6. Order the peaklists (Optional)
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#7-align-features" title="7. Align Features" class="md-nav__link">
    7. Align Features
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#8-detect-missing-peaks-gap-filling-optional" title="8. Detect Missing Peaks / Gap filling (Optional)" class="md-nav__link">
    8. Detect Missing Peaks / Gap filling (Optional)
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#9-filter-to-msms-peaks" title="9. Filter to MS/MS Peaks" class="md-nav__link">
    9. Filter to MS/MS Peaks
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#10-export-feature-abundances-filtered" title="10. Export Feature Abundances (filtered)" class="md-nav__link">
    10. Export Feature Abundances (filtered)
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#11-export-msms-as-mgf-for-gnps" title="11. Export MS/MS as MGF for GNPS" class="md-nav__link">
    11. Export MS/MS as MGF for GNPS
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#feature-based-molecular-networking-in-gnps" title="Feature Based Molecular Networking in GNPS" class="md-nav__link">
    Feature Based Molecular Networking in GNPS
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#video-tutorial" title="Video Tutorial" class="md-nav__link">
    Video Tutorial
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#results-tables" title="Results Tables" class="md-nav__link">
    Results Tables
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#visualizing-abundances-in-cytoscape-as-pie-charts" title="Visualizing Abundances in Cytoscape as Pie Charts" class="md-nav__link">
    Visualizing Abundances in Cytoscape as Pie Charts
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tutorials" title="Tutorials" class="md-nav__link">
    Tutorials
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#citation" title="Citation" class="md-nav__link">
    Citation
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
      
      
      
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          <div class="md-content">
            <article class="md-content__inner md-typeset">
              
                
                
                  <h1>Feature Based Molecular Networking</h1>
                
                <p>The Feature Finding Molecular Networking brings together LC-MS feature detection tools (e.g. MZmine2, OpenMS, MS-DIAL, MetaboScape), molecular networking (GNPS, http://gnps.ucsd.edu), and other in silico annotation tools, such as Sirius, CSI:FingerID, or Network Annotation Propagation.</p>
<h2 id="what-is-feature-based-molecular-networking">What is Feature Based Molecular Networking</h2>
<p>Feature Based Molecular Networking is a new way to create molecular networks that utilizes feature detection tools to provide quantification information paired with molecular networking. This is in difference from <a href="networking">classic molecular networking</a> that</p>
<h2 id="why-feature-based-molecular-networking">Why Feature Based Molecular Networking</h2>
<p>The key improvements of the new Feature Finding Molecular Networking are:</p>
<ol>
<li>Importing quantitative information derived from the feature detection tools into the molecular networks</li>
<li>Discriminate isomers by retention time and remove isotopic peak</li>
<li>Allows the annotation MS/MS of spectra with in silico tools and mapping in the molecular networks</li>
</ol>
<h2 id="feature-detection">Feature Detection</h2>
<p>In summary, these tools have been adapted by providing an .MGF export module for the result of LC-MS/MS feature detection.</p>
<ol>
<li>The data have to be processed as recommended by the developers.</li>
<li>The spectral data from the aligned data can be exported as .MGF, the aligned quantification table has to be exported as well.</li>
<li>The .MGF file can be used as input for any GNPS tools. The aligned quantification table can be imported into Cytoscape.</li>
<li>The metadata table groups can be generated automatically with a dedicated workflow for MZmine2, or post-processed with other solutions (Jupyter notebooks, excel).</li>
</ol>
<p>Currently, we are recommending using the MZmine2 workflow, as it has been thoroughly tested. See the documentation below.</p>
<h3 id="mass-spectrometry-data-feature-detection-with-mzmine2">Mass spectrometry data Feature Detection with MZmine2</h3>
<p>Download MZmine2 software (version MZmine v2.33 minimum) at <a href="https://github.com/mzmine/mzmine2/releases">https://github.com/mzmine/mzmine2/releases</a>.</p>
<p>See documentation and videos here: <a href="http://mzmine.github.io/documentation.html">http://mzmine.github.io/documentation.html</a>,
and <a href="tutorials/americangutmzmine/">the video tutorial about Feature Based Molecular Networking.</a></p>
<p>Please make sure your files are converted to mzXML or mzML.</p>
<h3 id="mzmine-batch-steps">MZMine Batch Steps</h3>
<p>In MZmine2, a sequence of steps must be performed. Here are several prototype for various instruments as a starting point:</p>
<table>
<thead>
<tr>
<th>Instrument</th>
<th>Gradient Length</th>
<th>Matrix Type</th>
<th>Sample Size</th>
<th>Download</th>
</tr>
</thead>
<tbody>
<tr>
<td>Bruker Maxis HD qTof</td>
<td>10 Min</td>
<td>Stool</td>
<td>20</td>
<td><a href="../static/maxis_12min_stool_20.xml">Batch</a></td>
</tr>
</tbody>
</table>
<p>These batch files can be imported into MZMine2.</p>
<!-- The prototype batch method for Bruker Maxis HD qTof can be [downloaded](static/qtof_batch.xml) and imported into MZmine2. -->

<p><img alt="complete workflow view" src="../img/mzmine/Workflow_mzmine.png" /></p>
<p>The steps required for data pre-processing with MZmine2 for GNPS are shown in the screen capture below.</p>
<p>See also the following <a href="https://ccms-ucsd.github.io/GNPSDocumentation/tutorials/americangutmzmine/">video tutorial</a>
<iframe width="800" height="500" src="https://www.youtube.com/embed/5jjMllbwD-U"> </iframe></p>
<p><img alt="img" src="../img/mzmine/batch_overview.png" /></p>
<p>Below is a walk through of all the steps</p>
<h4 id="1-import-files">1. Import Files</h4>
<p>Menu: Raw data methods / Raw data import / "Select the files"</p>
<p><img alt="img" src="../img/mzmine/2_import-raw.png" /></p>
<h4 id="2-mass-detection">2. Mass Detection</h4>
<p>Perform mass detection on MS level 1: Menu: Raw data methods / Mass detection / Set filter : MS level 1</p>
<p>[IMPORTANT] Set a appropriate intensity threshold. You cam use the preview window to asses your data. As a rule of thumb, the value should at least correspond to the minimum value set for the triggering of the MS2 scan event. (Example: MAXIS-QTOF: 1E3, Q-Exactive 1E4)</p>
<p>Perform mass detection on MS level 2. The same masslist name must be used: Menu: Raw data methods / Mass detection / Set filter : MS level 2.</p>
<p>[IMPORTANT] Make sure to set an intensity threshold representative of noise level in the MS2 spectrum. This is typically lower than for MS1. Inappropriate intensity threshold could hamper the GNPS and Sirius export modules. (Example: MAXIS-QTOF: 1E2; Q-Exactive 1E3). If you have any doubt, set it to 0.</p>
<p><img alt="img" src="../img/mzmine/mass_detection_ms2.png" /></p>
<h4 id="3-build-chromatogram">3. Build Chromatogram</h4>
<p>Menu: Raw data methods / Chromatogram builder</p>
<h4 id="4-deconvolve-the-chromatogram">4. Deconvolve the Chromatogram</h4>
<p>Menu: Peak list methods / Peak detection / Chromatogram deconvolution</p>
<p>[IMPORTANT] tick both options "m/z range for MS2 scan pairing (Da)" and "RT range for MS2 scan pairing (min)". Define the values according to your experimental setup. This depends on your expected chromatographic peak width and the MS mass accuracy. (Example: MAXIS-QTOF, 10 min grdainet, 0.15 min, 0.02 m/z; Q-Eaxtive, 5 min gradient, 0.1 min, 0.01 m/z)</p>
<p><img alt="img" src="../img/mzmine/deconvolve.png" /></p>
<h4 id="5-group-isotopes-and-co-eluting-ions">5. Group isotopes and co-eluting ions</h4>
<p>"Isotopic peaks grouper module" [recommended] or Camera module. Menu: Peak list methods / Isotopes / Isotopic peaks grouper.</p>
<p>[IMPORTANT]  This depends on your expected peak shapes, duty cycle time and the MS mass accuracy. (Example: MAXIS-QTOF, 10 min grdainet, 0.1 min, 0.02 m/z; Q-Eaxtive, 5 min gradient, 0.05 min, 0.01 m/z)</p>
<h4 id="6-order-the-peaklists-optional">6. Order the peaklists (Optional)</h4>
<p>Menu: Peak list methods / Order peak lists.</p>
<p>This is to ensure reproducibility of results. Final output might change a little if this is not performed.</p>
<h4 id="7-align-features">7. Align Features</h4>
<p>Menu: Peak list methods / Alignment / Join aligner</p>
<h4 id="8-detect-missing-peaks-gap-filling-optional">8. Detect Missing Peaks / Gap filling (Optional)</h4>
<p>Menu: Peak list methods / Gap filling / Peak finder</p>
<h4 id="9-filter-to-msms-peaks">9. Filter to MS/MS Peaks</h4>
<p>Use both filters in the peaklist row filter module: 'Keep only peaks with MS2 scan (GNPS)". Menu: Peak list methods / Filtering / Peak list row filter</p>
<p><img alt="img" src="../img/mzmine/ms2_filtering.png" /></p>
<h4 id="10-export-feature-abundances-filtered">10. Export Feature Abundances (filtered)</h4>
<p>Export the feature table containing filtered peaks in .CSV format. Menu: Peak list methods / Export / Export to CSV file.
See an example of MZmine feature table <a href="../tutorials/AG_tutorial_files/MZmine-GNPS_AG_test_featuretable.csv">here</a>.</p>
<p>The feature table must contain at least the row ID, the row m/z, and row retention time, along with the sample columns.
It is currently mandatory for the sample name headers string to have the following format: "filename Peak area". Depending on Note that depending on the steps used in MZmine the sample name header can be "filename baseline-corrected Peak area", but this has to changed back to "filename Peak area".</p>
<p><img alt="img" src="../img/mzmine/export_features.png" /></p>
<h4 id="11-export-msms-as-mgf-for-gnps">11. Export MS/MS as MGF for GNPS</h4>
<p>Export the .MGF file for GNPS. Menu: Peak list methods / Export / Export for GNPS.</p>
<p>See an example of MZmine .MGF file <a href="../tutorials/AG_tutorial_files/MZmine-GNPS_AG_test_GNPS.mgf">here</a>.</p>
<p><img alt="img" src="../img/mzmine/gnps_export.png" /></p>
<h3 id="feature-based-molecular-networking-in-gnps">Feature Based Molecular Networking in GNPS</h3>
<p>There is a special molecular networking workflow to handle MzMine2 features outputs. Try it out <a href="https://gnps.ucsd.edu/ProteoSAFe/index.jsp?params=%7B%22workflow%22:%22FEATURE-BASED-MOLECULAR-NETWORKING%22,%22library_on_server%22:%22d.speclibs;%22%7D">here</a>. You need to be logged in first!</p>
<p>You will need three items (test files are accessible <a href="https://github.com/CCMS-UCSD/GNPSDocumentation/tree/master/docs/tutorials/featurebasedgnps">here</a>):</p>
<ol>
<li>Feature Table from above</li>
<li>MGF for MS/MS from above</li>
<li>Metadata table - described <a href="networking#metadata">here</a></li>
</ol>
<p>There are several additional normalization options specifically for feature detection. We can normalize the features per LC/MS run and aggregate by groups with either a sum or average (recommended).</p>
<p><img alt="img" src="../img/mzmine/quant_options.png" /></p>
<p>Here is an example mzmine networking <a href="https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=747b36ba08434f0cb478b920decfe4ce">job</a> from a subset of the American Gut Project.</p>
<h4 id="video-tutorial">Video Tutorial</h4>
<p><strong>Running Feature Based Molecular Network at GNPS</strong></p>
<iframe width="800" height="500" src="https://www.youtube.com/embed/NTkQ0fS1aug"> </iframe>

<p><strong>Quick MZMine2 Export to GNPS Feature Based Molecular Networking</strong></p>
<iframe width="800" height="500" src="https://www.youtube.com/embed/vFcGG7T_44E"> </iframe>

<h4 id="results-tables">Results Tables</h4>
<p>Once the molecular networking is finished, there are many views to explore the analysis. These views are very similar to classic molecular networking.</p>
<p><img alt="img" src="../img/featurebased/feature_statuspage.png" /></p>
<p><strong>Default Views to Explore Molecular Networks</strong></p>
<table>
<thead>
<tr>
<th>View</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../networkingviews/#view-all-library-hits">View All Library Hits</a></td>
<td>All spectral library matches between consensus MS/MS spectra and the selected libraries</td>
</tr>
<tr>
<td>View All Analog Library Hits</td>
<td>All spectral library matches between consensus MS/MS spectra and the selected libraries With Analog Search</td>
</tr>
<tr>
<td><a href="../networkingviews/#view-all-clusters-with-ids">View All Clusters With IDs</a></td>
<td>All consensus MS/MS spectra created by Feature Detection Tool that were considered for library search and molecular networking.</td>
</tr>
</tbody>
</table>
<p><strong>Network Visualizations</strong></p>
<table>
<thead>
<tr>
<th>View</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../networkingviews/#view-spectral-families">View Spectral Families (In Browser Network Visualizer)</a></td>
<td>List of all Spectral Families (i.e. connected components in the network) as well as view the network visualized in the browser</td>
</tr>
</tbody>
</table>
<p><strong>Export/Download Network Files</strong></p>
<table>
<thead>
<tr>
<th>View</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td>Download Cytoscape Data</td>
<td>Download network files (graphML) for import into Cytoscape</td>
</tr>
</tbody>
</table>
<p><strong>Advanced Views - Experimental Views</strong></p>
<table>
<thead>
<tr>
<th>View</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td>Direct Cytoscape Preview/Download</td>
<td>Preview entire molecular network and download direct cys file to open in Cytoscape</td>
</tr>
</tbody>
</table>
<p><strong>Advanced Views - External Tools</strong></p>
<table>
<thead>
<tr>
<th>View</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td>View Dereplicator Results</td>
<td>View Dereplicator identification results</td>
</tr>
</tbody>
</table>
<h4 id="visualizing-abundances-in-cytoscape-as-pie-charts">Visualizing Abundances in Cytoscape as Pie Charts</h4>
<p>To visualize different group average abundances in Cytoscape, we'll use pie charts between different groups we care about. Open up the Molecular Network in Cytoscape.</p>
<p>Initially, all nodes will be blue</p>
<p><img alt="img" src="../img/featurebased/feature_cytoscape_load.png" /></p>
<p>Select the Style tab, and click the "Def." column under the Image/Chart 1 Row:</p>
<p><img alt="img" src="../img/featurebased/feature_style_select.png" /></p>
<p>This brings up a panel to select which groups you would like to display in the pie charts:</p>
<p><img alt="img" src="../img/featurebased/feature_group_select.png" /></p>
<p>Select the specific groups you want to compare (e.g. Case vs Control) and hit apply. Now the network will be colored with pies relevant to the groups you wanted to compare.</p>
<p><img alt="img" src="../img/featurebased/feature_cytoscape_pie_display.png" /></p>
<h2 id="tutorials">Tutorials</h2>
<p>See our <a href="tutorials/americangutmzmine">tutorial</a> on using MZmine2 to produce quantification/ms2 files for Feature Based Molecular Networking in the American Gut Project sample.</p>
<p>See our <a href="tutorials/featurebasedgnps">tutorial</a> on running Feature Based Molecular Networking within GNPS.</p>
<h3 id="citation">Citation</h3>
<p>This work builds on the efforts of our many colleagues, please check out their work:</p>
<p><a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-11-395">MZmine 2: Modular framework for processing, visualizing, and analyzing mass spectrometry-based molecular profile data. Pluskal et. al.</a></p>
                
                  
                
              
              
                


              
            </article>
          </div>
        </div>
      </main>
      
        
<footer class="md-footer">
  
    <div class="md-footer-nav">
      <nav class="md-footer-nav__inner md-grid">
        
          <a href="../nap/" title="Network Annotation Propagation (NAP)" class="md-flex md-footer-nav__link md-footer-nav__link--prev" rel="prev">
            <div class="md-flex__cell md-flex__cell--shrink">
              <i class="md-icon md-icon--arrow-back md-footer-nav__button"></i>
            </div>
            <div class="md-flex__cell md-flex__cell--stretch md-footer-nav__title">
              <span class="md-flex__ellipsis">
                <span class="md-footer-nav__direction">
                  Previous
                </span>
                Network Annotation Propagation (NAP)
              </span>
            </div>
          </a>
        
        
          <a href="../ms2lda/" title="MS2LDA and MotifDB Substructure Discovery" class="md-flex md-footer-nav__link md-footer-nav__link--next" rel="next">
            <div class="md-flex__cell md-flex__cell--stretch md-footer-nav__title">
              <span class="md-flex__ellipsis">
                <span class="md-footer-nav__direction">
                  Next
                </span>
                MS2LDA and MotifDB Substructure Discovery
              </span>
            </div>
            <div class="md-flex__cell md-flex__cell--shrink">
              <i class="md-icon md-icon--arrow-forward md-footer-nav__button"></i>
            </div>
          </a>
        
      </nav>
    </div>
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-footer-copyright">
        
        powered by
        <a href="https://www.mkdocs.org">MkDocs</a>
        and
        <a href="https://squidfunk.github.io/mkdocs-material/">
          Material for MkDocs</a>
      </div>
      
        
      
    </div>
  </div>
</footer>
      
    </div>
    
      <script src="../assets/javascripts/application.a353778b.js"></script>
      
      <script>app.initialize({version:"1.0.4",url:{base:".."}})</script>
      
    
  </body>
</html>