
<!DOCTYPE html>

<html class="no-js" lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width,initial-scale=1" name="viewport"/>
<link href="https://ccms-ucsd.github.io/GNPSDocumentation/gc-ms-data-processing-MZmine2/" rel="canonical"/>
<link href="../assets/images/favicon.png" rel="icon"/>
<meta content="mkdocs-1.2.3, mkdocs-material-8.1.2" name="generator"/>
<title>GC-MS data processing with MZmine - GNPS Documentation</title>
<link href="../assets/stylesheets/main.f7951f6f.min.css" rel="stylesheet"/>
<link href="../assets/stylesheets/palette.e6a45f82.min.css" rel="stylesheet"/>
<link crossorigin="" href="https://fonts.gstatic.com" rel="preconnect"/>
<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700%7CRoboto+Mono&amp;display=fallback" rel="stylesheet"/>
<style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
<script>__md_scope=new URL("..",location),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
<script>window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)},ga.l=+new Date,ga("create","UA-8412213-5","auto"),ga("set","anonymizeIp",!0),ga("send","pageview"),document.addEventListener("DOMContentLoaded",function(){document.forms.search&&document.forms.search.query.addEventListener("blur",function(){var e;this.value&&(e=document.location.pathname,ga("send","pageview",e+"?q="+this.value))}),"undefined"!=typeof location$&&location$.subscribe(function(e){ga("send","pageview",e.pathname)})})</script>
<script async="" src="https://www.google-analytics.com/analytics.js"></script>
</head>
<body data-md-color-accent="none" data-md-color-primary="none" data-md-color-scheme="" dir="ltr">
<input autocomplete="off" class="md-toggle" data-md-toggle="drawer" id="__drawer" type="checkbox"/>
<input autocomplete="off" class="md-toggle" data-md-toggle="search" id="__search" type="checkbox"/>
<label class="md-overlay" for="__drawer"></label>
<div data-md-component="skip">
<a class="md-skip" href="#gc-ms-data-processing-with-mzmine">
          Skip to content
        </a>
</div>
<div data-md-component="announce">
</div>
<header class="md-header" data-md-component="header">
<nav aria-label="Header" class="md-header__inner md-grid">
<a aria-label="GNPS Documentation" class="md-header__button md-logo" data-md-component="logo" href=".." title="GNPS Documentation">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"></path></svg>
</a>
<label class="md-header__button md-icon" for="__drawer">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2z"></path></svg>
</label>
<div class="md-header__title" data-md-component="header-title">
<div class="md-header__ellipsis">
<div class="md-header__topic">
<span class="md-ellipsis">
            GNPS Documentation
          </span>
</div>
<div class="md-header__topic" data-md-component="header-topic">
<span class="md-ellipsis">
            
              GC-MS data processing with MZmine
            
          </span>
</div>
</div>
</div>
<label class="md-header__button md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"></path></svg>
</label>
<div class="md-search" data-md-component="search" role="dialog">
<label class="md-search__overlay" for="__search"></label>
<div class="md-search__inner" role="search">
<form class="md-search__form" name="search">
<input aria-label="Search" autocapitalize="off" autocomplete="off" autocorrect="off" class="md-search__input" data-md-component="search-query" name="query" placeholder="Search" required="" spellcheck="false" type="text"/>
<label class="md-search__icon md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"></path></svg>
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"></path></svg>
</label>
<nav aria-label="Search" class="md-search__options">
<button aria-label="Clear" class="md-search__icon md-icon" tabindex="-1" type="reset">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"></path></svg>
</button>
</nav>
</form>
<div class="md-search__output">
<div class="md-search__scrollwrap" data-md-scrollfix="">
<div class="md-search-result" data-md-component="search-result">
<div class="md-search-result__meta">
            Initializing search
          </div>
<ol class="md-search-result__list"></ol>
</div>
</div>
</div>
</div>
</div>
<div class="md-header__source">
<a class="md-source" data-md-component="source" href="https://github.com/CCMS-UCSD/GNPSDocumentation/" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    CCMS-UCSD/GNPSDocumentation
  </div>
</a>
</div>
</nav>
</header>
<div class="md-container" data-md-component="container">
<main class="md-main" data-md-component="main">
<div class="md-main__inner md-grid">
<div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Navigation" class="md-nav md-nav--primary" data-md-level="0">
<label class="md-nav__title" for="__drawer">
<a aria-label="GNPS Documentation" class="md-nav__button md-logo" data-md-component="logo" href=".." title="GNPS Documentation">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"></path></svg>
</a>
    GNPS Documentation
  </label>
<div class="md-nav__source">
<a class="md-source" data-md-component="source" href="https://github.com/CCMS-UCSD/GNPSDocumentation/" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    CCMS-UCSD/GNPSDocumentation
  </div>
</a>
</div>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="..">
        GNPS Introduction
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../isgnpsright/">
        Is my data compatible with GNPS?
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../faq/">
        Frequently Asked Questions
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../superquickstart/">
        Superquick Start Guide
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../quickstart/">
        Quick Start Guide
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gnpsanalysisoverview/">
        GNPS Analysis Tools Overview
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../lexicon/">
        GNPS Lexicon
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_8" id="__nav_8" type="checkbox"/>
<label class="md-nav__link" for="__nav_8">
          Mass Spec/Computational Background
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Mass Spec/Computational Background" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_8">
<span class="md-nav__icon md-icon"></span>
          Mass Spec/Computational Background
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../massspecbackground/networkingtheory/">
        Molecular Networking
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_9" id="__nav_9" type="checkbox"/>
<label class="md-nav__link" for="__nav_9">
          Data Preparation/Upload
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Data Preparation/Upload" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_9">
<span class="md-nav__icon md-icon"></span>
          Data Preparation/Upload
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../fileconversion/">
        Mass Spectrometry File Conversion
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fileconversion_waters/">
        File Conversion (Waters)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fileupload/">
        GNPS File Upload
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../metadata/">
        Metadata
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_10" id="__nav_10" type="checkbox"/>
<label class="md-nav__link" for="__nav_10">
          Recommended Data Analysis
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Recommended Data Analysis" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_10">
<span class="md-nav__icon md-icon"></span>
          Recommended Data Analysis
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../networking/">
        Molecular Networking
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_10_2" id="__nav_10_2" type="checkbox"/>
<label class="md-nav__link" for="__nav_10_2">
          GC-MS Molecular Networking
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="GC-MS Molecular Networking" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_10_2">
<span class="md-nav__icon md-icon"></span>
          GC-MS Molecular Networking
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../gcanalysis/">
        GC-MS on GNPS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gc-ms-deconvolution/">
        GC-MS Deconvolution
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gc-ms-library-molecular-network/">
        GC-MS Library Search and Molecular Networking
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../networkingviews/">
        Molecular Networking Views
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../librarysearch/">
        Spectral Library Search
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../masst/">
        MASST Search (Mass Search Tool)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../resultviews/">
        Result Views Navigation
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_11" id="__nav_11" type="checkbox"/>
<label class="md-nav__link" for="__nav_11">
          Mass Spec Data Visualization
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Mass Spec Data Visualization" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_11">
<span class="md-nav__icon md-icon"></span>
          Mass Spec Data Visualization
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../lcms-dashboard/">
        GNPS Dashboard
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gnpsdatasetexplorer/">
        GNPS Dataset Explorer
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../usi/">
        Metabolomics Spectrum Resolver
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../dashboard_use_cases/">
        GNPS Dashboard Use Cases
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_12" id="__nav_12" type="checkbox"/>
<label class="md-nav__link" for="__nav_12">
          Advanced Data Analysis
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Advanced Data Analysis" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_12">
<span class="md-nav__icon md-icon"></span>
          Advanced Data Analysis
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../networkingfiles/">
        Molecular Networking Files
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_12_2" id="__nav_12_2" type="checkbox"/>
<label class="md-nav__link" for="__nav_12_2">
          Feature Based Molecular Networking
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Feature Based Molecular Networking" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_12_2">
<span class="md-nav__icon md-icon"></span>
          Feature Based Molecular Networking
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking/">
        FBMN Workflow
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-mzmine2/">
        FBMN with MZmine
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-ms-dial/">
        FBMN with MS-DIAL
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-xcms3/">
        FBMN with XCMS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-openms/">
        FBMN with OpenMS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-metaboscape/">
        FBMN with MetaboScape
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-progenesisQI/">
        FBMN with Progenesis QI
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-mztab-m/">
        FBMN with mzTab-M
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-cytoscape/">
        FBMN export to Cytoscape
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_12_3" id="__nav_12_3" type="checkbox"/>
<label class="md-nav__link" for="__nav_12_3">
          FBMN - Ion Identity Networking
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="FBMN - Ion Identity Networking" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_12_3">
<span class="md-nav__icon md-icon"></span>
          FBMN - Ion Identity Networking
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin/">
        IIN Workflow
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin-mzmine/">
        IIN with MZmine2
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin-xcms/">
        IIN with XCMS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin-msdial/">
        IIN with MS-DIAL
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../ms2lda/">
        MS2LDA and MotifDB Substructure Discovery
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../cytoscape/">
        Molecular Networking in Cytoscape
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../dereplicator/">
        DEREPLICATOR
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../dereplicator%2B/">
        DEREPLICATOR+
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../molDiscovery/">
        MolDiscovery
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../nap/">
        Network Annotation Propagation (NAP)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../ms2lda/">
        MS2LDA and MotifDB Substructure Discovery
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../molnetenhancer/">
        MolNetEnhancer
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../metaminer/">
        MetaMiner
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../sirius/">
        Sirius
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../ReDU/">
        Reanalysis of Data User Interface (ReDU)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../qemistree/">
        Qemistree
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../mmvec/">
        MMVEC
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../mergepolarity/">
        Merge Network Polarity
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../chemdir/">
        Chemical Directionality
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_13" id="__nav_13" type="checkbox"/>
<label class="md-nav__link" for="__nav_13">
          Spectral Libraries
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Spectral Libraries" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_13">
<span class="md-nav__icon md-icon"></span>
          Spectral Libraries
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../browselibraries/">
        Browsing Reference Spectral Libraries
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../spectrumcuration/">
        Add/Curate Reference Spectrum in the Library
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../batchupload/">
        Reference Spectral Library Batch Addition
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../batchuploadmzvault/">
        Reference Spectral Library mzVault Addition
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../downloadlibraries/">
        Downloading Reference Spectral Libraries
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_13_6" id="__nav_13_6" type="checkbox"/>
<label class="md-nav__link" for="__nav_13_6">
          Open-source Method for Reference Library Generation
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Open-source Method for Reference Library Generation" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_13_6">
<span class="md-nav__icon md-icon"></span>
          Open-source Method for Reference Library Generation
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../msmschooser/">
        MSMS-Chooser Workflow
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../msmschooserdataacquisitioncomputational/">
        Computational Data Acquisition Prep
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../msmschooserdataacquisitionQE/">
        Data Acqusition - Thermo QExactive (v1.0)
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tag_info/">
        GNPS Tag Contribution and Analyses
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_14" id="__nav_14" type="checkbox"/>
<label class="md-nav__link" for="__nav_14">
          Create a Public Datasets
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Create a Public Datasets" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_14">
<span class="md-nav__icon md-icon"></span>
          Create a Public Datasets
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../datasets/">
        Dataset Creation/Sharing
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../continuousid/">
        Continuous Identification
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../moleculeexplorer/">
        Molecule Explorer
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_15" id="__nav_15" type="checkbox"/>
<label class="md-nav__link" for="__nav_15">
          Community/Social Features
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Community/Social Features" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_15">
<span class="md-nav__icon md-icon"></span>
          Community/Social Features
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../user/">
        User Page
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../troubleshooting/">
        Troubleshooting
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_17" id="__nav_17" type="checkbox"/>
<label class="md-nav__link" for="__nav_17">
          Tutorial Guides
          <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Tutorial Guides" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_17">
<span class="md-nav__icon md-icon"></span>
          Tutorial Guides
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/bacterianetworking/">
        Molecular Networking - Analog Discovery in Microbial Extracts
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/americangutmzmine/">
        Feature-Based Molecular Networking (FBMN) with MZmine2 - Microbiome Molecules in Fecal Samples in American Gut Project
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/americangut-ms-dial/">
        FBMN with MS-DIAL - Microbiome Molecules in Fecal Samples in American Gut Project
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/coffee-tutorial-metaboscape/">
        FBMN with MetaboScape - Coffee Analysis
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/featurebasedgnps/">
        FBMN analysis at GNPS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/rdd/">
        Reference Data Driven Analysis
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../changelog/">
        Change Log
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gnps_community/">
        Join the GNPS community
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../workshops/">
        GNPS Workshops/Training
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../logos/">
        GNPS Logos
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../contact/">
        Contact/Office Hours
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../toolindex/">
        GNPS Tool Index
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../api/">
        API
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../developers/">
        For Developers
      </a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#conversion-to-mzml-with-msconvert">
    Conversion to mzML with MSConvert
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#installation-of-mzmine">
    Installation of MZmine
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#detection-of-masses">
    Detection of masses
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#detection-of-peaks">
    Detection of peaks
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#chromatogrampeak-deconvolution">
    Chromatogram/Peak deconvolution
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#spectral-deconvolution">
    Spectral deconvolution
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#alignment-of-the-peak-lists-accross-multiple-samples">
    Alignment of the peak lists accross multiple samples
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#missing-peaks-search">
    Missing peaks search
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#export-results">
    Export results
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#uploading-the-files-to-gnps-and-run-a-the-gc-ms-workflow">
    Uploading the files to GNPS and run a the GC-MS workflow
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#kovats-ri-marker-file-in-mzmine">
    Kovats RI marker file in MZmine
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#citation">
    Citation
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#page-contributors">
    Page Contributors
  </a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-content" data-md-component="content">
<article class="md-content__inner md-typeset">
<a class="md-content__button md-icon" href="https://github.com/CCMS-UCSD/GNPSDocumentation/edit/master/docs/gc-ms-data-processing-MZmine2.md" title="Edit this page">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25z"></path></svg>
</a>
<h1 id="gc-ms-data-processing-with-mzmine">GC-MS data processing with MZmine<a class="headerlink" href="#gc-ms-data-processing-with-mzmine" title="Permanent link">¶</a></h1>
<p>Below, we describe briefly how to perform processing GC-MS data. To do that, the following steps are required:</p>
<ol>
<li>
<p>The detection of masses from the .mzML files to build extracted ion chromatograms</p>
</li>
<li>
<p>The detection of features/peaks from the built chromatograms</p>
</li>
<li>
<p>The spectral deconvolution can be done with one of the following algorithms:</p>
</li>
</ol>
<p>​    - Hierarchical Clustering</p>
<p>​    - Multivariate Curve Resolution</p>
<ol>
<li>
<p>The alignment of the peaklists</p>
</li>
<li>
<p>The missing peaks search</p>
</li>
<li>
<p>Exporting the files for GNPS</p>
</li>
<li>
<p>Uploading the files to GNPS and run a the GC-MS workflow</p>
</li>
</ol>
<p>For detailed instructions and more information about parameters used in the steps 1/2/4, see the section of pre-processing LC-MS data, and for the step 3, see the section pre-processing GC-MS data available in <a href="https://mzmine.github.io/ADAP_user_manual.pdf">ADAP user manual</a>. At each step, click on "Help" to see how to fell out the parameters correctly. </p>
<p>Before starting, it is recommended to convert the files to .mzML format, but note that MZmine can also accept .netCDF and several proprietary vendor format. </p>
<h3 id="conversion-to-mzml-with-msconvert">Conversion to mzML with MSConvert<a class="headerlink" href="#conversion-to-mzml-with-msconvert" title="Permanent link">¶</a></h3>
<p>The GC-MS data is in the profile mode, then it has to be centroided. To convert and centroid data, please see our<a href="https://ccms-ucsd.github.io/GNPSDocumentation/fileconversion/"> conversion guide</a>. </p>
<h3 id="installation-of-mzmine">Installation of MZmine<a class="headerlink" href="#installation-of-mzmine" title="Permanent link">¶</a></h3>
<p>Download the latest version of <a href="https://mzmine.github.io/documentation.html">MZmine</a>. Once you have your .mzML files and MZmine opened, <strong>import your raw data files</strong> into a project by following “Raw data methods / raw data import” and select your files. Your .mzML files are now in the “Raw data” window. </p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture1.png"/></strong></p>
<p><strong>Save</strong> your current project by clicking on “Project / Save project as”. It is important to save your project regularly and for future access to the processing results. </p>
<p><strong>Reproducibility with the batch mode</strong> After having optimised the parameters in the GUI, it is strongly recommended to consolidate the parameters used for MZmine processing in a batch file (batch mode). This will ensure reproducibility of your analysis. </p>
<p><img alt="img" src="../img/GC-MS_documentation/Picture2.png"/></p>
<h3 id="detection-of-masses">Detection of masses<a class="headerlink" href="#detection-of-masses" title="Permanent link">¶</a></h3>
<p>The <strong>masses detection</strong> is going to detect individual ions in each scan and to generate a mass list of each scan. Select your .mzML files of interest on the “Raw data” window, then go to “Raw data methods / Feature detection / Mass detection” and set a <em>Noise level</em> as shown in the following picture. Make sure your threshold is not too high by clicking on “Show preview”. Then click “ok” to process. </p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture3.png"/></strong></p>
<h3 id="detection-of-peaks">Detection of peaks<a class="headerlink" href="#detection-of-peaks" title="Permanent link">¶</a></h3>
<p>Open the <strong>ADAP chromatogram builder</strong> window by following “Raw data methods / Feature detection / ADAP Chromatogram builder”. This module is going to connect data points from the previously created mass lists of each scan and generate an extracted ion chromatograms. Set the parameters accordingly to the instrument used. Click “Help” at the bottom of the pop-up window, or go on each parameter to see the details and how to set the parameters properly. When it is finished, click “OK”.</p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture4.png"/></strong></p>
<h3 id="chromatogrampeak-deconvolution">Chromatogram/Peak deconvolution<a class="headerlink" href="#chromatogrampeak-deconvolution" title="Permanent link">¶</a></h3>
<p>The <strong>chromatogram deconvolution</strong> consists of the detection and the separation of individual peaks in each chromatogram. After selecting your previously generated chromatograms in the “Feature lists” window, open the parameter window by following “Feature list methods / Feature detection / Chromatogram deconvolution”, then select <em>Wavelets (ADAP)</em> for the “Algorithm”. </p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture5.png"/></strong></p>
<p>Then, by clicking on the ellipse, a parameter window opens. Choose <em>Wavelet Coeff. SN</em> and set the parameters accordingly to the instrument used and previous parameters used to generate chromatograms.  The <em>RT wavelet range</em> parameter is chosen by looking at the peak-detection results when clicking on “Show preview”. By trying several different values, see when the picks-detection (colored picks) is optimal in some <em>Chromatogram</em> for each <em>Feature list</em>. Then click on the “ok” button. This step of peak-detection could take a while. </p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture6.png"/></strong></p>
<h3 id="spectral-deconvolution">Spectral deconvolution<a class="headerlink" href="#spectral-deconvolution" title="Permanent link">¶</a></h3>
<p>During the <strong>spectral deconvolution</strong>, similar peaks are combined and then, a pseudo fragmentation spectrum is built (also called deconvoluted spectrum), using peaks intensities. Two spectral deconvolution algorithms are available: <strong>Hierarchical Clustering</strong> and <strong>Multivariate Curve Resolution</strong>. </p>
<p>​       The <strong>Hierarchical Clustering</strong> is fast but requires users to specify multiple parameters. It can be run by following “Feature list methods / Spectral deconvolution / Hierarchical Clustering” when your previously generated chromatograms deconvoluted are selected. Check the box “Show preview” to preview the first clustering parameters (top right) and peaks that passed the filter with the second clustering parameters (bottom right), by selecting <em>Cluster list</em> from your <em>Feature list</em>. The first parameters <em>Min cluster distance</em> / <em>Min cluster size</em> / <em>Min cluster intensity</em> refer to the ability to combine peaks based on the proximity of their retention times. The filtering parameters are <em>Find shared peaks</em> / <em>Min edge-to-height ratio</em> / <em>Min delta-to-height ratio</em> / <em>Min sharpness</em> / <em>Exclude m/z values</em>. The last parameter <em>Shape-similarity tolerance (0..90)</em>, involved in the second clustering step, refines the clusters by calculating the similarity of the peak’s shapes. Finally, the module “Model based on” allows us to take the highest <em>Sharpness</em> or <em>M/Z value</em> passed peaks to propose as a model peak. When each parameter are set properly, process the spectral deconvolution by clicking on “OK”.</p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture7b.png"/></strong></p>
<p>​       The <strong>Multivariate Curve Resolution</strong> is slower but only a few parameters are used. Its model peaks that can have arbitrary shapes. Open the parameters window by following “Feature list methods / Spectral deconvolution / Multivariate Curve Resolution”. Because of the spectral deconvolution uses “Chromatograms” and detected “Peaks” previously generated, the user must select them using the “Specific feature lists” option for both, then click on the ellipsis button and check the boxes of the chromatograms and peaks list of interest, or using the “Feature list name pattern” option. After those parameters are set, click “Ok”.</p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture8.png"/></strong></p>
<p>When the selection is finished, crossing “Show preview” allows to see the influence of the parameters on the <em>Clusters</em> for each <em>Feature list</em>. Look at the preview plot (top right) for the <em>Deconvolution window tolerance width</em> parameter and the model peaks (right bottom) for <em>Retention time tolerance</em> and <em>Minimum Number of Peaks</em> parameters. Process the spectral deconvolution with the “Ok” button.</p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture9.png"/></strong></p>
<h3 id="alignment-of-the-peak-lists-accross-multiple-samples">Alignment of the peak lists accross multiple samples<a class="headerlink" href="#alignment-of-the-peak-lists-accross-multiple-samples" title="Permanent link">¶</a></h3>
<p>The <strong>ADAP alignment</strong> module calculates the similarity between fragmentation mass spectra to find the best alignment. When the previously generated spectral deconvolution lists are selected, set the alignment parameters in the window which opens when following “Feature list method / Alignment / ADAP Aligner (GC)”. Click “Ok”, then the aligned feature list appears in the “Feature lists” window. </p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture10.png"/></strong></p>
<h3 id="missing-peaks-search">Missing peaks search<a class="headerlink" href="#missing-peaks-search" title="Permanent link">¶</a></h3>
<p>Finally, a module named “<strong>Peak finder</strong> (multithreaded)”, available in “Feature list methods / Gap filling / Peak finder (multithreaded)” is used to search and find missing peaks in the feature list previously generated from the peaks in the raw data. Select your aligned feature list, then apply peak finder parameters by clicking “Ok”.</p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture11.png"/></strong></p>
<p>Open the generated aligned table by double-clicking on it. Taking a look at this table can help to define if the used parameters are good enough. This table can, not only be ordered by the <em>ID</em>, giving information about the number of detected features, but also by the <em>m/z</em> or the <em>RT</em>.</p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture12.png"/></strong></p>
<h3 id="export-results">Export results<a class="headerlink" href="#export-results" title="Permanent link">¶</a></h3>
<p>A new module (introduced in MZmine 2.52) allows to export the GC-MS data processing results (feature quantification table as .CSV and a spectral summary file as .MGF) into the correct input format for the GNPS GC-MS library search workflow. The <strong>export</strong> module is available at “Feature list methods / Export / Import / Export to GNPS-GC-MS (with ADAP)”. In the open window, complete the <em>filename</em> and press “Ok”.</p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture13.png"/></strong></p>
<p>The exported files are the .MGF file containing the deconvolved EI spectra and the .CSV feature quantification table. Those exported .MGF file and .CSV feature table can be used for the GNPS library search/molecular networking workflow directly, without any modifications. They need to be provided as input files by adding to the: “Deconvolved EI Spectra MGF File” and “Feature Quantification Table” correspondingly.</p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture14.png"/></strong></p>
<h3 id="uploading-the-files-to-gnps-and-run-a-the-gc-ms-workflow">Uploading the files to GNPS and run a the GC-MS workflow<a class="headerlink" href="#uploading-the-files-to-gnps-and-run-a-the-gc-ms-workflow" title="Permanent link">¶</a></h3>
<p>Please refer to the following page for more information on the <a href="../gc-ms-library-molecular-network/">GC-GNPS workflow</a>.</p>
<h3 id="kovats-ri-marker-file-in-mzmine">Kovats RI marker file in MZmine<a class="headerlink" href="#kovats-ri-marker-file-in-mzmine" title="Permanent link">¶</a></h3>
<p>MZmine (version 2.52 and later) also offers the extraction of Kovats RI markers into the GNPS GC-MS file format.</p>
<p>See <a href="https://youtu.be/XodHMJcuwnk">tutorial video</a> on Kovats RI marker file creation within MZmine.</p>
<h2 id="citation">Citation<a class="headerlink" href="#citation" title="Permanent link">¶</a></h2>
<p>Jiang  W,  Qiu  Y,  Ni  Y,  Su  M,  Jia  W,  Du  X: <a href="https://pubs.acs.org/doi/abs/10.1021/pr1007703">An  automated  data  analysis  pipeline  for GC-TOF-MS metabonomics studies.</a> J Proteome Res 2010, 9(11):5974-5981.</p>
<p>Ni Y, Qiu Y, Jiang W, Suttlemyre K, Su M, Zhang W, Jia W, Du X: <a href="https://pubs.acs.org/doi/abs/10.1021/ac300898h">ADAP-GC 2.0:  deconvolution of coeluting metabolites from GC/TOF-MS data for metabolomics studies.</a> Anal Chem 2012, 84(15):6619-6629.</p>
<p>Ni Y, Su M, Qiu Y, Jia W, Du X: <a href="https://pubs.acs.org/doi/abs/10.1021/acs.analchem.6b02222">ADAP-GC 3.0: Improved Peak Detection and De-convolution of Co-eluting Metabolites from GC/TOF-MS Data for Metabolomics Studies.</a> Anal Chem 2016, 88(17):8802-8811.</p>
<h2 id="page-contributors">Page Contributors<a class="headerlink" href="#page-contributors" title="Permanent link">¶</a></h2>
<p><span class="git-page-authors git-authors"><a href="mailto:mwang87@gmail.com">Ming Wang</a> (1.59%), <a href="mailto:43760206+melissaesposito@users.noreply.github.com">MelissaESPOSITO</a> (98.41%)</span></p>
<hr/>
<div class="md-source-file">
<small>
    
      Last update:
      <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-datetime">June 13, 2020 23:18:29</span>
</small>
</div>
</article>
</div>
</div>
</main>
<footer class="md-footer">
<div class="md-footer-meta md-typeset">
<div class="md-footer-meta__inner md-grid">
<div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" rel="noopener" target="_blank">
      Material for MkDocs
    </a>
</div>
</div>
</div>
</footer>
</div>
<div class="md-dialog" data-md-component="dialog">
<div class="md-dialog__inner md-typeset"></div>
</div>
<script id="__config" type="application/json">{"base": "..", "features": [], "translations": {"clipboard.copy": "Copy to clipboard", "clipboard.copied": "Copied to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.placeholder": "Type to start searching", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.term.missing": "Missing", "select.version.title": "Select version"}, "search": "../assets/javascripts/workers/search.0bbba5b5.min.js"}</script>
<script src="../assets/javascripts/bundle.649a939e.min.js"></script>
</body>
</html>