



<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      <meta http-equiv="x-ua-compatible" content="ie=edge">
      
      
        <link rel="canonical" href="https://ccms-ucsd.github.io/GNPSDocumentation/gc-ms-data-processing-MZmine2/">
      
      
      
        <meta name="lang:clipboard.copy" content="Copy to clipboard">
      
        <meta name="lang:clipboard.copied" content="Copied to clipboard">
      
        <meta name="lang:search.language" content="en">
      
        <meta name="lang:search.pipeline.stopwords" content="True">
      
        <meta name="lang:search.pipeline.trimmer" content="True">
      
        <meta name="lang:search.result.none" content="No matching documents">
      
        <meta name="lang:search.result.one" content="1 matching document">
      
        <meta name="lang:search.result.other" content="# matching documents">
      
        <meta name="lang:search.tokenizer" content="[\s\-]+">
      
      <link rel="shortcut icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.0.4, mkdocs-material-4.4.2">
    
    
      
        <title>GC-MS data processing with MZmine - GNPS Documentation</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/application.30686662.css">
      
      
    
    
      <script src="../assets/javascripts/modernizr.74668098.js"></script>
    
    
      
        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700|Roboto+Mono&display=fallback">
        <style>body,input{font-family:"Roboto","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
      
    
    <link rel="stylesheet" href="../assets/fonts/material-icons.css">
    
    
    
      
        
<script>
  window.ga = window.ga || function() {
    (ga.q = ga.q || []).push(arguments)
  }
  ga.l = +new Date
  /* Setup integration and send page view */
  ga("create", "UA-8412213-5", "ccms-ucsd.github.io")
  ga("set", "anonymizeIp", true)
  ga("send", "pageview")
  /* Register handler to log search on blur */
  document.addEventListener("DOMContentLoaded", () => {
    if (document.forms.search) {
      var query = document.forms.search.query
      query.addEventListener("blur", function() {
        if (this.value) {
          var path = document.location.pathname;
          ga("send", "pageview", path + "?q=" + this.value)
        }
      })
    }
  })
</script>
<script async src="https://www.google-analytics.com/analytics.js"></script>
      
    
    
  </head>
  
    <body dir="ltr">
  
    <svg class="md-svg">
      <defs>
        
        
      </defs>
    </svg>
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" data-md-component="overlay" for="__drawer"></label>
    
      <a href="#gc-ms-data-processing-with-mzmine" tabindex="1" class="md-skip">
        Skip to content
      </a>
    
    
      <header class="md-header" data-md-component="header">
  <nav class="md-header-nav md-grid">
    <div class="md-flex">
      <div class="md-flex__cell md-flex__cell--shrink">
        <a href="https://ccms-ucsd.github.io/GNPSDocumentation/" title="GNPS Documentation" class="md-header-nav__button md-logo">
          
            <i class="md-icon"></i>
          
        </a>
      </div>
      <div class="md-flex__cell md-flex__cell--shrink">
        <label class="md-icon md-icon--menu md-header-nav__button" for="__drawer"></label>
      </div>
      <div class="md-flex__cell md-flex__cell--stretch">
        <div class="md-flex__ellipsis md-header-nav__title" data-md-component="title">
          
            <span class="md-header-nav__topic">
              GNPS Documentation
            </span>
            <span class="md-header-nav__topic">
              
                GC-MS data processing with MZmine
              
            </span>
          
        </div>
      </div>
      <div class="md-flex__cell md-flex__cell--shrink">
        
          <label class="md-icon md-icon--search md-header-nav__button" for="__search"></label>
          
<div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="query" data-md-state="active">
      <label class="md-icon md-search__icon" for="__search"></label>
      <button type="reset" class="md-icon md-search__icon" data-md-component="reset" tabindex="-1">
        &#xE5CD;
      </button>
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="result">
          <div class="md-search-result__meta">
            Type to start searching
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
        
      </div>
      
        <div class="md-flex__cell md-flex__cell--shrink">
          <div class="md-header-nav__source">
            


  

<a href="/" title="Go to repository" class="md-source" data-md-source="">
  
  <div class="md-source__repository">
    
  </div>
</a>
          </div>
        </div>
      
    </div>
  </nav>
</header>
    
    <div class="md-container">
      
        
      
      
      <main class="md-main" role="main">
        <div class="md-main__inner md-grid" data-md-component="container">
          
            
              <div class="md-sidebar md-sidebar--primary" data-md-component="navigation">
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    <nav class="md-nav md-nav--primary" data-md-level="0">
  <label class="md-nav__title md-nav__title--site" for="__drawer">
    <a href="https://ccms-ucsd.github.io/GNPSDocumentation/" title="GNPS Documentation" class="md-nav__button md-logo">
      
        <i class="md-icon"></i>
      
    </a>
    GNPS Documentation
  </label>
  
    <div class="md-nav__source">
      


  

<a href="/" title="Go to repository" class="md-source" data-md-source="">
  
  <div class="md-source__repository">
    
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      


  <li class="md-nav__item">
    <a href=".." title="GNPS Introduction" class="md-nav__link">
      GNPS Introduction
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../isgnpsright/" title="Is my data compatible with GNPS?" class="md-nav__link">
      Is my data compatible with GNPS?
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../faq/" title="Frequently Asked Questions" class="md-nav__link">
      Frequently Asked Questions
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../superquickstart/" title="Superquick Start Guide" class="md-nav__link">
      Superquick Start Guide
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../quickstart/" title="Quick Start Guide" class="md-nav__link">
      Quick Start Guide
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../gnpsanalysisoverview/" title="GNPS Analysis Tools Overview" class="md-nav__link">
      GNPS Analysis Tools Overview
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-7" type="checkbox" id="nav-7">
    
    <label class="md-nav__link" for="nav-7">
      Mass Spec/Computational Background
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-7">
        Mass Spec/Computational Background
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../massspecbackground/networkingtheory/" title="Molecular Networking" class="md-nav__link">
      Molecular Networking
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-8" type="checkbox" id="nav-8">
    
    <label class="md-nav__link" for="nav-8">
      Data Preparation/Upload
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-8">
        Data Preparation/Upload
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../fileconversion/" title="Mass Spectrometry File Conversion" class="md-nav__link">
      Mass Spectrometry File Conversion
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../fileupload/" title="GNPS File Upload" class="md-nav__link">
      GNPS File Upload
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../metadata/" title="Metadata" class="md-nav__link">
      Metadata
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-9" type="checkbox" id="nav-9">
    
    <label class="md-nav__link" for="nav-9">
      Recommended Data Analysis
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-9">
        Recommended Data Analysis
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../networking/" title="Molecular Networking" class="md-nav__link">
      Molecular Networking
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-9-2" type="checkbox" id="nav-9-2">
    
    <label class="md-nav__link" for="nav-9-2">
      GC-MS Molecular Networking
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="2">
      <label class="md-nav__title" for="nav-9-2">
        GC-MS Molecular Networking
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../gcanalysis/" title="GC-MS on GNPS" class="md-nav__link">
      GC-MS on GNPS
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../gc-ms-deconvolution/" title="GC-MS Deconvolution" class="md-nav__link">
      GC-MS Deconvolution
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../gc-ms-library-molecular-network/" title="GC-MS Library Search and Molecular Networking" class="md-nav__link">
      GC-MS Library Search and Molecular Networking
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../networkingviews/" title="Molecular Networking Views" class="md-nav__link">
      Molecular Networking Views
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../librarysearch/" title="Spectral Library Search" class="md-nav__link">
      Spectral Library Search
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../masst/" title="MASST Search (Mass Search Tool)" class="md-nav__link">
      MASST Search (Mass Search Tool)
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../resultviews/" title="Result Views Navigation" class="md-nav__link">
      Result Views Navigation
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-10" type="checkbox" id="nav-10">
    
    <label class="md-nav__link" for="nav-10">
      Advanced Data Analysis
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-10">
        Advanced Data Analysis
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-10-1" type="checkbox" id="nav-10-1">
    
    <label class="md-nav__link" for="nav-10-1">
      Feature Based Molecular Networking
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="2">
      <label class="md-nav__title" for="nav-10-1">
        Feature Based Molecular Networking
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking/" title="FBMN Workflow" class="md-nav__link">
      FBMN Workflow
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-mzmine2/" title="FBMN with MZmine" class="md-nav__link">
      FBMN with MZmine
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-ms-dial/" title="FBMN with MS-DIAL" class="md-nav__link">
      FBMN with MS-DIAL
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-xcms3/" title="FBMN with XCMS" class="md-nav__link">
      FBMN with XCMS
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-openms/" title="FBMN with OpenMS" class="md-nav__link">
      FBMN with OpenMS
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-metaboscape/" title="FBMN with MetaboScape" class="md-nav__link">
      FBMN with MetaboScape
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-progenesisQI/" title="FBMN with Progenesis QI" class="md-nav__link">
      FBMN with Progenesis QI
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-with-mztab-m/" title="FBMN with mzTab-M" class="md-nav__link">
      FBMN with mzTab-M
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../featurebasedmolecularnetworking-cytoscape/" title="FBMN export to Cytoscape" class="md-nav__link">
      FBMN export to Cytoscape
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

        
          
          
          


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-10-2" type="checkbox" id="nav-10-2">
    
    <label class="md-nav__link" for="nav-10-2">
      FBMN - Ion Identity Networking
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="2">
      <label class="md-nav__title" for="nav-10-2">
        FBMN - Ion Identity Networking
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../fbmn-iin/" title="IIN Workflow" class="md-nav__link">
      IIN Workflow
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../fbmn-iin-mzmine/" title="IIN with MZmine2" class="md-nav__link">
      IIN with MZmine2
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../fbmn-iin-xcms/" title="IIN with XCMS" class="md-nav__link">
      IIN with XCMS
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../fbmn-iin-msdial/" title="IIN with MS-DIAL" class="md-nav__link">
      IIN with MS-DIAL
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../ms2lda/" title="MS2LDA and MotifDB Substructure Discovery" class="md-nav__link">
      MS2LDA and MotifDB Substructure Discovery
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../cytoscape/" title="Molecular Networking in Cytoscape" class="md-nav__link">
      Molecular Networking in Cytoscape
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../dereplicator/" title="DEREPLICATOR" class="md-nav__link">
      DEREPLICATOR
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../nap/" title="Network Annotation Propagation (NAP)" class="md-nav__link">
      Network Annotation Propagation (NAP)
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../ms2lda/" title="MS2LDA and MotifDB Substructure Discovery" class="md-nav__link">
      MS2LDA and MotifDB Substructure Discovery
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../molnetenhancer/" title="MolNetEnhancer" class="md-nav__link">
      MolNetEnhancer
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../metaminer/" title="MetaMiner" class="md-nav__link">
      MetaMiner
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../ReDU/" title="Reanalysis of Data User Interface (ReDU)" class="md-nav__link">
      Reanalysis of Data User Interface (ReDU)
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../qemistree/" title="Qemistree" class="md-nav__link">
      Qemistree
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-11" type="checkbox" id="nav-11">
    
    <label class="md-nav__link" for="nav-11">
      Spectral Libraries
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-11">
        Spectral Libraries
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../browselibraries/" title="Browsing Reference Spectral Libraries" class="md-nav__link">
      Browsing Reference Spectral Libraries
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../spectrumcuration/" title="Add/Curate Reference Spectrum in the Library" class="md-nav__link">
      Add/Curate Reference Spectrum in the Library
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../batchupload/" title="Reference Spectral Library Batch Addition" class="md-nav__link">
      Reference Spectral Library Batch Addition
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../downloadlibraries/" title="Downloading Reference Spectral Libraries" class="md-nav__link">
      Downloading Reference Spectral Libraries
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-11-5" type="checkbox" id="nav-11-5">
    
    <label class="md-nav__link" for="nav-11-5">
      Open-source Method for Reference Library Generation
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="2">
      <label class="md-nav__title" for="nav-11-5">
        Open-source Method for Reference Library Generation
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../msmschooserdataacquisitionQE/" title="Data Acqusition - Thermo QExactive (v1.0)" class="md-nav__link">
      Data Acqusition - Thermo QExactive (v1.0)
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../msmschooserdataacquisitioncomputational/" title="Computational Data Acquisition Prep" class="md-nav__link">
      Computational Data Acquisition Prep
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../msmschooser/" title="MSMS-Chooser Workflow" class="md-nav__link">
      MSMS-Chooser Workflow
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../tag_info/" title="GNPS Tag Contribution and Analyses" class="md-nav__link">
      GNPS Tag Contribution and Analyses
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-12" type="checkbox" id="nav-12">
    
    <label class="md-nav__link" for="nav-12">
      Create a Public Datasets
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-12">
        Create a Public Datasets
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../datasets/" title="Dataset Creation/Sharing" class="md-nav__link">
      Dataset Creation/Sharing
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../continuousid/" title="Continuous Identification" class="md-nav__link">
      Continuous Identification
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../moleculeexplorer/" title="Molecule Explorer" class="md-nav__link">
      Molecule Explorer
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-13" type="checkbox" id="nav-13">
    
    <label class="md-nav__link" for="nav-13">
      Community/Social Features
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-13">
        Community/Social Features
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../user/" title="User Page" class="md-nav__link">
      User Page
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../troubleshooting/" title="Troubleshooting" class="md-nav__link">
      Troubleshooting
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-15" type="checkbox" id="nav-15">
    
    <label class="md-nav__link" for="nav-15">
      Tutorial Guides
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-15">
        Tutorial Guides
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorials/bacterianetworking/" title="Molecular Networking - Analog Discovery in Microbial Extracts" class="md-nav__link">
      Molecular Networking - Analog Discovery in Microbial Extracts
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorials/americangutmzmine/" title="Feature-Based Molecular Networking (FBMN) with MZmine2 - Microbiome Molecules in Fecal Samples in American Gut Project" class="md-nav__link">
      Feature-Based Molecular Networking (FBMN) with MZmine2 - Microbiome Molecules in Fecal Samples in American Gut Project
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorials/americangut-ms-dial/" title="FBMN with MS-DIAL - Microbiome Molecules in Fecal Samples in American Gut Project" class="md-nav__link">
      FBMN with MS-DIAL - Microbiome Molecules in Fecal Samples in American Gut Project
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorials/coffee-tutorial-metaboscape/" title="FBMN with MetaboScape - Coffee Analysis" class="md-nav__link">
      FBMN with MetaboScape - Coffee Analysis
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorials/featurebasedgnps/" title="FBMN analysis at GNPS" class="md-nav__link">
      FBMN analysis at GNPS
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../changelog/" title="Change Log" class="md-nav__link">
      Change Log
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../gnps_community/" title="Join the GNPS community" class="md-nav__link">
      Join the GNPS community
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../workshops/" title="GNPS Workshops/Training" class="md-nav__link">
      GNPS Workshops/Training
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../logos/" title="GNPS Logos" class="md-nav__link">
      GNPS Logos
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../contact/" title="Contact/Office Hours" class="md-nav__link">
      Contact/Office Hours
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../toolindex/" title="GNPS Tool Index" class="md-nav__link">
      GNPS Tool Index
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../api/" title="API" class="md-nav__link">
      API
    </a>
  </li>

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              <div class="md-sidebar md-sidebar--secondary" data-md-component="toc">
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    
<nav class="md-nav md-nav--secondary">
  
  
    
  
  
    <label class="md-nav__title" for="__toc">Table of contents</label>
    <ul class="md-nav__list" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#conversion-to-mzml-with-msconvert" class="md-nav__link">
    Conversion to mzML with MSConvert
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#installation-of-mzmine" class="md-nav__link">
    Installation of MZmine
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#detection-of-masses" class="md-nav__link">
    Detection of masses
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#detection-of-peaks" class="md-nav__link">
    Detection of peaks
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#chromatogrampeak-deconvolution" class="md-nav__link">
    Chromatogram/Peak deconvolution
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#spectral-deconvolution" class="md-nav__link">
    Spectral deconvolution
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#alignment-of-the-peak-lists-accross-multiple-samples" class="md-nav__link">
    Alignment of the peak lists accross multiple samples
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#missing-peaks-search" class="md-nav__link">
    Missing peaks search
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#export-results" class="md-nav__link">
    Export results
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#uploading-the-files-to-gnps-and-run-a-the-gc-ms-workflow" class="md-nav__link">
    Uploading the files to GNPS and run a the GC-MS workflow
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kovats-ri-marker-file-in-mzmine" class="md-nav__link">
    Kovats RI marker file in MZmine
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#citation" class="md-nav__link">
    Citation
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#page-contributors" class="md-nav__link">
    Page contributors
  </a>
  
</li>
      
      
      
      
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          <div class="md-content">
            <article class="md-content__inner md-typeset">
              
                
                  <a href="https://github.com/CCMS-UCSD/GNPSDocumentation/blob/master/docs/gc-ms-data-processing-MZmine2.md" title="Edit this page" class="md-icon md-content__icon">&#xE3C9;</a>
                
                
                <h1 id="gc-ms-data-processing-with-mzmine">GC-MS data processing with MZmine</h1>
<p>Below, we describe briefly how to perform processing GC-MS data. To do that, the following steps are required:</p>
<ol>
<li>
<p>The detection of masses from the .mzML files to build extracted ion chromatograms</p>
</li>
<li>
<p>The detection of features/peaks from the built chromatograms</p>
</li>
<li>
<p>The spectral deconvolution can be done with one of the following algorithms:</p>
</li>
</ol>
<p>​    - Hierarchical Clustering</p>
<p>​    - Multivariate Curve Resolution</p>
<ol>
<li>
<p>The alignment of the peaklists</p>
</li>
<li>
<p>The missing peaks search</p>
</li>
<li>
<p>Exporting the files for GNPS</p>
</li>
<li>
<p>Uploading the files to GNPS and run a the GC-MS workflow</p>
</li>
</ol>
<p>For detailed instructions and more information about parameters used in the steps 1/2/4, see the section of pre-processing LC-MS data, and for the step 3, see the section pre-processing GC-MS data available in <a href="https://mzmine.github.io/ADAP_user_manual.pdf">ADAP user manual</a>. At each step, click on "Help" to see how to fell out the parameters correctly. </p>
<p>Before starting, it is recommended to convert the files to .mzML format, but note that MZmine can also accept .netCDF and several proprietary vendor format. </p>
<h3 id="conversion-to-mzml-with-msconvert">Conversion to mzML with MSConvert</h3>
<p>The GC-MS data is in the profile mode, then it has to be centroided. To convert and centroid data, please see our<a href="https://ccms-ucsd.github.io/GNPSDocumentation/fileconversion/"> conversion guide</a>. </p>
<h3 id="installation-of-mzmine">Installation of MZmine</h3>
<p>Download the latest version of <a href="https://mzmine.github.io/documentation.html">MZmine</a>. Once you have your .mzML files and MZmine opened, <strong>import your raw data files</strong> into a project by following “Raw data methods / raw data import” and select your files. Your .mzML files are now in the “Raw data” window. </p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture1.png" /></strong></p>
<p><strong>Save</strong> your current project by clicking on “Project / Save project as”. It is important to save your project regularly and for future access to the processing results. </p>
<p><strong>Reproducibility with the batch mode</strong> After having optimised the parameters in the GUI, it is strongly recommended to consolidate the parameters used for MZmine processing in a batch file (batch mode). This will ensure reproducibility of your analysis. </p>
<p><img alt="img" src="../img/GC-MS_documentation/Picture2.png" /></p>
<h3 id="detection-of-masses">Detection of masses</h3>
<p>The <strong>masses detection</strong> is going to detect individual ions in each scan and to generate a mass list of each scan. Select your .mzML files of interest on the “Raw data” window, then go to “Raw data methods / Feature detection / Mass detection” and set a <em>Noise level</em> as shown in the following picture. Make sure your threshold is not too high by clicking on “Show preview”. Then click “ok” to process. </p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture3.png" /></strong></p>
<h3 id="detection-of-peaks">Detection of peaks</h3>
<p>Open the <strong>ADAP chromatogram builder</strong> window by following “Raw data methods / Feature detection / ADAP Chromatogram builder”. This module is going to connect data points from the previously created mass lists of each scan and generate an extracted ion chromatograms. Set the parameters accordingly to the instrument used. Click “Help” at the bottom of the pop-up window, or go on each parameter to see the details and how to set the parameters properly. When it is finished, click “OK”.</p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture4.png" /></strong></p>
<h3 id="chromatogrampeak-deconvolution">Chromatogram/Peak deconvolution</h3>
<p>The <strong>chromatogram deconvolution</strong> consists of the detection and the separation of individual peaks in each chromatogram. After selecting your previously generated chromatograms in the “Feature lists” window, open the parameter window by following “Feature list methods / Feature detection / Chromatogram deconvolution”, then select <em>Wavelets (ADAP)</em> for the “Algorithm”. </p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture5.png" /></strong></p>
<p>Then, by clicking on the ellipse, a parameter window opens. Choose <em>Wavelet Coeff. SN</em> and set the parameters accordingly to the instrument used and previous parameters used to generate chromatograms.  The <em>RT wavelet range</em> parameter is chosen by looking at the peak-detection results when clicking on “Show preview”. By trying several different values, see when the picks-detection (colored picks) is optimal in some <em>Chromatogram</em> for each <em>Feature list</em>. Then click on the “ok” button. This step of peak-detection could take a while. </p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture6.png" /></strong></p>
<h3 id="spectral-deconvolution">Spectral deconvolution</h3>
<p>During the <strong>spectral deconvolution</strong>, similar peaks are combined and then, a pseudo fragmentation spectrum is built (also called deconvoluted spectrum), using peaks intensities. Two spectral deconvolution algorithms are available: <strong>Hierarchical Clustering</strong> and <strong>Multivariate Curve Resolution</strong>. </p>
<p>​       The <strong>Hierarchical Clustering</strong> is fast but requires users to specify multiple parameters. It can be run by following “Feature list methods / Spectral deconvolution / Hierarchical Clustering” when your previously generated chromatograms deconvoluted are selected. Check the box “Show preview” to preview the first clustering parameters (top right) and peaks that passed the filter with the second clustering parameters (bottom right), by selecting <em>Cluster list</em> from your <em>Feature list</em>. The first parameters <em>Min cluster distance</em> / <em>Min cluster size</em> / <em>Min cluster intensity</em> refer to the ability to combine peaks based on the proximity of their retention times. The filtering parameters are <em>Find shared peaks</em> / <em>Min edge-to-height ratio</em> / <em>Min delta-to-height ratio</em> / <em>Min sharpness</em> / <em>Exclude m/z values</em>. The last parameter <em>Shape-similarity tolerance (0..90)</em>, involved in the second clustering step, refines the clusters by calculating the similarity of the peak’s shapes. Finally, the module “Model based on” allows us to take the highest <em>Sharpness</em> or <em>M/Z value</em> passed peaks to propose as a model peak. When each parameter are set properly, process the spectral deconvolution by clicking on “OK”.</p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture7b.png" /></strong></p>
<p>​       The <strong>Multivariate Curve Resolution</strong> is slower but only a few parameters are used. Its model peaks that can have arbitrary shapes. Open the parameters window by following “Feature list methods / Spectral deconvolution / Multivariate Curve Resolution”. Because of the spectral deconvolution uses “Chromatograms” and detected “Peaks” previously generated, the user must select them using the “Specific feature lists” option for both, then click on the ellipsis button and check the boxes of the chromatograms and peaks list of interest, or using the “Feature list name pattern” option. After those parameters are set, click “Ok”.</p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture8.png" /></strong></p>
<p>When the selection is finished, crossing “Show preview” allows to see the influence of the parameters on the <em>Clusters</em> for each <em>Feature list</em>. Look at the preview plot (top right) for the <em>Deconvolution window tolerance width</em> parameter and the model peaks (right bottom) for <em>Retention time tolerance</em> and <em>Minimum Number of Peaks</em> parameters. Process the spectral deconvolution with the “Ok” button.</p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture9.png" /></strong></p>
<h3 id="alignment-of-the-peak-lists-accross-multiple-samples">Alignment of the peak lists accross multiple samples</h3>
<p>The <strong>ADAP alignment</strong> module calculates the similarity between fragmentation mass spectra to find the best alignment. When the previously generated spectral deconvolution lists are selected, set the alignment parameters in the window which opens when following “Feature list method / Alignment / ADAP Aligner (GC)”. Click “Ok”, then the aligned feature list appears in the “Feature lists” window. </p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture10.png" /></strong></p>
<h3 id="missing-peaks-search">Missing peaks search</h3>
<p>Finally, a module named “<strong>Peak finder</strong> (multithreaded)”, available in “Feature list methods / Gap filling / Peak finder (multithreaded)” is used to search and find missing peaks in the feature list previously generated from the peaks in the raw data. Select your aligned feature list, then apply peak finder parameters by clicking “Ok”.</p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture11.png" /></strong></p>
<p>Open the generated aligned table by double-clicking on it. Taking a look at this table can help to define if the used parameters are good enough. This table can, not only be ordered by the <em>ID</em>, giving information about the number of detected features, but also by the <em>m/z</em> or the <em>RT</em>.</p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture12.png" /></strong></p>
<h3 id="export-results">Export results</h3>
<p>A new module (introduced in MZmine 2.52) allows to export the GC-MS data processing results (feature quantification table as .CSV and a spectral summary file as .MGF) into the correct input format for the GNPS GC-MS library search workflow. The <strong>export</strong> module is available at “Feature list methods / Export / Import / Export to GNPS-GC-MS (with ADAP)”. In the open window, complete the <em>filename</em> and press “Ok”.</p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture13.png" /></strong></p>
<p>The exported files are the .MGF file containing the deconvolved EI spectra and the .CSV feature quantification table. Those exported .MGF file and .CSV feature table can be used for the GNPS library search/molecular networking workflow directly, without any modifications. They need to be provided as input files by adding to the: “Deconvolved EI Spectra MGF File” and “Feature Quantification Table” correspondingly.</p>
<p><strong><img alt="img" src="../img/GC-MS_documentation/Picture14.png" /></strong></p>
<h3 id="uploading-the-files-to-gnps-and-run-a-the-gc-ms-workflow">Uploading the files to GNPS and run a the GC-MS workflow</h3>
<p>Please refer to the following page for more information on the <a href="../gc-ms-library-molecular-network/">GC-GNPS workflow</a>.</p>
<h3 id="kovats-ri-marker-file-in-mzmine">Kovats RI marker file in MZmine</h3>
<p>MZmine (version 2.52 and later) also offers the extraction of Kovats RI markers into the GNPS GC-MS file format.</p>
<p>See <a href="https://youtu.be/XodHMJcuwnk">tutorial video</a> on Kovats RI marker file creation within MZmine.</p>
<h2 id="citation">Citation</h2>
<p>Jiang  W,  Qiu  Y,  Ni  Y,  Su  M,  Jia  W,  Du  X: <a href="https://pubs.acs.org/doi/abs/10.1021/pr1007703">An  automated  data  analysis  pipeline  for GC-TOF-MS metabonomics studies.</a> J Proteome Res 2010, 9(11):5974-5981.</p>
<p>Ni Y, Qiu Y, Jiang W, Suttlemyre K, Su M, Zhang W, Jia W, Du X: <a href="https://pubs.acs.org/doi/abs/10.1021/ac300898h">ADAP-GC 2.0:  deconvolution of coeluting metabolites from GC/TOF-MS data for metabolomics studies.</a> Anal Chem 2012, 84(15):6619-6629.</p>
<p>Ni Y, Su M, Qiu Y, Jia W, Du X: <a href="https://pubs.acs.org/doi/abs/10.1021/acs.analchem.6b02222">ADAP-GC 3.0: Improved Peak Detection and De-convolution of Co-eluting Metabolites from GC/TOF-MS Data for Metabolomics Studies.</a> Anal Chem 2016, 88(17):8802-8811.</p>
<h2 id="page-contributors">Page contributors</h2>
<p>Melissa Nothias-Esposito (UCSD), Louis Felix Nothias (UCSD), Aleksandr Smirnov (UNC Charlotte), Alexander Aksenov (UCSD)</p>
                
                  
                
              
              
                


              
            </article>
          </div>
        </div>
      </main>
      
        
<footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-footer-copyright">
        
        powered by
        <a href="https://www.mkdocs.org">MkDocs</a>
        and
        <a href="https://squidfunk.github.io/mkdocs-material/">
          Material for MkDocs</a>
      </div>
      
    </div>
  </div>
</footer>
      
    </div>
    
      <script src="../assets/javascripts/application.c648116f.js"></script>
      
      <script>app.initialize({version:"1.0.4",url:{base:".."}})</script>
      
    
  </body>
</html>