
<!DOCTYPE html>

<html class="no-js" lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width,initial-scale=1" name="viewport"/>
<link href="https://ccms-ucsd.github.io/GNPSDocumentation/dashboard_use_cases/" rel="canonical"/>
<link href="../assets/images/favicon.png" rel="icon"/>
<meta content="mkdocs-1.2.2, mkdocs-material-7.2.3" name="generator"/>
<title>GNPS Dashboard Use Cases - GNPS Documentation</title>
<link href="../assets/stylesheets/main.f7f47774.min.css" rel="stylesheet"/>
<link href="../assets/stylesheets/palette.3f5d1f46.min.css" rel="stylesheet"/>
<link crossorigin="" href="https://fonts.gstatic.com" rel="preconnect"/>
<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700%7CRoboto+Mono&amp;display=fallback" rel="stylesheet"/>
<style>:root{--md-text-font-family:"Roboto";--md-code-font-family:"Roboto Mono"}</style>
<script>window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)},ga.l=+new Date,ga("create","UA-8412213-5","auto"),ga("set","anonymizeIp",!0),ga("send","pageview"),document.addEventListener("DOMContentLoaded",function(){document.forms.search&&document.forms.search.query.addEventListener("blur",function(){var e;this.value&&(e=document.location.pathname,ga("send","pageview",e+"?q="+this.value))}),"undefined"!=typeof location$&&location$.subscribe(function(e){ga("send","pageview",e.pathname)})})</script>
<script async="" src="https://www.google-analytics.com/analytics.js"></script>
</head>
<body data-md-color-accent="none" data-md-color-primary="none" data-md-color-scheme="" dir="ltr">
<script>function __prefix(e){return new URL("..",location).pathname+"."+e}function __get(e,t=localStorage){return JSON.parse(t.getItem(__prefix(e)))}</script>
<input autocomplete="off" class="md-toggle" data-md-toggle="drawer" id="__drawer" type="checkbox"/>
<input autocomplete="off" class="md-toggle" data-md-toggle="search" id="__search" type="checkbox"/>
<label class="md-overlay" for="__drawer"></label>
<div data-md-component="skip">
<a class="md-skip" href="#use-case-1-mass-spectrometry-data-visualization-and-linking-to-molecular-networking">
          Skip to content
        </a>
</div>
<div data-md-component="announce">
</div>
<header class="md-header" data-md-component="header">
<nav aria-label="Header" class="md-header__inner md-grid">
<a aria-label="GNPS Documentation" class="md-header__button md-logo" data-md-component="logo" href=".." title="GNPS Documentation">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"></path></svg>
</a>
<label class="md-header__button md-icon" for="__drawer">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2z"></path></svg>
</label>
<div class="md-header__title" data-md-component="header-title">
<div class="md-header__ellipsis">
<div class="md-header__topic">
<span class="md-ellipsis">
            GNPS Documentation
          </span>
</div>
<div class="md-header__topic" data-md-component="header-topic">
<span class="md-ellipsis">
            
              GNPS Dashboard Use Cases
            
          </span>
</div>
</div>
</div>
<label class="md-header__button md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"></path></svg>
</label>
<div class="md-search" data-md-component="search" role="dialog">
<label class="md-search__overlay" for="__search"></label>
<div class="md-search__inner" role="search">
<form class="md-search__form" name="search">
<input aria-label="Search" autocapitalize="off" autocomplete="off" autocorrect="off" class="md-search__input" data-md-component="search-query" name="query" placeholder="Search" required="" spellcheck="false" type="text"/>
<label class="md-search__icon md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"></path></svg>
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"></path></svg>
</label>
<nav aria-label="Search" class="md-search__options">
<button aria-label="Clear" class="md-search__icon md-icon" tabindex="-1" type="reset">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"></path></svg>
</button>
</nav>
</form>
<div class="md-search__output">
<div class="md-search__scrollwrap" data-md-scrollfix="">
<div class="md-search-result" data-md-component="search-result">
<div class="md-search-result__meta">
            Initializing search
          </div>
<ol class="md-search-result__list"></ol>
</div>
</div>
</div>
</div>
</div>
<div class="md-header__source">
<a class="md-source" data-md-component="source" href="https://github.com/CCMS-UCSD/GNPSDocumentation/" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    CCMS-UCSD/GNPSDocumentation
  </div>
</a>
</div>
</nav>
</header>
<div class="md-container" data-md-component="container">
<main class="md-main" data-md-component="main">
<div class="md-main__inner md-grid">
<div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Navigation" class="md-nav md-nav--primary" data-md-level="0">
<label class="md-nav__title" for="__drawer">
<a aria-label="GNPS Documentation" class="md-nav__button md-logo" data-md-component="logo" href=".." title="GNPS Documentation">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"></path></svg>
</a>
    GNPS Documentation
  </label>
<div class="md-nav__source">
<a class="md-source" data-md-component="source" href="https://github.com/CCMS-UCSD/GNPSDocumentation/" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    CCMS-UCSD/GNPSDocumentation
  </div>
</a>
</div>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="..">
        GNPS Introduction
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../isgnpsright/">
        Is my data compatible with GNPS?
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../faq/">
        Frequently Asked Questions
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../superquickstart/">
        Superquick Start Guide
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../quickstart/">
        Quick Start Guide
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gnpsanalysisoverview/">
        GNPS Analysis Tools Overview
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../lexicon/">
        GNPS Lexicon
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_8" id="__nav_8" type="checkbox"/>
<label class="md-nav__link" for="__nav_8">
        Mass Spec/Computational Background
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Mass Spec/Computational Background" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_8">
<span class="md-nav__icon md-icon"></span>
          Mass Spec/Computational Background
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../massspecbackground/networkingtheory/">
        Molecular Networking
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_9" id="__nav_9" type="checkbox"/>
<label class="md-nav__link" for="__nav_9">
        Data Preparation/Upload
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Data Preparation/Upload" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_9">
<span class="md-nav__icon md-icon"></span>
          Data Preparation/Upload
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../fileconversion/">
        Mass Spectrometry File Conversion
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fileconversion_waters/">
        File Conversion (Waters)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fileupload/">
        GNPS File Upload
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../metadata/">
        Metadata
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_10" id="__nav_10" type="checkbox"/>
<label class="md-nav__link" for="__nav_10">
        Recommended Data Analysis
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Recommended Data Analysis" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_10">
<span class="md-nav__icon md-icon"></span>
          Recommended Data Analysis
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../networking/">
        Molecular Networking
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_10_2" id="__nav_10_2" type="checkbox"/>
<label class="md-nav__link" for="__nav_10_2">
        GC-MS Molecular Networking
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="GC-MS Molecular Networking" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_10_2">
<span class="md-nav__icon md-icon"></span>
          GC-MS Molecular Networking
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../gcanalysis/">
        GC-MS on GNPS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gc-ms-deconvolution/">
        GC-MS Deconvolution
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gc-ms-library-molecular-network/">
        GC-MS Library Search and Molecular Networking
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../networkingviews/">
        Molecular Networking Views
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../librarysearch/">
        Spectral Library Search
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../masst/">
        MASST Search (Mass Search Tool)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../resultviews/">
        Result Views Navigation
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--active md-nav__item--nested">
<input checked="" class="md-nav__toggle md-toggle" data-md-toggle="__nav_11" id="__nav_11" type="checkbox"/>
<label class="md-nav__link" for="__nav_11">
        Mass Spec Data Visualization
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Mass Spec Data Visualization" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_11">
<span class="md-nav__icon md-icon"></span>
          Mass Spec Data Visualization
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../lcms-dashboard/">
        GNPS Dashboard
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gnpsdatasetexplorer/">
        GNPS Dataset Explorer
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../usi/">
        Metabolomics Spectrum Resolver
      </a>
</li>
<li class="md-nav__item md-nav__item--active">
<input class="md-nav__toggle md-toggle" data-md-toggle="toc" id="__toc" type="checkbox"/>
<label class="md-nav__link md-nav__link--active" for="__toc">
          GNPS Dashboard Use Cases
          <span class="md-nav__icon md-icon"></span>
</label>
<a class="md-nav__link md-nav__link--active" href="./">
        GNPS Dashboard Use Cases
      </a>
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#daniel-petras-phd-post-doctoral-scholar-analytical-chemistry-and-chemical-ecology">
    Daniel Petras PhD - Post-Doctoral Scholar - Analytical Chemistry and Chemical Ecology
  </a>
</li>
</ul>
</nav>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_12" id="__nav_12" type="checkbox"/>
<label class="md-nav__link" for="__nav_12">
        Advanced Data Analysis
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Advanced Data Analysis" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_12">
<span class="md-nav__icon md-icon"></span>
          Advanced Data Analysis
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../networkingfiles/">
        Molecular Networking Files
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_12_2" id="__nav_12_2" type="checkbox"/>
<label class="md-nav__link" for="__nav_12_2">
        Feature Based Molecular Networking
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Feature Based Molecular Networking" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_12_2">
<span class="md-nav__icon md-icon"></span>
          Feature Based Molecular Networking
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking/">
        FBMN Workflow
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-mzmine2/">
        FBMN with MZmine
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-ms-dial/">
        FBMN with MS-DIAL
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-xcms3/">
        FBMN with XCMS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-openms/">
        FBMN with OpenMS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-metaboscape/">
        FBMN with MetaboScape
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-progenesisQI/">
        FBMN with Progenesis QI
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-with-mztab-m/">
        FBMN with mzTab-M
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../featurebasedmolecularnetworking-cytoscape/">
        FBMN export to Cytoscape
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_12_3" id="__nav_12_3" type="checkbox"/>
<label class="md-nav__link" for="__nav_12_3">
        FBMN - Ion Identity Networking
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="FBMN - Ion Identity Networking" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_12_3">
<span class="md-nav__icon md-icon"></span>
          FBMN - Ion Identity Networking
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin/">
        IIN Workflow
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin-mzmine/">
        IIN with MZmine2
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin-xcms/">
        IIN with XCMS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../fbmn-iin-msdial/">
        IIN with MS-DIAL
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../ms2lda/">
        MS2LDA and MotifDB Substructure Discovery
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../cytoscape/">
        Molecular Networking in Cytoscape
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../dereplicator/">
        DEREPLICATOR
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../dereplicator%2B/">
        DEREPLICATOR+
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../molDiscovery/">
        MolDiscovery
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../nap/">
        Network Annotation Propagation (NAP)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../ms2lda/">
        MS2LDA and MotifDB Substructure Discovery
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../molnetenhancer/">
        MolNetEnhancer
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../metaminer/">
        MetaMiner
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../sirius/">
        Sirius
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../ReDU/">
        Reanalysis of Data User Interface (ReDU)
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../qemistree/">
        Qemistree
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../mmvec/">
        MMVEC
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../mergepolarity/">
        Merge Network Polarity
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../chemdir/">
        Chemical Directionality
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_13" id="__nav_13" type="checkbox"/>
<label class="md-nav__link" for="__nav_13">
        Spectral Libraries
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Spectral Libraries" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_13">
<span class="md-nav__icon md-icon"></span>
          Spectral Libraries
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../browselibraries/">
        Browsing Reference Spectral Libraries
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../spectrumcuration/">
        Add/Curate Reference Spectrum in the Library
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../batchupload/">
        Reference Spectral Library Batch Addition
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../batchuploadmzvault/">
        Reference Spectral Library mzVault Addition
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../downloadlibraries/">
        Downloading Reference Spectral Libraries
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_13_6" id="__nav_13_6" type="checkbox"/>
<label class="md-nav__link" for="__nav_13_6">
        Open-source Method for Reference Library Generation
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Open-source Method for Reference Library Generation" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_13_6">
<span class="md-nav__icon md-icon"></span>
          Open-source Method for Reference Library Generation
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../msmschooser/">
        MSMS-Chooser Workflow
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../msmschooserdataacquisitioncomputational/">
        Computational Data Acquisition Prep
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../msmschooserdataacquisitionQE/">
        Data Acqusition - Thermo QExactive (v1.0)
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tag_info/">
        GNPS Tag Contribution and Analyses
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_14" id="__nav_14" type="checkbox"/>
<label class="md-nav__link" for="__nav_14">
        Create a Public Datasets
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Create a Public Datasets" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_14">
<span class="md-nav__icon md-icon"></span>
          Create a Public Datasets
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../datasets/">
        Dataset Creation/Sharing
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../continuousid/">
        Continuous Identification
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../moleculeexplorer/">
        Molecule Explorer
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_15" id="__nav_15" type="checkbox"/>
<label class="md-nav__link" for="__nav_15">
        Community/Social Features
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Community/Social Features" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_15">
<span class="md-nav__icon md-icon"></span>
          Community/Social Features
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../user/">
        User Page
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../troubleshooting/">
        Troubleshooting
      </a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" data-md-toggle="__nav_17" id="__nav_17" type="checkbox"/>
<label class="md-nav__link" for="__nav_17">
        Tutorial Guides
        <span class="md-nav__icon md-icon"></span>
</label>
<nav aria-label="Tutorial Guides" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_17">
<span class="md-nav__icon md-icon"></span>
          Tutorial Guides
        </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/bacterianetworking/">
        Molecular Networking - Analog Discovery in Microbial Extracts
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/americangutmzmine/">
        Feature-Based Molecular Networking (FBMN) with MZmine2 - Microbiome Molecules in Fecal Samples in American Gut Project
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/americangut-ms-dial/">
        FBMN with MS-DIAL - Microbiome Molecules in Fecal Samples in American Gut Project
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/coffee-tutorial-metaboscape/">
        FBMN with MetaboScape - Coffee Analysis
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/featurebasedgnps/">
        FBMN analysis at GNPS
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../tutorials/rdd/">
        Reference Data Driven Analysis
      </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../changelog/">
        Change Log
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../gnps_community/">
        Join the GNPS community
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../workshops/">
        GNPS Workshops/Training
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../logos/">
        GNPS Logos
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../contact/">
        Contact/Office Hours
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../toolindex/">
        GNPS Tool Index
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../api/">
        API
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../developers/">
        For Developers
      </a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#daniel-petras-phd-post-doctoral-scholar-analytical-chemistry-and-chemical-ecology">
    Daniel Petras PhD - Post-Doctoral Scholar - Analytical Chemistry and Chemical Ecology
  </a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-content" data-md-component="content">
<article class="md-content__inner md-typeset">
<a class="md-content__button md-icon" href="https://github.com/CCMS-UCSD/GNPSDocumentation/edit/master/docs/dashboard_use_cases.md" title="Edit this page">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25z"></path></svg>
</a>
<h1 id="use-case-1-mass-spectrometry-data-visualization-and-linking-to-molecular-networking"><strong>Use Case 1: Mass spectrometry data visualization and linking to molecular networking</strong><a class="headerlink" href="#use-case-1-mass-spectrometry-data-visualization-and-linking-to-molecular-networking" title="Permanent link">¶</a></h1>
<h2 id="daniel-petras-phd-post-doctoral-scholar-analytical-chemistry-and-chemical-ecology"><strong>Daniel Petras PhD - Post-Doctoral Scholar - Analytical Chemistry and Chemical Ecology</strong><a class="headerlink" href="#daniel-petras-phd-post-doctoral-scholar-analytical-chemistry-and-chemical-ecology" title="Permanent link">¶</a></h2>
<p><em>E. coli</em> Nissle is broadly used as a probiotic and the producer of a series of secondary metabolites including the DNA damaging cytotoxins colibactin and siderophores yersiniabactin. In the context of searching for novel zinc uptake mechanisms, we screened Nissle wildtype and yersiniabactin NRPS knock-out mutants culture extract for yersiniabactin production<a href="https://www.zotero.org/google-docs/?kusO1q">7</a>.</p>
<p>The molecular network of MS/MS data and MS1 XICs from the extracts are shown in <strong>SI Figure 2</strong>. Panel a shows the molecular network of yersiniabactins including a series of derivatives around the main compound (m/z 482.124) which are produced by the wildtype but not the NRPS KO mutant according to the network pie charts that visualize MS/MS spectral counts (red = WT, blue = KO). Upon inspection of the extracted ion chromatogram of the m/z 482.124, which is available as a direct link out in the molecular networking results, we could see the abundance difference is evident on the MS1 level in the XIC and the box-plot in panel b. Upon inspection of the XIC and corresponding MS/MS spectra of m/z 482.124, it became apparent that yersiniabactin is present as chromatographically separated isomers, most likely diastereomers with identical MS/MS spectra. Hence classical molecular networking (panel a) that makes use of MS/MS clustering, collapses the spectra to one node<a href="https://www.zotero.org/google-docs/?ZnfcHG">8</a>. In order to resolve the chromatographically separated features, we reanalyzed the data via feature-based molecular networking using MZmine and GNPS<a href="https://www.zotero.org/google-docs/?PFgpMD">9</a>, shown in panel c. In the FBMN results, the two yersiniabactin isomers (m/z 482.124) remain as individual nodes in the network. In summary, this is an example of the benefit of direct XIC inspection of classic molecular networking results to verify quantitative results based on MS1 level as well as control of chromatographic peak shape and eventual artifacts such as split peaks or collapsed MS/MS spectra from isomers.</p>
<p>Classic molecular Networking <a href="https://gnps.ucsd.edu/ProteoSAFe/result.jsp?view=network_displayer&amp;componentindex=76&amp;highlight_node=1509&amp;task=77c82c19571a49dc87e832a0789a2264#%7B%7D">link</a></p>
<p>GNPS Dashboard for Yersiniabactin <a href="https://gnps-lcms.ucsd.edu/?xicmz=482.124&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=ppm&amp;xic_rt_window=&amp;xic_norm=False&amp;xic_file_grouping=MZ&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3#%7B%22usi%22%3A%20%22mzspec%3AGNPS%3ATASK-77c82c19571a49dc87e832a0789a2264-f.MSV000086206/peak/mzml/S_N1.mzML%3Ascan%3A1%5Cnmzspec%3AGNPS%3ATASK-77c82c19571a49dc87e832a0789a2264-f.MSV000086206/peak/mzml/S_N2.mzML%3Ascan%3A1%5Cnmzspec%3AGNPS%3ATASK-77c82c19571a49dc87e832a0789a2264-f.MSV000086206/peak/mzml/S_N3.mzML%3Ascan%3A1%22%2C%20%22usi2%22%3A%20%22mzspec%3AGNPS%3ATASK-77c82c19571a49dc87e832a0789a2264-f.MSV000086206/peak/mzml/S_P1.mzML%3Ascan%3A1%5Cnmzspec%3AGNPS%3ATASK-77c82c19571a49dc87e832a0789a2264-f.MSV000086206/peak/mzml/S_P2.mzML%3Ascan%3A1%5Cnmzspec%3AGNPS%3ATASK-77c82c19571a49dc87e832a0789a2264-f.MSV000086206/peak/mzml/S_P3.mzML%3Ascan%3A1%22%7D">link</a></p>
<p>Feature-based molecular networking <a href="https://gnps.ucsd.edu/ProteoSAFe/result.jsp?view=network_displayer&amp;highlight_node=2&amp;componentindex=60&amp;task=5844c6bea85442b0a998f4d558caa310#%7B%7D">link</a></p>
<p><img src="img/dashboard_use_cases/SI_Figure_2.png" width="600"/></p>
<p><strong>Supplemental Figure 2: Mass spectrometry data visualization and linking to molecular networks from yersiniabactin from E. coli nissle extracts.</strong> Panel a shows the MS/MS molecular networking results from an <em>E. coli</em> nissle wild type and NRPS knock-out strain, indicating the presence of yersiniabactin (m/z 482.124) and its derivatives only in the wild type. Panel b shows the XIC visualization and peak area difference between the WT and KO group as box-plot through the GNPS Dashboard. Panel c shows the feature-based molecular networking results from the same data in which the yersiniabactin isomers (m/z 482.124) are maintained as individual nodes.</p>
<h1 id="use-case-2-quick-data-analysis-of-ms1-single-quad-data"><strong>Use Case 2: Quick data analysis of MS1 single quad data</strong><a class="headerlink" href="#use-case-2-quick-data-analysis-of-ms1-single-quad-data" title="Permanent link">¶</a></h1>
<h2 id="dale-a-cummings-jr-phd-student-biological-chemistry"><strong>Dale A. Cummings Jr - PhD Student - Biological Chemistry</strong><a class="headerlink" href="#dale-a-cummings-jr-phd-student-biological-chemistry" title="Permanent link">¶</a></h2>
<h2 id="aaron-w-puri-phd-assistant-professor-biological-chemistry"><strong>Aaron W. Puri PhD - Assistant Professor - Biological Chemistry</strong><a class="headerlink" href="#aaron-w-puri-phd-assistant-professor-biological-chemistry" title="Permanent link">¶</a></h2>
<p>While many labs now have access to tandem mass spectrometers and can therefore acquire data for MS2-based molecular networking, routine analysis is often still performed on unit resolution MS1 mass analyzers, such as single quadrupoles. However, proprietary software often makes it difficult to rapidly analyze and share these types of data. To address this we have added functionality for analyzing MS1 spectra in the GNPS Dashboard specifically in netCDF format, which is commonly used by single quadrupole LC-MS systems.</p>
<p>As an example use case, we analyzed the metabolome of the methylotrophic organism <em>Methylorubrum extorquens</em> PA1 (PA1) using an Agilent 6120 single quadrupole LC-MS. Methylotrophs have previously been reported to biosynthesize different natural products when grown on single versus multicarbon substrates<a href="https://www.zotero.org/google-docs/?pt4D4o">10,11</a>. We grew PA1 in minimal media containing either methanol or succinate as the sole source of carbon and energy, and subsequently extracted the supernatant with acidified ethyl acetate. We then analyzed this extract by LC-MS and examined the results using the GNPS Dashboard.</p>
<p>One difference in the metabolome under these two growth conditions appears to be the production of quorum sensing signals. PA1 possesses a single LuxI-family acyl-homoserine lactone (acyl-HSL) synthase with 100% amino acid identity to MlaI<a href="https://www.zotero.org/google-docs/?EKQz7P">10</a>. In the closely related strain <em>Methylorubrum extorquens</em> AM1 (AM1), Vorholt and coworkers demonstrated that MlaI produces the long-chain quorum-sensing signals 7_Z_-C14-HSL and 2_E_,7_Z_-C14-HSL when grown on methanol but not succinate<a href="https://www.zotero.org/google-docs/?WNjXLU">11</a>. We used the GNPS Dashboard to generate XICs for the protonated masses of these signals (<em>m/z</em> 308.1 for 2_E_,7_Z_-C14-HSL, and <em>m/z</em> 310.1 for 7_Z_-C14-HSL) (Figure S2a). These masses are apparent at retention times of 25.9 minutes for <em>m/z</em> 308 and 26.3 minutes for <em>m/z</em> 310 in the methanol-grown culture, but not in the succinate culture, which can be quantified by comparing the automatically generated peak integrations in the two conditions (Figure S2b). These features therefore likely correspond to quorum-sensing signals 7Z-C14-HSL and 2E,7Z-HSL, and demonstrate how the GNPS Dashboard can be used to rapidly analyze MS1 data.</p>
<p>GNPS Dashboard <a href="https://gnps-lcms.ucsd.edu/?xicmz=308.1%3B+310.1&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.25&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=25-27&amp;xic_norm=False&amp;xic_file_grouping=GROUP&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=MS1%3ASPECTRUM_1407&amp;show_lcms_2nd_map=True&amp;map_plot_zoom=%7B%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=f8635a981a5546eaaaf4dee795a1e95c#%7B%22usi%22%3A%20%22mzspec%3AMSV000086834%3Araw/20210210%20GNPS%20LMCS/PA1MeOH1.aia.CDF%5Cnmzspec%3AMSV000086834%3Araw/20210210%20GNPS%20LMCS/PA1MeOH2.aia.CDF%5Cnmzspec%3AMSV000086834%3Araw/20210210%20GNPS%20LMCS/PA1MeOH3.aia.CDF%22%2C%20%22usi2%22%3A%20%22mzspec%3AMSV000086834%3Araw/20210210%20GNPS%20LMCS/PA1SUC1.aia.CDF%5Cnmzspec%3AMSV000086834%3Araw/20210210%20GNPS%20LMCS/PA1SUC2.aia.CDF%5Cnmzspec%3AMSV000086834%3Araw/20210210%20GNPS%20LMCS/PA1SUC3.aia.CDF%22%7D">link</a>.</p>
<p><img src="img/dashboard_use_cases/SI_Figure_3.png" width="600"/></p>
<p><strong>Supplemental Figure 3: Quick data analysis of MS1 single quad data</strong></p>
<p>Identification of differences in the extracellular metabolomes of the methylotrophic bacterium_Methylorubrum extorquens_ PA1 (PA1) grown on different carbon and energy sources. (a) XICs for protonated masses corresponding to quorum-sensing signals produced by PA1. (b) Box plots of integrated peak areas for the same XICs from a retention time window of 25-27 minutes. Data are from three separate cultures for each growth condition.</p>
<h1 id="use-case-3-collaborative-exploration-of-lc-ms-data"><strong>Use Case 3: Collaborative Exploration of LC-MS Data</strong><a class="headerlink" href="#use-case-3-collaborative-exploration-of-lc-ms-data" title="Permanent link">¶</a></h1>
<h2 id="mingxun-wang-phd-post-doctoral-scholar-computer-science-and-bioinformatics"><strong>Mingxun Wang PhD - Post-Doctoral Scholar - Computer Science and Bioinformatics</strong><a class="headerlink" href="#mingxun-wang-phd-post-doctoral-scholar-computer-science-and-bioinformatics" title="Permanent link">¶</a></h2>
<h2 id="daniel-petras-phd-post-doctoral-scholar-analytical-chemistry-and-chemical-ecology_1"><strong>Daniel Petras PhD - Post-Doctoral Scholar - Analytical Chemistry and Chemical Ecology</strong><a class="headerlink" href="#daniel-petras-phd-post-doctoral-scholar-analytical-chemistry-and-chemical-ecology_1" title="Permanent link">¶</a></h2>
<p>The exploration of non-targeted mass spectrometry is often a long and collaborative process, e.g. between collaborating researchers or between mentors and mentees. In many instances, it is difficult to collaboratively explore and share results when it is impractical or impossible to physically sit together, for example during a global pandemic. With the GNPS Dashboard, it is possible to share exactly the settings and visualization that a researcher is seeing on their own screen either by our collaborative sync feature or by simply copying and pasting a url. This sharing ability enables others to build upon this visualization and reshare to collaboratively enhance and dive deeper into the data, whether in joint data analysis sessions with collaborators, reports, or peer-reviewed publications. One particular example and collaborative visualization dialog can be seen below with quality control (QC) sample of a mixture of 6 standard compounds ( <strong>SI Figure 4</strong> ):</p>
<p>Researcher A starts by initial data loading of the QC data- <a href="https://gnps-lcms.ucsd.edu/?xicmz=&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=&amp;xic_norm=False&amp;xic_file_grouping=FILE&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3#%7B%22usi%22%3A%20%22mzspec%3AMSV000085852%3AQC_0%22%2C%20%22usi2%22%3A%20%22%22%7D">link</a>.</p>
<p>Next, Researcher B has the information of exact masses and adds XIC for the 6 standard compound used (m/z 271.0315; 278.1902; 279.0909; 285.0205; 311.0805; 314.1381) - <a href="https://gnps-lcms.ucsd.edu/?usi=mzspec%3AMSV000085852%3AQC_0&amp;xicmz=271.0315%3B278.1902%3B279.0909%3B285.0205%3B311.0805%3B314.1381&amp;xic_tolerance=0.5&amp;xic_norm=No&amp;show_ms2_markers=1&amp;ms2_identifier=">link</a>.</p>
<p>To verify the ID of one of the standards through its isotope pattern, Researcher A then visualizes of MS1 spectrum for the m/z 314.1381 standard - <a href="https://gnps-lcms.ucsd.edu/?xicmz=271.0315%3B278.1902%3B279.0909%3B285.0205%3B311.0805%3B314.1381&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=&amp;xic_norm=False&amp;xic_file_grouping=FILE&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=MS1%3A3347&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%22xaxis.autorange%22%3A+true%2C+%22xaxis.showspikes%22%3A+false%2C+%22yaxis.autorange%22%3A+true%2C+%22yaxis.showspikes%22%3A+false%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3#%7B%22usi%22%3A%20%22mzspec%3AMSV000085852%3AQC_0%22%2C%20%22usi2%22%3A%20%22%22%7D">link</a>.</p>
<p>Finally, Researcher B inspects co-eluting features by zooming in on 2D LC-MS heatmap for elution of the m/z 314.1381 standard - <a href="https://gnps-lcms.ucsd.edu/?xicmz=271.0315%3B278.1902%3B279.0909%3B285.0205%3B311.0805%3B314.1381&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=&amp;xic_norm=False&amp;xic_file_grouping=FILE&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=MS1%3A3347&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%22xaxis.range%5B0%5D%22%3A+5.533460597113934%2C+%22xaxis.range%5B1%5D%22%3A+5.859496359439452%2C+%22yaxis.range%5B0%5D%22%3A+148.29089374509707%2C+%22yaxis.range%5B1%5D%22%3A+368.99624703728165%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3#%7B%22usi%22%3A%20%22mzspec%3AMSV000085852%3AQC_0%22%2C%20%22usi2%22%3A%20%22%22%7D">link</a>.</p>
<p>The final url can then be shared with others in the team and used to protocol the instrument performance for this particular experiment.</p>
<p><img src="img/dashboard_use_cases/SI_Figure_4.png" width="600"/></p>
<p><strong>Supplemental Figure 4:</strong> In panel a we see an example communication between two users to collaboratively analyze a mass spectrometry sample. This communication can be done purely remotely by sharing URLs or the synchronization feature in GNPS Dashboard. In panel b we can see how the discrete steps line up with updating different parts of the dashboard as the analysis progresses.</p>
<h1 id="use-case-4-inspection-and-validation-of-published-results-based-upon-public-data-sharinginspection"><strong>Use Case 4: Inspection and validation of published results based upon public data sharing/inspection</strong><a class="headerlink" href="#use-case-4-inspection-and-validation-of-published-results-based-upon-public-data-sharinginspection" title="Permanent link">¶</a></h1>
<h2 id="mingxun-wang-phd-post-doctoral-scholar-computer-science-and-bioinformatics_1"><strong>Mingxun Wang PhD - Post-Doctoral Scholar - Computer Science and Bioinformatics</strong><a class="headerlink" href="#mingxun-wang-phd-post-doctoral-scholar-computer-science-and-bioinformatics_1" title="Permanent link">¶</a></h2>
<h2 id="daniel-petras-phd-post-doctoral-scholar-analytical-chemistry-and-chemical-ecology_2"><strong>Daniel Petras PhD - Post-Doctoral Scholar - Analytical Chemistry and Chemical Ecology</strong><a class="headerlink" href="#daniel-petras-phd-post-doctoral-scholar-analytical-chemistry-and-chemical-ecology_2" title="Permanent link">¶</a></h2>
<p>Data transparency and verification of software-derived results have become key aspects in recent years in proteomics and metabolomics publications thanks to policy and cultural shifts within these fields. It has even become the standard for many scientific communities where publishers require data sharing for papers to be accepted for publication. While there are differences in guidelines with regards to data content (e.g. raw and processed data), as well as accepted repositories (e.g. ProteoXchange<a href="https://www.zotero.org/google-docs/?m4f2IF">12</a> initiative or Nature Publishing Group guidelines), there still exist multiple barriers for reviewers and readers of scientific articles to make use of the shared data for inspection or reproduction of the published results.</p>
<p>The GNPS Dashboard in combination with the GNPS Dataset Explorer enables direct visualization of public data from the most common mass spec data repositories (PRIDE<a href="https://www.zotero.org/google-docs/?cHbAJd">13</a>, MassIVE<a href="https://www.zotero.org/google-docs/?QcCaat">14</a>/GNPS<a href="https://www.zotero.org/google-docs/?BSP0w4">8</a>, MetaboLights<a href="https://www.zotero.org/google-docs/?NBiNql">15</a>, Metabolomics Workbench<a href="https://www.zotero.org/google-docs/?rosPTf">16</a>) in a vendor-independent fashion. The GNPS Dashboard removes the need to install software and to download MS data locally and enables inspection of and reproduction of peak visualization and integration as well as downstream data analysis in the GNPS environment within a few clicks in the web browser.</p>
<p>Between October 2020 and January 2021, we inspected 10 LC-MS datasets from peer-reviewed articles as well as preprints in the fields of natural product research, metabolomics, and proteomics that were shared through the social media platform Twitter, which is quickly becoming a fundamental part of scientific literature sharing and discussions. In all cases, we inspected the raw MS/MS data using the GNPS Dashboard and created extracted ion chromatograms (XIC) for the masses of the main compounds discussed in the manuscripts. Depending on the experimental design of each study we also reproduced simple quantitative analysis by integrating the XICs and comparing the peak areas of the experimental groups as box plots directly in the GNPS Dashboard. Upon positive validation, we provided direct feedback through the comment and retweet function in the social media platform, including confirmation of the presence of described masses, MS/MS matching, and quantitative differences between sample groups ( <strong>SI Figure 5</strong> ), as well as comparison to other public data through repository scale analysis ( <strong>SI Figure 6</strong> ), and recommendations for additional data analysis options such as molecular networking in the case of one of the preprints ( <strong>SI Figure 7</strong> ).</p>
<p><strong>Example Tweets List</strong></p>
<ol>
<li><a href="https://twitter.com/mingxunwang/status/1359643001211920386">Tweet</a>- <a href="https://gnps-lcms.ucsd.edu/?xicmz=547.3815&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.05&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=1.2&amp;xic_norm=False&amp;xic_file_grouping=MZ&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=4d57bd4d0f034b578e89d559c37e3c5e#%7B%22usi%22%3A%20%22mzspec%3AMSV000086584%3Araw/KRD174_ISP2.mzXML%5Cnmzspec%3AMSV000086584%3Araw/KRD174_ISP3.mzXML%5Cnmzspec%3AMSV000086584%3Araw/KRD174_M1.mzXML%5Cnmzspec%3AMSV000086584%3Araw/KRD174_TSB.mzXML%22%2C%20%22usi2%22%3A%20%22%22%7D">Dashboard</a> - Comparative Metabologenomics Analysis of Polar Actinomycetes</li>
<li><a href="https://twitter.com/mingxunwang/status/1357879902385491972">Tweet</a> - <a href="https://gnps-lcms.ucsd.edu/?xicmz=&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=&amp;xic_norm=False&amp;xic_file_grouping=FILE&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=BPI&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=MZmine2&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=21d00bbc259b416a817b7350fc7bfb62#%7B%22usi%22%3A%20%22mzspec%3AMSV000086453%3Accms_peak/Raw%20files/940-1C.mzML%5Cn%22%2C%20%22usi2%22%3A%20%22%22%7D">Dashboard</a>- Exploring the Chemical Space of Macro- and Micro-Algae Using Comparative Metabolomics</li>
<li><a href="https://twitter.com/mingxunwang/status/1356757386975866880">Tweet</a>- <a href="https://gnps-lcms.ucsd.edu/?xicmz=1255.6630&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.05&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=&amp;xic_norm=False&amp;xic_file_grouping=FILE&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=MS2%3A4335&amp;show_lcms_2nd_map=True&amp;map_plot_zoom=%7B%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=BPI&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=#%7B%22usi%22%3A%20%22mzspec%3AMSV000085974%3Accms_peak/KMRB16041505_neg.mzML%22%2C%20%22usi2%22%3A%20%22mzspec%3AMSV000085974%3Accms_peak/CV_FruitingBodies_Extract.mzML%22%7D">Dashboard</a>- Species Prioritization Based on Spectral Dissimilarity: A Case Study of Polyporoid Fungal Species</li>
<li><a href="https://twitter.com/mingxunwang/status/1354508639713718272">Tweet</a>- <a href="https://gnps-lcms.ucsd.edu/?xicmz=610.3%3B596.31%3B612.31&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=&amp;xic_norm=False&amp;xic_file_grouping=GROUP&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=#%7B%22usi%22%3A%20%22mzspec%3AMSV000085376%3Apeak/mod-mzXML/6030_6034_mod.mzXML%5Cnmzspec%3AMSV000085376%3Apeak/mod-mzXML/6031_6035_mod.mzXML%5Cnmzspec%3AMSV000085376%3Apeak/mod-mzXML/6029_6033_mod.mzXML%5Cnmzspec%3AMSV000085376%3Apeak/mod-mzXML/6028_6032_mod.mzXML%22%2C%20%22usi2%22%3A%20%22mzspec%3AMSV000085376%3Apeak/mod-mzXML/5425_5426_mod.mzXML%5Cnmzspec%3AMSV000085376%3Apeak/mod-mzXML/5430_5429_mod.mzXML%22%7D">Dashboard</a>- Planomonospora: A Metabolomics Perspective on an Underexplored Actinobacteria Genus</li>
<li><a href="https://twitter.com/mingxunwang/status/1352777404700184576">Tweet</a>- <a href="https://gnps-lcms.ucsd.edu/?xicmz=496.3397&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=&amp;xic_norm=False&amp;xic_file_grouping=FILE&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=True&amp;map_plot_zoom=%7B%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3#%7B%22usi%22%3A%20%22mzspec%3AMSV000086729%3Araw/raw/Identification/Polar_QEx_POS_SAT_ob_1.raw%22%2C%20%22usi2%22%3A%20%22mzspec%3AMSV000086729%3Araw/raw/Quantification/Polar_Quant_POS_SAT_ob_1.raw%22%7D">Dashboard</a>- AdipoAtlas: A Reference Lipidome for Human White Adipose Tissue</li>
<li><a href="https://twitter.com/mingxunwang/status/1352389864465915904">Tweet</a>- <a href="https://gnps-lcms.ucsd.edu/?xicmz=1327.7&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=1&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=&amp;xic_norm=False&amp;xic_file_grouping=FILE&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%22xaxis.range%5B0%5D%22%3A+-2.879949588410682%2C+%22xaxis.range%5B1%5D%22%3A+70.31716098817184%2C+%22yaxis.range%5B0%5D%22%3A+924.2868147514461%2C+%22yaxis.range%5B1%5D%22%3A+1809.8933588174573%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3#%7B%22usi%22%3A%20%22mzspec%3AMSV000081885%3Araw/Raw%20Files/51_Naja_nigricollis_Liverpool_Nigeria_red_1.raw%5Cnmzspec%3AMSV000081885%3Araw/Raw%20Files/51_Naja_nigricollis_Liverpool_Nigeria_red_2.raw%22%2C%20%22usi2%22%3A%20%22%22%7D">Dashboard</a>- Convergent evolution of pain-inducing defensive venom components in spitting cobras</li>
<li><a href="https://twitter.com/mingxunwang/status/1334665271131262979">Tweet</a>- <a href="https://gnps-lcms.ucsd.edu/?usi=mzspec%3AMSV000085618%3Accms_peak%2FBLANK_P2-F-1_01_13263.mzML%3Ascan%3A1%0Amzspec%3AMSV000085618%3Accms_peak%2FED103_P1-D-8_01_3762.mzML%3Ascan%3A1%0Amzspec%3AMSV000085618%3Accms_peak%2FED104_P1-D-9_01_3763.mzML%3Ascan%3A1%0Amzspec%3AMSV000085618%3Accms_peak%2FED105_P1-A-1_01_3781.mzML%3Ascan%3A1%0A&amp;usi2=mzspec%3AMSV000085618%3Accms_peak%2FED24_P1-B-2_01_2171.mzML%3Ascan%3A1%0Amzspec%3AMSV000085618%3Accms_peak%2FED25_P1-B-3_01_2172.mzML%3Ascan%3A1%0Amzspec%3AMSV000085618%3Accms_peak%2FED26_P1-B-4_01_2173.mzML%3Ascan%3A1%0Amzspec%3AMSV000085618%3Accms_peak%2FED36_P1-C-3_01_2181.mzML%3Ascan%3A1%0Amzspec%3AMSV000085618%3Accms_peak%2FED37_P1-C-4_01_2182.mzML%3Ascan%3A1%0Amzspec%3AMSV000085618%3Accms_peak%2FED38_P1-C-5_01_2183.mzML%3Ascan%3A1%0Amzspec%3AMSV000085618%3Accms_peak%2FSJ-123_P1-A-6_01_13658.mzML%3Ascan%3A1&amp;xicmz=799.437&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=ppm&amp;xic_rt_window=2.5&amp;xic_norm=False&amp;xic_file_grouping=MZ&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%22autosize%22%3A+true%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC">Dashboard</a>- Iron-meditated fungal starvation by lupine rhizosphere-associated and extremotolerant Streptomyces sp. S29 desferrioxamine production</li>
<li><a href="https://twitter.com/mingxunwang/status/1327321800485654530">Tweet</a>- <a href="https://gnps-lcms.ucsd.edu/?usi=mzspec%3AMSV000085070%3Apeak%2F12_alone.mzXML%3Ascan%3A1%0Amzspec%3AMSV000085070%3Apeak%2F1655_alone.mzXML%3Ascan%3A1%0Amzspec%3AMSV000085070%3Apeak%2FCCA.mzXML%3Ascan%3A1%0Amzspec%3AMSV000085070%3Apeak%2FCan_alone.mzXML%3Ascan%3A1%0Amzspec%3AMSV000085070%3Apeak%2FDeb_alone.mzXML%3Ascan%3A1%0Amzspec%3AMSV000085070%3Apeak%2FFus_alone.mzXML%3Ascan%3A1%0Amzspec%3AMSV000085070%3Apeak%2FJB370_alone.mzXML%3Ascan%3A1%0Amzspec%3AMSV000085070%3Apeak%2FRS17_alone.mzXML%3Ascan%3A1%0Amzspec%3AMSV000085070%3Apeak%2FSAM_alone.mzXML%3Ascan%3A1%0Amzspec%3AMSV000085070%3Apeak%2Fmeoh_blank.mzXML%3Ascan%3A1%0Amzspec%3AMSV000085070%3Apeak%2Fmeoh_blank_end.mzXML%3Ascan%3A1&amp;usi2=&amp;xicmz=965.55400&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.05&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=5.52&amp;xic_norm=False&amp;xic_file_grouping=MZ&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%22autosize%22%3A+true%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC">Dashboard</a>- Bacterial–fungal interactions revealed by genome-wide analysis of bacterial mutant fitness</li>
<li><a href="https://twitter.com/mingxunwang/status/1317243139212550145">Tweet</a>- <a href="https://gnps-lcms.ucsd.edu/?usi=mzspec%3AMSV000086092%3A190308JSH05_MS1-neg.mzXML%3Ascan%3A1%0A&amp;usi2=mzspec%3AMSV000086092%3A190312JSH01-10ml_neg_MS2.mzXML%3Ascan%3A1&amp;xicmz=399.18875&amp;xic_tolerance=0.05&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=5.15&amp;xic_norm=False&amp;xic_file_grouping=MZ&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%22autosize%22%3A+true%7D&amp;tic_option=TIC">Dashboard</a>- Deep Interrogation of Metabolism Using a Pathway-Targeted Click-Chemistry Approach</li>
<li><a href="https://twitter.com/mingxunwang/status/1305652815952445441">Tweet</a>- <a href="https://gnps-lcms.ucsd.edu/?usi=mzspec%3AMSV000079843%3AJD_06232014_sample1-A%3Ascan%3A1%0Amzspec%3AMSV000079843%3AJD_06232014_sample1_B%3Ascan%3A1%0Amzspec%3AMSV000079843%3AJD_06232014_sample1_C%3Ascan%3A1&amp;usi2=mzspec%3AMSV000079843%3AJD_06232014_sample2_A%3Ascan%3A1%0Amzspec%3AMSV000079843%3AJD_06232014_sample2_B%3Ascan%3A1%0Amzspec%3AMSV000079843%3AJD_06232014_sample2_C%3Ascan%3A1&amp;xicmz=729.365&amp;xic_tolerance=0.05&amp;xic_rt_window=53-56&amp;xic_norm=False&amp;xic_file_grouping=MZ&amp;show_ms2_markers=True&amp;ms2_identifier=None">Dashboard</a>- MassIVE.quant: a community resource of quantitative mass spectrometry-based proteomics datasets</li>
</ol>
<p><img src="img/dashboard_use_cases/SI_Figure_5.png" width="600"/></p>
<p><strong>Supplemental Figure 5 -</strong> Inspection of publicly shared mass spectrometry data from Deep Interrogation of Metabolism Using a Pathway-Targeted Click-Chemistry Approach</p>
<p><img src="img/dashboard_use_cases/SI_Figure_6.png" width="600"/></p>
<p><strong>Supplemental Figure 6:</strong> Inspection of publicly shared mass spectrometry data from Bacterial–fungal interactions revealed by genome-wide analysis of bacterial mutant fitness</p>
<p><img src="img/dashboard_use_cases/SI_Figure_7.png" width="600"/></p>
<p><strong>Supplemental Figure 7:</strong> Inspection of publicly shared mass spectrometry data of AdipoAtlas: A Reference Lipidome for Human White Adipose Tissue</p>
<h1 id="use-case-5a-gnps-dashboard-usage-in-teaching-setting"><strong>Use Case 5A: GNPS Dashboard usage in Teaching Setting</strong><a class="headerlink" href="#use-case-5a-gnps-dashboard-usage-in-teaching-setting" title="Permanent link">¶</a></h1>
<h2 id="katherine-maloney-phd-professor-natural-product-chemistry"><strong>Katherine Maloney PhD - Professor - Natural Product Chemistry,</strong><a class="headerlink" href="#katherine-maloney-phd-professor-natural-product-chemistry" title="Permanent link">¶</a></h2>
<h2 id="allegra-aron-phd-post-doctoral-scholar-biological-chemistry"><strong>Allegra Aron PhD - Post-Doctoral Scholar - Biological Chemistry</strong><a class="headerlink" href="#allegra-aron-phd-post-doctoral-scholar-biological-chemistry" title="Permanent link">¶</a></h2>
<p>GNPS Dashboard proved useful in a teaching setting, as we developed a virtual, interactive laboratory exercise for undergraduate students to explore LC-MS data generated by the instructor, as part of Tiny Earth, an initiative to crowdsource antibiotic discovery in a Course-based Undergraduate Research Experience (CURE) GNPS Dashboard was an ideal tool for this interactive (or a virtually interactive) lab because software installation was not necessary and software licenses did not need to be obtained, both challenging obstacles when working with a class. One option frequently employed is the use of a virtual desktop that has the requisite software installed. For example, we used such a system for the processing of NMR data in the Tiny Earth<a href="https://www.zotero.org/google-docs/?6Hjvxl">17</a> course. However, students found the virtual desktop to be clunky and error-prone. The web-based interface of the GNPS Dashboard proved to be markedly more user-friendly, giving reproducible and shareable results. Moreover, the instructor can equip students with path links to data for exploration in GNPS Dashboard or can take students through data exploration using the synchronization feature.</p>
<p>The interactive lab exercise making use of the GNPS Dashboard was part of the Tiny Earth Chemistry pilot course (CHEM 1003L) developed and taught by Katherine Maloney and Allegra Aron at Point Loma Nazarene University in San Diego, CA. The goal of the exercise was to teach students about LC-MS experiments and data structure by providing hands-on experience with LC-MS data generated from pure antibiotic standards and antibiotics in culture extracts. Briefly, the exercise asks students to determine the exact mass of antibiotics in question; once this has been completed students explore the LC-MS of a mixture of antibiotics using the LC-MS Viewer and a path link provided in the lab protocol. Students explore the TIC plot to load the MS1 spectrum for each antibiotic ( <strong>SI Figure 8</strong> ). Next, students explore the XIC for each antibiotic then hone in to inspect the tandem MS (or MS2) spectrum of each antibiotic, trying to annotate fragment peaks with particular molecular fragments. Finally, students are given another path link for the LC-MS of culture extracts. Students are tasked with first looking through the TIC and XIC plots; from here, students dive into further exploration of compounds of interest in GNPS, including the use of Mass Spectrometry Search Tool (MASST)<a href="https://www.zotero.org/google-docs/?d6YsNM">18</a>. The full laboratory exercise is provided as <strong>SI Data 1</strong>.</p>
<p>GNPS Dashboard - <a href="https://gnps-lcms.ucsd.edu/?xic_mz=321.005%3B443.146%3B333.0915%3B696.3025&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=&amp;xic_norm=False&amp;xic_file_grouping=FILE&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=6b74adb2b9cf47e490ad4c5b035eb4ec&amp;comment=&amp;map_plot_color_scale=Hot_r&amp;map_plot_quantization_level=Medium#%7B%22usi%22%3A%20%22mzspec%3AMSV000086079%3Aantibiotics_1ug_ml_neg_20200902112656%3Ascan%3A1000%22%2C%20%22usi2%22%3A%20%22%22%7D">Link</a></p>
<p><img src="img/dashboard_use_cases/SI_Figure_8.png" width="600"/></p>
<p><strong>Supplemental Figure 8:</strong> Students use the GNPS Dashboard (<a href="https://gnps-lcms.ucsd.edu/?xic_mz=321.005%3B443.146%3B333.0915%3B696.3025&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=&amp;xic_norm=False&amp;xic_file_grouping=FILE&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=6b74adb2b9cf47e490ad4c5b035eb4ec&amp;comment=&amp;map_plot_color_scale=Hot_r&amp;map_plot_quantization_level=Medium#%7B%22usi%22%3A%20%22mzspec%3AMSV000086079%3Aantibiotics_1ug_ml_neg_20200902112656%3Ascan%3A1000%22%2C%20%22usi2%22%3A%20%22%22%7D">link</a>) to visualize XIC plots of antibiotics of interest, which include chloramphenicol ([H-H+], <em>m/z</em> 321.005), tetracycline ([H-H+], <em>m/z</em> 443.146), penicillin G ([H-H+], <em>m/z</em> 333.0915), and rifamycin ([H-H+], <em>m/z</em> 696.3025).</p>
<h1 id="use-case-5b-gnps-dashboard-for-targeted-ms-in-a-teaching-setting"><strong>Use Case 5B: GNPS Dashboard for Targeted MS in a Teaching Setting</strong><a class="headerlink" href="#use-case-5b-gnps-dashboard-for-targeted-ms-in-a-teaching-setting" title="Permanent link">¶</a></h1>
<h1 id="michael-marty-phd-assistant-professor-analytical-chemistry"><strong>Michael Marty PhD – Assistant Professor – Analytical Chemistry</strong><a class="headerlink" href="#michael-marty-phd-assistant-professor-analytical-chemistry" title="Permanent link">¶</a></h1>
<h1 id="deirdre-belle-oudry-phd-senior-lecturer-analytical-chemistry"><strong>Deirdre Belle-Oudry PhD – Senior Lecturer – Analytical Chemistry</strong><a class="headerlink" href="#deirdre-belle-oudry-phd-senior-lecturer-analytical-chemistry" title="Permanent link">¶</a></h1>
<p>In addition to analyzing LC-MS data generated by the instructor or in public data sets, the GNPS Dashboard is also useful for undergraduate students analyzing targeted data for quantitative analysis. For example, the GNPS Dashboard was used as the data analysis tool for two new labs in the <em>Quantitative Analysis Lab</em> (CHEM 326) course at the University of Arizona in Tucson, AZ developed by Michael Marty and Deirdre Belle-Oudry. Earlier in the semester, the students used HPLC with UV detection to quantify aspirin, acetaminophen, and caffeine<a href="https://www.zotero.org/google-docs/?GugTMT">19</a>. However, additional peaks can be present in the UV chromatogram due to hydrolysis of acetyl-salicylic acid into salicylic acid. In the first LC/MS lab, students use LC/MS/MS with a triple quadrupole mass spectrometer to first identify the degraded product from product ion scans and then use single reaction monitoring (SRM) scans for targeted quantitation. After collecting the data and converting it to .mzML format using MSConvert<a href="https://www.zotero.org/google-docs/?0I2w5d">3</a> students use the GNPS Dashboard to view the MS1 and MS2 data to assign the peaks. Example data is posted to MassIVE MSV000087058 to allow students to access training data from a direct link. Next, they use the Dashboard to view and integrate the SRM chromatograms. Because the LC/MS/MS system has a UV detector, students can also load the UV chromatograms into the GNPS Dashboard to compare retention times from both detection methods. In the second lab, the students apply the skills they develop in the first lab to quantify amino acids in food samples based on targeted SRM scans. Overall, the GNPS Dashboard provides a fast and user-friendly method to quantify targeted SRM scans and view MS data. The web-based interface provides easy and inclusive access for students with all different types of computers.</p>
<p>Tutorial Video - <a href="https://www.youtube.com/watch?v=tT5b1IIJCE8">https://www.youtube.com/watch?v=tT5b1IIJCE8</a></p>
<p>GNPS Dashboard - <a href="https://gnps-lcms.ucsd.edu/?xic_mz=&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=&amp;xic_norm=False&amp;xic_file_grouping=FILE&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=ca12499c5aa54d2cbd98551634fd62b0&amp;chromatogram_options=%5B%22SRM+SIC+152%2C111%22%5D&amp;comment=&amp;map_plot_color_scale=Hot_r&amp;map_plot_quantization_level=Medium#%7B%22usi%22%3A%20%22mzspec%3AMSV000087058%3Apeak/peak/std1_022721.mzML%5Cnmzspec%3AMSV000087058%3Apeak/peak/std2_022721.mzML%5Cnmzspec%3AMSV000087058%3Apeak/peak/std3_022721.mzML%5Cnmzspec%3AMSV000087058%3Apeak/peak/std4_022721.mzML%22%2C%20%22usi2%22%3A%20%22%22%7D">Link</a></p>
<h1 id="use-case-6-data-visualization-for-quantitative-proteomics-results"><strong>Use Case 6: Data Visualization for Quantitative Proteomics Results</strong><a class="headerlink" href="#use-case-6-data-visualization-for-quantitative-proteomics-results" title="Permanent link">¶</a></h1>
<h2 id="ben-pullman-phd-student-computational-mass-spectrometry"><strong>Ben Pullman - PhD Student - Computational Mass Spectrometry</strong><a class="headerlink" href="#ben-pullman-phd-student-computational-mass-spectrometry" title="Permanent link">¶</a></h2>
<h2 id="nuno-bandeira-phd-professor-computational-mass-spectrometry"><strong>Nuno Bandeira PhD - Professor - Computational Mass Spectrometry</strong><a class="headerlink" href="#nuno-bandeira-phd-professor-computational-mass-spectrometry" title="Permanent link">¶</a></h2>
<p>MassIVE.quant<a href="https://www.zotero.org/google-docs/?AZNqlb">20</a> provides a platform for the sharing of quantitative re-analyses, from RAW data to features to differential expression (DE) analysis. The complete set of scripts and commands used to run the experiment enables reproducible data reanalysis and also facilitates altering parameters and algorithms used for slightly-modified future reanalyses. Incorporating the GNPS Dashboard provides a more interactive exploration of these stages of analysis and further facilitates investigating reported findings.</p>
<p>In the IPRG2015<a href="https://www.zotero.org/google-docs/?2ArJb5">21</a> study (MSV000079843/PXD015300), four samples are spiked into a consistent yeast background in six known, different concentrations. There are four conditions, each with a different known combination of the previous four spikes in samples (C1-C4), and each condition is run three times as technical replicates. The experiment design proves itself useful when considering quantifications pipelines, as we can assess if the differential expression found matches what is expected. In the reanalysis: <a href="https://massive.ucsd.edu/ProteoSAFe/reanalysis.jsp?task=855dcc3741ae42e9a6f6229c2a6eddab">RMSV000000249.18</a>, which uses MaxQuant<a href="https://www.zotero.org/google-docs/?KOUorz">22</a> for feature finding, almost all true positives (TP) are found, and the number of false positives (FP) is overall very low. However, there are still some FPs to investigate. sp|P07834|CDC4_YEAST is not a spiked in protein but is DE between two conditions in the experiment, C4 as compared to C2, and this expression is supported by only a single feature, though the feature does have consistent intensity across all three replicates in the sample ( <strong>SI Figure 9a</strong> , <a href="https://proteomics.ucsd.edu/ProteoSAFe/result.jsp?view=view_differential&amp;task=c3e158d9f77d4cc38cbd337598db6f89#%7B%22_dyn_%23Label_input%22%3A%22C4-C2%22%2C%22_dyn_%23issue_input%22%3A%22NA%22%2C%22_dyn_%23adj.pvalue_upperinput%22%3A%220.05%22%2C%22_dyn_%23Protein_input%22%3A%22sp%7CP07834%7CCDC4_YEAST%22%7D">Protein Differential Expression Results</a>). We can find the location of the features supporting the peptide by looking in the MaxQuant<a href="https://www.zotero.org/google-docs/?kfkeE6">22</a> analysis ( <strong>SI Figure 9b</strong> , <a href="https://proteomics.ucsd.edu/ProteoSAFe/result.jsp?task=870e0c5c6e0b433993a37858e78956c0&amp;view=view_spectra#%7B%22_dyn_%23Proteins_input%22%3A%22CDC4_YEAST%22%2C%22_dyn_%23Raw%20file_input%22%3A%22sample4%22%2C%22_dyn_%23Type_input%22%3A%22MULTI%22%7D">Peptide Differential Expression Results</a>).</p>
<p>To further examine the connection between the differential results and the raw data, the GNPS Dashboard enables us to examine the region where this peptide was detected to be, and inspect if there might be any other artifacts that might be influencing this feature intensity. We see that there's a feature for a different protein group that appears in exactly the same area and the first isotope of that feature overlaps completely with the monoisotopic <em>m/z</em> of the sp|P07834|CDC4_YEAST feature. Also generally it seems that the region around where the feature was acquired has many features ( <strong>SI Figure 9c</strong> , <a href="https://gnps-lcms.ucsd.edu/?xic_mz=432.25213528544%3B432.25116284419&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.05&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=26&amp;xic_norm=False&amp;xic_file_grouping=FILE&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=MS2%3A8576&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%22xaxis.range%5B0%5D%22%3A+26.071442063945774%2C+%22xaxis.range%5B1%5D%22%3A+26.295005228234334%2C+%22yaxis.range%5B0%5D%22%3A+427.87914733366677%2C+%22yaxis.range%5B1%5D%22%3A+435.0938398740948%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=mzspec%3AGNPS%3ATASK-870e0c5c6e0b433993a37858e78956c0-processed_output%2Fevidence.tsv&amp;overlay_mz=m%2Fz&amp;overlay_rt=Calibrated+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=Sequence&amp;overlay_filter_column=Raw+file&amp;overlay_filter_value=JD_06232014_sample4_C&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=26dc1db5f90b4a348ffd3475f9f19715&amp;comment=&amp;map_plot_color_scale=Hot_r&amp;map_plot_quantization_level=High#%7B%22usi%22%3A%20%22mzspec%3AMSV000079843%3AJD_06232014_sample4_C%22%2C%20%22usi2%22%3A%20%22%22%7D">GNPS Dashboard Link</a>).</p>
<p>Finally, to confirm, when looking at the MS2 mapped to the feature, we also see that most of the intensity for the MS2 being considered cannot be explained by the peptide which it is assigned, lending more evidence to the fact that this feature might be improperly considered. (SI Figure 8d, <a href="https://massive.ucsd.edu/ProteoSAFe/usi.jsp?usi=mzspec%3AMSV000079843%3AJD_06232014_sample4_C%3Ascan%3A8493%3ASQQILEK#%7B%22usi%22%3A%22mzspec%3AMSV000079843%3AJD_06232014_sample4_C%3Ascan%3A8576%3ALSQKYPK%2F2%22%2C%22searched_button%22%3A%22spectra%22%7D">Peptide Fragmentation Visualization - mzspec:PXD015300:JD_06232014_sample4_C:scan:8576:LSQKYPK/2</a>).</p>
<p><img src="img/dashboard_use_cases/SI_Figure_9.png" width="600"/></p>
<p><strong>Supplementary Figure 9: Connecting Protein Differential Expression with Raw Proteomics Data -</strong> a) shows the log2 intensity for features that map to sp|P07834|CDC4_YEAST in all 4 conditions from the IPRG2015 experiment, illustrating how the intensity is consistent across conditions 1-3 but significantly upregulated in condition 4, though only due to one features abundance, b) shows all features in Condition 4 that are for this protein, 6 in total, corresponding to 2 peptides, each with 3 replicates c) shows the LC-MS map for the region near the feature for LSQKYPK, the feature with very high abundance, in the 3rd replicate, and d) shows the annotated MS/MS spectrum for the peptide, and how the annotated peaks don't show much of the intensity of the spectrum.</p>
<h1 id="use-case-7-data-visualization-and-parameter-optimization-for-large-scale-feature-finding"><strong>Use Case 7: Data visualization and parameter optimization for large scale feature finding</strong><a class="headerlink" href="#use-case-7-data-visualization-and-parameter-optimization-for-large-scale-feature-finding" title="Permanent link">¶</a></h1>
<h2 id="carlos-molina-santiago-phd-post-doctoral-scholar-microbiology"><strong>Carlos Molina-Santiago PhD, Post-Doctoral Scholar, Microbiology</strong><a class="headerlink" href="#carlos-molina-santiago-phd-post-doctoral-scholar-microbiology" title="Permanent link">¶</a></h2>
<p><em>Bacillus subtilis</em> is a well-known Gram-positive strain that exhibits great potential as a biocontrol agent against bacterial and fungal pathogens in plants, such as in cucurbit crops<a href="https://www.zotero.org/google-docs/?khRfy7">23</a>. In the example, we showcase the study of the metabolic changes occurring on melon leaves after the inoculation of <em>B. subtilis</em> by non-targeted LC-MS/MS. We compared the metabolomes of untreated and treated melon leaves 5 days after inoculation with <em>B. subtilis</em>.</p>
<p>The optimization of feature finding parameters is one of the most important steps in the analysis of non-targeted LC-MS data, as it dictates much of the downstream quality of results. The GNPS Dashboard enables the tweaking of feature-finding parameters and immediate visualization of them for further downstream large-scale feature-finding via MZmine<a href="https://www.zotero.org/google-docs/?haDYcy">6</a> as a GNPS workflow.</p>
<p>We have used the GNPS Dashboard as a fast and easy tool to firstly compare the samples. We initially checked the successful colonization of the melon plant by <em>B. subtilis</em> by displaying the presence of surfactin (<em>m/z</em> 1036.69), a secondary metabolite known to be produced by this bacterium<a href="https://www.zotero.org/google-docs/?xYRLfF">24,25</a>. The GNPS Dashboard ( <strong>SI Fig 10A and B</strong> ) confirming its presence and abundance in treated leaves. With all the data already loaded in the GNPS dashboard, we can now perform Feature Finding directly modifying the settings in this tool ( <strong>SI Fig. 10C</strong> ) obtaining a heatmap where we can identify all the features detected (green diamonds) and their intensity coverage (Fig. XD). Once confirmed that the data are of enough quality to perform a deepest analysis, we can now perform Feature-Based Molecular Networking (FBMN)<a href="https://www.zotero.org/google-docs/?koQwFI">9</a> and MZmine to all the datasets directly from the GNPS dashboard. With that, we can now compare all the samples of the dataset using PcoA analysis ( <strong>SI Fig. 10E</strong> ) confirming the pattern differences between non-treated and treated melon leaves and visualizing the results using Cytoscape. These tools have permitted us, for instance, to highlight the differences observed in a cluster where we can find known potential antimicrobial compounds produced by plants such as 3,5-Dimethoxy-4-hydroxycinnamic acid, neochlorogenic acid, and trans-ferulic acid ( <strong>SI Fig. 10F</strong> ) In this case, we find a higher abundance of these compounds when melon leaves are treated with <em>B. subtilis</em>, a result that we can also confirm using box plots from the GNPS dashboard ( <strong>SI Fig. 10G</strong> ), showcasing the seamless interlinking of the GNPS Dashboard to optimize and launch data analysis as well as downstream manual inspection and confirmation of results from FBMN<a href="https://www.zotero.org/google-docs/?BS7Lbo">9</a>.</p>
<p><a href="https://gnps-lcms.ucsd.edu/?xicmz=1036.7&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.1&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=5-6&amp;xic_norm=False&amp;xic_file_grouping=GROUP&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%22xaxis.range%5B0%5D%22%3A+4.408429663319255%2C+%22xaxis.range%5B1%5D%22%3A+6.980701943615583%2C+%22yaxis.range%5B0%5D%22%3A+967.8475616707162%2C+%22yaxis.range%5B1%5D%22%3A+1123.4699224328826%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=MZmine2&amp;feature_finding_ppm=10&amp;feature_finding_noise=50000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.1&amp;sychronization_session_id=4ff8c0bd65a64245ae760aad607c7d6a#%7B%22usi%22%3A%20%22mzspec%3AMSV000082378%3Apeak/mzxml/3610_portion1_5days_2.mzXML%5Cnmzspec%3AMSV000082378%3Apeak/mzxml/3610_portion2_5days_1.mzXML%5Cnmzspec%3AMSV000082378%3Apeak/mzxml/3610_portion2_5days_2.mzXML%5Cnmzspec%3AMSV000082378%3Apeak/mzxml/3610_portion3_5days_1.mzXML%5Cnmzspec%3AMSV000082378%3Apeak/mzxml/3610_portion3_5days_2.mzXML%5Cnmzspec%3AMSV000082378%3Apeak/mzxml/3610_portion4_5days_1.mzXML%5Cnmzspec%3AMSV000082378%3Apeak/mzxml/3610_portion4_5days_2.mzXML%5Cnmzspec%3AMSV000082378%3Apeak/mzxml/3610_portion5_5days_1.mzXML%5Cnmzspec%3AMSV000082378%3Apeak/mzxml/3610_portion5_5days_2.mzXML%5Cnmzspec%3AMSV000082378%3Apeak/mzxml/3610_portion6_5days_1.mzXML%5Cnmzspec%3AMSV000082378%3Apeak/mzxml/3610_portion6_5days_2.mzXML%5Cnmzspec%3AMSV000082378%3Apeak/mzxml/3610_portion1_5days_1.mzXML%22%2C%20%22usi2%22%3A%20%22mzspec%3AMSV000082378%3Apeak/mzxml/Control_portion1_5days_1.mzXML%5Cnmzspec%3AMSV000082378%3Apeak/mzxml/Control_portion1_5days_2.mzXML%5Cnmzspec%3AMSV000082378%3Apeak/mzxml/Control_portion2_5days_1.mzXML%5Cnmzspec%3AMSV000082378%3Apeak/mzxml/Control_portion2_5days_2.mzXML%5Cnmzspec%3AMSV000082378%3Apeak/mzxml/Control_portion3_5days_1.mzXML%5Cnmzspec%3AMSV000082378%3Apeak/mzxml/Control_portion3_5days_2.mzXML%5Cnmzspec%3AMSV000082378%3Apeak/mzxml/Control_portion4_5days_1.mzXML%5Cnmzspec%3AMSV000082378%3Apeak/mzxml/Control_portion4_5days_2.mzXML%5Cnmzspec%3AMSV000082378%3Apeak/mzxml/Control_portion5_5days_1.mzXML%5Cnmzspec%3AMSV000082378%3Apeak/mzxml/Control_portion5_5days_2.mzXML%5Cnmzspec%3AMSV000082378%3Apeak/mzxml/Control_portion6_5days_1.mzXML%5Cnmzspec%3AMSV000082378%3Apeak/mzxml/Control_portion6_5days_2.mzXML%22%7D">GNPS Dashboard link</a></p>
<p><a href="https://proteomics2.ucsd.edu/ProteoSAFe/status.jsp?task=223f6e79690f42e585cafe05e68a96ac">FBMN GNPS Task link</a></p>
<p><img src="img/dashboard_use_cases/SI_Figure_10.png" width="600"/></p>
<p><strong>Supplemental Figure 10. Optimization of feature finding settings and feature-based molecular networking through the GNPS dashboard.</strong> Panel a shows the comparison of XIC of Surfactin (<em>m/z</em> 1036.69) of B. subtilis treated plants (blue) and control plants without treatment (orange) in the GNPS dashboard. Box plots of integrated peak areas of Surfactin of treatment vs. control are shown in panel b and show that surfactin was detected in the treated sample but not the control. In the GNPS Dashboard MZmine parameters were visually optimized and shown as green boxes in the 2D LC-MS heatmap in panel d for a representative sample and then directly applied in a link out for feature finding using the GNPS MZmine workflow. Panel e shows the multivariate (PCoA)<a href="https://www.zotero.org/google-docs/?5GRHEu">26</a> visualization of sample-to-sample distance based on all features in the data set. Panel f shows a selected network family that show different abundances between treatment and control group which was then further inspected by manually integrating the XICs of ferulic acid methyl ester (<em>m/z</em> 177) and neochlorogenic acid (m/z 163) in the GNPS Dashboard as shown in panel g.</p>
<h1 id="use-case-8-outliers-in-data-analysis-visualization-of-lc-ms-can-diagnose-upstream-data-issues"><strong>Use Case 8: Outliers in data analysis - Visualization of LC-MS can diagnose upstream data issues</strong><a class="headerlink" href="#use-case-8-outliers-in-data-analysis-visualization-of-lc-ms-can-diagnose-upstream-data-issues" title="Permanent link">¶</a></h1>
<h2 id="rachel-neve-phd-student-microbiology"><strong>Rachel Neve - PhD Student - Microbiology</strong><a class="headerlink" href="#rachel-neve-phd-student-microbiology" title="Permanent link">¶</a></h2>
<h2 id="vanessa-phelan-phd-assistant-professor-chemical-ecology"><strong>Vanessa Phelan PhD - Assistant Professor - Chemical Ecology</strong><a class="headerlink" href="#vanessa-phelan-phd-assistant-professor-chemical-ecology" title="Permanent link">¶</a></h2>
<p><img src="img/dashboard_use_cases/SI_Figure_11.png" width="600"/></p>
<p><strong>Supplemental Figure 11. Using GNPS Dashboard to confirm that low abundance data is due to poor extraction or improper injection.</strong> TOP: Boxplots with data points indicating the area under the curve value of features corresponding to <em>m/z</em> 225.0658, 244.1693, and 527.3184 of three biological replicates of <em>P. aeruginosa</em> SH2D overlaid on the boxes. BOTTOM: Extracted ion chromatograms of <em>m/z</em> 225.0658, 244.1693, and 527.3184 corresponding to each replicate.</p>
<p>Visualization of quantitative LC-MS data using boxplots or scatterplots can reveal outlier values. Due to the number of processing steps involved to generate quantitative values from raw LC-MS data, those outlier values may originate from poor sample extraction, improper injection volume, or inappropriate feature finding settings for the dataset. However, the GNPS Dashboard can be used to quickly visualize the raw data to identify the origin of the outlier values. For the example in <strong>Supplemental Figure 11</strong> , we analyzed the metabolome of <em>Pseudomonas aeruginosa</em> strain SH2D grown in synthetic cystic fibrosis medium 2 (SCFM2) containing dialyzed bovine submaxillary mucin (BSM). We subjected the LC-MS data to feature finding and generated box plots to visualize the area under the curve values for each replicate ( <strong>Supplemental Figure 11 TOP</strong> ). For many of the features quantified, biological replicate 1 (SH2D-1) values were near zero, while values for replicate 2 (SH2D-2) and 3 (SH2D-3) were not. This result suggested that there was likely an issue with either sample quality or data processing. We then visualized the extracted ion chromatograms (EIC) of <em>m/z</em> values associated with select features ( <strong>Supplemental Figure 11 BOTTOM</strong> ). Overlaying the EIC from each replicate for each <em>m/z</em> revealed that the boxplots faithfully represented the raw data, indicating an experimental issue associated with either sample extraction or injection. <a href="https://gnps-lcms.ucsd.edu/?xic_mz=225.0658%3B244.1693%3B527.3184&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=&amp;xic_norm=False&amp;xic_file_grouping=MZ&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2marker_color=blue&amp;ms2marker_size=5&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=2d65aa1ef8064b6abe8383136209130b&amp;chromatogram_options=%5B%5D&amp;comment=&amp;map_plot_color_scale=Hot_r&amp;map_plot_quantization_level=Medium#%7B%22usi%22%3A%20%22mzspec%3AMSV000087157%3Apeak/Strains%20mzXMLs/SH2D-1_P2-G-4_01_11825.mzXML%5Cnmzspec%3AMSV000087157%3Apeak/Strains%20mzXMLs/SH2D-2_P2-G-5_01_11826.mzXML%5Cnmzspec%3AMSV000087157%3Apeak/Strains%20mzXMLs/SH2D-3_P2-G-6_01_11827.mzXML%22%2C%20%22usi2%22%3A%20%22%22%7D">GNPS Dashboard Link</a></p>
<h1 id="use-case-9-cross-laboratoryinstrument-sharingvisualization-of-ms-data"><strong>Use Case 9: Cross Laboratory/Instrument sharing/visualization of MS data</strong><a class="headerlink" href="#use-case-9-cross-laboratoryinstrument-sharingvisualization-of-ms-data" title="Permanent link">¶</a></h1>
<h2 id="part-1-multiple-instruments-within-different-laboratories"><strong>Part 1: Multiple instruments within different laboratories</strong><a class="headerlink" href="#part-1-multiple-instruments-within-different-laboratories" title="Permanent link">¶</a></h2>
<h2 id="monica-thukral-phd-student-microbial-ecology"><strong>Monica Thukral - PhD Student - Microbial Ecology</strong><a class="headerlink" href="#monica-thukral-phd-student-microbial-ecology" title="Permanent link">¶</a></h2>
<p><em>Pseudo-nitzschia australis (P. australis</em>) is a cosmopolitan marine microalga capable of producing a potent neurotoxin domoic acid, particularly when undergoing nutrient starvation<a href="https://www.zotero.org/google-docs/?V0b3MD">27</a>. Domoic acid (DA) bioaccumulates up the food chain and can cause mass mortality to marine mammals and birds and also to humans if they consume DA-containing shellfish or crustaceans<a href="https://www.zotero.org/google-docs/?7f6Oj8">27,28</a>. In the context of studying <em>P. australis'</em> metabolome under nutrient starvation, three LC-MS platforms are typically used by our group: A Bruker Amazon Ion Trap, an Agilent 6530 qToF, and a Thermo Fisher Scientific Q-Exactive qOrbitrap. As all three platforms come with their own vendor-specific data analysis software, the GNPS Dashboard facilitates this multi-laboratory and multi-instrument data analysis.</p>
<p>Visualization of mass spectrometry data of aqueous methanol extracts of <em>Pseudo-nitzschia australis are shown</em> in <strong>SI Figure 12</strong>. Panel a shows extracted ion chromatograms (XICs) (left) and mass spectra (right) extracted for m/z = 314.1441, the mass of domoic acid from ion trap (top) qToF (middle) and qOrbitrap (bottom). Due to the use of three different platforms for data acquisition, we expect the absolute value of the intensity of peaks to differ. To resolve this problem and ensure proper comparison, the XIC normalization toggle feature is turned on, to clearly and easily compare visual data. <em>A mass trace corresponding to domoic acids was clearly detected with all platforms and MS/MS spectra share the most intense fragment (m/z 266) but differ especially in the low m/z range in the ion trap data.</em> <strong>SI Figure 12b</strong> shows the comparison of data of extracts acquired on the Ion Trap and the two high-resolution qTOF and qOrbitrap platforms. Depicted are heat maps and total ion chromatograms (TICs) from qToF (left) and Orbitrap (right) provide a direct comparison of globe data structure, including MS/MS events. The heat maps show <em>m/z</em> vs retention time as a 2D LC-MS map. Blue crosses represent the triggering of MS/MS data collection. While in the qToF data, the blue crosses are denser, the qOrbitrap data shows blue crosses well distributed across the data. Also clearly visible is that dynamic exclusion is not utilized in the qToF analysis, causing multiple MS/MS to be triggered along one peak and reducing the metabolome coverage of a molecular network in downstream analysis. Hence, it is not surprising that the coverage of chemical space in the downstream molecular network data analysis has a significantly higher contribution from the qOrbitrap data ( <strong>SI Figure 12c</strong> ) than from the qToF.</p>
<p>In summary, the GNPS Dashboard directly benefited the investigation into <em>P. australis</em> metabolomics by allowing for rapid and efficient comparison between three instruments with different data analysis platforms. The GNPS Dashboard will serve as a valuable data integration tool for analysis going forward.</p>
<p><a href="https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=d82c42e148734958848b0284c0d726f8">Classic Molecular Networking Link</a><a href="https://gnps-lcms.ucsd.edu/?usi=mzspec:GNPS:TASK-b3af2429890643c392765d440225e1c0-f.monicathukral/DA_15NDLNGG_Exp_Paus_70mL_21_01_15338.mzML&amp;overlay_usi=mzspec:GNPS:TASK-b3af2429890643c392765d440225e1c0-clusterinfosummarygroup_attributes_withIDs_withcomponentID/&amp;overlay_size=number%20of%20spectra&amp;overlay_mz=precursor%20mass&amp;overlay_rt=RTMean_min#%7B%7D">GNPS Dashboard IonTrap</a><a href="https://gnps-lcms.ucsd.edu/?usi=mzspec:GNPS:TASK-b3af2429890643c392765d440225e1c0-f.monicathukral/210111/australis.mzXML&amp;overlay_usi=mzspec:GNPS:TASK-b3af2429890643c392765d440225e1c0-clusterinfosummarygroup_attributes_withIDs_withcomponentID/&amp;overlay_size=number%20of%20spectra&amp;overlay_mz=precursor%20mass&amp;overlay_rt=RTMean_min#%7B%7D">GNPS Dashboard qToF</a><a href="https://gnps-lcms.ucsd.edu/?usi=mzspec:GNPS:TASK-b3af2429890643c392765d440225e1c0-f.MSV000084738/peak/mzxml/Australis_MT_120319_1.mzXML&amp;overlay_usi=mzspec:GNPS:TASK-b3af2429890643c392765d440225e1c0-clusterinfosummarygroup_attributes_withIDs_withcomponentID/&amp;overlay_size=number%20of%20spectra&amp;overlay_mz=precursor%20mass&amp;overlay_rt=RTMean_min#%7B%7D">GNPS Dashboard Orbitrap</a></p>
<p><img src="img/dashboard_use_cases/SI_Figure_12.png" width="600"/></p>
<p><strong>Supplemental Figure 12: Cross-platform comparison of LC-MS/MS from the same sample</strong>. Panel a shows the comparison of MS data from Domoic acid From extracts from <em>Pseudo-nitzschia australis</em> extracts that were recorded on three different instrument platforms (Ion Trap qToF and qOrbitrap). On the left side, the Extracted Ion Chromatograms (XIC) and corresponding MS/MS of domoic acid spectra on the right can be seen. Platform comparison of global DDA MS/MS data is shown in Panel b as heatmaps of the LC-MS run and MS/MS scans indicated as blue crosses. A molecular network from the global MS/MS data from all three platforms and zoom-ins to the domoic acids networks is shown in panel c.</p>
<h2 id="part-2-multiple-instrument-within-the-same-laboratory"><strong>Part 2: Multiple instrument within the same laboratory</strong><a class="headerlink" href="#part-2-multiple-instrument-within-the-same-laboratory" title="Permanent link">¶</a></h2>
<h2 id="alan-jarmusch-phd-post-doctoral-scholar-analytical-chemistry"><strong>Alan Jarmusch PhD - Post-Doctoral Scholar - Analytical Chemistry</strong><a class="headerlink" href="#alan-jarmusch-phd-post-doctoral-scholar-analytical-chemistry" title="Permanent link">¶</a></h2>
<p>The multitude of mass spectrometers in existence is reflective of their commonplace in modern chemical measurement as well as their flexibility. Different mass spectrometers are defined, principally, by the mass analyzer of which there are traps, time-of-flights, quadruples, sectors, and hybrids. The different analytical performances (<em>e.g.</em> sensitivity, specificity, speed, duty cycle, mass resolution) of these mass analyzers often necessitate multiple mass spectrometers in a laboratory in order to access the different types of experiments available. The practical implication of this fact is that a laboratory is often forced to purchase instruments from different manufacturers and learn each manufacturer's control and data analysis software. The data analysis software is often complex to learn, knowledge does not readily translate from one manufacturer's software to another, and is plagued by different file formats. In reverse order, the utilization of an open-source file format (e.g. .mzML<a href="https://www.zotero.org/google-docs/?Rf7BBQ">29</a>) is a good initial step toward overcoming this challenge. By learning a single data analysis and visualization software, particularly one that exists in a web browser, it is hugely advantageous and nearly eliminates the need to learn multiple pieces of manufacturer software for simple visualizations and data processing. This of course will not immediately eliminate the need for manufacturer software, but it does make the initial steps towards a democratized, open-source, web-enabled, trans-manufacturer, and trans-mass spectrometer piece of software to facilitate chemical measurement and minimize time spent learning disparate software.</p>
<h1 id="use-case-10-integration-of-dashboard-with-other-tools-sirius-and-comparison-with-standard-to-identify-isomerizationdegradation"><strong>Use Case 10: Integration of Dashboard with other tools (SIRIUS) and comparison with standard to identify isomerization/degradation</strong><a class="headerlink" href="#use-case-10-integration-of-dashboard-with-other-tools-sirius-and-comparison-with-standard-to-identify-isomerizationdegradation" title="Permanent link">¶</a></h1>
<h2 id="jessica-m-deutsch-phd-student-chemistry"><strong>Jessica M. Deutsch - PhD Student - Chemistry</strong><a class="headerlink" href="#jessica-m-deutsch-phd-student-chemistry" title="Permanent link">¶</a></h2>
<h2 id="neha-garg-phd-assistant-professor-natural-product-chemistry"><strong>Neha Garg PhD - Assistant Professor - Natural Product Chemistry</strong><a class="headerlink" href="#neha-garg-phd-assistant-professor-natural-product-chemistry" title="Permanent link">¶</a></h2>
<p>In non-targeted metabolomics experiments, one can typically annotate 1- 10% of the detected metabolites. To enhance our ability to decode the dark metabolome, we utilize various <em>in silico</em> methods such as SIRIUS 4<a href="https://www.zotero.org/google-docs/?hnR9Aa">30</a> with CSI:FingerID<a href="https://www.zotero.org/google-docs/?XmJIDW">31</a> among others. The use of GNPS Dashboard to generate XIC on chemical formulas suggested by SIRIUS followed by feature detection and interrogation of the presence of chemical formula across samples allows us to integrate output from tools such as SIRIUS 4 with CSI:FingerID with rapid inspection of MS1 and MS2 spectra across multiple samples. Analysis performed directly via the GNPS Dashboard eliminated the need to install and toggle between multiple software to extract features and visualize multiple XIC alongside SIRIUS 4 with CSI:FingerID. The option to generate XICs from molecular formulas in the GNPS Dashboard also enables automated visualization of potential adducts, such as [M+Na]+ and [M+K]+, which enables further validation of the proposed chemical formula. Furthermore, access to individual LC-MS files on GNPS Dashboard also enables visualization of the MS2 spectra to inspect the fragmentation pattern and validation with analytical standards. Lastly, we can share these results with collaborators, additional lab members, and generate publication-quality figures without the use of expensive vendor software.</p>
<p>The use of GNPS Dashboard has been instrumental in a remote teaching setting where I teach a potpourri of multiple mass spectrometry-based data annotation strategies and has enabled remote training of graduate students that joined my laboratory during the COVID shut down. Thus, the advantages of GNPS Dashboard are multi-fold and extend beyond the research setting. In the example presented in <strong>SI Figure 13</strong> , we interrogated the annotation of korormicin A as suggested by SIRIUS with CSI:FingerID in organic extracts of a bacterium isolated from the coral tissue by our collaborator, Dr. Valerie Paul. The chemical formula was suggested as C25H39NO5 by SIRIUS 4 with a score of 99.95% and the top-most annotation belonged to korormicin A ( <strong>SI Figure 13a</strong> ). The XIC for this chemical formula was generated in the organic extract of bacterium ( <strong>SI Figure 13b</strong> ), which revealed detection of both [M+H]+ and [M+Na]+ adducts. This feature was used to optimize parameters for MZmine-based<a href="https://www.zotero.org/google-docs/?uFuwfh">6</a> feature detection directly in GNPS Dashboard. These parameters were used to interrogate the presence of korormicin A in extracts of various microbes extracted from the same coral to identify likely isolation of the same species of bacteria ( <strong>SI Figure 13c</strong> ). Lastly, XIC comparison with analytical standards using GNPS Dashboard allowed us to observe isomerization of the purified molecule, which was also observed at low abundance in the culture extracts ( <strong>SI Figure 13c</strong> ).</p>
<p>GNPS Dashboard Figure 12b - <a href="https://gnps-lcms.ucsd.edu/?xic_mz=&amp;xic_formula=C25H39NO5&amp;xic_peptide=&amp;xic_tolerance=0.05&amp;xic_ppm_tolerance=5&amp;xic_tolerance_unit=Da&amp;xic_rt_window=&amp;xic_norm=False&amp;xic_file_grouping=FILE&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=4158bc4e43924fe18fa2713ff94c268c&amp;comment=&amp;map_plot_color_scale=Hot_r&amp;map_plot_quantization_level=Medium#%7B%22usi%22%3A%20%22mzspec%3AGNPS%3ATASK-41dae82204f64ba492d3cdf8556068a1-f.jdeutsch79/2020/Korormicin/124-1_1_P2-B-2_01_6955.mzXML%22%2C%20%22usi2%22%3A%20%22%22%7D">Link</a></p>
<p>GNPS Dashboard Figure 12c - <a href="https://gnps-lcms.ucsd.edu/?xic_mz=434.290&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.05&amp;xic_ppm_tolerance=5&amp;xic_tolerance_unit=Da&amp;xic_rt_window=&amp;xic_norm=False&amp;xic_file_grouping=MZ&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=a45611a025824f87a0fa02214a16c8b5&amp;comment=&amp;map_plot_color_scale=Hot_r&amp;map_plot_quantization_level=Medium#%7B%22usi%22%3A%20%22mzspec%3AGNPS%3ATASK-41dae82204f64ba492d3cdf8556068a1-jdeutsch79/2020/Korormicin/Korormicin_1_redo_P1-A-1_01_6614.mzXML%5Cnmzspec%3AGNPS%3ATASK-41dae82204f64ba492d3cdf8556068a1-jdeutsch79/2020/Korormicin/Korormicin_2_P1-A-1_01_6615.mzXML%5Cnmzspec%3AGNPS%3ATASK-41dae82204f64ba492d3cdf8556068a1-jdeutsch79/2020/Korormicin/124-1_1_P2-B-2_01_6955.mzXML%5Cnmzspec%3AGNPS%3ATASK-41dae82204f64ba492d3cdf8556068a1-jdeutsch79/2020/Korormicin/124-1_2_P2-B-2_01_6961.mzXML%5Cnmzspec%3AGNPS%3ATASK-41dae82204f64ba492d3cdf8556068a1-jdeutsch79/2020/Korormicin/128-1_1_P2-B-3_01_6956.mzXML%5Cnmzspec%3AGNPS%3ATASK-41dae82204f64ba492d3cdf8556068a1-jdeutsch79/2020/Korormicin/128-1_2_P2-B-3_01_6962.mzXML%5Cnmzspec%3AGNPS%3ATASK-41dae82204f64ba492d3cdf8556068a1-jdeutsch79/2020/Korormicin/SOD_Blank1_P1-F-1_01_6780.mzXML%22%2C%20%22usi2%22%3A%20%22%22%7D">Link</a></p>
<p><img src="img/dashboard_use_cases/SI_Figure_13.png" width="600"/></p>
<p><strong>Supplemental Figure 13. Integration of</strong> _ <strong>in silico</strong> _ <strong>annotation tool SIRIUS 4 with CSI:FingerID with GNPS Dashboard</strong>. a) The output ofSIRIUS 4 with CSI:FingerID showing prediction of chemical formula as C25H39NO5, with plausible annotation as korormicin A as the top candidate for <em>m/z</em> 434.290 Da. b) The XIC plot for a chemical formula using LC-MS viewer shows the possible adducts as [M+H]+ (blue trace) and [M+Na]+ (orange trace). The trace for [M+K]+ does not show peaks at the same retention time as [M+H]+ confirming [M+K]+ is not detected. c) The multiple XIC plot reveals that only one bacterial extract (green trace) produces korormicin A suggesting that the isolated bacteria is different from the other isolates. The same two isomers are observed in the bacterial extract. The isomer at 11 min is present in lower relative abundance in the bacterial extract than the standard suggesting conversion between the isomers during purification.</p>
<h1 id="use-case-11-targeted-dereplication-and-evaluation-of-culture-conditions"><strong>Use Case 11: Targeted dereplication and evaluation of culture conditions</strong><a class="headerlink" href="#use-case-11-targeted-dereplication-and-evaluation-of-culture-conditions" title="Permanent link">¶</a></h1>
<h2 id="scott-jarmusch-phd-post-doctoral-scholar-natural-product-chemistry"><strong>Scott Jarmusch PhD - Post-Doctoral Scholar - Natural Product Chemistry</strong><a class="headerlink" href="#scott-jarmusch-phd-post-doctoral-scholar-natural-product-chemistry" title="Permanent link">¶</a></h2>
<p>Dereplication remains one of the principal target areas for metabolomics-based tools to accommodate the natural products community. GNPS molecular networking integrates fast, automated dereplication into its workflow, yet often, results still require the need to evaluate raw data for more critical interrogation. Utilizing the public data from a recent study on desferrioxamine metabolites produced by <em>Streptomyces</em> sp. S29<a href="https://www.zotero.org/google-docs/?B6Pb5m">32</a>, we can visualize how a targeted dereplication approach may be carried out. Using hits from the GNPS library as well as the work done by Traxler et al.<a href="https://www.zotero.org/google-docs/?w7C25N">33</a>, the GNPS Dashboard allows for fast evaluation regarding the presence or absence of accurate mass XIC as well as providing context in which samples these metabolites are present. Taking desferrioxamine D (<em>m/z</em> 603.37), we expect it to be produced and it is, in any of the three conditions tested ( <strong>SI Figure 14a</strong> ): monoculture, coculture with <em>Aspergillus niger</em> and coculture with <em>Botrytis cinerea</em>. Furthermore, in microbial natural products, evaluation of culture conditions is vital to gain a better understanding of metabolite elicitation, especially when focusing on discovery. The two studies above observe the production of long-chain acyl desferrioxamines exclusively under coculture conditions. When we evaluate the presence/absence of three acyl desferrioxamines (C11-C13) in the fungal coculture study, we can clearly see the presence of the three metabolites in the cocultivation with <em>B. cinerea</em>, a presence of one metabolite in cocultivation with <em>A. niger</em> and an absence of the three in monoculture conditions ( <strong>SI Figure 14b</strong> ). Beyond the evaluation of knowns, we can also easily visualize and allow for prioritization of extracts when hunting for unknowns. An unknown desferrioxamine-like metabolite was identified in this study and we observe that its presence is related to cocultivation with <em>B. cinerea</em>, providing a guide to future isolation of this unknown ( <strong>SI Figure 14c</strong> ). Allowing for this rapid observation in the GNPS Dashboard interface that can then link into further GNPS tools makes this functionality invaluable when analyzing data that would normally require multiple software types to be opened simultaneously, requiring high computing power that some researchers do not have access to. These basic functionalities shown are essential in the community and the ability of the GNPS Dashboard to facilitate these are paramount to the service it provides.</p>
<p>GNPS Dashboard Figure 13a - <a href="https://gnps-lcms.ucsd.edu/?xic_mz=603.3714&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=&amp;xic_norm=False&amp;xic_file_grouping=MZ&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=ee437b47cb744940bbd29f84ea6d58e4&amp;comment=&amp;map_plot_color_scale=Hot_r&amp;map_plot_quantization_level=Medium#%7B%22usi%22%3A%20%22mzspec%3AMSV000085618%3Accms_peak/ED104_P1-D-9_01_3763.mzML%22%2C%20%22usi2%22%3A%20%22mzspec%3AMSV000085618%3Accms_peak/ED25_P1-B-3_01_2172.mzML%5Cnmzspec%3AMSV000085618%3Accms_peak/ED37_P1-C-4_01_2182.mzML%22%7D">Link</a></p>
<p>GNPS Dashboard Figure 13b - <a href="https://gnps-lcms.ucsd.edu/?xic_mz=701.5174%3B715.5328%3B729.5484&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=&amp;xic_norm=False&amp;xic_file_grouping=FILE&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=ee437b47cb744940bbd29f84ea6d58e4&amp;comment=&amp;map_plot_color_scale=Hot_r&amp;map_plot_quantization_level=Medium#%7B%22usi%22%3A%20%22mzspec%3AMSV000085618%3Accms_peak/ED105_P1-A-1_01_3781.mzML%5Cn%22%2C%20%22usi2%22%3A%20%22mzspec%3AMSV000085618%3Accms_peak/ED26_P1-B-4_01_2173.mzML%5Cnmzspec%3AMSV000085618%3Accms_peak/ED38_P1-C-5_01_2183.mzML%22%7D">Link</a></p>
<p>GNPS Dashboard Figure 13c - <a href="https://gnps-lcms.ucsd.edu/?xic_mz=650.37&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=&amp;xic_norm=False&amp;xic_file_grouping=MZ&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=ee437b47cb744940bbd29f84ea6d58e4&amp;comment=&amp;map_plot_color_scale=Hot_r&amp;map_plot_quantization_level=Medium#%7B%22usi%22%3A%20%22mzspec%3AMSV000085618%3Accms_peak/ED105_P1-A-1_01_3781.mzML%5Cn%22%2C%20%22usi2%22%3A%20%22mzspec%3AMSV000085618%3Accms_peak/ED26_P1-B-4_01_2173.mzML%5Cnmzspec%3AMSV000085618%3Accms_peak/ED38_P1-C-5_01_2183.mzML%22%7D">Link</a></p>
<p><img src="img/dashboard_use_cases/SI_Figure_14.png" width="600"/></p>
<p><strong>Supplemental Figure 14. Targeted dereplication and evaluation of culture conditions.</strong> a) XIC plot of desferrioxamine D (<em>m/z</em> 603.3714) using LC-MS viewer shows the presence of the archetypal siderophore in monoculture (blue trace), cocultivation with <em>A. niger</em> (orange trace), and cocultivation with <em>B. cinerea</em> (green trace). b) XIC plot grouped by file showing the presence/absence of acyl desferrioxamines C11-C13 in each culture environment, clearly showing production in the presence of <em>B. cinerea</em>, some production in the presence of <em>A. niger,</em> and no production in the monoculture. c) targeting the culture conditions where an unknown metabolite (<em>m/z</em> 650.37 [M+2H]2+) could be isolated for future studies.</p>
<h1 id="use-case-12-large-scale-xic-inspection-in-gnps-dashboard"><strong>Use Case 12 - Large Scale XIC Inspection in GNPS Dashboard</strong><a class="headerlink" href="#use-case-12-large-scale-xic-inspection-in-gnps-dashboard" title="Permanent link">¶</a></h1>
<h2 id="morgan-panitchpakdi-staff-research-associate-metabolomics"><strong>Morgan Panitchpakdi - Staff Research Associate - Metabolomics</strong><a class="headerlink" href="#morgan-panitchpakdi-staff-research-associate-metabolomics" title="Permanent link">¶</a></h2>
<p>Analysis of non-targeted LC-MS data often requires a thorough examination of the quality of each data file generated. Generating Extracted Ion Chromatograms (XIC) for internal standards and compounds of interest using vendor software is possible but becomes slower as the project size increases (\&gt;20 files). The GNPS Dashboard helps alleviate these issues and enables quick inspection and analysis of LC-MS data and can generate XICs for large-scale projects within minutes.</p>
<p>With the goal of reanalyzing published data from a clinical study investigating drug metabolism<a href="https://www.zotero.org/google-docs/?R9lSvV">34</a>, the GNPS Dashboard was used to quickly analyze LC-MS data quality and to confirm the presence of five reported internal standards in 140 urine samples (<a href="https://massive.ucsd.edu/ProteoSAFe/dataset.jsp?task=b8ffc66bab8d435aa863a0fed8fff6fa">MSV000082493</a>) before continuing on with downstream processing with MZmine<a href="https://www.zotero.org/google-docs/?KGC4U6">6</a>. A single sample file was first inspected for the presence of the internal standards to determine retention time range. XIC's for all five internal standards, Omeprazole-d3 (<em>m/z</em> 349.1409), Caffeine-d3 (<em>m/z</em> 197.0992), Midazolam-d4 (<em>m/z</em> 330.1112), Dextromethorphan-d3 (<em>m/z</em> 275.2198), and Dextrophan-d3 (<em>m/z</em> 261.2035) were generated for all 140 urine samples. To obtain the XIC for all sample files, 140 USI's for these files were input into the GNPS USI field, <em>m/z</em> value and RT range were specified for each internal standard and visualized by grouping XIC's by <em>m/z</em> ( <strong>Figure 15a</strong> ). The GNPS Dashboard was used to visualize plotted AUC values for each of the five internal standards in all 140 urine samples ( <strong>Figure 15b</strong> ). In summary, the GNPS Dashboard generated XIC's for 140 sample files faster than the vendor-provided data analysis software. Visual and shareable AUC value plots allow for quick analysis of internal standard presence and data quality before downstream processing and analysis begins.</p>
<p><a href="https://gnps-lcms.ucsd.edu/?xic_mz=349.1409&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=2.7-2.78&amp;xic_norm=False&amp;xic_file_grouping=FILE&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=90f11807e4fc437c82ea6da956f747c6&amp;comment=&amp;map_plot_color_scale=Hot_r&amp;map_plot_quantization_level=Medium#%7B%22usi%22%3A%20%22mzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076790_RE5_01_29156.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076791_RH2_01_29192.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076792_RA8_01_29107.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076793_RD10_01_29092.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076794_RH10_01_29200.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076795_RC6_01_29131.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076796_RA9_01_29108.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076797_RB6_01_29062.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076798_RC1_01_29070.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076799_RC8_01_29077.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076800_RG4_01_29181.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076801_RE2_01_29153.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076802_RD6_01_29144.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076803_RH6_01_29196.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076804_RF8_01_29172.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076805_RA2_01_29101.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076806_RB7_01_29119.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076807_RD4_01_29086.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076808_RD3_01_29085.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076809_RA7_01_29106.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076810_RD7_01_29145.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076811_RF5_01_29169.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076812_RG7_01_29184.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076813_RB11_01_29123.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076814_RC12_01_29081.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076815_RH3_01_29193.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076816_RD4_01_29142.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076817_RA5_01_29048.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076818_RE7_01_29158.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076819_RC9_01_29078.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076820_RC4_01_29073.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076821_RC12_01_29137.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076822_RB1_01_29057.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076823_RB12_01_29124.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076825_RF2_01_29166.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076826_RA2_01_29045.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076827_RA6_01_29049.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076830_RE8_01_29159.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076831_RG2_01_29179.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076832_RD10_01_29148.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076833_RC4_01_29129.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076834_RB8_01_29064.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076835_RA1_01_29044.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076837_RC3_01_29072.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076838_RB2_01_29058.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076839_RG10_01_29187.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078262_RF3_01_29167.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078263_RC2_01_29071.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078264_RG8_01_29185.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078265_RD7_01_29089.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078699_RB6_01_29118.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078700_RF6_01_29170.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078702_RE3_01_29154.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078706_RD2_01_29140.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078707_RB3_01_29115.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078708_RC10_01_29079.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078710_RA3_01_29102.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078711_RB10_01_29066.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078716_RF10_01_29174.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078718_RD5_01_29143.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078719_RA11_01_29054.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078722_RA10_01_29109.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078724_RC5_01_29074.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078726_RG5_01_29182.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078729_RF7_01_29171.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078730_RH7_01_29197.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078732_RD5_01_29087.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078735_RA9_01_29052.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078737_RB10_01_29122.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078738_RB8_01_29120.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085587_RE12_01_29163.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085590_RD1_01_29083.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085591_RH9_01_29199.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085592_RD1_01_29139.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085593_RA4_01_29047.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085594_RC3_01_29128.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085595_RA5_01_29104.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085596_RG6_01_29183.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085597_RA8_01_29051.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085598_RB12_01_29068.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085599_RD12_01_29150.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086566_RB11_01_29067.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086567_RC7_01_29076.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086569_RD11_01_29149.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086570_RC9_01_29134.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086571_RB3_01_29059.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086576_RC1_01_29126.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086578_RA7_01_29050.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086579_RD8_01_29146.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086580_RF12_01_29176.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088099_RB7_01_29063.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088101_RH1_01_29191.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088103_RG3_01_29180.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088106_RE1_01_29152.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088108_RG1_01_29178.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088109_RD2_01_29084.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088111_RC7_01_29132.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088113_RA6_01_29105.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088115_RB4_01_29060.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088117_RF9_01_29173.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088425_RE9_01_29160.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088426_RF11_01_29175.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088427_RA10_01_29053.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088428_RH4_01_29194.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088429_RD3_01_29141.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088430_RF1_01_29165.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088431_RG12_01_29189.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088432_RC10_01_29135.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088433_RB4_01_29116.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088434_RB9_01_29065.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088435_RC2_01_29127.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088436_RC11_01_29080.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088437_RE11_01_29162.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088438_RC11_01_29136.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088439_RC6_01_29075.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088440_RB5_01_29061.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088441_RA12_01_29111.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088442_RA3_01_29046.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088443_RD9_01_29091.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088444_RF4_01_29168.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088445_RC5_01_29130.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088446_RG9_01_29186.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088447_RH8_01_29198.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088449_RG11_01_29188.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088474_RH5_01_29195.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088901_RC8_01_29133.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088903_RB2_01_29114.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088906_RE10_01_29161.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088910_RB1_01_29113.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088912_RD6_01_29088.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088915_RB9_01_29121.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088918_RE6_01_29157.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088921_RE4_01_29155.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088924_RA1_01_29100.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088928_RD9_01_29147.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089111_RB5_01_29117.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089112_RA11_01_29110.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089120_RD8_01_29090.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089124_RA12_01_29055.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089125_RA4_01_29103.mzML%22%2C%20%22usi2%22%3A%20%22%22%7D">GNPS Dashboard Link</a> - Omeprazole-d3</p>
<p><a href="https://gnps-lcms.ucsd.edu/?xic_mz=197.0992&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=2.10-2.16&amp;xic_norm=False&amp;xic_file_grouping=FILE&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=90f11807e4fc437c82ea6da956f747c6&amp;comment=&amp;map_plot_color_scale=Hot_r&amp;map_plot_quantization_level=Medium#%7B%22usi%22%3A%20%22mzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076790_RE5_01_29156.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076791_RH2_01_29192.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076792_RA8_01_29107.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076793_RD10_01_29092.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076794_RH10_01_29200.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076795_RC6_01_29131.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076796_RA9_01_29108.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076797_RB6_01_29062.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076798_RC1_01_29070.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076799_RC8_01_29077.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076800_RG4_01_29181.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076801_RE2_01_29153.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076802_RD6_01_29144.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076803_RH6_01_29196.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076804_RF8_01_29172.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076805_RA2_01_29101.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076806_RB7_01_29119.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076807_RD4_01_29086.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076808_RD3_01_29085.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076809_RA7_01_29106.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076810_RD7_01_29145.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076811_RF5_01_29169.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076812_RG7_01_29184.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076813_RB11_01_29123.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076814_RC12_01_29081.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076815_RH3_01_29193.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076816_RD4_01_29142.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076817_RA5_01_29048.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076818_RE7_01_29158.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076819_RC9_01_29078.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076820_RC4_01_29073.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076821_RC12_01_29137.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076822_RB1_01_29057.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076823_RB12_01_29124.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076825_RF2_01_29166.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076826_RA2_01_29045.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076827_RA6_01_29049.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076830_RE8_01_29159.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076831_RG2_01_29179.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076832_RD10_01_29148.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076833_RC4_01_29129.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076834_RB8_01_29064.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076835_RA1_01_29044.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076837_RC3_01_29072.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076838_RB2_01_29058.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076839_RG10_01_29187.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078262_RF3_01_29167.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078263_RC2_01_29071.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078264_RG8_01_29185.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078265_RD7_01_29089.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078699_RB6_01_29118.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078700_RF6_01_29170.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078702_RE3_01_29154.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078706_RD2_01_29140.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078707_RB3_01_29115.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078708_RC10_01_29079.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078710_RA3_01_29102.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078711_RB10_01_29066.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078716_RF10_01_29174.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078718_RD5_01_29143.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078719_RA11_01_29054.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078722_RA10_01_29109.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078724_RC5_01_29074.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078726_RG5_01_29182.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078729_RF7_01_29171.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078730_RH7_01_29197.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078732_RD5_01_29087.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078735_RA9_01_29052.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078737_RB10_01_29122.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078738_RB8_01_29120.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085587_RE12_01_29163.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085590_RD1_01_29083.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085591_RH9_01_29199.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085592_RD1_01_29139.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085593_RA4_01_29047.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085594_RC3_01_29128.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085595_RA5_01_29104.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085596_RG6_01_29183.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085597_RA8_01_29051.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085598_RB12_01_29068.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085599_RD12_01_29150.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086566_RB11_01_29067.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086567_RC7_01_29076.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086569_RD11_01_29149.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086570_RC9_01_29134.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086571_RB3_01_29059.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086576_RC1_01_29126.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086578_RA7_01_29050.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086579_RD8_01_29146.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086580_RF12_01_29176.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088099_RB7_01_29063.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088101_RH1_01_29191.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088103_RG3_01_29180.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088106_RE1_01_29152.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088108_RG1_01_29178.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088109_RD2_01_29084.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088111_RC7_01_29132.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088113_RA6_01_29105.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088115_RB4_01_29060.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088117_RF9_01_29173.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088425_RE9_01_29160.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088426_RF11_01_29175.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088427_RA10_01_29053.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088428_RH4_01_29194.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088429_RD3_01_29141.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088430_RF1_01_29165.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088431_RG12_01_29189.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088432_RC10_01_29135.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088433_RB4_01_29116.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088434_RB9_01_29065.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088435_RC2_01_29127.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088436_RC11_01_29080.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088437_RE11_01_29162.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088438_RC11_01_29136.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088439_RC6_01_29075.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088440_RB5_01_29061.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088441_RA12_01_29111.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088442_RA3_01_29046.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088443_RD9_01_29091.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088444_RF4_01_29168.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088445_RC5_01_29130.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088446_RG9_01_29186.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088447_RH8_01_29198.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088449_RG11_01_29188.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088474_RH5_01_29195.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088901_RC8_01_29133.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088903_RB2_01_29114.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088906_RE10_01_29161.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088910_RB1_01_29113.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088912_RD6_01_29088.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088915_RB9_01_29121.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088918_RE6_01_29157.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088921_RE4_01_29155.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088924_RA1_01_29100.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088928_RD9_01_29147.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089111_RB5_01_29117.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089112_RA11_01_29110.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089120_RD8_01_29090.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089124_RA12_01_29055.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089125_RA4_01_29103.mzML%22%2C%20%22usi2%22%3A%20%22%22%7D">GNPS Dashboard Link</a> - Caffeine-d3</p>
<p><a href="https://gnps-lcms.ucsd.edu/?xic_mz=330.1112&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=3.0-3.1&amp;xic_norm=False&amp;xic_file_grouping=FILE&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=90f11807e4fc437c82ea6da956f747c6&amp;comment=&amp;map_plot_color_scale=Hot_r&amp;map_plot_quantization_level=Medium#%7B%22usi%22%3A%20%22mzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076790_RE5_01_29156.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076791_RH2_01_29192.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076792_RA8_01_29107.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076793_RD10_01_29092.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076794_RH10_01_29200.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076795_RC6_01_29131.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076796_RA9_01_29108.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076797_RB6_01_29062.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076798_RC1_01_29070.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076799_RC8_01_29077.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076800_RG4_01_29181.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076801_RE2_01_29153.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076802_RD6_01_29144.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076803_RH6_01_29196.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076804_RF8_01_29172.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076805_RA2_01_29101.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076806_RB7_01_29119.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076807_RD4_01_29086.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076808_RD3_01_29085.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076809_RA7_01_29106.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076810_RD7_01_29145.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076811_RF5_01_29169.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076812_RG7_01_29184.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076813_RB11_01_29123.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076814_RC12_01_29081.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076815_RH3_01_29193.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076816_RD4_01_29142.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076817_RA5_01_29048.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076818_RE7_01_29158.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076819_RC9_01_29078.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076820_RC4_01_29073.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076821_RC12_01_29137.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076822_RB1_01_29057.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076823_RB12_01_29124.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076825_RF2_01_29166.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076826_RA2_01_29045.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076827_RA6_01_29049.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076830_RE8_01_29159.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076831_RG2_01_29179.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076832_RD10_01_29148.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076833_RC4_01_29129.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076834_RB8_01_29064.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076835_RA1_01_29044.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076837_RC3_01_29072.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076838_RB2_01_29058.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076839_RG10_01_29187.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078262_RF3_01_29167.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078263_RC2_01_29071.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078264_RG8_01_29185.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078265_RD7_01_29089.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078699_RB6_01_29118.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078700_RF6_01_29170.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078702_RE3_01_29154.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078706_RD2_01_29140.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078707_RB3_01_29115.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078708_RC10_01_29079.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078710_RA3_01_29102.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078711_RB10_01_29066.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078716_RF10_01_29174.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078718_RD5_01_29143.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078719_RA11_01_29054.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078722_RA10_01_29109.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078724_RC5_01_29074.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078726_RG5_01_29182.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078729_RF7_01_29171.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078730_RH7_01_29197.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078732_RD5_01_29087.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078735_RA9_01_29052.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078737_RB10_01_29122.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078738_RB8_01_29120.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085587_RE12_01_29163.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085590_RD1_01_29083.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085591_RH9_01_29199.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085592_RD1_01_29139.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085593_RA4_01_29047.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085594_RC3_01_29128.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085595_RA5_01_29104.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085596_RG6_01_29183.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085597_RA8_01_29051.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085598_RB12_01_29068.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085599_RD12_01_29150.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086566_RB11_01_29067.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086567_RC7_01_29076.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086569_RD11_01_29149.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086570_RC9_01_29134.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086571_RB3_01_29059.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086576_RC1_01_29126.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086578_RA7_01_29050.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086579_RD8_01_29146.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086580_RF12_01_29176.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088099_RB7_01_29063.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088101_RH1_01_29191.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088103_RG3_01_29180.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088106_RE1_01_29152.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088108_RG1_01_29178.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088109_RD2_01_29084.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088111_RC7_01_29132.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088113_RA6_01_29105.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088115_RB4_01_29060.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088117_RF9_01_29173.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088425_RE9_01_29160.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088426_RF11_01_29175.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088427_RA10_01_29053.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088428_RH4_01_29194.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088429_RD3_01_29141.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088430_RF1_01_29165.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088431_RG12_01_29189.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088432_RC10_01_29135.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088433_RB4_01_29116.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088434_RB9_01_29065.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088435_RC2_01_29127.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088436_RC11_01_29080.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088437_RE11_01_29162.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088438_RC11_01_29136.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088439_RC6_01_29075.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088440_RB5_01_29061.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088441_RA12_01_29111.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088442_RA3_01_29046.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088443_RD9_01_29091.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088444_RF4_01_29168.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088445_RC5_01_29130.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088446_RG9_01_29186.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088447_RH8_01_29198.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088449_RG11_01_29188.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088474_RH5_01_29195.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088901_RC8_01_29133.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088903_RB2_01_29114.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088906_RE10_01_29161.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088910_RB1_01_29113.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088912_RD6_01_29088.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088915_RB9_01_29121.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088918_RE6_01_29157.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088921_RE4_01_29155.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088924_RA1_01_29100.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088928_RD9_01_29147.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089111_RB5_01_29117.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089112_RA11_01_29110.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089120_RD8_01_29090.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089124_RA12_01_29055.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089125_RA4_01_29103.mzML%22%2C%20%22usi2%22%3A%20%22%22%7D">GNPS Dashboard Link</a> - Midazolam-d4</p>
<p><a href="https://gnps-lcms.ucsd.edu/?xic_mz=275.2198&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=2.99-3.1&amp;xic_norm=False&amp;xic_file_grouping=FILE&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=90f11807e4fc437c82ea6da956f747c6&amp;comment=&amp;map_plot_color_scale=Hot_r&amp;map_plot_quantization_level=Medium#%7B%22usi%22%3A%20%22mzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076790_RE5_01_29156.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076791_RH2_01_29192.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076792_RA8_01_29107.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076793_RD10_01_29092.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076794_RH10_01_29200.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076795_RC6_01_29131.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076796_RA9_01_29108.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076797_RB6_01_29062.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076798_RC1_01_29070.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076799_RC8_01_29077.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076800_RG4_01_29181.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076801_RE2_01_29153.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076802_RD6_01_29144.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076803_RH6_01_29196.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076804_RF8_01_29172.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076805_RA2_01_29101.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076806_RB7_01_29119.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076807_RD4_01_29086.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076808_RD3_01_29085.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076809_RA7_01_29106.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076810_RD7_01_29145.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076811_RF5_01_29169.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076812_RG7_01_29184.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076813_RB11_01_29123.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076814_RC12_01_29081.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076815_RH3_01_29193.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076816_RD4_01_29142.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076817_RA5_01_29048.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076818_RE7_01_29158.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076819_RC9_01_29078.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076820_RC4_01_29073.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076821_RC12_01_29137.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076822_RB1_01_29057.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076823_RB12_01_29124.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076825_RF2_01_29166.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076826_RA2_01_29045.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076827_RA6_01_29049.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076830_RE8_01_29159.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076831_RG2_01_29179.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076832_RD10_01_29148.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076833_RC4_01_29129.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076834_RB8_01_29064.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076835_RA1_01_29044.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076837_RC3_01_29072.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076838_RB2_01_29058.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076839_RG10_01_29187.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078262_RF3_01_29167.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078263_RC2_01_29071.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078264_RG8_01_29185.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078265_RD7_01_29089.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078699_RB6_01_29118.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078700_RF6_01_29170.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078702_RE3_01_29154.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078706_RD2_01_29140.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078707_RB3_01_29115.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078708_RC10_01_29079.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078710_RA3_01_29102.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078711_RB10_01_29066.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078716_RF10_01_29174.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078718_RD5_01_29143.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078719_RA11_01_29054.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078722_RA10_01_29109.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078724_RC5_01_29074.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078726_RG5_01_29182.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078729_RF7_01_29171.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078730_RH7_01_29197.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078732_RD5_01_29087.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078735_RA9_01_29052.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078737_RB10_01_29122.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078738_RB8_01_29120.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085587_RE12_01_29163.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085590_RD1_01_29083.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085591_RH9_01_29199.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085592_RD1_01_29139.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085593_RA4_01_29047.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085594_RC3_01_29128.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085595_RA5_01_29104.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085596_RG6_01_29183.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085597_RA8_01_29051.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085598_RB12_01_29068.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085599_RD12_01_29150.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086566_RB11_01_29067.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086567_RC7_01_29076.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086569_RD11_01_29149.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086570_RC9_01_29134.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086571_RB3_01_29059.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086576_RC1_01_29126.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086578_RA7_01_29050.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086579_RD8_01_29146.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086580_RF12_01_29176.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088099_RB7_01_29063.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088101_RH1_01_29191.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088103_RG3_01_29180.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088106_RE1_01_29152.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088108_RG1_01_29178.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088109_RD2_01_29084.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088111_RC7_01_29132.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088113_RA6_01_29105.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088115_RB4_01_29060.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088117_RF9_01_29173.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088425_RE9_01_29160.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088426_RF11_01_29175.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088427_RA10_01_29053.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088428_RH4_01_29194.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088429_RD3_01_29141.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088430_RF1_01_29165.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088431_RG12_01_29189.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088432_RC10_01_29135.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088433_RB4_01_29116.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088434_RB9_01_29065.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088435_RC2_01_29127.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088436_RC11_01_29080.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088437_RE11_01_29162.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088438_RC11_01_29136.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088439_RC6_01_29075.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088440_RB5_01_29061.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088441_RA12_01_29111.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088442_RA3_01_29046.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088443_RD9_01_29091.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088444_RF4_01_29168.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088445_RC5_01_29130.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088446_RG9_01_29186.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088447_RH8_01_29198.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088449_RG11_01_29188.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088474_RH5_01_29195.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088901_RC8_01_29133.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088903_RB2_01_29114.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088906_RE10_01_29161.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088910_RB1_01_29113.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088912_RD6_01_29088.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088915_RB9_01_29121.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088918_RE6_01_29157.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088921_RE4_01_29155.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088924_RA1_01_29100.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088928_RD9_01_29147.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089111_RB5_01_29117.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089112_RA11_01_29110.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089120_RD8_01_29090.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089124_RA12_01_29055.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089125_RA4_01_29103.mzML%22%2C%20%22usi2%22%3A%20%22%22%7D">GNPS Dashboard Link</a> - Dextromethorphan-d3</p>
<p><a href="https://gnps-lcms.ucsd.edu/?xic_mz=261.2035&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=2.5-2.6&amp;xic_norm=False&amp;xic_file_grouping=FILE&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=90f11807e4fc437c82ea6da956f747c6&amp;comment=&amp;map_plot_color_scale=Hot_r&amp;map_plot_quantization_level=Medium#%7B%22usi%22%3A%20%22mzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076790_RE5_01_29156.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076791_RH2_01_29192.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076792_RA8_01_29107.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076793_RD10_01_29092.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076794_RH10_01_29200.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076795_RC6_01_29131.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076796_RA9_01_29108.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076797_RB6_01_29062.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076798_RC1_01_29070.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076799_RC8_01_29077.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076800_RG4_01_29181.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076801_RE2_01_29153.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076802_RD6_01_29144.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076803_RH6_01_29196.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076804_RF8_01_29172.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076805_RA2_01_29101.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076806_RB7_01_29119.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076807_RD4_01_29086.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076808_RD3_01_29085.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076809_RA7_01_29106.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076810_RD7_01_29145.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076811_RF5_01_29169.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076812_RG7_01_29184.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076813_RB11_01_29123.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076814_RC12_01_29081.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076815_RH3_01_29193.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076816_RD4_01_29142.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076817_RA5_01_29048.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076818_RE7_01_29158.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076819_RC9_01_29078.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076820_RC4_01_29073.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076821_RC12_01_29137.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076822_RB1_01_29057.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076823_RB12_01_29124.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076825_RF2_01_29166.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076826_RA2_01_29045.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076827_RA6_01_29049.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076830_RE8_01_29159.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076831_RG2_01_29179.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076832_RD10_01_29148.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076833_RC4_01_29129.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076834_RB8_01_29064.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076835_RA1_01_29044.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076837_RC3_01_29072.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076838_RB2_01_29058.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000076839_RG10_01_29187.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078262_RF3_01_29167.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078263_RC2_01_29071.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078264_RG8_01_29185.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078265_RD7_01_29089.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078699_RB6_01_29118.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078700_RF6_01_29170.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078702_RE3_01_29154.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078706_RD2_01_29140.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078707_RB3_01_29115.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078708_RC10_01_29079.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078710_RA3_01_29102.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078711_RB10_01_29066.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078716_RF10_01_29174.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078718_RD5_01_29143.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078719_RA11_01_29054.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078722_RA10_01_29109.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078724_RC5_01_29074.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078726_RG5_01_29182.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078729_RF7_01_29171.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078730_RH7_01_29197.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078732_RD5_01_29087.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078735_RA9_01_29052.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078737_RB10_01_29122.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000078738_RB8_01_29120.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085587_RE12_01_29163.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085590_RD1_01_29083.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085591_RH9_01_29199.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085592_RD1_01_29139.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085593_RA4_01_29047.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085594_RC3_01_29128.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085595_RA5_01_29104.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085596_RG6_01_29183.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085597_RA8_01_29051.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085598_RB12_01_29068.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000085599_RD12_01_29150.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086566_RB11_01_29067.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086567_RC7_01_29076.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086569_RD11_01_29149.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086570_RC9_01_29134.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086571_RB3_01_29059.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086576_RC1_01_29126.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086578_RA7_01_29050.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086579_RD8_01_29146.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000086580_RF12_01_29176.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088099_RB7_01_29063.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088101_RH1_01_29191.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088103_RG3_01_29180.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088106_RE1_01_29152.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088108_RG1_01_29178.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088109_RD2_01_29084.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088111_RC7_01_29132.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088113_RA6_01_29105.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088115_RB4_01_29060.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088117_RF9_01_29173.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088425_RE9_01_29160.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088426_RF11_01_29175.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088427_RA10_01_29053.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088428_RH4_01_29194.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088429_RD3_01_29141.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088430_RF1_01_29165.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088431_RG12_01_29189.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088432_RC10_01_29135.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088433_RB4_01_29116.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088434_RB9_01_29065.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088435_RC2_01_29127.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088436_RC11_01_29080.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088437_RE11_01_29162.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088438_RC11_01_29136.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088439_RC6_01_29075.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088440_RB5_01_29061.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088441_RA12_01_29111.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088442_RA3_01_29046.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088443_RD9_01_29091.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088444_RF4_01_29168.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088445_RC5_01_29130.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088446_RG9_01_29186.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088447_RH8_01_29198.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088449_RG11_01_29188.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088474_RH5_01_29195.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088901_RC8_01_29133.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088903_RB2_01_29114.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088906_RE10_01_29161.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088910_RB1_01_29113.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088912_RD6_01_29088.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088915_RB9_01_29121.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088918_RE6_01_29157.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088921_RE4_01_29155.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088924_RA1_01_29100.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000088928_RD9_01_29147.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089111_RB5_01_29117.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089112_RA11_01_29110.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089120_RD8_01_29090.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089124_RA12_01_29055.mzML%5Cnmzspec%3AGNPS%3ATASK-bde4547d96c3453dac207716b2b54dec-f.MSV000082493/ccms_peak/urine/DM000089125_RA4_01_29103.mzML%22%2C%20%22usi2%22%3A%20%22%22%7D">GNPS Dashboard Link</a> - Dextrophan-d3</p>
<p><img src="img/dashboard_use_cases/SI_Figure_15.png" width="600"/></p>
<p><strong>Supplemental Figure 15. Large Scale XIC in GNPS Dashboard</strong></p>
<p>a) Ovelrayed Extracted Ion Chromatograms (XIC) for each of the five internal standards in all 140 urine samples b) Plotted AUC values to confirm the presence of five internal standards in all 140 urine samples.</p>
<h1 id="use-case-13-validating-multiply-eluting-peptides"><strong>Use Case 13 - Validating Multiply-Eluting Peptides</strong><a class="headerlink" href="#use-case-13-validating-multiply-eluting-peptides" title="Permanent link">¶</a></h1>
<h2 id="ben-pullman-phd-student-computational-mass-spectrometry_1"><strong>Ben Pullman - PhD Student - Computational Mass Spectrometry</strong><a class="headerlink" href="#ben-pullman-phd-student-computational-mass-spectrometry_1" title="Permanent link">¶</a></h2>
<h2 id="nuno-bandeira-phd-professor-computational-mass-spectrometry_1"><strong>Nuno Bandeira PhD - Professor - Computational Mass Spectrometry</strong><a class="headerlink" href="#nuno-bandeira-phd-professor-computational-mass-spectrometry_1" title="Permanent link">¶</a></h2>
<p>When analyzing a label-free proteomics experiment, the assumption is often made that each peptide should elute a single time in a given fraction, and the intensity of the peptide is that of the feature. However, in some cases, peptides can elute multiple times over a single run. Multiple elutions can complicate the calculation of feature intensity as well as create issues for the transfer of features between different runs. The GNPS Dashboard allows us to validate these multiply eluted peptides when examining the feature intensities.</p>
<p>In the following example (<a href="https://gnps-lcms.ucsd.edu/?xic_mz=1132.010986%3B1132.505737%3B1133.012939&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=ppm&amp;xic_rt_window=50-95&amp;xic_norm=False&amp;xic_file_grouping=FILE&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%22xaxis.range%5B0%5D%22%3A+50.0%2C+%22xaxis.range%5B1%5D%22%3A+95.0%2C+%22yaxis.range%5B0%5D%22%3A+1132.0%2C+%22yaxis.range%5B1%5D%22%3A+1134.0%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=aa76e51252f5403285ae9da7a219c560&amp;comment=&amp;map_plot_color_scale=Hot_r&amp;map_plot_quantization_level=High#%7B%22usi%22%3A%20%22mzspec%3AMSV000083508%3A01275_D03_P013127_S00_N20_R1%22%2C%20%22usi2%22%3A%20%22%22%7D">GNPS Dashboard link</a>), from the recent experiment looking at 31 tissues of the human proteome<a href="https://www.zotero.org/google-docs/?yagVWO">35</a> run through MaxQuant<a href="https://www.zotero.org/google-docs/?0YemI1">22</a> for feature finding and identification, we see an example of a peptide:</p>
<p>NDGYLM[+16]FQQVPM[+16]VEIDGM[+16]K</p>
<p>mapping to protein group:</p>
<p>sp|P08263|GSTA1_HUMAN;sp|P09210|GSTA2_HUMAN</p>
<p>that elutes 10 times over a 30-minute span in a single LC-MS run. All of these features have a mapped MS2 that could be identified to the peptide as well. Further, we see that the intensity of all the features is at max only a single order of magnitude lower than the highest intensity, and if the features beyond the highest two are not considered, the total intensity for the peptide would be off by ~30%.</p>
<p>Using the GNPS Dashboard, we can examine the MS2 spectra for each feature in the XIC, and see that while there are likely some co-eluting peptides, the MS2 spectra generally look similar, confirming that indeed this feature likely elutes multiple times. The USIs for these MS2s are below:</p>
<p><a href="https://massive.ucsd.edu/ProteoSAFe/usi.jsp#%7B%22usi%22%3A%22mzspec%3AMSV000083508%3A01275_D03_P013127_S00_N20_R1%3Ascan%3A24647%3ANDGYLM%5B%2B15.994915%5DFQQVPM%5B%2B15.994915%5DVEIDGM%5B%2B15.994915%5DK%2F2%22%7D">mzspec:MSV000083508:01275_D03_P013127_S00_N20_R1:scan:24647:NDGYLM[+15.994915]FQQVPM[+15.994915]VEIDGM[+15.994915]K/2</a></p>
<p><a href="https://massive.ucsd.edu/ProteoSAFe/usi.jsp#%7B%22usi%22%3A%22mzspec%3AMSV000083508%3A01275_D03_P013127_S00_N20_R1%3Ascan%3A30728%3ANDGYLM%5B%2B15.994915%5DFQQVPM%5B%2B15.994915%5DVEIDGM%5B%2B15.994915%5DK%2F2%22%2C%22searched_button%22%3A%22spectra%22%7D">mzspec:MSV000083508:01275_D03_P013127_S00_N20_R1:scan:30728:NDGYLM[+15.994915]FQQVPM[+15.994915]VEIDGM[+15.994915]K/2</a></p>
<p><a href="https://massive.ucsd.edu/ProteoSAFe/usi.jsp#%7B%22usi%22%3A%22mzspec%3AMSV000083508%3A01275_D03_P013127_S00_N20_R1%3Ascan%3A35674%3ANDGYLM%5B%2B15.994915%5DFQQVPM%5B%2B15.994915%5DVEIDGM%5B%2B15.994915%5DK%2F2%22%2C%22searched_button%22%3A%22spectra%22%7D">mzspec:MSV000083508:01275_D03_P013127_S00_N20_R1:scan:35674:NDGYLM[+15.994915]FQQVPM[+15.994915]VEIDGM[+15.994915]K/2</a></p>
<p><a href="https://massive.ucsd.edu/ProteoSAFe/usi.jsp#%7B%22usi%22%3A%22mzspec%3AMSV000083508%3A01275_D03_P013127_S00_N20_R1%3Ascan%3A41691%3ANDGYLM%5B%2B15.994915%5DFQQVPM%5B%2B15.994915%5DVEIDGM%5B%2B15.994915%5DK%2F2%22%2C%22searched_button%22%3A%22spectra%22%7D">mzspec:MSV000083508:01275_D03_P013127_S00_N20_R1:scan:41691:NDGYLM[+15.994915]FQQVPM[+15.994915]VEIDGM[+15.994915]K/2</a></p>
<p><img src="img/dashboard_use_cases/SI_Figure_16.png" width="600"/></p>
<p><strong>Supplemental Figure 16</strong> : <strong>Example of a long eluting peptide</strong></p>
<p>a) Shows the LC-MS map for the features of NDGYLM[+16]FQQVPM[+16]VEIDGM[+16]K zoomed in to show only RT 50-95 min and <em>m/z</em> 1132-1134. b) Shows the XIC for the mono-isotopic signal of the feature (<em>m/z</em> 1132.505737) with a retention time of 50-95 min, c) 1-4 show the corresponding MS/MS spectra that map to the identified features, color-coordinated with the arrows showing where the MS/MS was acquired on both a) the LC-MS map and b) the XIC.</p>
<h1 id="use-case-14-implementation-of-third-party-tool-in-the-gnps-dashboard-for-rapid-prototyping"><strong>Use Case 14 - Implementation of third party tool in the GNPS Dashboard for rapid prototyping</strong><a class="headerlink" href="#use-case-14-implementation-of-third-party-tool-in-the-gnps-dashboard-for-rapid-prototyping" title="Permanent link">¶</a></h1>
<h2 id="robin-schmid-phd-post-doctoral-scholar-computational-mass-spectrometry"><strong>Robin Schmid PhD - Post-Doctoral Scholar - Computational Mass Spectrometry</strong><a class="headerlink" href="#robin-schmid-phd-post-doctoral-scholar-computational-mass-spectrometry" title="Permanent link">¶</a></h2>
<p>Software solutions for mass spectrometry data analysis have become essential to cope with the complex data generated by non-targeted LC-MS/MS. The GNPS dashboard offers collaborative visualization and integrative data analysis options (e.g., feature finding, molecular networking, and repository scale spectrum searches). Beyond a visualization tool for end-users, the GNPS Dashboard provides a new platform for software developers to quickly test new MS data analysis tools and make them readily accessible to the broader community. The access of public metabolomics and proteomics data in mass spectrometry repositories, rich visualization options, and the GNPS Dashboard's web-interface are beneficial for the testing and development of new software tools. Open-source licensing and effortless local deployment of the GNPS Dashboard enable rapid prototyping to integrate other tools and tailor the modular dashboard with specific graphical and tabular outputs. To exemplify a use case for developers, we used the GNPS Dashboard to integrate and test the latest development branch of the feature finding tool MZmine 3, which is currently undergoing a comprehensive redesign of its data model. The goal was to provide a framework to deploy, run, and visually compare feature finding results from MZmine 3 and the latest release of MZmine 2 as a consistency check.</p>
<p>After downloading the open-source code for GNPS Dashboard, the dashboard can be executed locally in debugging mode (<a href="https://github.com/mwang87/GNPS_LCMSDashboard#development">Documentation</a>). MZmine 3 was integrated based on the existing feature finding options. Instead of running MZmine 2 only, both versions were executed sequentially to yield feature tables. Here, the dashboard provided UI controls to set essential parameters (e.g., mass tolerance, noise level, and feature width constraints) and rerun the comparison. New parameters that were introduced in MZmine 3 were set to reflect the default behavior of MZmine 2 (e.g., the minimum number of data points in the local minimum search feature resolver). We adjusted the LC-MS 2D heatmap ( <strong>Supplementary Figure 17</strong> ) and feature finding results table to overlay the results of both MZmine versions.</p>
<p>The presented integration was beneficial for comparing the latest development version and release of MZmine on different metabolomics datasets. As a result, we were able to track behavior changes in data processing and reporting. Some identified as planned changes (e.g., new parameters), while others resulted from introducing a new data model (e.g., changed accuracy (double\&lt;\&gt;float) for various data types) or changes in calculation methods. The latter case led to the feature area being calculated based on the retention time in minutes instead of seconds. The dashboard helped identify these small differences and verify that feature finding resulted in the same number of features for both MZmine versions. A discrepancy in the number of detected features was evident based on missing shapes in the LC-MS map for MZmine 3 when a new filter parameter was not turned off ( <strong>Supplementary Figure 17</strong> ).</p>
<p>Even for a rich graphical user interface tool, such as MZmine, the integration into the LC-MS dashboard facilitated the software development and testing. Key points were the provided data access from various repositories, the LC-MS map that showed overlapping or missing features, and the integration of MZmine 3 into the dashboard's modular framework. Furthermore, the dashboard's deployment in a local development environment was effortless and followed a few documented steps (<a href="https://github.com/mwang87/GNPS_LCMSDashboard#development">Documentation</a>) only.</p>
<p><img src="img/dashboard_use_cases/SI_Figure_17.png" width="600"/></p>
<p><strong>Supplemental Figure 17: Consistency testing by overlaying feature finding results from MZmine 2 and the latest development version of MZmine 3.</strong> A complete results overlap was achieved after adjustment of the MZmine 3 preferences and batch mode. The specialized version of the GNPS dashboard informed the development of MZmine 3 to provide comparable results to MZmine 2.</p>
<h1 id="use-case-15-quantification-of-isobutanol-produced-by-metabolically-engineered-_-ecoli-_-strains"><strong>Use Case 15 - Quantification of isobutanol produced by metabolically engineered</strong> _ <strong>E.coli</strong> _ <strong>strains</strong><a class="headerlink" href="#use-case-15-quantification-of-isobutanol-produced-by-metabolically-engineered-_-ecoli-_-strains" title="Permanent link">¶</a></h1>
<h2 id="simon-boecker-phd-post-doctoral-scholar-metabolic-engineering"><strong>Simon Boecker, PhD - Post-doctoral Scholar - Metabolic Engineering</strong><a class="headerlink" href="#simon-boecker-phd-post-doctoral-scholar-metabolic-engineering" title="Permanent link">¶</a></h2>
<p>The performance of engineered microbial cell factories for the synthesis of a product of interest is usually characterized by the three parameters titer (product per volume), productivity (product per volume (or per biomass) and time), and yield (product per substrate). To determine these parameters, an exact quantification of the produced metabolite secreted to the medium is indispensable.</p>
<p>For this use case, we compared the performance of two <em>E. coli</em> strains, which were engineered for the production of the biofuel isobutanol (<em>i_BuOH). _i_BuOH is a volatile compound and can be quantified directly from the culture broth by GC-MS using head-space injection. External standards and medium sampled at different time points were analyzed on an Agilent Technologies GC-MS instrument in SIM mode. In our lab, we only own one license of the MassHunter software for data analysis, complicating the evaluation of the obtained data on your office or home PC. Instead of using MassHunter, we converted the data to mzML-format via the MSConvert tool<a href="https://www.zotero.org/google-docs/?PFWwKE">3</a>, uploaded it to the MassIVE repository, and used the GNPS Dashboard for data analysis. XICs were extracted for the quantifier ion (_m/z</em> 56.1) and the areas of the selected peaks were automatically integrated and exported as csv-file and used for generating a calibration curve ( <strong>SI Figure 18a</strong> ). The amount of _i_BuOH produced by the two strains was quantified with the calibration curve, demonstrating that strain 2 shows slightly superior performance in terms of _i_BuOH production ( <strong>SI Figure 18b</strong> ).</p>
<p>In summary, the GNPS Dashboard is a very useful tool that facilitates data analysis for compound quantification by GC-MS, especially when the number of available software licenses is limited in the lab.</p>
<p>GNPS Dashboard integration results - <a href="https://gnps-lcms.ucsd.edu/?xic_mz=56.1&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=2.3-2.9&amp;xic_norm=False&amp;xic_file_grouping=MZ&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=MS1%3A669&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=b638a51388bd4ca8805628cc90696eba&amp;chromatogram_options=%5B%5D&amp;comment=&amp;map_plot_color_scale=Hot_r&amp;map_plot_quantization_level=Medium#%7B%22usi%22%3A%20%22mzspec%3AMSV000087056%3Apeak/std_0010_iBuOH.mzML%5Cnmzspec%3AMSV000087056%3Apeak/std_0050_iBuOH.mzML%5Cnmzspec%3AMSV000087056%3Apeak/std_0100_iBuOH.mzML%5Cnmzspec%3AMSV000087056%3Apeak/std_0250_iBuOH.mzML%5Cnmzspec%3AMSV000087056%3Apeak/std_0500_iBuOH.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_0abfae11/peak/std_0010_iBuOH_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_0abfae11/peak/std_0050_iBuOH_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_0abfae11/peak/std_0100_iBuOH_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_0abfae11/peak/std_0250_iBuOH_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_0abfae11/peak/std_0500_iBuOH_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time2_IBU64_1.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time2_IBU64_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time2_IBU64_3.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time2_IBU66_1.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time2_IBU66_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time2_IBU66_3.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time3_1zu10_IBU64_1.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time3_1zu10_IBU64_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time3_1zu10_IBU64_3.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time3_1zu10_IBU66_1.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time3_1zu10_IBU66_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time3_1zu10_IBU66_3.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time4_1zu10_IBU64_1.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time4_1zu10_IBU64_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time4_1zu10_IBU64_3.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time4_1zu10_IBU66_1.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time4_1zu10_IBU66_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time4_1zu10_IBU66_3.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time5_1zu10_IBU64_1.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time5_1zu10_IBU64_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time5_1zu10_IBU64_3.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time5_1zu10_IBU66_1.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time5_1zu10_IBU66_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time5_1zu10_IBU66_3.mzML%22%2C%20%22usi2%22%3A%20%22mzspec%3AMSV000087056%3Apeak/iBuOH_producer_sample1.mzML%5Cnmzspec%3AMSV000087056%3Apeak/iBuOH_producer_sample2.mzML%5Cnmzspec%3AMSV000087056%3Apeak/std_0010_iBuOH.mzML%5Cnmzspec%3AMSV000087056%3Apeak/std_0050_iBuOH.mzML%5Cnmzspec%3AMSV000087056%3Apeak/std_0100_iBuOH.mzML%5Cnmzspec%3AMSV000087056%3Apeak/std_0250_iBuOH.mzML%5Cnmzspec%3AMSV000087056%3Apeak/std_0500_iBuOH.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_0abfae11/peak/std_0010_iBuOH_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_0abfae11/peak/std_0050_iBuOH_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_0abfae11/peak/std_0100_iBuOH_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_0abfae11/peak/std_0250_iBuOH_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_0abfae11/peak/std_0500_iBuOH_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time2_IBU64_1.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time2_IBU64_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time2_IBU64_3.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time2_IBU66_1.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time2_IBU66_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time2_IBU66_3.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time3_1zu10_IBU64_1.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time3_1zu10_IBU64_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time3_1zu10_IBU64_3.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time3_1zu10_IBU66_1.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time3_1zu10_IBU66_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time3_1zu10_IBU66_3.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time4_1zu10_IBU64_1.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time4_1zu10_IBU64_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time4_1zu10_IBU64_3.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time4_1zu10_IBU66_1.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time4_1zu10_IBU66_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time4_1zu10_IBU66_3.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time5_1zu10_IBU64_1.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time5_1zu10_IBU64_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time5_1zu10_IBU64_3.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time5_1zu10_IBU66_1.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time5_1zu10_IBU66_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_6d9ca600/peak/Time5_1zu10_IBU66_3.mzML%22%7D">Link</a></p>
<p>GNPS Dashboard Figure 17 - <a href="https://gnps-lcms.ucsd.edu/?xic_mz=56.1&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=2.3-2.9&amp;xic_norm=False&amp;xic_file_grouping=MZ&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=MS1%3A669&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=b638a51388bd4ca8805628cc90696eba&amp;chromatogram_options=%5B%5D&amp;comment=&amp;map_plot_color_scale=Hot_r&amp;map_plot_quantization_level=Medium#%7B%22usi%22%3A%20%22mzspec%3AMSV000087056%3Apeak/std_0050_iBuOH.mzML%5Cnmzspec%3AMSV000087056%3Apeak/std_0100_iBuOH.mzML%5Cnmzspec%3AMSV000087056%3Apeak/std_0250_iBuOH.mzML%5Cnmzspec%3AMSV000087056%3Apeak/std_0500_iBuOH.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_0abfae11/peak/std_0050_iBuOH_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_0abfae11/peak/std_0100_iBuOH_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_0abfae11/peak/std_0250_iBuOH_2.mzML%5Cnmzspec%3AMSV000087056%3Aupdates/2021-03-16_sboecker_0abfae11/peak/std_0500_iBuOH_2.mzML%5Cn%22%2C%20%22usi2%22%3A%20%22%22%7D">Link</a></p>
<p><img src="img/dashboard_use_cases/SI_Figure_18.png" width="600"/></p>
<p><strong>Supplemental Figure 18:</strong> a) Overlaid XICs for <em>m/z</em> 56.1 of <em>i_BuOH standards (in duplicates) visualized in the GNPS Dashboard and generated calibration curve from integrated peak areas. b) Time course of _i_BuOH production and specific productivity of both _E. coli</em> strains.</p>
<h1 id="use-case-16-exploring-new-directions-in-previously-analyzed-gc-ms-data"><strong>Use Case 16 - Exploring new directions in previously-analyzed GC-MS data</strong><a class="headerlink" href="#use-case-16-exploring-new-directions-in-previously-analyzed-gc-ms-data" title="Permanent link">¶</a></h1>
<h2 id="rachel-gregor-phd-post-doctoral-scholar-chemical-ecology"><strong>Rachel Gregor, PhD - Post-doctoral Scholar - Chemical Ecology</strong><a class="headerlink" href="#rachel-gregor-phd-post-doctoral-scholar-chemical-ecology" title="Permanent link">¶</a></h2>
<h2 id="michael-m-meijler-phd-professor-chemical-biology"><strong>Michael M. Meijler, PhD - Professor - Chemical Biology</strong><a class="headerlink" href="#michael-m-meijler-phd-professor-chemical-biology" title="Permanent link">¶</a></h2>
<h2 id="itzhak-mizrahi-phd-professor-microbial-ecology"><strong>Itzhak Mizrahi, PhD - Professor - Microbial Ecology</strong><a class="headerlink" href="#itzhak-mizrahi-phd-professor-microbial-ecology" title="Permanent link">¶</a></h2>
<p>Large-scale 'omics projects generate vast quantities of mass spectrometry data, more than can be comprehensively analyzed within the scope of one or even a series of publications. The same data can often be re-analyzed later to explore different questions or angles, especially if it is uploaded on public data depositories such as MassIVE. For example, as part of a large fecal microbiome-metabolome dataset collected in collaboration with a local zoo, we included food samples from the animals' diets and analyzed them by GC-MS. The food GC-MS metabolomes were not characterized in detail in the original data analysis, other than to quantify the overlap between fecal and food metabolites. Therefore, this portion of the dataset would be an excellent candidate for future analyses, for example in order to explore specific classes of nutrients found in different animal diet components.</p>
<p>However, unexpected setbacks can arise when re-analyzing data at a later point, even for the original team who collected the data. In this case, we had originally visualized the chromatograms using Agilent's proprietary MassHunter software and performed deconvolution and molecular networking using the GNPS MS-Hub workflow. Since then, the lead author in this project finished her PhD and moved to a different institution, and no longer has access to an Agilent workstation with MassHunter, presenting a barrier in visualizing the chromatography data. This stage is especially crucial for the analysis of GC-MS data with electron impact ionization, both in order to manually verify the quality of the deconvolution for peaks of interest, and because different classes of compounds often have characteristic retention time ranges that are evident on the chromatograms.</p>
<p>Using the GNPS Dashboard, we were able to load the GC-MS dataset from MassIVE (<a href="https://gnps.ucsd.edu/ProteoSAFe/result.jsp?task=a1111d1e17884b4ca516444125f22c28&amp;view=advanced_view#%7B%7D">MSV000083859</a>) and immediately explore metabolites of interest in the food samples. For example, the herbivores in the study consumed a variety of foods, including fresh fruits and two types of pellets. We examined the sugar content of three types of fruit and the two pellet samples by creating an XIC for m/z 217, a characteristic sugar fragment, resulting in three groups of peaks corresponding to mono-, di-, and tri-saccharides ( <strong>SI figure 19a</strong> ). It was immediately evident that fruits were enriched in mono- and some di-saccharides, while the pellets contained primarily di- and tri-saccharides. This was further strengthened by the molecular network, which showed a similar trend in a large cluster of sugar compounds ( <strong>SI figure 19b</strong> ). Using the GNPS Dashboard and the identifications from the molecular networking, we were able to easily examine specific examples of sugars from each type, such as the mono-saccharide fructose, the di-saccharide sucrose, and the tri-saccharide maltotriose, and compare the relative quantification in each sample ( <strong>SI figure 19c</strong> ). This analysis was possible quickly and easily upon selecting these samples from the MassIVE dataset on the GNPS Dashboard, thus removing barriers and making this data immediately accessible and ready for analysis.</p>
<p><img src="img/dashboard_use_cases/SI_Figure_19.png" width="600"/></p>
<p><strong>Supplemental figure 19.</strong> a) Overlaid XIC's for m/z 217 for fruit samples (top) and pellet samples (bottom). Characteristic elution ranges for mono-, di-, and tri-saccharides are indicated in grey. b) Molecular network of saccharide cluster with relative abundances of fruit (green) and pellets (purple). (c) Overlaid XIC's for m/z 217 for all samples, for peaks identified by spectral library search as fructose (top), sucrose (middle), and maltotriose (bottom).</p>
<h1 id="use-case-17-inspection-of-data-during-the-review-process"><strong>Use Case 17 - Inspection of data during the review process.</strong><a class="headerlink" href="#use-case-17-inspection-of-data-during-the-review-process" title="Permanent link">¶</a></h1>
<h2 id="pieter-c-dorrestein-phd-professor-small-molecule-mass-spectrometry"><strong>Pieter C. Dorrestein, PhD - Professor - Small Molecule Mass Spectrometry</strong><a class="headerlink" href="#pieter-c-dorrestein-phd-professor-small-molecule-mass-spectrometry" title="Permanent link">¶</a></h2>
<p>The review process is an active process of the scientific enterprise. The goal of peer review of manuscripts is to ensure that the science that is presented is accurate and, when needed, to provide suggestions to improve the work. One of the key challenges as a reviewer of research articles that include mass spectrometry (MS) data is to get insight into the methods and settings used for data acquisition and quality or details of the actual data that is collected. Even when data is made public, many reviewers do not have or take the time to look at the raw MS data as there is a high barrier to entry (e.g., as this entails downloading and converting the data into the proper format and then loading the data into specific software to view). The GNPS Dashboard makes this process a simple single step in the web browser. Further, if the authors include a direct url, then it is as easy as clicking the link to allow direct data inspection. As the review process is confidential, in this use case I will not show data, but rather describe recent examples that have been observed during recent reviews.</p>
<p>Being able to inspect the data during a review enables one to gain insight if there are alternative explanations possible. Most methods in papers have insufficient detail to understand how the data was acquired. A quick inspection of the <em>m/z</em> and retention time (rt) data with MS/MS points marked will reveal specific details about the data dependent acquisition (DDA), starting with the length of separation, how well the TIC appears to separate, general noise levels, how much data was also present in the background or blank samples, etc. Quick inspection using the GNPS Dashboard also gives insight into the use of data dependent exclusion parameters. For example continuous "lines'' of MS/MS at a given m/z value, can reveal that dynamic exclusion was not employed or not properly set, and the DDA perhaps cycled through the top n most intense ions only. Being able to quickly inspect the data in this way will enable reviewers to rapidly judge if the methods and data are appropriate for the task presented in the paper. For example if the paper wants to get molecular networks from as many different molecules as possible then it is critical to set the DDA including dynamic exclusion appropriately, however if the task is to verify the presence of a few molecules and their analogs, other settings may be appropriate.</p>
<p>In addition, with the GNPS Dashboard it is possible to quickly check how the data was used and if the data analysis is reproducible. For example, a paper may show an zoomed XIC derived feature but upon inspection by zooming out there are multiple XIC's with the same mass. Thus, integration of peak area needs to be carefully considered as there could be artefacts due to a different molecule with the same mass (which will also give rise to a different MS/MS spectrum), an isomer which would likely have a nearly identical MS/MS spectrum, but could also be the same molecule but with different ion adducts, rotamers of molecules. In other words a single molecule can give rise to multiple peaks during chromatography. This could affect the quantification results and thus is important to be able to assess this as a reviewer. Being able to inspect the data will not only allow reviewers to inspect the details of the data and data acquisition parameters but also share their observations as hyperlinks in the reviews so that reviewers can help the authors to improve their papers or to provide suggestions on what to consider in future experiments. This way we all grow scientifically due to the review process.</p>
<h1 id="use-case-18-structure-and-fragmentation-pattern-analysis-through-the-full-collaborative-synchronization-mode"><strong>Use Case 18: Structure and fragmentation pattern analysis through the full collaborative synchronization mode</strong><a class="headerlink" href="#use-case-18-structure-and-fragmentation-pattern-analysis-through-the-full-collaborative-synchronization-mode" title="Permanent link">¶</a></h1>
<h2 id="mirtha-navarro-phd-professor-natural-product-chemistry-bioactivity-and-nanotechnology"><strong>Mirtha Navarro PhD – Professor – Natural Product Chemistry, Bioactivity, and Nanotechnology</strong><a class="headerlink" href="#mirtha-navarro-phd-professor-natural-product-chemistry-bioactivity-and-nanotechnology" title="Permanent link">¶</a></h2>
<h2 id="felipe-vasquez-castro-msc-candidate-natural-product-chemistry-and-environmental-microbiology"><strong>Felipe Vasquez-Castro – MSc candidate – Natural Product Chemistry and Environmental Microbiology</strong><a class="headerlink" href="#felipe-vasquez-castro-msc-candidate-natural-product-chemistry-and-environmental-microbiology" title="Permanent link">¶</a></h2>
<p>Depending on the structure of research groups, principal investigators (PI) are directly involved in the day-to-day knowledge transfer to junior researchers. Therefore, tools for efficient collaborative work are of the utmost importance to support these interactions. For instance, the analysis of complex mass spectrometry data requires particular supervision, and this time-consuming effort becomes a challenge for Principal Investigators' tight schedules. GNPS Dashboard facilitates this task through the fully collaborative synchronization mode, which allows a multi-user simultaneous visualization and examination of data, thus enhancing the learning experience. Hereby we illustrate such a case, in which a PI and a junior researcher use GNPS Dashboard to perform a remote multi-user analysis of a LC-MS/MS spectrum with the purpose of identifying the structural differences and fragmentation patterns of procyanidin oligomers. The researcher needed to learn and apply these interpretation skills in proanthocyanidin-enriched extracts from the PI's research group (BIODESS). For this, the spectrum (MSV000087075) of a <em>P. domestica</em> enriched-polyphenolic extract obtained using Pressure Liquid Extraction and Amberlite resin chromatographic purification<a href="https://www.zotero.org/google-docs/?AlvS4x">36</a>, was loaded onto the GNPS Dashboard, and the full collaborative synchronization mode under Sync Options was initiated ( <strong>SI Figure 20</strong> ).</p>
<p>During inspection of the Data Exploration panel, the PI highlighted that a peak at m/z 577.13 (12.64 min) indicated the presence of a procyanidin B-type dimer holding one single interflavanic bond between two units of (epi)catechin, while a peak at m/z 575.12 (24.96 min) indicated a procyanidin with 2 Da difference. Thus, corresponding to an A-type dimer with two single bonds present between the flavan-3-ol units. Based on this observation, the researcher was able to assign a peak at m/z 865.20 (17.88 min) to a procyanidin B-type trimer and to observe the peak at m/z 863.18 (18.80 min) with 2 Da lower mass. Therefore indicating a mixed procyanidin type-A trimer with one and two single interflavanic bonds respectively between the three (epi)catechin units. ( <strong>SI Figure 21</strong> ).</p>
<p>The utilization of the full collaborative synchronization mode focused afterwards in the PI explanation of the different fragmentation pathways these compounds undergo, namely Retro Diels-Alder (RDA), Heterocyclic Ring Fission (HRF) with the loss of phluoroglucinol (126 Da), and Quinone-methide (QM). The spectrum analysis using the GNPS Dashboard allowed the researcher to observe the more abundant ion fragment at m/z 425 in the procyanidin B-type dimer corresponding to the RDA neutral loss of 152 Da. The loss of water (18 Da) after RDA fragmentation produced the more abundant ion fragment at m/z 695 in the procyanidin B-type trimer. In addition, the joint spectrum analysis allowed the PI and researcher to highlight differences in respect to the second preferred fragmentation pathway. In fact, the second more abundant ion fragment at m/z 451 in the B-type dimer corresponded to the HRF pathway after the neutral loss of phloroglucinol (126 Da), while the QM pathway delivered the second more important ion fragment at m/z 577 in the case of the B-trimer ( <strong>SI Figure 22</strong> ).</p>
<p>In summary, GNPS Dashboard permitted the simultaneous exploration and analysis of mass spectrometry data through the fully synchronized collaborative mode, which proved useful for a Principal Investigator to share information in an efficient way with a researcher in MS data analysis, enhancing the learning process through the possibility of reviewing all actions and provide simultaneous feedback on the data produced.</p>
<p>Link to XIC:<a href="https://gnps-lcms.ucsd.edu/?xic_mz=577.13%3B865.20%3B575.12%3B863.18&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=&amp;xic_norm=No&amp;xic_file_grouping=FILE&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%22xaxis.range%5B0%5D%22%3A+8.198592732740764%2C+%22xaxis.range%5B1%5D%22%3A+27.923797706628058%2C+%22yaxis.range%5B0%5D%22%3A+437.15993824812324%2C+%22yaxis.range%5B1%5D%22%3A+886.8000906770754%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=fc02496b0307452d928ba8231c6886d6&amp;chromatogram_options=%5B%5D&amp;comment=&amp;map_plot_color_scale=Hot_r&amp;map_plot_quantization_level=Medium#%7B%22usi%22%3A%20%22mzspec%3AMSV000087075%3Apeak/pdomestica_mnavarro.mzML%22%2C%20%22usi2%22%3A%20%22%22%7D">GNPS - LCMS Browser (ucsd.edu)</a></p>
<p>Link 2D LC-MS<a href="https://gnps-lcms.ucsd.edu/?xic_mz=577.13%3B865.20%3B575.12%3B863.18&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=&amp;xic_norm=No&amp;xic_file_grouping=FILE&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=None&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%22xaxis.range%5B0%5D%22%3A+8.198592732740764%2C+%22xaxis.range%5B1%5D%22%3A+27.923797706628058%2C+%22yaxis.range%5B0%5D%22%3A+437.15993824812324%2C+%22yaxis.range%5B1%5D%22%3A+886.8000906770754%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=fc02496b0307452d928ba8231c6886d6&amp;chromatogram_options=%5B%5D&amp;comment=&amp;map_plot_color_scale=Hot_r&amp;map_plot_quantization_level=Medium#%7B%22usi%22%3A%20%22mzspec%3AMSV000087075%3Apeak/pdomestica_mnavarro.mzML%22%2C%20%22usi2%22%3A%20%22%22%7D">GNPS - LCMS Browser (ucsd.edu)</a></p>
<p>MS2 at 12.64 min:<a href="https://gnps-lcms.ucsd.edu/?xic_mz=577.13&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=&amp;xic_norm=No&amp;xic_file_grouping=FILE&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=MS2%3A971&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%22xaxis.range%5B0%5D%22%3A+8.198592732740764%2C+%22xaxis.range%5B1%5D%22%3A+27.923797706628058%2C+%22yaxis.range%5B0%5D%22%3A+437.15993824812324%2C+%22yaxis.range%5B1%5D%22%3A+886.8000906770754%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=fc02496b0307452d928ba8231c6886d6&amp;chromatogram_options=%5B%5D&amp;comment=&amp;map_plot_color_scale=Hot_r&amp;map_plot_quantization_level=Medium#%7B%22usi%22%3A%20%22mzspec%3AMSV000087075%3Apeak/pdomestica_mnavarro.mzML%22%2C%20%22usi2%22%3A%20%22%22%7D">GNPS - LCMS Browser (ucsd.edu)</a></p>
<p>MS2 at 17.88 min:<a href="https://gnps-lcms.ucsd.edu/?xic_mz=577.13&amp;xic_formula=&amp;xic_peptide=&amp;xic_tolerance=0.5&amp;xic_ppm_tolerance=10&amp;xic_tolerance_unit=Da&amp;xic_rt_window=&amp;xic_norm=No&amp;xic_file_grouping=FILE&amp;xic_integration_type=AUC&amp;show_ms2_markers=True&amp;ms2_identifier=MS2%3A971&amp;show_lcms_2nd_map=False&amp;map_plot_zoom=%7B%22xaxis.range%5B0%5D%22%3A+8.198592732740764%2C+%22xaxis.range%5B1%5D%22%3A+27.923797706628058%2C+%22yaxis.range%5B0%5D%22%3A+437.15993824812324%2C+%22yaxis.range%5B1%5D%22%3A+886.8000906770754%7D&amp;polarity_filtering=None&amp;polarity_filtering2=None&amp;tic_option=TIC&amp;overlay_usi=None&amp;overlay_mz=row+m%2Fz&amp;overlay_rt=row+retention+time&amp;overlay_color=&amp;overlay_size=&amp;overlay_hover=&amp;overlay_filter_column=&amp;overlay_filter_value=&amp;feature_finding_type=Off&amp;feature_finding_ppm=10&amp;feature_finding_noise=10000&amp;feature_finding_min_peak_rt=0.05&amp;feature_finding_max_peak_rt=1.5&amp;feature_finding_rt_tolerance=0.3&amp;sychronization_session_id=fc02496b0307452d928ba8231c6886d6&amp;chromatogram_options=%5B%5D&amp;comment=&amp;map_plot_color_scale=Hot_r&amp;map_plot_quantization_level=Medium#%7B%22usi%22%3A%20%22mzspec%3AMSV000087075%3Apeak/pdomestica_mnavarro.mzML%22%2C%20%22usi2%22%3A%20%22%22%7D">GNPS - LCMS Browser (ucsd.edu)</a></p>
<p><img src="img/dashboard_use_cases/SI_Figure_20.png" width="600"/></p>
<p><strong>Supplemental Figure 20</strong> : Step-by-step setup of the bidirectional synchronization mode. A spectrum is loaded onto GNPS Dashboard using a MassIVE accession number (as in this case) or by dragging and dropping the file. The spectrum can either be loaded before or after initiating the bidirectional synchronization mode. The button ''Sync Options'' reveals the different synchronization types available. The synchronization is set to ''Bidirectional sync'' under ''Dashboard Synchronization Type'', and the URL to the session is obtained by clicking on the ''Collab URL'' button.</p>
<p><img src="img/dashboard_use_cases/SI_Figure_21.png" width="600"/></p>
<p><strong>Supplemental Figure 21</strong>. a) Example structures of a procyanidin A-type dimer constituted by two units of (epi)catechin with C4-C8 and C2-C7 bonds and a procyanidin B-type trimer composed of three units of (epi)catechin with C4-C8 bonds. b) 2D LC-MS heatmap for procyanidin A-type and B-type dimers and trimers c) grouped XICs for procyanidin A-type and B-type dimers and trimers.</p>
<p><img src="img/dashboard_use_cases/SI_Figure_22.png" width="600"/></p>
<p><strong>Supplemental Figure 22</strong>. a) Fragmentation pathways for procyanidins: Retro Diels-Alder (RDA), Heterocyclic Ring Fission (HRF), and Quinone-methide (QM). b) MS2 spectrum of a procyanidin B-type dimer at 12.64 min (m/z 577.13). c) MS2 spectrum of a procyanidin B-type trimer at 17.88 min (m/z 865.20). The fragments highlighted in both spectrums represent the respective fragmentation pathways illustrated in panel a.</p>
<h1 id="references"><strong>References</strong><a class="headerlink" href="#references" title="Permanent link">¶</a></h1>
<ol>
<li>Deutsch, E. W. et al. Universal Spectrum Identifier for mass spectra. bioRxiv 2020.12.07.415539 (2020) doi:10.1101/2020.12.07.415539.</li>
<li>Bald, T. et al. pymzML—Python module for high-throughput bioinformatics on mass spectrometry data. Bioinformatics 28, 1052–1053 (2012).</li>
<li>Chambers, M. C. et al. A cross-platform toolkit for mass spectrometry and proteomics. Nat. Biotechnol. 30, 918–920 (2012).</li>
<li>ThermoRawFileParser: Modular, Scalable, and Cross-Platform RAW File Conversion | Journal of Proteome Research. https://pubs.acs.org/doi/10.1021/acs.jproteome.9b00328.</li>
<li>Teleman, J., Chawade, A., Sandin, M., Levander, F. &amp; Malmström, J. Dinosaur: A Refined Open-Source Peptide MS Feature Detector. J. Proteome Res. 15, 2143–2151 (2016).</li>
<li>Pluskal, T., Castillo, S., Villar-Briones, A. &amp; Orešič, M. MZmine 2: Modular framework for processing, visualizing, and analyzing mass spectrometry-based molecular profile data. BMC Bioinformatics 11, 395 (2010).</li>
<li>Zhi, H. et al. Siderophore-mediated zinc acquisition enhances enterobacterial colonization of the inflamed gut. bioRxiv 2020.07.20.212498 (2020) doi:10.1101/2020.07.20.212498.</li>
<li>Wang, M. et al. Sharing and community curation of mass spectrometry data with Global Natural Products Social Molecular Networking. Nat. Biotechnol. 34, 828–837 (2016).</li>
<li>Nothias, L.-F. et al. Feature-based molecular networking in the GNPS analysis environment. Nat. Methods 1–4 (2020).</li>
<li>Nieto Penalver, C. G. et al. Methylobacterium extorquens AM1 produces a novel type of acyl-homoserine lactone with a double unsaturated side chain under methylotrophic growth conditions. FEBS Lett. 580, 561–567 (2006).</li>
<li>Ueoka, R., Bortfeld-Miller, M., Morinaka, B. I., Vorholt, J. A. &amp; Piel, J. Toblerols: Cyclopropanol-Containing Polyketide Modulators of Antibiosis in Methylobacteria. Angew. Chem. Int. Ed. 57, 977–981 (2018).</li>
<li>Vizcaíno, J. A. et al. ProteomeXchange provides globally coordinated proteomics data submission and dissemination. Nat. Biotechnol. 32, 223–226 (2014).</li>
<li>PRIDE database and related tools and resources in 2019: improving support for quantification data | Nucleic Acids Research | Oxford Academic. https://academic.oup.com/nar/article/47/D1/D442/5160986?login=true.</li>
<li>Wang, M. et al. Assembling the Community-Scale Discoverable Human Proteome. Cell Syst. 7, 412-421.e5 (2018).</li>
<li>Haug, K. et al. MetaboLights: a resource evolving in response to the needs of its scientific community. Nucleic Acids Res. 48, D440–D444 (2020).</li>
<li>Sud, M. et al. Metabolomics Workbench: An international repository for metabolomics data and metadata, metabolite standards, protocols, tutorials and training, and analysis tools. Nucleic Acids Res. 44, D463–D470 (2016).</li>
<li>Hurley, A. et al. Tiny Earth: A Big Idea for STEM Education and Antibiotic Discovery. mBio 12, (2021).</li>
<li>Wang, M. et al. Mass spectrometry searches using MASST. Nat. Biotechnol. 38, 23–26 (2020).</li>
<li>Identification and Quantitative Analysis of Acetaminophen, Acetylsalicylic Acid, and Caffeine in Commercial Analgesic Tablets by LC−MS | Journal of Chemical Education. https://pubs.acs.org/doi/10.1021/ed100280y.</li>
<li>Choi, M. et al. MassIVE. quant: A community resource of quantitative mass spectrometry–based proteomics datasets. Nat. Methods 17, 981–984 (2020).</li>
<li>ABRF Proteome Informatics Research Group (iPRG) 2015 Study: Detection of Differentially Abundant Proteins in Label-Free Quantitative LC–MS/MS Experiments | Journal of Proteome Research. https://pubs.acs.org/doi/abs/10.1021/acs.jproteome.6b00881.</li>
<li>Tyanova, S., Temu, T. &amp; Cox, J. The MaxQuant computational platform for mass spectrometry-based shotgun proteomics. Nat. Protoc. 11, 2301 (2016).</li>
<li>Cámara-Almirón, J. et al. Dual functionality of the amyloid protein TasA in Bacillus physiology and fitness on the phylloplane. Nat. Commun. 11, 1–21 (2020).</li>
<li>Molina-Santiago, C. et al. The extracellular matrix protects Bacillus subtilis colonies from Pseudomonas invasion and modulates plant co-colonization. Nat. Commun. 10, 1–15 (2019).</li>
<li>Ongena, M. et al. Surfactin and fengycin lipopeptides of Bacillus subtilis as elicitors of induced systemic resistance in plants. Environ. Microbiol. 9, 1084–1090 (2007).</li>
<li>Bolyen, E. et al. Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2. Nat. Biotechnol. 37, 852–857 (2019).</li>
<li>Bates, S. S., Hubbard, K. A., Lundholm, N., Montresor, M. &amp; Leaw, C. P. Pseudo-nitzschia, Nitzschia, and domoic acid: new research since 2011. Harmful Algae 79, 3–43 (2018).</li>
<li>Lefebvre, K. A. &amp; Robertson, A. Domoic acid and human exposure risks: A review. Toxicon 56, 218–230 (2010).</li>
<li>Martens, L. et al. mzML--a community standard for mass spectrometry data. Mol. Cell. Proteomics MCP 10, R110.000133 (2011).</li>
<li>Dührkop, K. et al. SIRIUS 4: a rapid tool for turning tandem mass spectra into metabolite structure information. Nat. Methods 16, 299–302 (2019).</li>
<li>Dührkop, K., Shen, H., Meusel, M., Rousu, J. &amp; Böcker, S. Searching molecular structure databases with tandem mass spectra using CSI: FingerID. Proc. Natl. Acad. Sci. 112, 12580–12585 (2015).</li>
<li>A. Jarmusch, S. et al. Iron-meditated fungal starvation by lupine rhizosphere-associated and extremotolerant Streptomyces sp. S29 desferrioxamine production. Mol. Omics 17, 95–107 (2021).</li>
<li>Traxler, M. F., Watrous, J. D., Alexandrov, T., Dorrestein, P. C. &amp; Kolter, R. Interspecies Interactions Stimulate Diversification of the Streptomyces coelicolor Secreted Metabolome. mBio 4, (2013).</li>
<li>Jarmusch, A. K. et al. Enhanced Characterization of Drug Metabolism and the Influence of the Intestinal Microbiome: A Pharmacokinetic, Microbiome, and Untargeted Metabolomics Study. Clin. Transl. Sci. 13, 972–984 (2020).</li>
<li>Wang, D. et al. A deep proteome and transcriptome abundance atlas of 29 healthy human tissues. Mol. Syst. Biol. 15, e8503 (2019).</li>
<li>Navarro, M. et al. Polyphenolic Characterization and Antioxidant Activity of Malus domestica and Prunus domestica Cultivars from Costa Rica. Foods 7, (2018).</li>
</ol>
<hr/>
<div class="md-source-date">
<small>
    
      Last update: <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-datetime">August 11, 2021 17:57:39</span>
</small>
</div>
</article>
</div>
</div>
</main>
<footer class="md-footer">
<nav aria-label="Footer" class="md-footer__inner md-grid">
<a aria-label="Previous: Metabolomics Spectrum Resolver" class="md-footer__link md-footer__link--prev" href="../usi/" rel="prev">
<div class="md-footer__button md-icon">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"></path></svg>
</div>
<div class="md-footer__title">
<div class="md-ellipsis">
<span class="md-footer__direction">
                Previous
              </span>
              Metabolomics Spectrum Resolver
            </div>
</div>
</a>
<a aria-label="Next: Molecular Networking Files" class="md-footer__link md-footer__link--next" href="../networkingfiles/" rel="next">
<div class="md-footer__title">
<div class="md-ellipsis">
<span class="md-footer__direction">
                Next
              </span>
              Molecular Networking Files
            </div>
</div>
<div class="md-footer__button md-icon">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4z"></path></svg>
</div>
</a>
</nav>
<div class="md-footer-meta md-typeset">
<div class="md-footer-meta__inner md-grid">
<div class="md-footer-copyright">
        
        Made with
        <a href="https://squidfunk.github.io/mkdocs-material/" rel="noopener" target="_blank">
          Material for MkDocs
        </a>
</div>
</div>
</div>
</footer>
</div>
<div class="md-dialog" data-md-component="dialog">
<div class="md-dialog__inner md-typeset"></div>
</div>
<script id="__config" type="application/json">{"base": "..", "features": [], "translations": {"clipboard.copy": "Copy to clipboard", "clipboard.copied": "Copied to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.placeholder": "Type to start searching", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.term.missing": "Missing", "select.version.title": "Select version"}, "search": "../assets/javascripts/workers/search.709b4209.min.js", "version": null}</script>
<script src="../assets/javascripts/bundle.29db7785.min.js"></script>
</body>
</html>